<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Biotechnol Biofuels</span> <span class="iso-abbrev" title="iso-abbrev">Biotechnol Biofuels</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Biotechnology for Biofuels</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1754-6834</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">BioMed Central</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6613240">6613240</a> </span> <span class="publisher-id" title="publisher-id">1516</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1186/s13068-019-1516-6">10.1186/s13068-019-1516-6</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Amelioration of biomass and lipid in marine alga by an endophytic fungus <i>Piriformospora indica</i> </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bhatnagar</span> <span class="given-names" tagx="given-names" title="given-names">Vipul Swarup</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">vipul140492@gmail.com</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bandyopadhyay</span> <span class="given-names" tagx="given-names" title="given-names">Prasun</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">prasun.banerjee1986@gmail.com</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rajacharya</span> <span class="given-names" tagx="given-names" title="given-names">Girish H.</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">girish@icgeb.res.in</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sarkar</span> <span class="given-names" tagx="given-names" title="given-names">Sharanya</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">sharanya.mukta@gmail.com</span> </span> <a href="#Aff2">2</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Poluri</span> <span class="given-names" tagx="given-names" title="given-names">Krishna Mohan</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">mohanpmk@gmail.com</span> </span> <a href="#Aff2">2</a> </span> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-7192-1526</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumar</span> <span class="given-names" tagx="given-names" title="given-names">Shashi</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">skrhode@icgeb.res.in</span> </span> <a href="#Aff1">1</a> </span> <span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0004 0498 7682</span> <span class="institution-id" title="institution-id">grid.425195.e</span> <span class="institution" title="institution">International Centre for Genetic Engineering and Biotechnology, </span> </div> </span> <span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0000 9429 752X</span> <span class="institution-id" title="institution-id">grid.19003.3b</span> <span class="institution" title="institution">Department of Biotechnology &amp;amp; Centre for Transportation Systems (CTRANS), </span> <span class="institution" title="institution">Indian Institute of Technology Roorkee, </span> </div> </span> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-7-7</span> </span> <span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2019-7-7</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2019</span> </span> <span class="volume" tagx="volume" title="volume">12</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">176</span> <span class="history" title="history"> <span class="received" title="received">received: 2019-2-28</span> <span class="accepted" title="accepted">accepted: 2019-6-23</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , </span> <span class="license" title="license"> <span class="license-p" title="license-p"> <b>Open Access</b>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License ( <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver ( <a href="http://creativecommons.org/publicdomain/zero/1.0/">http://creativecommons.org/publicdomain/zero/1.0/</a>) applies to the data made available in this article, unless otherwise stated. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p>Many studies have been carried out on the growth-modulating efficiency of plants by the colonization of an endophytic fungus <i>Piriformospora indica.</i> However, studies involving the co-culture of alga with endophytic fungal strains for enhanced biodiesel production are rare. In this study, the interaction between <i>P. indica</i> and <i>Parachlorella kessleri</i>-I, a marine algal strain, was assessed at metabolic level. </p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p>In association with an endophytic fungus, the algal biomass enhanced from 471.6 to 704 mg/L, and the fatty acid methyl ester (FAME) profile of <i>P. kessleri</i>-I increased substantially. In case of FAME profile of co-cultured <i>P. kessleri</i>-I, two essential components of biodiesel, i.e. elaidic acid and oleic acid, increased by 1.4- and 1.8-fold, respectively. To ascertain changes in the metabolic profile of <i>P. kessleri</i>-I by <i>P. indica</i> co-culture, gas chromatography–mass spectrometry (GC–MS)-based untargeted metabolomics study was performed to identify the metabolites involved; and differential nature of the essential metabolites was also confirmed using HPLC and LC–MS. Significant modulation of the bioactive metabolites such as succinate, oxo-propanoate, <div class="sc_UNKNOWN">
l</div>-alanine, glutamate, acetate and 1,2 propanediol, hydroxy butane was observed. </p> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>The metabolites like glutamate and succinate that usually belong to the GABA shunt pathway were observed to be upregulated. The pathway links nitrogen metabolism and carbon metabolism, thus influencing the growth and lipid profile of the algae. These differential metabolites thus indicated the important commensal association between the endophytic fungus and autotrophic marine alga, and established that endophytic fungus can be handy for the sustainability of algal biofuel industries. </p> </div> <div class="electronicsupplementarymaterial" title="sec"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <p>The online version of this article (10.1186/s13068-019-1516-6) contains supplementary material, which is available to authorized users. </p> </div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
Commensalism</div> <div class="kwd" title="kwd">
Metabolomics</div> <div class="kwd" title="kwd">
Endophytic fungus</div> <div class="kwd" title="kwd">
Biofuel</div> <div class="kwd" title="kwd">
Co-culture</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100001407</span> <span class="institution" title="institution">Department of Biotechnology , Ministry of Science and Technology</span> </div> </div> <span class="award-id" tagx="award-id" title="award-id">BT/PB/Centre/03/ICGEB/2011-II Phase</span> <span class="principal-award-recipient" title="principal-award-recipient"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumar</span> <span class="given-names" tagx="given-names" title="given-names">Shashi</span> </span> </span> </div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2019" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Algae-based biodiesel is a new energy source that has been explored worldwide as the third-generation biofuel. The oil content in algae can be readily converted to petroleum-like products for serving the drop-in replacements for gasoline, diesel, jet fuel, etc. [ <a href="#CR1">1</a>]. Moreover, compared to diesel, petroleum and other sources of fossil fuels, the combustion of algal biofuel does not produce any sulfur oxide or nitrous oxide. It generates reduced quantity of carbon monoxide and unburned hydrocarbons. Moreover, the associated greenhouse gases (GHG) footprint is way lesser than the other conventional fossil fuels [ <a href="#CR2">2</a>]. Algae-based fuel yields considerably more energy per unit area and can be cultivated easily in the land that is unsuitable for agriculture [ <a href="#CR3">3</a>]. However, this technology is not yet established completely for commercial-scale production. At present, enhancement of biomass and fatty acid methyl ester (FAME) content is also a major target for biofuel industries apart from harvesting and lipid extraction [ <a href="#CR4">4</a>– <a href="#CR7">7</a>]. </p> <p>Currently, most of the studies have focused on the use of fungal pellets formed in the suspension medium for harvesting algal cells. These fungal pellets or para-globules are formed due to the varying morphology of fungal strains. The variation in morphology can be of different types, for instance, microscopic aggregates, dispersed hyphae, clump formation, pellet formation or denser spherical aggregates. Pelletization due to filamentous fungi can be of various types, namely fluffy pellets, smooth-compact pellets or smooth-hollow pellets. The formation of pellets usually takes place either from one spore, termed as a non-coagulation pellet or from the aggregation of spores at an early stage, which is termed as coagulation pellet [ <a href="#CR8">8</a>, <a href="#CR9">9</a>]. Few studies, that have been carried out for depicting harvesting efficiency of fungal pellets, suggest that algal cells have a net negative surface charge due to the presence of carboxylic, phosphor diester, phosphoric, hydroxyl and amine groups. It may be neutralized by the surface charge of mycelia and hyphae of fungal strains that contain positively charged polysaccharides [ <a href="#CR10">10</a>]. This, in turn, might enable physical interaction between the two completely different organisms, thus paving the way toward the understanding of their metabolic profile changes [ <a href="#CR11">11</a>]. </p> <p>The most common example of algal and fungal interaction of two complicated populations in nature is lichen, which is the symbiotic association of two or more than two species [ <a href="#CR10">10</a>, <a href="#CR12">12</a>]. Both autotroph–heterotroph symbionts thrive by carrying out various functions and by regulating their metabolic pathways. The symbiotic interaction of endophytic fungi with marine microalgae resulted in a bioactive metabolite that has the potential to act as an active pharmaceutical ingredient [ <a href="#CR13">13</a>]. However, the modulating effect of endophytic fungal strain on microalgae has not yet been reported. </p> <p> <i>Piriformospora indica</i> is a well-studied basidiomycete endophytic fungus that can be grown axenically, thus making it the most suitable among endophytic fungal species for studying the mechanism and evolution of mutualistic symbiosis [ <a href="#CR14">14</a>]. It colonizes the monocotyledonous and dicotyledonous plants easily and stimulates the growth of many plants [ <a href="#CR15">15</a>– <a href="#CR17">17</a>]. <i>P. indica</i> helps in mobilizing the insoluble phosphate in an energy-dependent process and translocates phosphate into the host, thereby mimicking the mutualistic relation between two species [ <a href="#CR14">14</a>]. The fungus receives carbohydrate and growth factors from the plant while it helps plants by improving its growth-enhancing functions, including increased nutrient absorption. Studies deciphering the fungal genome have revealed that <i>P. indica</i> depicts the missing relation between saprophytic and obligate biotrophic mutualistic fungal species. Furthermore, few other studies on <i>P. indica</i> focus on the co-existence of characteristics of both symbiotic and obligate biotrophic fungi, which is attributed to the presence of genes responsible for biotrophic lifestyle and the absence of genes for nitrogen metabolism [ <a href="#CR14">14</a>, <a href="#CR18">18</a>]. </p> <p>The current study is designed to elucidate the effect of <i>P. indica</i> on <i>Parachlorella kessleri</i>-I ( <i>P. kessleri</i>-I) algal biomass and lipid profile under a co-cultivation environment. Further, to observe the variation in the metabolic profile of algal cells under the influence of <i>P. indica</i>, gas chromatography–mass spectrometry (GC–MS)-based untargeted metabolomics studies were conducted. High-pressure liquid chromatography (HPLC) and liquid chromatography–mass spectrometry (LC–MS) were used to substantiate the differential presence of the metabolites identified by GC–MS. The results evidenced for enhanced lipid accumulation profile of the algal cells under the biotic stress of <i>P. indica</i>. Thus, the endophytic fungus may serve as an important symbiont for the algal biofuel industries. </p> </div> <div class="materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
Materials and methods</div> <p>The marine micro algal <i>P. kessleri</i>-I cells were previously isolated in our lab and maintained in Tris–Acetate Phosphate (TAP) medium [ <a href="#CR19">19</a>]. <i>P. kessleri</i>-I cells were inoculated with a dilution of 3 × 10 <sup>5</sup> cells/mL in 250-mL flasks in triplicates for each set of TAP medium with working volume of 150 mL and then incubated at 30 °C, 150 rpm under photosynthetic light of 179.8 µmol/s/m <sup>2</sup> intensity for 12 days. The fungal strain— <i>P. indica</i>—was a kind gift from Dr. Narendra Tuteja, ICGEB, New Delhi and it was maintained on Hill and Kaefer medium [ <a href="#CR20">20</a>]. To prepare the fungal inoculums, chlamydospores were harvested with 0.02% (v/v) Tween-20 aqueous solution from 7-day-old incubated <i>P. indic</i>a culture and washed thrice with autoclaved water. Spores were then inoculated in Hill and Kaefer Petri dishes and incubated for 7 days at 30 °C for sporulation [ <a href="#CR21">21</a>]. After 7 days of incubation, spores were harvested from the Petri dishes with 10 mL of autoclaved Milli-Q water and filtered through Mira cloth of 20–25 µm pore size, supplied by Calbio chem. Then, these harvested chlamydospores of <i>P. indica</i> were inoculated at a dilution of 2.85 × 10 <sup>5</sup> spores/mL (approximately), for pellet formation in Hill and Kaefer medium. After 7 days of incubation, pellets were harvested and washed with autoclaved Milli-Q and co-inoculated with <i>P. kessleri</i>-I cultures in Tris–Phosphate medium. These co-cultures along with pure cultures of <i>P. kessleri</i>-I and <i>P. indica</i> as controls were incubated at 30 °C, 150 rpm with 179.8 µmol/s/m <sup>2</sup> light intensity for 12 days respectively in Tris–Phosphate (TP) medium. </p> <div class="observationbylightmicroscope" title="sec"> <div class="title" tagx="title" title="title">
Observation by light microscope</div> <p> <i>Parachlorella kessleri</i>-I and <i>P. indica</i> conjugated pellets were harvested and dipped in Lactophenol Cotton Blue dye to stain chitin present in hyphal cell wall of <i>P. indica</i>. The mesh network was observed under 60× and 100× magnifications under upright Nikon Eclipse Ni microscope (Nikon Corporation, Japan). </p> </div> <div class="estimationofgrowthbychlorophyllcontentanddrycellweight" title="sec"> <div class="title" tagx="title" title="title">
Estimation of growth by chlorophyll content and dry cell weight</div> <p>The controls and co-cultures of <i>P. kessleri</i>-I and <i>P. indica</i> were incubated for 12 days. Total chlorophyll content and dry cell weight were estimated at regular time intervals on the 3rd, 6th, 9th and 12th day. Estimation of total chlorophyll content was done using the method described by Porra et al. [ <a href="#CR22">22</a>]. Extraction of total chlorophyll from algal–fungal mesh was done by taking 1 mL of the macerated culture in 2 mL Eppendorf tubes from each of the setups and then centrifuging them at 5000 rpm for 10 min. The supernatant was discarded and the pellets were washed with Milli-Q water and re-suspended in 1 mL methanol with gentle tapping. Then, the extracts were incubated in dark for 30 min and centrifuged at 5000 rpm at 20 °C. Then from each of the extracts, 200 μL of the upper methanol layer was taken in 96-well EIA/RIA Corning plate (Corning Incorporation, USA) for absorbance measurements at 665 and 652 nm wavelengths simultaneously in Multimode micro-plate reader Spectramax M3 (Molecular Devices, USA). </p> <p>Chlorophyll content was calculated using the following model equations [ <a href="#CR22">22</a>]: </p> <p>Chl a: 18.22A <sup>665</sup> – 9.55A <sup>652</sup> </p> <p>Chl b: 33.75A <sup>652</sup> – 14.96A <sup>665</sup> </p> <p>Chla+b: 24.23A <sup>652</sup> + 3.26A <sup>665</sup> </p> <p>Dry cell weight was estimated after harvesting co-cultures and controls at 5000 rpm, 20 °C on the 3rd, 6th, 9th and 12th day. Then, the harvested biomass of each experimental set was dried at 60 °C in hot air oven for 8 h and their respective weights were recorded. </p> </div> <div class="estimationoflipidaccumulationbyfluorescence" title="sec"> <div class="title" tagx="title" title="title">
Estimation of lipid accumulation by fluorescence</div> <p>Lipid accumulation in the co-culture and pure culture of <i>P. kessleri</i>-I was estimated at different stages by Nile Red-based neutral lipid fluorescence intensity. Quantification of neutral lipid was performed in 96-well costar plate in triplicates. At each time interval (on 3rd, 6th, 9th and 12th day), 1 mL of culture was taken from each set and centrifuged at 5000 rpm, 20 °C for 10 min. Then, the supernatant was discarded and the pellets were re-suspended in 1 mL Milli-Q water. About 150 µL was mixed with 20 µL of Nile Red containing 25% DMSO solution (6 µg/mL). Simultaneously, 150 µL of algal culture was mixed with 20 µL of 25% DMSO with no Nile Red as a control. The control unstained cells were incubated at 40 °C, 150 rpm for 10 min. Then, the fluorescence was recorded using Spectra max spectra-fluorometer at excitation wavelength of 485 nm and emission wavelength of 552 nm [ <a href="#CR23">23</a>]. Fold increase of the lipid content is calculated as follows: </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M1">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\frac{{{\text{Flu}} . {\text{intensity of stained cells}} - {\text{Flu}} . {\text{intensity of unstained cells}}}}{{{\text{Optical density of the unstained cells at }}750\;{\text{nm}}}}$$\end{document} </div> </div> <div class="math_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
Flu</div> <div class="mo_UNKNOWN">
.</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
intensity of stained cells</div> </div> <div class="mo_UNKNOWN">
-</div> <div class="mtext_UNKNOWN">
Flu</div> <div class="mo_UNKNOWN">
.</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
intensity of unstained cells</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
Optical density of the unstained cells at</div> <div class="mspace_UNKNOWN" /> </div> <div class="mn_UNKNOWN">
750</div> <div class="mspace_UNKNOWN" /> <div class="mtext_UNKNOWN">
nm</div> </div> </div> </div> <div class="graphic_UNKNOWN" /> </div> </div> <p /> </div> <div class="lipidextractionandestimationoffattyacidmethylesterprofile" title="sec"> <div class="title" tagx="title" title="title">
Lipid extraction and estimation of fatty acid methyl ester profile</div> <p>Total lipid was extracted as reported [ <a href="#CR24">24</a>]. After 12 days of incubation, the wet biomass is harvested by centrifuging at 5000 rpm and dried at 60 °C. Around 1 g of dry biomass of each experimental set was crushed using mortar and pestle, and the crushed biomass was mixed with 1 mL of chloroform along with 2 mL of methanol, which was then kept for shaking at 150 rpm for 12 h. The mixtures were again shaken at 150 rpm for 1 h after adding 1.5 mL of distilled water, to separate the mixture into bi-phasic layers. From this bi-phasic layer, the bottom layer of chloroform was separated and filtered out using Whattmann filter paper which was then evaporated by Nitrogen flushing. Transesterification of neutral lipid was done using 2 M methanolic-KOH, added in the ratio of 200 µL per 20 mg of total lipid. Heptadecanoic acid methyl ester was added as an internal standard at a concentration of 200 µg/mL to each of the samples for quantification of the fatty acid methyl ester (FAME) content. Transesterified neutral lipids were extracted by 1 mL <i>n</i>-Hexane (HPLC grade) for performing GC–MS. For quantification of FAME, Agilent 7890A series GC system (Agilent Technologies; Singapore) comprising an Omega Wax 250 column (30 m × 0.25 mm 0.25 µm) coupled with an Agilent 7000 QQQ MS was used. All essential parameters for GC–MS experiments are included in Additional file <a href="#MOESM1">1</a>: Methods section. </p> </div> <div class="uatr-ftirmeasurementofcell–cellinteraction" title="sec"> <div class="title" tagx="title" title="title">
UATR-FTIR measurement of cell–cell interaction</div> <p>The co-culture and pure culture biomasses were harvested at 5000 rpm after complete growth. The samples were collected and washed with Milli-Q water and centrifuged. The pellets were then freeze dried and stored for further analysis by Mid Infrared range (MIR range) Perkin Elmer’s MIR/NIR Frontier DTGC/KBr UATR (Perkin Elmer, Singapore) with built-in diamond universal attenuated total reflectance with wave number ranging from 600 to 4000 cm <sup>−1</sup>. Total 100 scans were averaged for each sample and background noise with spectral contribution of water from paste was subtracted from Universal Attenuated Transform Reflectance-Fourier Transform Infrared (UATR-FTIR) spectrum of individual samples [ <a href="#CR25">25</a>, <a href="#CR26">26</a>]. </p> </div> <div class="metaboliteextractionandgc–ms-baseduntargetedmetabolitedetection" title="sec"> <div class="title" tagx="title" title="title">
Metabolite extraction and GC–MS-based untargeted metabolite detection</div> <p>The metabolites of the samples were extracted using the protocol reported by Villas-Boas et al. [ <a href="#CR27">27</a>]. About 500 mg of wet biomass for 18 replicates of both co-culture and controls was collected and then quenched overnight at − 20 °C in a solution containing glycerol and wate0r in a ratio of 5:1 with 13.5 gm/L of sodium chloride added in it. Then, pellet from each replicate was washed twice with a 1:1 solution of glycerol and water. Pellets were then suspended in super-cooled methanol and vortexed for 5 min and then methanol was evaporated using nitrogen flushing. Metabolites for each sample were concentrated by repeating the methanol extraction step for three times. Derivatization was carried out using 100 µL of BSTFA ( <i>N</i>, <i>O</i>-Bis(trimethylsilyl)trifluoroacetamide) mixed with 1% TMCS (Trimethylchlorosilane) of Cerilliant (Sigma Aldrich) followed by incubation at 80 °C on a hot plate for approximately 30 min. 100 µL of derivatized extracts of each sample with internal standard Heptadecanoic methyl ester was used for GC–MS-based analysis. </p> <p>GC–MS equipped with an HP-5 column (30 m * 0.25 mm ID, 0.25 μm thickness, Varian) coupled with an Agilent 7000 QQQ MS was used for the analyses of metabolic profiles of the samples. Electron ionization system with ionization energy of 70 eV was used and 99.99% pure helium was used as a carrier gas at a constant flow rate of 1.1 mL/min. Mass transfer line and injector temperature were set at 220 °C and 250 °C, respectively, and the oven temperature was programmed at 60 °C for 1 min, then increased at the rate of 5 °C/min to 180 °C for 1 min, then again increased at 10 °C/min to finally 310 °C for 2 min. 1 µL of sample from each set was injected in the split mode 5:1 during analysis. The signals were recorded in full-scan mode ( <i>m</i>/ <i>z</i> 20–600, 250 scans/ms). </p> <p>GC–MS raw data were obtained and deconvolution and metabolite identification of this complex data were done by comparing their mass spectra with those obtained from authentic samples and/or the NIST (National Institute of Standards and Technology) mass spectral database using AMDIS (Automated Mass spectral Deconvolution and Identification System) and MassHunter software. This whole process was repeated at least for 18 biological replicates. Once the data were preprocessed using AMDIS software, the metabolites were screened for bioactive compounds using KEGG (Kyoto Encyclopedia of Genes and Genomes) compound portal of KEGG database. Relative abundance percentages of these bioactive metabolites were calculated using AMDIS software averaged manually and then plotted in the clustered heatmap. The heatmap was generated using BioConductor’s R script package gplots combined with R ColorBrewer’s palettes (Warnes et al. [ <a href="#CR28">28</a>]). </p> </div> <div class="metaboliteextractionandliquidchromatography–massspectrometry(lc–ms)-baseduntargetedmetabolitedetection" title="sec"> <div class="title" tagx="title" title="title">
Metabolite extraction and liquid chromatography–mass spectrometry (LC–MS)-based untargeted metabolite detection </div> <p>Samples were grown for 12 days and harvested at 5000 rpm (2348× <i>g</i>) 4 °C for 10 min using Eppendorf centrifuge 5424R (Eppendorf, Germany). Pellets were rinsed with Milli-Q water and ~ 500 mg of each was re-suspended in 0.5 mL of Methanol:water (80:20) solvent and kept in − 80 °C overnight. After quenching, samples were homogenized by a sonicator (Sonics-VCX 500, USA) using 30% pulse for 15 min. Samples were centrifuged and passed through 0.45-µm filter. 100 µg/mL no-valine was used as an internal standard along with samples for LC–MS–MS analysis. </p> <p>Orbitrap Fusion Lumos Ttribrid Mass Spectrometer equipped with column Acclaim Trinity P2 (100 × 2.1 mm, 3 µm) (Thermo Fischer Scientific, Singapore) was used in mix mode for metabolic analysis. Essential parameters for buffer system and LC–MS–MS are mentioned in Additional file <a href="#MOESM1">1</a>: Table S1. The xcalibur software was used for the LC–MS–MS data acquisition (Thermo Scientific Version 3.0) and exported in RAW format. After data acquisition, each RAW file was imported in Compound Discoverer 3.0 (Thermo Scientific) and non-nested grouping was done with 7 replicates. Each sample group, i.e. algal, fungal, and co-culture were then assigned with ratio type (co-culture/alga; co-culture/fungal) for depicting <i>p</i> value. Untargeted metabolomic analysis with peak filtering compound annotation by online databases was selected as the workflow template as shown in Additional file related to workflow. Multivariate analysis was performed using the Compound Discoverer 3.0 (Thermo Scientific). </p> </div> <div class="high-pressureliquidchromatography(hplc)-basedanalysisofcellextracts" title="sec"> <div class="title" tagx="title" title="title">
High-pressure liquid chromatography (HPLC)-based analysis of cell extracts</div> <p>After 12 days of incubation, samples were harvested at 5000 rpm 25 °C using Eppendorf centrifuge (Eppendorf, Germany). Then pellets were washed and re-suspended in 1 mL of Milli-Q water and then sonicated at 30% pulse for 10 min with Sonics-VCX 500 sonicator (USA). Cell extract was collected and filtered using a 0.45-µm syringe filter. We analyzed the concentration of hydroxy-glutamate and succinate in cell extracts using high-pressure liquid chromatography equipped with Aminex HPX-87H column. Derivatization of glutamate to form hydroxy-glutamate was carried out in 2-mL microcentrifuge tubes at 45 °C for 90 min. Samples were mixed with 0.2 mL of 1 M KNO <sub>3</sub>, and reaction was started by adding 0.04 mL of 12 M HCl and stopped by adding 0.2 mL 2 M NaOH. Quantification of derivatized glutamate was studied using an Aminex HPX-87H column (300 × 7.8 mm, 10 µ) Bio-Rad, USA; Ion Exchange column and RI detector. HPLC parameters used for glutamate and succinate study are mentioned in Additional file <a href="#MOESM1">1</a>: Methods section. </p> </div> <div class="metabolicpathwayanalysis(metpa)" title="sec"> <div class="title" tagx="title" title="title">
Metabolic pathway analysis (MetPA)</div> <p>Uncommon compounds identified in co-culture of <i>P. indica</i> and <i>P. kessleri</i>-I with respect to their controls were searched by their KEGG IDs using KEGG database [ <a href="#CR29">29</a>, <a href="#CR30">30</a>]. The KEGG IDs of these compounds were then used in Metabopathway (MetPa) analysis web interface for positioning them in global metabolic pathways. Compound hits obtained from MetPA were plotted in the stacked bar graph against total metabolites present in respective pathways of the KEGG database with <i>Chlamydomonas reinhardtii</i> taken as a model organism [ <a href="#CR31">31</a>]. </p> </div> <div class="statisticaldataanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistical data analysis</div> <p>Each statistical datum was calculated from biological triplicates of each experimental set and error line was calculated by standard deviation between each set plotted using GraphpadPrism version 5. For all the statistical analyses, ANOVA test was used. </p> </div> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="co-culturingofp.kessleri-iwithp.indica" title="sec"> <div class="title" tagx="title" title="title">
Co-culturing of <i>P. kessleri</i>-I with <i>P. indica</i> </div> <p>Using the co-culture of <i>P. kessleri</i>-I with <i>P. indica</i>, changes in growth, lipid content, UATR-based physical interaction and untargeted metabolic profile were studied. Tris–Phosphate (TP), a standard maintenance medium, was used to co-culture <i>P. indica</i> pellets with <i>P. kessleri</i>-I along with their pure cultures as controls (Fig. <a href="#Fig1">1</a>A, B). We observed there was no change in the growth of <i>P. indica</i> pellets in TP medium, due to the absence of peptone and carbon contents that are important for spore formation. Under Nikon Ni light microscope at 100× magnification, we observed hyphal networks of <i>P. indica</i> interacting with <i>P. kessleri</i>-I cells (Fig. <a href="#Fig1">1</a>C). Formation of a biofilm of fungal hyphae with algal cells entrapped in it resulted in their physical interaction. It was essential to validate that there is cell–cell interaction between the two species. For this, Universal Attenuated Fourier Transform Infrared Spectroscopy (UATR) was used to determine the cell surface functional groups responsible for adhesion of the two different cells. In the case of co-cultures, we observed higher peak intensities of C–O bond vibration of carbohydrate (Additional file <a href="#MOESM1">1</a>: Figs. S1–S3, Tables S2–S4) [ <a href="#CR25">25</a>]. This UATR-based observation suggested the strong possibility for physical interaction-based commensalism between the two species [ <a href="#CR25">25</a>, <a href="#CR32">32</a>]. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 1</span> </a> </div>   <p> <b>A</b>– <b>C</b> Culture of endophytic fungus <i>Piriformospora indica</i> ( <i>Pi</i>) and para-globules formation in Hill and Kaefer medium for studying fungal interaction with marine alga <i>Parachlorella kessleri</i>-I ( <i>Pk</i>-I). <b>A</b> Spore formation of <i>Pi</i> on Hill and Kaefer medium after 7 days. Then, <i>Pi</i> spores (2.8 × 10 <sup>5</sup> spores/mL) were subjected to form fungal pellets. <b>B</b> After 7 days of <i>Pi</i> pellet formation, they were co-cultured with <i>Pk</i>-I on 0th day culture and incubated for 12 days. <b>C</b> Interaction of <i>Pi</i> and <i>Pk</i>-I studies under Nikon Ni light microscope (100× magnification) after 12th day. <i>Pk</i>-I cells (green) interacting with <i>Pi</i> hyphae (stained with cotton blue dye). The interacting <i>Pk</i>-I cells are shown with black arrows and <i>Pi</i> hyphae are indicated by red arrows </p>   </div> <p /> </div> <div class="estimationofgrowthbycorrelationratioofchlorophyllcontentanddrycellweight" title="sec"> <div class="title" tagx="title" title="title">
Estimation of growth by correlation ratio of Chlorophyll content and dry cell weight</div> <p>We have analyzed the total chlorophyll content for co-culture and the control in nmol/mL. According to previously published studies, the increase in biomass is relevantly equivalent to the increase in total chlorophyll content (nmol/mL) [ <a href="#CR33">33</a>, <a href="#CR34">34</a>]. We observed that there was a substantial increase in the total chlorophyll content of <i>P. kessleri</i>-I in co-culture, i.e. approximately 11.7 nmol/mL higher than the biomass of pure culture (Table <a href="#Tab1">1</a>). Then, the correlation ratio of total chlorophyll content and their consecutive dry cell weight at different time points (3rd, 6th, 9th and 12th day) were plotted (Fig. <a href="#Fig2">2</a>). Under the association of an endophytic fungus, the algal biomass was enhanced by 1.5-fold (from 471.6 to 704 mg/L) on the 12th day (Table <a href="#Tab2">2</a>). We have also validated the correlation ratio data with standard curve of total chlorophyll content vs. dry biomass of pure culture (Additional file <a href="#MOESM1">1</a>: Fig. S4). <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  </div></p><p>Total chlorophyll content (average) estimation of co-cultured and pure culture of <i>P. kessleri</i>-I at different time points </p>  <table> <thead> <tr> <th>Time point</th> <th>3rd day</th> <th>6th day</th> <th>9th day</th> <th>12th day</th> </tr> </thead> <tbody> <tr> <td> <i>P. kessleri</i>-I (nmol/mL) </td> <td>2.06365</td> <td>3.9715</td> <td>5.2109</td> <td>7.13305</td> </tr> <tr> <td> <i>P. kessleri</i>-I + <i>P. Indica</i> (nmol/mL) </td> <td>2.7694</td> <td>5.67105</td> <td>7.41755</td> <td>18.455</td> </tr> </tbody> </table>  <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 2</span> </a> </div>   <p>Estimation of correlation ratio of chlorophyll content to biomass of co-cultured <i>P. kessleri</i>-I ( <i>Pk</i>-I) with respect to pure culture. To assess the effective enhancement in growth yield under the influence of biotic stress given by <i>P. indica</i>, dry correlation ratio of chlorophyll content to biomass of <i>P. kessleri</i>-I at pH 7 was analyzed at each time point. There was about twofold increase with respect to <i>P. kessleri</i>-I control </p>   </div> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 2</span>  <p>Dry cell weight (average) estimation of co-cultured and pure culture of <i>P. kessleri</i>-I and <i>P. indica</i> at different time points </p>  <table> <thead> <tr> <th>Time point</th> <th>3rd day</th> <th>6th day</th> <th>9th day</th> <th>12th day</th> </tr> </thead> <tbody> <tr> <td> <i>P. kessleri</i>-I (mg/L) </td> <td>260 ± 10</td> <td>276.6 ± 5.7</td> <td>316.6 ± 10.59</td> <td>471.6 ± 9.6</td> </tr> <tr> <td> <i>P. indica</i> (mg/L) </td> <td>152 ± 7.2</td> <td>169 ± 11.53</td> <td>160.6 ± 7.09</td> <td>164 ± 16</td> </tr> <tr> <td> <i>P. kessleri</i>-I + <i>P. indica</i> (mg/L) </td> <td>520 ± 10</td> <td>570 ± 10</td> <td>619 ± 14.17</td> <td>704.4 ± 6.02</td> </tr> </tbody> </table> </div> <p /> </div> <div class="estimationofmodulationinfameprofile" title="sec"> <div class="title" tagx="title" title="title">
Estimation of modulation in FAME profile</div> <p>We have measured the changes in the neutral lipid content at different time points for each experiment set up by Nile Red fluorescence normalized intensity. We observed maximum fluorescence intensity in co-cultured algal cells between day 6 and day 12, whereas in algal controls the fluorescence intensity decreased on the 12th day (Table <a href="#Tab3">3</a>), may be due to the cessation of lipid biosynthesis. We observed about onefold increase in Nile Red fluorescence intensity for co-culture on day 12. Increase in neutral lipid fluorescence intensity of co-cultured algae has motivated us to analyze the GC–MS-based FAME profile, which reflects the changes in the neutral lipid content. The oleic and elaidic acid contents were observed to increase drastically by 1.4- to 1.8-fold in the FAME profile of co-cultures (Fig. <a href="#Fig3">3</a>). Both of them are essential for improving the quality of biodiesel. Internal standard heptadecanoic acid methyl ester was used for correcting system error; its chromatogram has been shown in Additional file <a href="#MOESM1">1</a>: Fig. S5. <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 3</span>  </div></p><p>Neutral lipid normalized Nile Red fluorescence intensity at different time points</p>  <table> <thead> <tr> <th>Time point</th> <th>Day 3</th> <th>Day 6</th> <th>Day 12</th> </tr> </thead> <tbody> <tr> <td>Control</td> <td>15.81 ± 5.28</td> <td>23.89 ± 10.142</td> <td>20.93 ± 17.402</td> </tr> <tr> <td> <i>Pi</i> </td> <td>6.27 ± 12.02</td> <td>5.68 ± 14.732</td> <td>5.98 ± 1.37</td> </tr> <tr> <td> <i>Pk</i>+ <i>Pi</i> </td> <td>20.48 ± 9.105</td> <td>26.20 ± 5.418</td> <td>29.42 ± 13.729</td> </tr> </tbody> </table>  <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 3</span> </a> </div>   <p>Estimation of FAME content of <i>P. kessleri</i>-I co-cultured with <i>P. indica</i>after 12 days. It was found that there was modulation in fatty acid methyl ester content of <i>P. kessleri</i>-I under the commensal effect of <i>P. indica.</i> Approximately onefold enhancement in the FAME profile of <i>P. kessleri</i>-I was observed under the influence of <i>P. indica</i> </p>   </div> <p /> </div> <div class="gaschromatography–massspectrometry(gc–ms)-baseduntargetedmetabolomicanalysis" title="sec"> <div class="title" tagx="title" title="title">
Gas chromatography–mass spectrometry (GC–MS)-based untargeted metabolomic analysis</div> <p>After assessing the modulation in growth and neutral lipid content of <i>P. kessleri</i>-I by co-culturing with <i>P. indica</i>, it was also essential to decipher the changes that were taking place at the metabolic level. To identify induced and repressed metabolites in co-culture, we have generated three metabolic libraries as follows: two control metabolic libraries each from pure cultures of alga, fungus and one library from co-culture. Pre-processing and manual screening of GC–MS data gave a total of 366 hits, among which 152 active metabolites were detected. We observed about 57 metabolites with 19 bioactive metabolites after manual screening through KEGG database with <i>Chlamydomonas reinhardtii</i> taken as the model organism. Among these 57 bioactive compounds, 42 were identified in co-culture, 32 in <i>P. kessleri</i>-I control and 25 in <i>P. indica</i> control (Table <a href="#Tab4">4</a>). Venn diagram plotted using Venny 2.1.0 shows the set of compounds common or uncommon in the case of co-culture and their controls (Fig. <a href="#Fig4">4</a>) [ <a href="#CR35">35</a>]. PCA score plot from XCMS database was used to determine the relationship between samples and utilized the loading plot to observe the correlation between the metabolites and the sample types. We have observed the clustering of dots in PCA-2D plot due to divergence in the metabolite and sample type [ <a href="#CR36">36</a>]. Each dot in PCA-2D, score plot and loading plot denotes a particular metabolite. They belong to different experimental sets and the clustering is done based on the sample types (Additional file <a href="#MOESM1">1</a>: Fig. S6). A major number of metabolites, with high probability of their occurrence, were clustered in co-culture and algal control groups. We observed about 60% variance in the distribution of metabolites between different sample groups. Also, this interactive PCA-based clustering of metabolite hits in their sample types depicted the difference in the profile of the co-culture metabolic library (Additional file <a href="#MOESM1">1</a>: Fig. S7). Further, to determine the relative hierarchical cluster analysis, we have generated the heatmap using abundance percentage (%) values of bioactive metabolites. In our untargeted approach, we have subtracted the abundance percentage of identified bioactive compounds detected in co-culture from those of the controls. After subtracting the abundance percentage of co-culture metabolites from the combined sum of their abundance in the controls, the higher ones were being considered to be induced. Interestingly in our study, metabolites such as succinate, oxo-propanoate, <div class="sc_UNKNOWN">
l</div>-alanine, glutamate, and acetate were induced in co-culture. These metabolites were depicted in heatmap using R studio-based gplots (Fig. <a href="#Fig5">5</a>), and also in carbon internetworking pathway mapping (Fig. <a href="#Fig6">6</a>). The repressed metabolites with a notable decrease in relative abundance such as 1,2 propanediol, hydroxy butane, acetone and the abundance percentage values of all the metabolites are presented in Additional file <a href="#MOESM1">1</a>: Table S5. <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 4</span>  </div></p><p>Bioactive compounds identified after AMDIS-based pre-processing</p>  <table> <thead> <tr> <th /> <th>Total metabolites detected</th> <th>Functional metabolites screened</th> </tr> </thead> <tbody> <tr> <td> <i>P. kessleri</i>-I </td> <td>177</td> <td>32</td> </tr> <tr> <td> <i>P. indica</i> </td> <td>85</td> <td>25</td> </tr> <tr> <td> <i>P. kessleri</i>-I + <i>P. indica</i> </td> <td>110</td> <td>42</td> </tr> </tbody> </table>  <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 4</span> </a> </div>   <p>Venn diagram was drawn of metabolomic analysis of <i>P. indica</i>– <i>P. kessleri</i>-I co-culture after 12 days of incubation. The number of bioactive metabolites from <i>P. indica</i> and <i>P. kessleri</i>-I during co-culture as well as in an individual cultures is shown using Venn diagram </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 5</span> </a> </div>   <p>Abundance of relative % values of upregulated and downregulated bioactive metabolites in <i>P. kessleri</i>-I– <i>P. indica</i> co-culture. Bioactive compounds screened from co-cultured metabolome are depicted in heatmap with respect to the Abundance value of <i>P. kessleri</i>-I and <i>P. indica</i> controls. Ranges of colors are specified as follows: − 10 to − 0.1 as Red, 0.09 to 0.1 as Black and 0.51 to 10 as Green. Heatmap was generated using gplots of R script. Heatmap was used to visualize the comparative relative increase in the abundance value of induced metabolites with respect to <i>Pk</i>-I and <i>Pi</i> controls thus making it easy for visualizing the comparison </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 6</span> </a> </div>   <p>Carbon metabolism internetworking; in the figure, red-colored metabolites are the induced metabolites and blue=colored ones are the repressed metabolites </p>   </div> <p /> </div> <div class="classificationofmetabolitehitsinglobalmetabolicpathways" title="sec"> <div class="title" tagx="title" title="title">
Classification of metabolite hits in global metabolic pathways</div> <p>The induced metabolites were categorized into their cognate metabolic pathways by MetPa online tool using their KEGG IDs. Metabolites classified in global metabolic pathways were plotted in a stacked bar graph (Fig. <a href="#Fig7">7</a>) and their FDR values and <i>p</i> values are shown in Fig. <a href="#Fig8">8</a> and Table <a href="#Tab5">5</a>. Most of the induced metabolites belong to carbon metabolism, fatty acid biosynthesis and amino acid metabolism. The observation supported our hypothesis that co-culturing of <i>P. indica</i> with <i>P. kessleri</i>-I induced metabolites which improve the growth of algal cells and neutral lipid biosynthesis inside the cell. However, for further reaffirmation and deep insight into the cellular metabolism, we had performed HPLC and LC–MS–MS based profiling of few essential metabolites [ <a href="#CR29">29</a>, <a href="#CR30">30</a>]. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 7</span> </a> </div>   <p>Global metabolic pathway classification of metabolomics data of <i>P. indica</i>– <i>P. kessleri</i>-I co-culture. Total number of bioactive metabolites identified in co-culture of <i>Pi</i> and <i>Pk</i>-I ( <i>P. indica</i> and <i>P. kessleri</i>-I), positioned in global metabolic pathways from KEGG database using MetPA online tool of MetaboAnalyst. The bioactive metabolites of co-cultured <i>Pk</i>-I– <i>Pi</i> are classified according to their biological functions </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 8</span> </a> </div>   <p>Enrichment analysis of co-culture metabolites classified in global pathways using <i>p</i> value and fold enrichment pathways as classified in Table <a href="#Tab5">5</a>. Carbon metabolism and glutamate linked pathways showed increased <i>p</i> value with a significant increase in their fold change in the co-culture metabolite hits </p>   </div> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 5</span>  <p>Metabolites classified in global pathways with their <i>p</i> values and FDR values </p>  <table> <thead> <tr> <th>Pathway name</th> <th>Total</th> <th>Hits</th> <th> <i>p</i> value </th> <th>− log( <i>p</i>) </th> <th>FDR</th> <th>Impact</th> </tr> </thead> <tbody> <tr> <td>Alanine, aspartate and glutamate metabolism</td> <td>22</td> <td>3</td> <td>0.0021</td> <td>6.1658</td> <td>0.1827</td> <td>0.33333</td> </tr> <tr> <td>Propanoate metabolism</td> <td>15</td> <td>2</td> <td>0.013831</td> <td>4.2808</td> <td>0.38074</td> <td>0</td> </tr> <tr> <td>Nitrogen metabolism</td> <td>15</td> <td>2</td> <td>0.013831</td> <td>4.2808</td> <td>0.38074</td> <td>0</td> </tr> <tr> <td>Butanoate metabolism</td> <td>18</td> <td>2</td> <td>0.019721</td> <td>3.9261</td> <td>0.38074</td> <td>0</td> </tr> <tr> <td>Selenoamino acid metabolism</td> <td>19</td> <td>2</td> <td>0.021882</td> <td>3.8221</td> <td>0.38074</td> <td>0.00254</td> </tr> <tr> <td>Aminoacyl-tRNA biosynthesis</td> <td>1</td> <td>3</td> <td>0.046322</td> <td>3.0721</td> <td>0.67166</td> <td>0</td> </tr> <tr> <td>Tropane, piperidine and pyridine alkaloid biosynthesis</td> <td>8</td> <td>1</td> <td>0.095412</td> <td>2.3495</td> <td>1</td> <td>0</td> </tr> <tr> <td>Phenylalanine metabolism</td> <td>8</td> <td>1</td> <td>0.095412</td> <td>2.3495</td> <td>1</td> <td>0.5</td> </tr> <tr> <td>Sulfur metabolism</td> <td>12</td> <td>1</td> <td>0.13985</td> <td>1.9672</td> <td>1</td> <td>0.13333</td> </tr> <tr> <td>Ascorbate and aldarate metabolism</td> <td>15</td> <td>1</td> <td>0.17183</td> <td>1.7613</td> <td>1</td> <td>0.22535</td> </tr> <tr> <td>Glyoxylate and dicarboxylate metabolism</td> <td>17</td> <td>1</td> <td>0.19252</td> <td>1.6475</td> <td>1</td> <td>0</td> </tr> <tr> <td>Tyrosine metabolism</td> <td>18</td> <td>1</td> <td>0.20269</td> <td>1.5961</td> <td>1</td> <td>0</td> </tr> <tr> <td>Citrate cycle (TCA cycle)</td> <td>20</td> <td>1</td> <td>0.22266</td> <td>1.5021</td> <td>1</td> <td>0.04626</td> </tr> <tr> <td>Phenylalanine, tyrosine and tryptophan biosynthesis</td> <td>21</td> <td>1</td> <td>0.23247</td> <td>1.459</td> <td>1</td> <td>0</td> </tr> <tr> <td>Carbon fixation in photosynthetic organisms</td> <td>21</td> <td>1</td> <td>0.23247</td> <td>1.459</td> <td>1</td> <td>0</td> </tr> <tr> <td>Pyruvate metabolism</td> <td>21</td> <td>1</td> <td>0.23247</td> <td>1.459</td> <td>1</td> <td>0.09114</td> </tr> <tr> <td>Glycolysis or gluconeogenesis</td> <td>25</td> <td>1</td> <td>0.27056</td> <td>1.3073</td> <td>1</td> <td>0.00147</td> </tr> <tr> <td>Glutathione metabolism</td> <td>26</td> <td>1</td> <td>0.2798</td> <td>1.2737</td> <td>1</td> <td>0.07756</td> </tr> <tr> <td>Tryptophan metabolism</td> <td>27</td> <td>1</td> <td>0.28893</td> <td>1.2416</td> <td>1</td> <td>0.11765</td> </tr> <tr> <td>Porphyrin and chlorophyll metabolism</td> <td>29</td> <td>1</td> <td>0.30686</td> <td>1.1813</td> <td>1</td> <td>0</td> </tr> <tr> <td>Arginine and proline metabolism</td> <td>1</td> <td>1</td> <td>0.38246</td> <td>0.96114</td> <td>1</td> <td>0.14004</td> </tr> <tr> <td>Phenylpropanoid biosynthesis</td> <td>1</td> <td>1</td> <td>0.43583</td> <td>0.8305</td> <td>1</td> <td>0</td> </tr> <tr> <td>Glucosinolate biosynthesis</td> <td>1</td> <td>1</td> <td>0.49812</td> <td>0.69692</td> <td>1</td> <td>0</td> </tr> </tbody> </table> </div> <p /> </div> <div class="quantificationofsuccinateandhydroxy-glutamateconcentrationincellextracts" title="sec"> <div class="title" tagx="title" title="title">
Quantification of succinate and hydroxy-glutamate concentration in cell extracts</div> <p>The concentration of succinate and hydroxy-glutamate (derivatized amino acid) was quantified by High-pressure liquid chromatography (Additional file <a href="#MOESM1">1</a>: Figs. S8–S16). The amount of succinate was increased drastically in the co-cultured cells in comparison to control pure algal cultures and fungal cell extracts (Table <a href="#Tab6">6</a>). The enhanced amount of hydroxy-glutamate (Table <a href="#Tab7">7</a>) was also observed in co-cultured cells’ extract, though it was not too high in comparison to succinate accumulation. <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 6</span>  </div></p><p>Quantification of succinate concentration in controls and co-cultures samples by HPLC-Aminex</p>  <table> <thead> <tr> <th>Samples</th> <th>Succinate amount (µg/mL)</th> </tr> </thead> <tbody> <tr> <td>Algal extract</td> <td>0.39 ± 0.07</td> </tr> <tr> <td>Fungal extract</td> <td>1.00 ± 1.33</td> </tr> <tr> <td>Co-cultured cells extract</td> <td>544.4 ± 45.9</td> </tr> </tbody> </table>  <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 7</span>  <p>Quantification of hydroxy-glutamate concentration in controls and co-cultures samples by HPLC-Aminex </p>  <table> <thead> <tr> <th>Samples</th> <th>Hydroxy-glutamate amount (µg/mL)</th> </tr> </thead> <tbody> <tr> <td>Algal extract</td> <td>1.6 ± 1.2</td> </tr> <tr> <td>Fungal extract</td> <td>0.11 ± 0.06</td> </tr> <tr> <td>Co-cultured cells extract</td> <td>2.23 ± 1.88</td> </tr> </tbody> </table> </div> <p /> </div> <div class="multivariateanalysisoflc–ms–ms" title="sec"> <div class="title" tagx="title" title="title">
Multivariate analysis of LC–MS–MS</div> <p>We generated the multivariate plots after processing the non-nested LC–MS–MS data by Compound Discoverer 3.0 software. The scatter plot shown in Fig. <a href="#Fig9">9</a> was generated to depict the fold change difference of each compound hit, while color predicting its <i>p</i> value. In this plot, each point represents the compound hit followed by their presence in each data set of sample/control ratio. Based on the scatter plot, we observed that the maximum number of hits belongs to the sample (co-culture) with significantly higher fold change and <i>p</i> value having maximum similarity with the compounds found in algal control. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 9</span> </a> </div>   <p>Scatter plot for the compounds exhibiting fold change in the co-culture with respect to individual algal and fungal controls. Each data point in scatter plot represents compound hits and is arranged based on their Log2 fold change for samples in comparison to controls. Changes in the <i>p</i> value of compounds hits are shown by color scale. The increase in Log2 fold changes with significant <i>p</i> value for compounds mostly belongs to the co-culture samples </p>   </div> <p /> <p>Scoring plot generated simultaneously with loading plot during Principal Component Analysis (PCA) was used to group the compound hits together based on sample/control profiles. The PCA score plot also depicted the presence of three separate data set points (containing alga, fungus and co-culture shown in Fig. <a href="#Fig10">10</a>a, b). The PCA score plot, generated to categorize between three profiles, has showed total variance of 53% among the compound data set points. In our multivariate study, the higher fold change was observed in the compound hits with a significant <i>p</i> value particularly in case of co-culture. Further, Partial Least Squares Discriminant Analysis (PLS-DA) plot was generated to differentiate the varying compound hits of groups’ profile. In PLS-DA plot, varying compound hits of groups were highlighted with orange color (Fig. <a href="#Fig11">11</a>). The above data set points were also visualized through the hierarchical clustered analysis of a heatmap. We have observed a significant increase in the <i>p</i> value of succinate (Fig. <a href="#Fig12">12</a>), glutamate (Fig. <a href="#Fig13">13</a>), and GABA (gamma-aminobutyric acid) (Fig. <a href="#Fig14">14</a>). Bioactive compound hits were detected using LC–MS–MS study to support the GC–MS data shown in Additional file <a href="#MOESM1">1</a>: Figs. S17. Detected compound hits in LC–MS–MS metabolomic analysis were classified into their global pathways by Metabolika database linked in Compound Discoverer 3.0 software (Fig. <a href="#Fig15">15</a>). We observed that maximum number of compound hits belonged to carbon metabolism and amino acid biosynthesis pathways. Thus, using LC–MS–MS study, the role of succinate and glutamate was reaffirmed in enhancing the metabolic pathways’ efficiency related to growth modulation in <i>P. kessleri</i>-I when co-cultured with <i>P. indica</i>. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 10</span> </a> </div>   <p> <b>a</b>, <b>b</b> Principal Component Analysis (PCA) score plot and loading plot; <b>a</b> PCA score plot showed the total variance about 53.1% among data sets between each sample group. Dotted line circles are used for grouping the data sets on species profile basis (blue represents algal profile, orange for co-culture, and cyan for fungal profile). Most of the data sets were observed common in each group, only few were diversified. <b>b</b> Loading plot represents the region where peaks showed variation with respect to other sample groups and relative to the data set points shown on PCA score plot </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 11</span> </a> </div>   <p>PLSDA (Partial least square discriminant analysis) plot with each data point representing a compound hit. Highlighted data points with orange color are the compounds which were found to be the cause of variation among sample groups as shown in PCA plot </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 12</span> </a> </div>   <p>Hierarchical cluster analysis visualization through heat map showing significantly high <i>p</i> value for succinate compound in co-culture extracts (highlighted with red color) </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 13</span> </a> </div>   <p>Hierarchical cluster analysis visualization through heat map showing significantly high <i>p</i> value for glutamate and carboxylic acid in co-culture extracts (highlighted with red color) </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 14</span> </a> </div>   <p>Hierarchical cluster analysis visualization through heat map showing significantly high <i>p</i> value for GABA (Gamma Amino glutamyl butyrate) in co-culture extracts (highlighted with red color) </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 15</span> </a> </div>   <p>Global metabolic pathway classification of LC–MS–MS metabolomics data of <i>P. indica</i>– <i>P. kessleri</i>-I co-culture. Total number of bioactive metabolites identified in co-culture of <i>Pi</i> and <i>Pk</i>-I ( <i>P. indica</i> and <i>P. kessleri</i>-I), positioned in global metabolic pathways using Metabolika database of Discoverer 3.0 </p>   </div> <p /> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Owing to interaction with the endophytic fungal strain, the microalga adapts to the biotic stress and usually undergoes a series of changes at the metabolic levels. Indeed, the linkage between the two species has been also confirmed by the double-layer semi-solid agar culturing, where the agar plates that co-cultured <i>P. kessleri</i>-I showed higher number of colonies with respect to control plates (Additional file <a href="#MOESM1">1</a>: Fig. S18 A, B). The associated metabolic changes cause the modulation in the algal growth and lipid profile. Henceforth, deciphering the metabolic profile under stress conditions will be crucial for acquiring the interaction knowledge of the unicellular photoautotrophs with fungal species. Association of endophytic fungal strains with plants or other species has gained attention due to its synergistic and commensal effects. Such studies give insight toward the dynamic relation of cell–cell interaction and its effect at physiological scale [ <a href="#CR37">37</a>, <a href="#CR38">38</a>]. Thus, in our study, we have tried to elucidate the effect of <i>P. indica</i> on <i>P. kessleri</i>-I biomass, lipid profile and associated metabolic changes. Hua and co-authors previously reported the effect of the colonization of Chinese cabbage by <i>P. indica</i>, and revealed the increase in metabolites associated with the TCA (tri-carboxylic acid) cycle of the host organism [ <a href="#CR39">39</a>]. There was also an increase in the intermediate metabolites related to GABA (gamma-aminobutyric acid) synthesis, Tryptophan, and Phenylalanine metabolism. On a similar note, we have also observed an increase in the abundance percentage of TCA cycle-related metabolites like succinate and acetate. The reaction of acetate with coenzyme A (CoA), catalyzed by enzyme Acetyl CoA synthetase, helps in the formation of Acetyl CoA which is involved in many biochemical reactions, especially energy production [ <a href="#CR40">40</a>]. Succinate is a central metabolite since it is an intermediate of TCA cycle and GABA shunt as well as linked with amino acid biosynthesis pathway via the formation of pyruvate or oxaloacetate [ <a href="#CR41">41</a>]. </p> <p>Further, it has been observed that both glutamate and succinate levels are increased, which are major components of the GABA shunt. Therefore, they play major roles in nitrogen metabolism and primary carbon metabolism [ <a href="#CR42">42</a>, <a href="#CR43">43</a>]. There are three catalytic conversion steps in GABA shunt which are as follows: (i) glutamate decarboxylase enzyme catalytically converts <div class="sc_UNKNOWN">
l</div>-glutamate into ɤ-aminobutyric acid (GABA) (ii) GABA is converted to succinate semialdehyde by the action of GABA transaminase (iii) finally, succinic semialdehyde dehydrogenase enzyme converts succinate semialdehyde to succinate [ <a href="#CR43">43</a>, <a href="#CR44">44</a>]. Increase in the abundance of glutamate could be the probable result of the diversion of the TCA cycle to glutamate formation through citrate thus leading to a decrease in the abundance 2-hydroxy butane. Several studies based on plants have evidenced an increase in growth due to GABA shunt internetworking with carbon/nitrogen balance [ <a href="#CR45">45</a>]. Increase in the GABA level is an outcome of biotic or abiotic stress on plant tissues. These outcomes from metabolic profiling depict that modulation of biomass and lipid is the effect of induction of intermediate metabolites of GABA shunt and carbon metabolism that play major roles in sequestering carbon, nitrogen and phosphate inside an algal cell. In our LC–MS–MS-based metabolomics study, we have observed maximum hits related to compounds having phosphate as their functional group. We also observed the abundance of <div class="sc_UNKNOWN">
l</div>-glutamate and succinate in the intracellular extract of co-culture cells in comparison to controls. Other then deciphering the changes occurring at metabolic, physical and morphologically level, the harvesting efficiency of mixed <i>P. indica</i> pellets with <i>Parachlorella kessleri</i>-I at pH 3 after an incubation of ~ 72 h evidenced for ~ 60% harvesting efficiency (Additional file <a href="#MOESM1">1</a>: Fig. S19). However, further detailed analysis at the proteome and transcriptome level needs to be performed in order to elucidate the comprehensive molecular changes of algae under the commensal influence of an endophytic fungus. </p> </div> <div class="concludingremarks" title="sec"> <div class="title" tagx="title" title="title">
Concluding remarks</div> <p>Based on the study carried out to observe the effect of <i>P. indica</i> on <i>P. kessleri</i>-I, we concluded the existence of commensalism between them that helps <i>P. kessleri</i>-I in the modulation of biomass and FAME content. This modulation is an effect of directed changes or enhancement in intracellular metabolic networks. We observed the changes in the central backbone of the carbon metabolic pathway comprising the inter-networks of carbon fixation, pyruvate metabolism, TCA pathway and stress-based propionate metabolism pathway. Thus, this study provided the strong evidence that interaction between the fungal and the algal cells modulated the growth and lipid accumulation of the latter. Thus, the endophytic fungus co-culturing technique may serve as an important advancement for biofuel industries to enhance the biomass, and thus the lipid content of algal strains at commercial level. </p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Additional file</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"> <a href="13068_2019_1516_MOESM1_ESM.doc">LINK</a>  <p> <b>Additional file 1.</b> LCMS and GCMS parameters + Multivariate analysis of GCMS. </p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p> <b>Publisher's Note</b> </p> <p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. </p> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We would like to acknowledge Department of Biotechnology Biofuel Centre of International Center for Genetic Engineering and Biotechnology, New Delhi, India for financial and technical support. </p> </div> <div class="title" tagx="title" title="title">
Authors’ contributions</div> <p>VSB carried out the research experiments and interpreted results and prepared the manuscript draft along with SS. PB provided the fungal strain and helped in experimental study. GHR helped in GCMS study and deconvoluted raw data. KMP assisted in interpreting the results, reviewed and gave suggestions on study, and helped in preparing the final manuscript. SK conceptualized the study and supervised the whole experimental research, data and analysis of results. All authors read and approved the final manuscript. </p> <div class="title" tagx="title" title="title">
Funding</div> <p>This work was supported by the Department of Biotechnology, Government of India, and Grant No. BT/PB/Centre/03/ICGEB/2011-II Phase to SK. </p> <div class="title" tagx="title" title="title">
Availability of data and materials</div> <p>All data generated or analyzed during this study are included in this published article and its additional files. </p> <div class="title" tagx="title" title="title">
Ethics approval and consent to participate</div> <p>Not applicable.</p> <div class="title" tagx="title" title="title">
Consent for publication</div> <p>Not applicable.</p> <div class="title" tagx="title" title="title">
Competing interests</div> <p>The authors declare that they have no competing interests.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="CR1" /> <span class="label" tagx="label" title="label">1.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gendy</span> <span class="given-names" tagx="given-names" title="given-names">TS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">El-temtamy</span> <span class="given-names" tagx="given-names" title="given-names">SA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Commercialization potential aspects of microalgae for biofuel production : an overview</span> <span class="source" tagx="source" title="source">Egypt J Pet</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">22</span> <span class="fpage" tagx="fpage" title="fpage">43</span> <span class="lpage" tagx="lpage" title="lpage">51</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ejpe.2012.07.001">10.1016/j.ejpe.2012.07.001</a> </span> </span> </li> <li tag="ref"> <a name="CR2" /> <span class="label" tagx="label" title="label">2.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ioelovich</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Recent findings and the energetic potential of plant biomass as a renewable source of biofuels - a review </span> <span class="source" tagx="source" title="source">BioResources.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">1879</span> <span class="lpage" tagx="lpage" title="lpage">1914</span> </span> </li> <li tag="ref"> <a name="CR3" /> <span class="label" tagx="label" title="label">3.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mata</span> <span class="given-names" tagx="given-names" title="given-names">TM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martins</span> <span class="given-names" tagx="given-names" title="given-names">AA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Caetano</span> <span class="given-names" tagx="given-names" title="given-names">NS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Microalgae for biodiesel production and other applications: a review</span> <span class="source" tagx="source" title="source">Renew Sustain Energy Rev.</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">217</span> <span class="lpage" tagx="lpage" title="lpage">232</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.rser.2009.07.020">10.1016/j.rser.2009.07.020</a> </span> </span> </li> <li tag="ref"> <a name="CR4" /> <span class="label" tagx="label" title="label">4.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Singh</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nigam</span> <span class="given-names" tagx="given-names" title="given-names">PS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Murphy</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Renewable fuels from algae: an answer to debatable land based fuels</span> <span class="source" tagx="source" title="source">Bioresour Technol.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">102</span> <span class="fpage" tagx="fpage" title="fpage">10</span> <span class="lpage" tagx="lpage" title="lpage">16</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.biortech.2010.06.032">10.1016/j.biortech.2010.06.032</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20615690">20615690</a> </span> </span> </li> <li tag="ref"> <a name="CR5" /> <span class="label" tagx="label" title="label">5.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lam</span> <span class="given-names" tagx="given-names" title="given-names">MK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">KT</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Microalgae biofuels: a critical review of issues, problems and the way forward</span> <span class="source" tagx="source" title="source">Biotechnol Adv</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">30</span> <span class="fpage" tagx="fpage" title="fpage">673</span> <span class="lpage" tagx="lpage" title="lpage">690</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.biotechadv.2011.11.008">10.1016/j.biotechadv.2011.11.008</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22166620">22166620</a> </span> </span> </li> <li tag="ref"> <a name="CR6" /> <span class="label" tagx="label" title="label">6.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Olguín</span> <span class="given-names" tagx="given-names" title="given-names">EJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Dual purpose microalgae–bacteria-based systems that treat wastewater and produce biodiesel and chemical products within a biorefinery </span> <span class="source" tagx="source" title="source">Biotechnol Adv</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">30</span> <span class="fpage" tagx="fpage" title="fpage">1031</span> <span class="lpage" tagx="lpage" title="lpage">1046</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.biotechadv.2012.05.001">10.1016/j.biotechadv.2012.05.001</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22609182">22609182</a> </span> </span> </li> <li tag="ref"> <a name="CR7" /> <span class="label" tagx="label" title="label">7.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rajkumar</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yaakob</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Takriff</span> <span class="given-names" tagx="given-names" title="given-names">MS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Potential of the micro and macro algae for biofuel production: a brief review</span> <span class="source" tagx="source" title="source">BioResources</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">1606</span> <span class="lpage" tagx="lpage" title="lpage">1633</span> </span> </li> <li tag="ref"> <a name="CR8" /> <span class="label" tagx="label" title="label">8.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gultom</span> <span class="given-names" tagx="given-names" title="given-names">SO</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hu</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Review of microalgae harvesting via co-pelletization with filamentous fungus</span> <span class="source" tagx="source" title="source">Energies</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">5921</span> <span class="lpage" tagx="lpage" title="lpage">5939</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3390/en6115921">10.3390/en6115921</a> </span> </span> </li> <li tag="ref"> <a name="CR9" /> <span class="label" tagx="label" title="label">9.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Veiter</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rajamanickam</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Herwig</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The filamentous fungal pellet—relationship between morphology and productivity</span> <span class="source" tagx="source" title="source">Appl Microbiol Biotechnol</span> <span class="year" tagx="year" title="year">2018</span> <span class="volume" tagx="volume" title="volume">102</span> <span class="fpage" tagx="fpage" title="fpage">2997</span> <span class="lpage" tagx="lpage" title="lpage">3006</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00253-018-8818-7">10.1007/s00253-018-8818-7</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29473099">29473099</a> </span> </span> </li> <li tag="ref"> <a name="CR10" /> <span class="label" tagx="label" title="label">10.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Muradov</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Taha</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miranda</span> <span class="given-names" tagx="given-names" title="given-names">AF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wrede</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kadali</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gujar</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Fungal-assisted algal flocculation: application in wastewater treatment and biofuel production </span> <span class="source" tagx="source" title="source">Biotechnol Biofuels</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">24</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/s13068-015-0210-6">10.1186/s13068-015-0210-6</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25763102">25763102</a> </span> </span> </li> <li tag="ref"> <a name="CR11" /> <span class="label" tagx="label" title="label">11.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Metz</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kossen</span> <span class="given-names" tagx="given-names" title="given-names">NWF</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The growth of molds in the form of pellets—a literature review</span> <span class="source" tagx="source" title="source">Biotechnol Bioeng</span> <span class="year" tagx="year" title="year">1977</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="fpage" tagx="fpage" title="fpage">781</span> <span class="lpage" tagx="lpage" title="lpage">799</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/bit.260190602">10.1002/bit.260190602</a> </span> </span> </li> <li tag="ref"> <a name="CR12" /> <span class="label" tagx="label" title="label">12.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oksanen</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Ecological and biotechnological aspects of lichens</span> <span class="source" tagx="source" title="source">Appl Microbiol Biotechnol</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">73</span> <span class="fpage" tagx="fpage" title="fpage">723</span> <span class="lpage" tagx="lpage" title="lpage">734</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00253-006-0611-3">10.1007/s00253-006-0611-3</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17082931">17082931</a> </span> </span> </li> <li tag="ref"> <a name="CR13" /> <span class="label" tagx="label" title="label">13.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Flewelling</span> <span class="given-names" tagx="given-names" title="given-names">AJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Currie</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gray</span> <span class="given-names" tagx="given-names" title="given-names">CA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johnson</span> <span class="given-names" tagx="given-names" title="given-names">JA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Endophytes from marine macroalgae: promising sources of novel natural products</span> <span class="source" tagx="source" title="source">Curr Sci</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">109</span> <span class="fpage" tagx="fpage" title="fpage">88</span> <span class="lpage" tagx="lpage" title="lpage">111</span> </span> </li> <li tag="ref"> <a name="CR14" /> <span class="label" tagx="label" title="label">14.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gill</span> <span class="given-names" tagx="given-names" title="given-names">SS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gill</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Trivedi</span> <span class="given-names" tagx="given-names" title="given-names">DK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Anjum</span> <span class="given-names" tagx="given-names" title="given-names">NA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharma</span> <span class="given-names" tagx="given-names" title="given-names">KK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ansari</span> <span class="given-names" tagx="given-names" title="given-names">MW</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title"> <i>Piriformospora indica</i>: potential and significance in plant stress tolerance </span> <span class="source" tagx="source" title="source">Front Microbiol</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">20</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3389/fmicb.2016.00332">10.3389/fmicb.2016.00332</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26834723">26834723</a> </span> </span> </li> <li tag="ref"> <a name="CR15" /> <span class="label" tagx="label" title="label">15.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Varma</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Verma</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sudha</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sahay</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bütehorn</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Franken</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title"> <i>Piriformospora indica</i>, a cultivable plant-growth-promoting root endophyte </span> <span class="source" tagx="source" title="source">Appl Environ Microbiol</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">65</span> <span class="fpage" tagx="fpage" title="fpage">2741</span> <span class="lpage" tagx="lpage" title="lpage">2744</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10347070">10347070</a> </span> </span> </li> <li tag="ref"> <a name="CR16" /> <span class="label" tagx="label" title="label">16.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peškan-Berghöfer</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shahollari</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pham</span> <span class="given-names" tagx="given-names" title="given-names">HG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hehl</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Markert</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blanke</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Association of <i>Piriformospora indica</i> with <i>Arabidopsis thaliana</i> roots represents a novel system to study beneficial plant–microbe interactions and involves early plant protein modifications in the endoplasmic reticulum and at the plasma membrane </span> <span class="source" tagx="source" title="source">Physiol Plant</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">122</span> <span class="fpage" tagx="fpage" title="fpage">465</span> <span class="lpage" tagx="lpage" title="lpage">477</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1399-3054.2004.00424.x">10.1111/j.1399-3054.2004.00424.x</a> </span> </span> </li> <li tag="ref"> <a name="CR17" /> <span class="label" tagx="label" title="label">17.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Akum</span> <span class="given-names" tagx="given-names" title="given-names">FN</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Steinbrenner</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Biedenkopf</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Imani</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kogel</span> <span class="given-names" tagx="given-names" title="given-names">K-H</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The Piriformospora indica effector PIIN{_}08944 promotes the mutualistic <i>Sebacinalean symbiosis</i> </span> <span class="source" tagx="source" title="source">Front Plant Sci</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">906</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3389/fpls.2015.00906">10.3389/fpls.2015.00906</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26579156">26579156</a> </span> </span> </li> <li tag="ref"> <a name="CR18" /> <span class="label" tagx="label" title="label">18.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zuccaro</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lahrmann</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Güldener</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Langen</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pfiffi</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Biedenkopf</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Endophytic life strategies decoded by genome and transcriptome analyses of the mutualistic root symbiont <i>Piriformospora indica</i> </span> <span class="source" tagx="source" title="source">PLoS Pathog</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">e1002290</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.ppat.1002290">10.1371/journal.ppat.1002290</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22022265">22022265</a> </span> </span> </li> <li tag="ref"> <a name="CR19" /> <span class="label" tagx="label" title="label">19.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ahmad</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fatma</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yazdani</span> <span class="given-names" tagx="given-names" title="given-names">SS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumar</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">DNA barcode and lipid analysis of new marine algae potential for biofuel</span> <span class="source" tagx="source" title="source">Algal</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="fpage" tagx="fpage" title="fpage">10</span> <span class="lpage" tagx="lpage" title="lpage">15</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.algal.2012.10.003">10.1016/j.algal.2012.10.003</a> </span> </span> </li> <li tag="ref"> <a name="CR20" /> <span class="label" tagx="label" title="label">20.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hill</span> <span class="given-names" tagx="given-names" title="given-names">TW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kafer</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Improved protocols for <i>Aspergillus</i> minimal medium: trace element and minimal medium salt stock solutions </span> <span class="source" tagx="source" title="source">Fungal Genet Newsl</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">48</span> <span class="fpage" tagx="fpage" title="fpage">20</span> <span class="lpage" tagx="lpage" title="lpage">21</span> </span> </li> <li tag="ref"> <a name="CR21" /> <span class="label" tagx="label" title="label">21.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johnson</span> <span class="given-names" tagx="given-names" title="given-names">JM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sherameti</span> <span class="given-names" tagx="given-names" title="given-names">I</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nongbri</span> <span class="given-names" tagx="given-names" title="given-names">PL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oelmüller</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Varma</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kost</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oelmüller</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Standardized conditions to study beneficial and nonbeneficial traits in the <i>Piriformospora indica</i>/ <i>Arabidopsis thaliana</i> interaction </span> <span class="source" tagx="source" title="source">Piriformospora indica</span> <span class="year" tagx="year" title="year">2013</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Berlin, Heidelberg</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span> <span class="fpage" tagx="fpage" title="fpage">325</span> <span class="lpage" tagx="lpage" title="lpage">43</span> </span> </li> <li tag="ref"> <a name="CR22" /> <span class="label" tagx="label" title="label">22.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Porra</span> <span class="given-names" tagx="given-names" title="given-names">RJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thompson</span> <span class="given-names" tagx="given-names" title="given-names">WA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kriedemann</span> <span class="given-names" tagx="given-names" title="given-names">PE</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Determination of accurate extinction coefficients and simultaneous-equations for assaying chlorophyll-A and chlorophyll-B extracted with 4 different solvents—verification of the concentration of chlorophyll standards by atomic-absorption spectroscopy </span> <span class="source" tagx="source" title="source">Biochim Biophys Acta</span> <span class="year" tagx="year" title="year">1989</span> <span class="volume" tagx="volume" title="volume">975</span> <span class="fpage" tagx="fpage" title="fpage">384</span> <span class="lpage" tagx="lpage" title="lpage">394</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0005-2728(89)80347-0">10.1016/S0005-2728(89)80347-0</a> </span> </span> </li> <li tag="ref"> <a name="CR23" /> <span class="label" tagx="label" title="label">23.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kimura</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamaoka</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kamisaka</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Rapid estimation of lipids in oleaginous fungi and yeasts using Nile red fluorescence</span> <span class="source" tagx="source" title="source">J Microbiol Methods</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">56</span> <span class="fpage" tagx="fpage" title="fpage">331</span> <span class="lpage" tagx="lpage" title="lpage">338</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.mimet.2003.10.018">10.1016/j.mimet.2003.10.018</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14967224">14967224</a> </span> </span> </li> <li tag="ref"> <a name="CR24" /> <span class="label" tagx="label" title="label">24.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bligh</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dyer</span> <span class="given-names" tagx="given-names" title="given-names">W</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A rapid method of total lipid extraction and purification</span> <span class="source" tagx="source" title="source">Can J Biochem Physiol</span> <span class="year" tagx="year" title="year">1959</span> <span class="volume" tagx="volume" title="volume">37</span> <span class="fpage" tagx="fpage" title="fpage">911</span> <span class="lpage" tagx="lpage" title="lpage">917</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1139/y59-099">10.1139/y59-099</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/13671378">13671378</a> </span> </span> </li> <li tag="ref"> <a name="CR25" /> <span class="label" tagx="label" title="label">25.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rajendran</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hu</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Mycoalgae biofilm: development of a novel platform technology using algae and fungal cultures </span> <span class="source" tagx="source" title="source">Biotechnol Biofuels</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">112</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/s13068-016-0533-y">10.1186/s13068-016-0533-y</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27247623">27247623</a> </span> </span> </li> <li tag="ref"> <a name="CR26" /> <span class="label" tagx="label" title="label">26.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmitt</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Flemming</span> <span class="given-names" tagx="given-names" title="given-names">HC</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">FTIR-spectroscopy in microbial and material analysis</span> <span class="source" tagx="source" title="source">Int Biodeterior Biodegrad</span> <span class="year" tagx="year" title="year">1998</span> <span class="volume" tagx="volume" title="volume">41</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">11</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0964-8305(98)80002-4">10.1016/S0964-8305(98)80002-4</a> </span> </span> </li> <li tag="ref"> <a name="CR27" /> <span class="label" tagx="label" title="label">27.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Villas-Bôas</span> <span class="given-names" tagx="given-names" title="given-names">SG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bruheim</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Cold glycerol-saline: the promising quenching solution for accurate intracellular metabolite analysis of microbial cells </span> <span class="source" tagx="source" title="source">Anal Biochem</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">370</span> <span class="fpage" tagx="fpage" title="fpage">87</span> <span class="lpage" tagx="lpage" title="lpage">97</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ab.2007.06.028">10.1016/j.ab.2007.06.028</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17643383">17643383</a> </span> </span> </li> <li tag="ref"> <a name="CR28" /> <span class="label" tagx="label" title="label">28.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Warnes GR, Bolker B, Bonebakker L, Gentleman R, Huber W, et al. Package ‘gplots’; 2012. </span> </li> <li tag="ref"> <a name="CR29" /> <span class="label" tagx="label" title="label">29.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kanehisa</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Furumichi</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tanabe</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sato</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morishima</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">KEGG: new perspectives on genomes, pathways, diseases and drugs</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">45</span> <span class="fpage" tagx="fpage" title="fpage">D353</span> <span class="lpage" tagx="lpage" title="lpage">D361</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkw1092">10.1093/nar/gkw1092</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27899662">27899662</a> </span> </span> </li> <li tag="ref"> <a name="CR30" /> <span class="label" tagx="label" title="label">30.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Okuda</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamada</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hamajima</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Itoh</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Katayama</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bork</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">KEGG Atlas mapping for global analysis of metabolic pathways</span> <span class="source" tagx="source" title="source">Nucleic Acids Res</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">36</span> <span class="fpage" tagx="fpage" title="fpage">423</span> <span class="lpage" tagx="lpage" title="lpage">426</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkn282">10.1093/nar/gkn282</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18039709">18039709</a> </span> </span> </li> <li tag="ref"> <a name="CR31" /> <span class="label" tagx="label" title="label">31.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xia</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wishart</span> <span class="given-names" tagx="given-names" title="given-names">DS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Using MetaboAnalyst 3.0 for comprehensive metabolomics data analysis</span> <span class="source" tagx="source" title="source">Curr Protoc Bioinform.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">55</span> <span class="fpage" tagx="fpage" title="fpage">14.10.1</span> <span class="lpage" tagx="lpage" title="lpage">4.10.91</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/cpbi.11">10.1002/cpbi.11</a> </span> </span> </li> <li tag="ref"> <a name="CR32" /> <span class="label" tagx="label" title="label">32.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Abo-Shady</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mohamed</span> <span class="given-names" tagx="given-names" title="given-names">YA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lasheen</span> <span class="given-names" tagx="given-names" title="given-names">T</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Chemical composition of the cell wall in some green algae species</span> <span class="source" tagx="source" title="source">Biol Plant</span> <span class="year" tagx="year" title="year">1993</span> <span class="volume" tagx="volume" title="volume">35</span> <span class="fpage" tagx="fpage" title="fpage">629</span> <span class="lpage" tagx="lpage" title="lpage">632</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/BF02928041">10.1007/BF02928041</a> </span> </span> </li> <li tag="ref"> <a name="CR33" /> <span class="label" tagx="label" title="label">33.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mayew</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cuhel</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nyholm</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A simple in vitro fluorescence method for biomass measurements in algal growth inhibition tests </span> <span class="source" tagx="source" title="source">War Res</span> <span class="year" tagx="year" title="year">1997</span> <span class="volume" tagx="volume" title="volume">31</span> <span class="fpage" tagx="fpage" title="fpage">2525</span> <span class="lpage" tagx="lpage" title="lpage">2531</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0043-1354(97)00084-5">10.1016/S0043-1354(97)00084-5</a> </span> </span> </li> <li tag="ref"> <a name="CR34" /> <span class="label" tagx="label" title="label">34.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gustavs</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schumann</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eggert</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Karsten</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">In vivo growth fluorometry: accuracy and limits of microalgal growth rate measurements in ecophysiological investigations </span> <span class="source" tagx="source" title="source">Aquat Microb Ecol</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">55</span> <span class="fpage" tagx="fpage" title="fpage">95</span> <span class="lpage" tagx="lpage" title="lpage">104</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3354/ame01291">10.3354/ame01291</a> </span> </span> </li> <li tag="ref"> <a name="CR35" /> <span class="label" tagx="label" title="label">35.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Oliveros JC. Venny. BioinfoGP Service; 2007. Available from: <a href="http://bioinfogp.cnb.csic.es/tools/venny/index.html">http://bioinfogp.cnb.csic.es/tools/venny/index.html</a>. </span> </li> <li tag="ref"> <a name="CR36" /> <span class="label" tagx="label" title="label">36.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gowda</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ivanisevic</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johnson</span> <span class="given-names" tagx="given-names" title="given-names">CH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kurczy</span> <span class="given-names" tagx="given-names" title="given-names">ME</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Benton</span> <span class="given-names" tagx="given-names" title="given-names">HP</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rinehart</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Interactive XCMS online: simplifying advanced metabolomic data processing and subsequent statistical analyses </span> <span class="source" tagx="source" title="source">Anal Chem.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">86</span> <span class="fpage" tagx="fpage" title="fpage">6931</span> <span class="lpage" tagx="lpage" title="lpage">6939</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/ac500734c">10.1021/ac500734c</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24934772">24934772</a> </span> </span> </li> <li tag="ref"> <a name="CR37" /> <span class="label" tagx="label" title="label">37.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schulthess</span> <span class="given-names" tagx="given-names" title="given-names">FM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Faeth</span> <span class="given-names" tagx="given-names" title="given-names">SH</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Distribution, abundances, and associations of the endophytic fungal community of Arizona Fescue ( <i>Festuca arizonica</i>) </span> <span class="source" tagx="source" title="source">Mycologia</span> <span class="year" tagx="year" title="year">1998</span> <span class="volume" tagx="volume" title="volume">90</span> <span class="fpage" tagx="fpage" title="fpage">569</span> <span class="lpage" tagx="lpage" title="lpage">578</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1080/00275514.1998.12026945">10.1080/00275514.1998.12026945</a> </span> </span> </li> <li tag="ref"> <a name="CR38" /> <span class="label" tagx="label" title="label">38.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Faeth</span> <span class="given-names" tagx="given-names" title="given-names">SH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fagan</span> <span class="given-names" tagx="given-names" title="given-names">WF</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Fungal endophytes: common host plant symbionts but uncommon mutualists</span> <span class="source" tagx="source" title="source">Integr Comp Biol</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">42</span> <span class="fpage" tagx="fpage" title="fpage">360</span> <span class="lpage" tagx="lpage" title="lpage">368</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/icb/42.2.360">10.1093/icb/42.2.360</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21708729">21708729</a> </span> </span> </li> <li tag="ref"> <a name="CR39" /> <span class="label" tagx="label" title="label">39.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hua</span> <span class="given-names" tagx="given-names" title="given-names">MDS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Senthil Kumar</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shyur</span> <span class="given-names" tagx="given-names" title="given-names">LF</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheng</span> <span class="given-names" tagx="given-names" title="given-names">YB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tian</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oelmüller</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Metabolomic compounds identified in <i>Piriformospora indica</i>—colonized Chinese cabbage roots delineate symbiotic functions of the interaction </span> <span class="source" tagx="source" title="source">Sci Rep</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">14</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/s41598-016-0028-x">10.1038/s41598-016-0028-x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28127051">28127051</a> </span> </span> </li> <li tag="ref"> <a name="CR40" /> <span class="label" tagx="label" title="label">40.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Behal</span> <span class="given-names" tagx="given-names" title="given-names">RH</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lin</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Back</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oliver</span> <span class="given-names" tagx="given-names" title="given-names">DJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Role of acetyl-coenzyme A synthetase in leaves of <i>Arabidopsis thaliana</i> </span> <span class="source" tagx="source" title="source">Arch Biochem Biophys</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">402</span> <span class="fpage" tagx="fpage" title="fpage">259</span> <span class="lpage" tagx="lpage" title="lpage">267</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0003-9861(02)00086-3">10.1016/S0003-9861(02)00086-3</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12051672">12051672</a> </span> </span> </li> <li tag="ref"> <a name="CR41" /> <span class="label" tagx="label" title="label">41.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huber</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gallenberger</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jahn</span> <span class="given-names" tagx="given-names" title="given-names">U</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eylert</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Berg</span> <span class="given-names" tagx="given-names" title="given-names">IA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kockelkorn</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A dicarboxylate/4-hydroxybutyrate autotrophic carbon assimilation cycle in the hyperthermophilic Archaeum Ignicoccus hospitalis </span> <span class="source" tagx="source" title="source">Proc Natl Acad Sci</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">105</span> <span class="fpage" tagx="fpage" title="fpage">7851</span> <span class="lpage" tagx="lpage" title="lpage">7856</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.0801043105">10.1073/pnas.0801043105</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18511565">18511565</a> </span> </span> </li> <li tag="ref"> <a name="CR42" /> <span class="label" tagx="label" title="label">42.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Narayan</span> <span class="given-names" tagx="given-names" title="given-names">VS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nair</span> <span class="given-names" tagx="given-names" title="given-names">PM</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Metabolism, enzymology and possible roles of 4-aminobutyrate in higher plants</span> <span class="source" tagx="source" title="source">Phytochemistry</span> <span class="year" tagx="year" title="year">1990</span> <span class="volume" tagx="volume" title="volume">29</span> <span class="fpage" tagx="fpage" title="fpage">367</span> <span class="lpage" tagx="lpage" title="lpage">375</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/0031-9422(90)85081-P">10.1016/0031-9422(90)85081-P</a> </span> </span> </li> <li tag="ref"> <a name="CR43" /> <span class="label" tagx="label" title="label">43.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Busch</span> <span class="given-names" tagx="given-names" title="given-names">KB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fromm</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Plant succinic semialdehyde dehydrogenase. Cloning, purification, localization in mitochondria, and regulation by adenine nucleotides </span> <span class="source" tagx="source" title="source">Plant Physiol</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">121</span> <span class="fpage" tagx="fpage" title="fpage">589</span> <span class="lpage" tagx="lpage" title="lpage">598</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.121.2.589">10.1104/pp.121.2.589</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10517851">10517851</a> </span> </span> </li> <li tag="ref"> <a name="CR44" /> <span class="label" tagx="label" title="label">44.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fait</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fromm</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Walter</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Galili</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fernie</span> <span class="given-names" tagx="given-names" title="given-names">AR</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Highway or byway: the metabolic role of the GABA shunt in plants</span> <span class="source" tagx="source" title="source">Trends Plant Sci</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">13</span> <span class="fpage" tagx="fpage" title="fpage">14</span> <span class="lpage" tagx="lpage" title="lpage">19</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.tplants.2007.10.005">10.1016/j.tplants.2007.10.005</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18155636">18155636</a> </span> </span> </li> <li tag="ref"> <a name="CR45" /> <span class="label" tagx="label" title="label">45.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shelp</span> <span class="given-names" tagx="given-names" title="given-names">BJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Walton</span> <span class="given-names" tagx="given-names" title="given-names">CS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Snedden</span> <span class="given-names" tagx="given-names" title="given-names">WA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tuin</span> <span class="given-names" tagx="given-names" title="given-names">LG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oresnik</span> <span class="given-names" tagx="given-names" title="given-names">IJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Layzell</span> <span class="given-names" tagx="given-names" title="given-names">DB</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Gaba shunt in developing soybean seeds is associated with hypoxia</span> <span class="source" tagx="source" title="source">Physiol Plant</span> <span class="year" tagx="year" title="year">1995</span> <span class="volume" tagx="volume" title="volume">94</span> <span class="fpage" tagx="fpage" title="fpage">219</span> <span class="lpage" tagx="lpage" title="lpage">228</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1399-3054.1995.tb05304.x">10.1111/j.1399-3054.1995.tb05304.x</a> </span> </span> </li> </ul> </div> </div>  </body></html>