<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Biotechnol Res Int</span> <span class="iso-abbrev" title="iso-abbrev">Biotechnol Res Int</span> <span class="publisher-id" title="publisher-id">BTRI</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Biotechnology Research International</span> </div> <span class="issn-ppub" tagx="issn" title="issn-ppub">2090-3138</span> <span class="issn-epub" tagx="issn" title="issn-epub">2090-3146</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Hindawi Publishing Corporation</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5225391">5225391</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1155/2016/4769542">10.1155/2016/4769542</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Optimization of Crude Oil and PAHs Degradation by <i> Stenotrophomonas rhizophila</i> KX082814 Strain through Response Surface Methodology Using Box-Behnken Design </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Virupakshappa</span> <span class="given-names" tagx="given-names" title="given-names">Praveen Kumar Siddalingappa</span> </span> <a href="#I1" /> </span> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-4721-2062</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Krishnaswamy</span> <span class="given-names" tagx="given-names" title="given-names">Manjunatha Bukkambudhi</span> </span> <a href="#I1" /> <a href="#cor1"> <sup>*</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mishra</span> <span class="given-names" tagx="given-names" title="given-names">Gaurav</span> </span> <a href="#I1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mehkri</span> <span class="given-names" tagx="given-names" title="given-names">Mohammed Ameenuddin</span> </span> <a href="#I1" /> </span> </div> <span class="citation_author_institution" id="I1">[], </span> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
*Manjunatha Bukkambudhi Krishnaswamy: <span class="email" tagx="email" title="email">professorbkm@gmail.com</span> </div> <div class="fn-type-other" title="other"> <p>Academic Editor: Felipe Garcia-Rodriguez</p> </div> </div> <span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2016</span> </span> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2016-12-12</span> </span> <span class="volume" tagx="volume" title="volume">2016</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">4769542</span> <span class="history" title="history"> <span class="received" title="received">received: 2016-6-22</span> <span class="rev-recd" title="rev-recd">rev-recd: 2016-9-27</span> <span class="accepted" title="accepted">accepted: 2016-10-13</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2016</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open access article distributed under the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>The present paper describes the process optimization study for crude oil degradation which is a continuation of our earlier work on hydrocarbon degradation study of the isolate <i> Stenotrophomonas rhizophila </i>(PM-1) with GenBank accession number <i> KX082814</i>. Response Surface Methodology with Box-Behnken Design was used to optimize the process wherein temperature, pH, salinity, and inoculum size (at three levels) were used as independent variables and Total Petroleum Hydrocarbon, Biological Oxygen Demand, and Chemical Oxygen Demand of crude oil and PAHs as dependent variables (response). The statistical analysis, via ANOVA, showed coefficient of determination <i>R</i> <sup>2</sup> as 0.7678 with statistically significant <i>P</i> value 0.0163 fitting in second-order quadratic regression model for crude oil removal. The predicted optimum parameters, namely, temperature, pH, salinity, and inoculum size, were found to be 32.5°C, 9, 12.5, and 12.5 mL, respectively. At this optimum condition, the observed and predicted PAHs and crude oil removal were found to be 71.82% and 79.53% in validation experiments, respectively. The % TPH results correlate with GC/MS studies, BOD, COD, and TPC. The validation of numerical optimization was done through GC/MS studies and % removal of crude oil. </p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
Naval Research Board</div> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
DRDO</div> </div> </div> </div> </div> <div class="body" title="body"> <div class="1.introduction" title="sec"> <div class="title" tagx="title" title="title">
1. Introduction</div> <p>Bioremediation is an ecologically acceptable technology that employs the use of microorganisms to efficiently degrade pollutants [ <a href="#B1">1</a>]. The strain <i> Stenotrophomonas rhizophila</i> (PM-1) showed potential crude oil and PAHs degrading ability in our earlier report [ <a href="#B5">2</a>]. In the present study, an attempt has been made to optimize the process of bioremediation through Response Surface Methodology (RSM), which is a reliable and powerful tool for modelling and optimization of bioremediation processes [ <a href="#B1">1</a>]. </p> <p>In RSM, the Box-Behnken Design is having the maximum efficiency for an experiment involving three factors and three levels; further, the number of experiments conducted for this is much less compared to a central composite design. Box-Behnken Designs always have three levels for each factor and are purpose built to fit a quadratic model [ <a href="#B2">3</a>, <a href="#B3">4</a>]. The Box-Behnken Design does not have runs at the extreme combinations of all the factors but compensates by having better prediction precision in the centre of the factor space. While a run or two can be botched in these designs the accuracy of the observations in the remaining runs is critical to the dependability of the model. Categoric factors can be added to these designs; however, the design is duplicated for every categoric treatment combination. It is well established that, in biological treatment processes, various operational parameters such as the level of temperature, salinity concentration, inoculum size, and pH directly influence the bacterial degradation performance of PAHs and crude oil [ <a href="#B7">5</a>]. Thus, to make the process more efficient, faster, and practically applicable, studies on the effect of each factor on the bacterial degradation of PAHs and crude oil appear essential [ <a href="#B4">6</a>]. Hence in this context, the present study was designed with an attempt to optimize cultural (pH, temperature, dose of inoculum, and salinity concentration) factors using conventional (one-factor-at-a-time) and statistical Response Surface Methodologies (RSMs) for degradation of crude oil and PAHs by the strain <i> Stenotrophomonas rhizophila </i>PM-1. </p> <p>The present work discusses the use of Box-Behnken Design approach to plan the experiments for crude oil degradation with an overall objective of optimizing the process to degrade the crude oil. </p> </div> <div class="2.materialsandmethod" title="sec"> <div class="title" tagx="title" title="title">
2. Materials and Method</div> <div class="2.1.soilsamplingandisolationofbacteria" title="sec"> <div class="title" tagx="title" title="title">
2.1. Soil Sampling and Isolation of Bacteria</div> <p>The soil samples were collected from different localities of Western Ghats of Karnataka State, covering the oil spilled areas and the hydrocarbon degrading bacteria were isolated using R <sub>2</sub>A media followed by serial dilution using standard protocols. </p> </div> <div class="2.2.preliminarydegradationstudies,pahs,andcrudeoilutilizationstudies" title="sec"> <div class="title" tagx="title" title="title">
2.2. Preliminary Degradation Studies, PAHs, and Crude Oil Utilization Studies</div> <p>The preliminary degradation was studied using redox indicator 2,6-dichlorophenolindophenol (DCPIP) [ <a href="#B8">7</a>]. The 2% PAHs and crude oil utilization by the bacterial isolate using Bacto Bushnell Hass broth was studied using decanol, hexadecane, toluene, dodecane, engine oil, benzene, octane, oleic acid, and naphthalene as sole carbon source and checked for utilization. </p> </div> <div class="2.3.experimentaldesign" title="sec"> <div class="title" tagx="title" title="title">
2.3. Experimental Design</div> <p>The Box-Behnken factorial experimental design had employed four independent variables, namely, temperature (25, 30, 35, and 40°C), pH (5, 7, 9, and 11), salinity (5, 10, 15, and 20), and inoculum size (5, 10, 15, and 20 ml) as mentioned in <a href="#tab1">Table 1</a>. Each of the independent variables was studied at three levels (1, 0, and +1), with 29 experimental runs and one control. The levels were selected based on the results of experimental designs as shown in <a href="#tab2">Table 2</a>. The full strength media with 2% crude oil/PAHs act as a control. The statistical software Design-Expert® 10 (Stat-Ease, Inc., Minneapolis, MN, USA) was used to evaluate the analysis of variance ( <i>P</i> &amp;lt; 0.05) to determine the significance of each term in the fitted equations and to estimate the goodness of fit in each case. In order to visualise the relationship between the experimental variables and responses, 3D plots are generated from the models. The optimum variables are obtained from the response surface. </p> </div> <div class="2.4.extractionofresidualoilandtotalpetroleumhydrocarbon(tph)analysis" title="sec"> <div class="title" tagx="title" title="title">
2.4. Extraction of Residual Oil and Total Petroleum Hydrocarbon (TPH) Analysis</div> <p>Based on the preliminary degradation results, the potential bacterial strains were selected and checked for the utilization capability of the crude oil. The isolates were inoculated into the conical flask containing Bacto Bushnell Hass broth in artificial sea water along with 2% crude oil and dextrose as additional carbon source [ <a href="#B5">2</a>]. The flask was monitored at regular intervals of time up to 15 days. The flasks were observed for any changes in the physical nature of the oil. </p> </div> <div class="2.5.biologicaloxygendemand(bod)andchemicaloxygendemand(cod)analysis" title="sec"> <div class="title" tagx="title" title="title">
2.5. Biological Oxygen Demand (BOD) and Chemical Oxygen Demand (COD) Analysis</div> <p>In order to assess the rate of degradation and PAHs utilization by the isolate <i> S. rhizophila</i> KX082814, the BOD and COD analysis were performed by following standard methods (APHA, 2001, and IS-3025). </p> </div> <div class="2.6.gc/msanalysistovalidatethersmdesign" title="sec"> <div class="title" tagx="title" title="title">
2.6. GC/MS Analysis to Validate the RSM Design</div> <p>The GC/MS analysis was performed using a MS-5973 spectrometer coupled to a Hewlett-Packard Model 6890 and GC equipped with a cool-on-column inlet and capillary direct interface. The instrument conditions were the following: capillary column HP-1MS, 60 m × 0.2 mm; helium column flow 1 ml/min; pressure 18.5 psi; and split ratio 20 : 1. The initial temperature was 70°C and kept for 5 minutes with a temperature ramp of 14°C per minute and final temperature of 280°C was kept for 10 minutes with total run time 3024 minutes. A solvent delay was employed in order to prolong detector lifetime from 0 to 4.5 minutes. The solvent front reached the detector in 4.0 minutes and initial analyte retention time was approximately 7 minutes, so there was no loss of resolution due to initial solvent delay. The solvent used in all analyses was mixture of hexanes. </p> </div> <div class="2.7.16srrnasequencingandncbigenebankdeposition" title="sec"> <div class="title" tagx="title" title="title">
2.7. 16s rRNA Sequencing and NCBI Gene Bank Deposition</div> <p>Bacterial identification was carried through 16s rRNA sequencing. Bacterial genomic DNA was isolated using the Insta GeneTM Matrix genomic DNA isolation kit Catalog # 732-6030. Using 16s rRNA universal primers gene fragment was amplified using MJ Research PTC-225 Peltier Thermal Cycler. The sequence obtained is deposited in the NCBI gene bank using the tool Sequin. </p> </div> </div> <div class="3.resultsanddiscussions" title="sec"> <div class="title" tagx="title" title="title">
3. Results and Discussions</div> <p>The present study was undertaken to examine the cumulative effect of four different parameters on degradation of crude oil and PAHs. The second-order polynomial coefficient for each term of the equation was determined through multiple regression analysis using the Design-Expert v.10. The experimental design and response for each trail were mentioned in <a href="#tab3">Table 3</a>. Maximum degradation was observed in case of run number 10 followed by 11, 17, and 7, where % degradation was found to be 70.25, 69.05, 68.54, and 67.84, respectively. It was noticed that the similar results were observed in case of runs number 10 and 11; this may be due to the identical experimental conditions, whereas in case of runs number 7 and 17, although the experimental conditions are different, the results show the highest rate of degradation was almost equal to the runs number 10 and 11. Hence according to the model, the runs number 10 and 11 as standard optimized conditions were selected as the optimum conditions to enhance the degradation. Further, the obtained optimized conditions from the RSM-BBD were confirmed and validated by experimental studies at standard conditions. </p> <p>The obtained results were checked for their fitness to the model and obtained data was illustrated. The predicted and actual values for the model, Cook's distance, and studentized residuals illustrate the normal distribution and constant variance of the residuals; according to <a href="#tab4">Table 4</a> and <a href="#fig4">Figure 4</a>, there were no points that were potentially powerful due to their location in the factor indicating the goodness of fit. </p> <p>By the model, <i>F</i>-value of 3.31 implies that the model is significant ( <a href="#tab5">Table 5</a>). Values of "Prob &amp;gt; <i>F</i>" less than 0.0500 indicate model terms are significant. In this case <i>A</i>, <i>C</i>, <i>A</i> <sup>2</sup>, and <i>C</i> <sup>2</sup> are significant model terms as values greater than 0.1000 indicate the model terms are not significant. The lack of fit <i>F</i>-value of 0.19 implies the lack of fit is not significantly relative to the pure error. There is a 98.50% chance that a lack of fit <i>F</i>-value this large could occur due to noise. Nonsignificant lack of fit is good and we want the model to fit. The "Pred <i>R</i> <sup>2</sup>" of 0.3240 is not as close to the "Adj <i>R</i> <sup>2</sup>" of 0.5355 as one might normally expect; that is, the difference is more than 0.2. This may indicate a large block effect or a possible problem with our model and/or data. Things to consider are model reduction, response transformation, outliers, and so forth. All empirical models should be tested by confirmation runs. "Adeq Precision" measures the signal to noise ratio. </p> <p> <i>Final Equation in Terms of Coded Factors</i> </p><div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">(1)</span> <div class="math_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="malignmark_UNKNOWN" /> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
 </div> <div class="mo_UNKNOWN">
 </div> <div class="mi_UNKNOWN">
P</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
l</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
u</div> <div class="mi_UNKNOWN">
m</div> <div class="mo_UNKNOWN">
 </div> <div class="mo_UNKNOWN">
 </div> <div class="mi_UNKNOWN">
H</div> <div class="mi_UNKNOWN">
y</div> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
b</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
 </div> <div class="mo_UNKNOWN">
 </div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
%</div> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
P</div> <div class="mi_UNKNOWN">
H</div> <div class="mspace_UNKNOWN" /> </div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="maligngroup_UNKNOWN" /> <div class="malignmark_UNKNOWN" /> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
64.76</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
2.67</div> <div class="mi_UNKNOWN">
∗</div> <div class="mi_UNKNOWN">
A</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1.34</div> <div class="mi_UNKNOWN">
∗</div> <div class="mi_UNKNOWN">
B</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
2.53</div> <div class="mi_UNKNOWN">
∗</div> <div class="mi_UNKNOWN">
C</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
0.013</div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="maligngroup_UNKNOWN" /> <div class="malignmark_UNKNOWN" /> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
∗</div> <div class="mi_UNKNOWN">
D</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
0.42</div> <div class="mi_UNKNOWN">
∗</div> <div class="mi_UNKNOWN">
A</div> <div class="mi_UNKNOWN">
B</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
2.04</div> <div class="mi_UNKNOWN">
∗</div> <div class="mi_UNKNOWN">
A</div> <div class="mi_UNKNOWN">
C</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
3.77</div> <div class="mi_UNKNOWN">
∗</div> <div class="mi_UNKNOWN">
A</div> <div class="mi_UNKNOWN">
D</div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="maligngroup_UNKNOWN" /> <div class="malignmark_UNKNOWN" /> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
0.48</div> <div class="mi_UNKNOWN">
∗</div> <div class="mi_UNKNOWN">
B</div> <div class="mi_UNKNOWN">
C</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
2.14</div> <div class="mi_UNKNOWN">
∗</div> <div class="mi_UNKNOWN">
B</div> <div class="mi_UNKNOWN">
D</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
2.89</div> <div class="mi_UNKNOWN">
∗</div> <div class="mi_UNKNOWN">
C</div> <div class="mi_UNKNOWN">
D</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
5.98</div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="maligngroup_UNKNOWN" /> <div class="malignmark_UNKNOWN" /> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
∗</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
0.81</div> <div class="mi_UNKNOWN">
∗</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
B</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
3.96</div> <div class="mi_UNKNOWN">
∗</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
0.52</div> <div class="mi_UNKNOWN">
∗</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
,</div> </div> </div> </div> </div> </div>where <i>A</i>, <i>B</i>, <i>C</i>, and <i>D</i> are the coded values of the test variables ( <a href="#tab6">Table 6</a>), temperature (°C), pH, salinity, and inoculum size (ml), respectively. The equation in terms of coded factors can be used to make predictions about the response for given levels of each factor. By default, the high levels of the factors are coded as +1 and the low levels of the factors are coded as −1. The coded equation is useful for identifying the relative impact of the factors by comparing the factor coefficients. <p /> <p> <i>Final Equation in Terms of Actual Factors</i> </p><div class="disp-formula" title="disp-formula"> <span class="label" tagx="label" title="label">(2)</span> <div class="math_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="malignmark_UNKNOWN" /> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
 </div> <div class="mo_UNKNOWN">
 </div> <div class="mi_UNKNOWN">
P</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
l</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
u</div> <div class="mi_UNKNOWN">
m</div> <div class="mo_UNKNOWN">
 </div> <div class="mo_UNKNOWN">
 </div> <div class="mi_UNKNOWN">
H</div> <div class="mi_UNKNOWN">
y</div> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
b</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
 </div> <div class="mo_UNKNOWN">
 </div> <div class="mfenced_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
P</div> <div class="mi_UNKNOWN">
H</div> <div class="mspace_UNKNOWN" /> </div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="maligngroup_UNKNOWN" /> <div class="malignmark_UNKNOWN" /> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
71.82338</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
7.08541</div> <div class="mi_UNKNOWN">
∗</div> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
m</div> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
u</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
0.17307</div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="maligngroup_UNKNOWN" /> <div class="malignmark_UNKNOWN" /> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
∗</div> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
H</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1.05685</div> <div class="mi_UNKNOWN">
∗</div> <div class="mi_UNKNOWN">
S</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
l</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
n</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
y</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
3.81374</div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="maligngroup_UNKNOWN" /> <div class="malignmark_UNKNOWN" /> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
∗</div> <div class="mi_UNKNOWN">
I</div> <div class="mi_UNKNOWN">
n</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
u</div> <div class="mi_UNKNOWN">
l</div> <div class="mi_UNKNOWN">
u</div> <div class="mi_UNKNOWN">
m</div> <div class="mo_UNKNOWN">
 </div> <div class="mo_UNKNOWN">
 </div> <div class="mi_UNKNOWN">
S</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
z</div> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
0.018667</div> <div class="mi_UNKNOWN">
∗</div> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
m</div> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
u</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="maligngroup_UNKNOWN" /> <div class="malignmark_UNKNOWN" /> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
∗</div> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
H</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
0.036222</div> <div class="mi_UNKNOWN">
∗</div> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
m</div> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
u</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
∗</div> <div class="mi_UNKNOWN">
S</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
l</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
n</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
y</div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="maligngroup_UNKNOWN" /> <div class="malignmark_UNKNOWN" /> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
0.067067</div> <div class="mi_UNKNOWN">
∗</div> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
m</div> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
u</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
∗</div> <div class="mi_UNKNOWN">
I</div> <div class="mi_UNKNOWN">
n</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
u</div> <div class="mi_UNKNOWN">
l</div> <div class="mi_UNKNOWN">
u</div> <div class="mi_UNKNOWN">
m</div> <div class="mo_UNKNOWN">
 </div> <div class="mo_UNKNOWN">
 </div> <div class="mi_UNKNOWN">
S</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
z</div> <div class="mi_UNKNOWN">
e</div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="maligngroup_UNKNOWN" /> <div class="malignmark_UNKNOWN" /> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
0.021222</div> <div class="mi_UNKNOWN">
∗</div> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
H</div> <div class="mi_UNKNOWN">
∗</div> <div class="mi_UNKNOWN">
S</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
l</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
n</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
y</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
0.095222</div> <div class="mi_UNKNOWN">
∗</div> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
H</div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="maligngroup_UNKNOWN" /> <div class="malignmark_UNKNOWN" /> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
∗</div> <div class="mi_UNKNOWN">
I</div> <div class="mi_UNKNOWN">
n</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
u</div> <div class="mi_UNKNOWN">
l</div> <div class="mi_UNKNOWN">
u</div> <div class="mi_UNKNOWN">
m</div> <div class="mo_UNKNOWN">
 </div> <div class="mo_UNKNOWN">
 </div> <div class="mi_UNKNOWN">
S</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
z</div> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
0.051422</div> <div class="mi_UNKNOWN">
∗</div> <div class="mi_UNKNOWN">
S</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
l</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
n</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
y</div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="maligngroup_UNKNOWN" /> <div class="malignmark_UNKNOWN" /> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
∗</div> <div class="mi_UNKNOWN">
I</div> <div class="mi_UNKNOWN">
n</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
u</div> <div class="mi_UNKNOWN">
l</div> <div class="mi_UNKNOWN">
u</div> <div class="mi_UNKNOWN">
m</div> <div class="mo_UNKNOWN">
 </div> <div class="mo_UNKNOWN">
 </div> <div class="mi_UNKNOWN">
S</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
z</div> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
0.10625</div> <div class="mi_UNKNOWN">
∗</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
m</div> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
u</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="maligngroup_UNKNOWN" /> <div class="malignmark_UNKNOWN" /> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
0.090120</div> <div class="mi_UNKNOWN">
∗</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
H</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
0.070359</div> <div class="mi_UNKNOWN">
∗</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
l</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
n</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
y</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="maligngroup_UNKNOWN" /> <div class="malignmark_UNKNOWN" /> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
9.24741</div> <div class="mi_UNKNOWN">
E</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
003</div> <div class="mi_UNKNOWN">
∗</div> <div class="mi_UNKNOWN">
I</div> <div class="mi_UNKNOWN">
n</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
u</div> <div class="mi_UNKNOWN">
l</div> <div class="mi_UNKNOWN">
u</div> <div class="mi_UNKNOWN">
m</div> <div class="mo_UNKNOWN">
 </div> <div class="mo_UNKNOWN">
 </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
z</div> <div class="mi_UNKNOWN">
e</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> </div> </div> </div> </div>The equation in terms of actual factors can be used to make predictions about the response for given levels of each factor. Here, the levels should be specified in the original units for each factor. This equation should not be used to determine the relative impact of each factor because the coefficients are scaled to accommodate the units of each factor and the intercept is not at the centre of the design space. The Diagnostic Plots are as shown in Figures <a href="#fig1">1</a> <a href="#fig2" /> <a href="#fig3" /> <a href="#fig4" />– <a href="#fig5">5</a>. <a href="#fig1"> Figure 1</a> shows the normal probability plot of the studentized residuals to check for normality of residuals. <a href="#fig2"> Figure 2</a> shows studentized residuals versus predicted values to check for constant error in the design. <a href="#fig3"> Figure 3</a> shows externally studentized residuals to look for outliers, that is, influential values of the design. <a href="#fig4">Figure 4</a> shows Box-Cox plot for power transformations, where Cook's distance and studentized residual illustrate the normal distribution and constant variance of the residuals and <a href="#fig5">Figure 5</a> shows the interactions among factors that influence crude oil and PAHs degradation by the isolate <i> S. rhizophila</i> (PM-1) KX082814. <p /> <p>The response surface curves show the relative effects of two variables, by keeping the other variable at fixed level, on crude oil degradation. The 3D plots and cubic designs are shown in Figures <a href="#fig6">6</a> <a href="#fig7" /> <a href="#fig8" />– <a href="#fig9">9</a>. The result obtained shows that pH of 8, temperature of 32.5°C, inoculum size of 12.5 ml, and salinity concentration of 12.5 were the best conditions to obtain maximum degradation of crude oil and PAHs using the bacterial isolate <i> S. rhizophila</i> KX082814. The optimal values for the variables as predicted by the RSM were found within the Box-Behnken Design region. </p> <p> <i>Validation of Optimization Process</i>. Validation experiments were conducted in triplicate to determine the performance of <i> S. rhizophila</i> KX082814 and reproducibility of the results by evaluating the level of BOD, COD, TPC, % TPH, and GC/MS studies at the optimum favourable conditions through Box-Behnken Design and RSM ( <a href="#tab7">Table 7</a>; <a href="#fig10">Figure 10</a>). The results showed 79.53 ± 2.5% of crude oil removal efficiency. The percentage error between the predicted and actual values was found to be 0.75%. The GC/MS study clearly indicates the removal of majority of both aromatic and aliphatic hydrocarbons present in the crude oil (Figures <a href="#fig11">11</a> and <a href="#fig12">12</a>; <a href="#tab8">Table 8</a>). A total of 42 different hydrocarbon components were observed in control sample, where after the bioremediation treatment with the isolate <i> S. rhizophila (KX082814),</i> it was noticed that only 20 hydrocarbon components remained in the test sample after 15th day. It was observed that majority of the components in the control samples are completely degraded and some peaks are converted to simpler hydrocarbon moieties. From the results the noticeable difference in the degradation process was clearly observed. </p> </div> <div class="4.conclusion" title="sec"> <div class="title" tagx="title" title="title">
4. Conclusion</div> <p>This study reveals the bioremediation of crude oil and PAHs utilization by the isolate <i> Stenotrophomonas rhizophila</i> (PM-1) with gene bank accession number KX082814 could be achieved up to 79.53 ± 2.5% by maintaining the optimum parameters, namely, temperature, pH, salinity, and inoculum size. The GC/MS studies also indicate that the degradation of majority of the hydrocarbon components, the statistical analyses, and the closeness of the experimental results and model predictions show the reliability of the regression model. </p> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>The authors thank Naval Research Board (Material Panel), DRDO, (DNRD/05/4003/NRB/291), the government of India for the financial support of this work. The authors are thankful to SAIF Centre, IIT Madras for GCMS facility, and management, principal, and head of department of Biotechnology, The Oxford College of Engineering, Bengaluru, for their constant support and facility. </p> </div> <div class="COI-statement" title="COI-statement"> <div class="title" tagx="title" title="title">
Competing Interests</div> <p>The authors declare that there is no conflict of interests.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <li tag="ref"> <a name="B1" /> <span class="label" tagx="label" title="label">1</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Agarry</span> <span class="given-names" tagx="given-names" title="given-names">S. E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ogunleye</span> <span class="given-names" tagx="given-names" title="given-names">O. O.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Box-Behnken design application to study enhanced bioremediation of soil artificially contaminated with spent engine oil using biostimulation strategy </span> <span class="source" tagx="source" title="source"> <i>International Journal of Energy and Environmental Engineering</i> </span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">3, article 31</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/2251-6832-3-31">10.1186/2251-6832-3-31</a> </span> <span class="pub-id" /> </span> </li> <li tag="ref"> <a name="B5" /> <span class="label" tagx="label" title="label">2</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumar</span> <span class="given-names" tagx="given-names" title="given-names">P. S. V.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Manjunatha</span> <span class="given-names" tagx="given-names" title="given-names">B. K.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Studies on hydrocarbon degradation by the bacterial isolate <i>Stenotrophomonas rhizophila</i> (PM-1) from the oil spilled regions of Western Ghats of Karnataka </span> <span class="source" tagx="source" title="source"> <i>Science, Technology and Arts Research Journal</i> </span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">139</span> <span class="lpage" tagx="lpage" title="lpage">144</span> <span class="pub-id"> <a href="https://dx.doi.org/10.4314/star.v4i3.21">10.4314/star.v4i3.21</a> </span> </span> </li> <li tag="ref"> <a name="B2" /> <span class="label" tagx="label" title="label">3</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Agarry</span> <span class="given-names" tagx="given-names" title="given-names">S. E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Owabor</span> <span class="given-names" tagx="given-names" title="given-names">C. N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yusuf</span> <span class="given-names" tagx="given-names" title="given-names">R. O.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Enhanced bioremediation of soil artificially contaminated with kerosene: optimization of biostimulation agents through statistical experimental design </span> <span class="source" tagx="source" title="source"> <i>Journal of Petroleum &amp;amp; Environmental Biotechnology</i> </span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">3, article 120</span> <span class="pub-id"> <a href="https://dx.doi.org/10.4172/2157-7463.1000120">10.4172/2157-7463.1000120</a> </span> </span> </li> <li tag="ref"> <a name="B3" /> <span class="label" tagx="label" title="label">4</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Debasmita</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rajasimman</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Optimization and kinetics studies on biodegradation of atrazine using mixed microorganisms</span> <span class="source" tagx="source" title="source"> <i>Alexandria Engineering Journal</i> </span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">52</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">499</span> <span class="lpage" tagx="lpage" title="lpage">505</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.aej.2013.06.008">10.1016/j.aej.2013.06.008</a> </span> <span class="pub-id" /> </span> </li> <li tag="ref"> <a name="B7" /> <span class="label" tagx="label" title="label">5</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Olawale</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oyawale</span> <span class="given-names" tagx="given-names" title="given-names">F. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Adepoju</span> <span class="given-names" tagx="given-names" title="given-names">T. F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aikulolu</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Optimisation of diesel polluted soil using response surface methodology</span> <span class="source" tagx="source" title="source"> <i>International Journal of Environmental Protection and Policy</i> </span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="issue" tagx="issue" title="issue">6</span> <span class="fpage" tagx="fpage" title="fpage">194</span> <span class="lpage" tagx="lpage" title="lpage">202</span> <span class="pub-id"> <a href="https://dx.doi.org/10.11648/j.ijepp.20150306.13">10.11648/j.ijepp.20150306.13</a> </span> </span> </li> <li tag="ref"> <a name="B4" /> <span class="label" tagx="label" title="label">6</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garg</span> <span class="given-names" tagx="given-names" title="given-names">S. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tripathi</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lal</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Response surface methodology for optimization of process variable for reactive orange 4 dye discoloration by <i>Pseudomonas putida</i> SKG-1 strain and bioreactor trial for its possible use in large-scale bioremediation </span> <span class="source" tagx="source" title="source"> <i>Desalination and Water Treatment</i> </span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">54</span> <span class="issue" tagx="issue" title="issue">11</span> <span class="fpage" tagx="fpage" title="fpage">3122</span> <span class="lpage" tagx="lpage" title="lpage">3133</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1080/19443994.2014.905975">10.1080/19443994.2014.905975</a> </span> <span class="pub-id" /> </span> </li> <li tag="ref"> <a name="B8" /> <span class="label" tagx="label" title="label">7</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Youssef</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">El-Taweel</span> <span class="given-names" tagx="given-names" title="given-names">G. E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">El-Naggar</span> <span class="given-names" tagx="given-names" title="given-names">A. Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">El-Hawary</span> <span class="given-names" tagx="given-names" title="given-names">S. E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">El-Meleigy</span> <span class="given-names" tagx="given-names" title="given-names">M. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ahmed</span> <span class="given-names" tagx="given-names" title="given-names">S. A.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Hydrocarbon degrading bacteria as indicator of petroleum pollutionismailia canal, Egypt </span> <span class="source" tagx="source" title="source"> <i>World Applied Sciences Journal</i> </span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="issue" tagx="issue" title="issue">10</span> <span class="fpage" tagx="fpage" title="fpage">1226</span> <span class="lpage" tagx="lpage" title="lpage">1233</span> </span> </li> </ul> </div> </div> <div class="floats-group" title="floats-group"> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <p>The Diagnostic Plots obtained by the Box-Behnken Design to evaluate the normal plot residuals using the normal % probability versus externally studentized residuals by the bacterial isolate <i> S. rhizophila KX082814</i> for its crude oil degradation ability. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <p>Predicted versus actual plot obtained by the Box-Behnken Design based on the % TPH of crude oil and PAHs degradation using the bacterial isolate <i> S. rhizophila KX082814.</i> </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <p>The Diagnostic Plots showing the recommended studentized residuals obtained by the Box-Behnken Design using the residual versus predicted by the bacterial isolate <i> S. rhizophila KX082814</i> for its crude oil degradation ability. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 4</span> </a> </div>   <p>The Diagnostic Plots showing Cook's distance plot obtained by the Box-Behnken Design using Cook's distance versus run number by the bacterial isolate <i> S. rhizophila KX082814</i> for its crude oil degradation ability. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 5</span> </a> </div>   <p>The Diagnostic Plots obtained by the Box-Behnken Design showing the interactions among factors that influence crude oil and PAHs degradation by the bacterial isolate <i> S. rhizophila KX082814</i> for its crude oil degradation ability. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 6</span> </a> </div>   <p>3D response surface plot and Cube representation showing the standard error of design and interaction effect of factors that influence crude oil and PAHs degradation by the bacterial isolate <i> S. rhizophila KX082814.</i> </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 7</span> </a> </div>   <p>3D response surface plot and Cube representation showing the interaction effect of factors that influence BOD of crude oil and PAHs degradation by the bacterial isolate <i> S. rhizophila KX082814.</i> </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 8</span> </a> </div>   <p>3D response surface plot and Cube representation showing the interaction effect of factors that influence COD of crude oil and PAHs degradation by the bacterial isolate <i> S. rhizophila KX082814.</i> </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 9</span> </a> </div>   <p>3D response surface plot and Cube representation showing the interaction effect of factors that influence % TPH of crude oil and PAHs degradation by the bacterial isolate <i> S. rhizophila KX082814.</i> </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 10</span> </a> </div>   <p>Validation of degradation using predicted optimum conditions (at temperature of 32.5°C, pH of 9, 12.5 of salinity, and 12.5 ml of inoculum size). From left the Media Control (FSM-ASW), Positive Control (FSM-ASW + 2% CO), and Test (FSM-ASW + 2%CO + PM-1 bacterial isolate) on 15th day of incubation time. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 11</span> </a> </div>   <p>GC/MS chromatogram for 2% crude oil at 15th day of incubation shows persistence of total 40 different hydrocarbon components. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 12</span> </a> </div>   <p>GC/MS chromatogram for 2% crude oil at 15th day of incubation shows persistence of total 23 different hydrocarbon components after being treated with the bacterial isolate <i> S. rhizophila</i> KX082814 on 15th day. </p>   </div> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  <p>Experimental range and the variables showing the limiting factors.</p>  <table> <thead> <tr> <th>Name</th> <th>Unit</th> <th>Type</th> <th>Low</th> <th>High</th> </tr> </thead> <tbody> <tr> <td>Temperature</td> <td>°C</td> <td>Factor</td> <td>25</td> <td>40</td> </tr> <tr> <td>pH</td> <td> </td> <td>Factor</td> <td>5</td> <td>11</td> </tr> <tr> <td>Salinity</td> <td> </td> <td>Factor</td> <td>5</td> <td>20</td> </tr> <tr> <td>Inoculum size</td> <td>ml</td> <td>Factor</td> <td>5</td> <td>20</td> </tr> <tr> <td>Total Petroleum Hydrocarbon (TPH)</td> <td>%</td> <td>Response</td> <td> </td> <td> </td> </tr> <tr> <td>Biological Oxygen Demand (BOD)</td> <td>mg/L</td> <td>Response</td> <td> </td> <td> </td> </tr> <tr> <td>Chemical Oxygen Demand (COD)</td> <td>mg/L</td> <td>Response</td> <td> </td> <td> </td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 2</span>  <p>Full-factorial Box-Behnken Design levels for the four independent variables showing total of 29 sets of experimentation work. </p>  <table> <thead> <tr> <th>Std</th> <th>Run </th> <th>Temp (°C) </th> <th>pH</th> <th>Salinity</th> <th>Inoculum size (ml)</th> </tr> </thead> <tbody> <tr> <td>8</td> <td>1</td> <td>32.5</td> <td>8</td> <td>20</td> <td>20</td> </tr> <tr> <td>16</td> <td>2</td> <td>32.5</td> <td>11</td> <td>20</td> <td>12.5</td> </tr> <tr> <td>12</td> <td>3</td> <td>40</td> <td>8</td> <td>12.5</td> <td>20</td> </tr> <tr> <td>18</td> <td>4</td> <td>40</td> <td>8</td> <td>5</td> <td>12.5</td> </tr> <tr> <td>23</td> <td>5</td> <td>32.5</td> <td>5</td> <td>12.5</td> <td>20</td> </tr> <tr> <td>6</td> <td>6</td> <td>32.5</td> <td>8</td> <td>20</td> <td>5</td> </tr> <tr> <td>11</td> <td>7</td> <td>25</td> <td>8</td> <td>12.5</td> <td>20</td> </tr> <tr> <td>20</td> <td>8</td> <td>40</td> <td>8</td> <td>20</td> <td>12.5</td> </tr> <tr> <td>14</td> <td>9</td> <td>32.5</td> <td>11</td> <td>5</td> <td>12.5</td> </tr> <tr> <td>29</td> <td>10</td> <td>32.5</td> <td>8</td> <td>12.5</td> <td>12.5</td> </tr> <tr> <td>25</td> <td>11</td> <td>32.5</td> <td>8</td> <td>12.5</td> <td>12.5</td> </tr> <tr> <td>22</td> <td>12</td> <td>32.5</td> <td>11</td> <td>12.5</td> <td>5</td> </tr> <tr> <td>21</td> <td>13</td> <td>32.5</td> <td>5</td> <td>12.5</td> <td>5</td> </tr> <tr> <td>19</td> <td>14</td> <td>25</td> <td>8</td> <td>20</td> <td>12.5</td> </tr> <tr> <td>1</td> <td>15</td> <td>25</td> <td>5</td> <td>12.5</td> <td>12.5</td> </tr> <tr> <td>28</td> <td>16</td> <td>32.5</td> <td>8</td> <td>12.5</td> <td>12.5</td> </tr> <tr> <td>13</td> <td>17</td> <td>32.5</td> <td>5</td> <td>5</td> <td>12.5</td> </tr> <tr> <td>5</td> <td>18</td> <td>32.5</td> <td>8</td> <td>5</td> <td>5</td> </tr> <tr> <td>10</td> <td>19</td> <td>40</td> <td>8</td> <td>12.5</td> <td>5</td> </tr> <tr> <td>7</td> <td>20</td> <td>32.5</td> <td>8</td> <td>5</td> <td>20</td> </tr> <tr> <td>4</td> <td>21</td> <td>40</td> <td>11</td> <td>12.5</td> <td>12.5</td> </tr> <tr> <td>9</td> <td>22</td> <td>25</td> <td>8</td> <td>12.5</td> <td>5</td> </tr> <tr> <td>27</td> <td>23</td> <td>32.5</td> <td>8</td> <td>12.5</td> <td>12.5</td> </tr> <tr> <td>15</td> <td>24</td> <td>32.5</td> <td>5</td> <td>20</td> <td>12.5</td> </tr> <tr> <td>3</td> <td>25</td> <td>25</td> <td>11</td> <td>12.5</td> <td>12.5</td> </tr> <tr> <td>24</td> <td>26</td> <td>32.5</td> <td>11</td> <td>12.5</td> <td>20</td> </tr> <tr> <td>2</td> <td>27</td> <td>40</td> <td>5</td> <td>12.5</td> <td>12.5</td> </tr> <tr> <td>17</td> <td>28</td> <td>25</td> <td>8</td> <td>5</td> <td>12.5</td> </tr> <tr> <td>26</td> <td>29</td> <td>32.5</td> <td>8</td> <td>12.5</td> <td>12.5</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 3</span>  <p>Showing the % TPH, BOD, and COD results of 29 experimental designs using <i>S. rhizophila </i>KX082814 when crude oil was used as sole carbon source on 15th day of incubation period. </p>  <table> <thead> <tr> <th>Std</th> <th>Run </th> <th>Temp (°C) </th> <th>pH</th> <th>Salinity </th> <th>Inoculum size (ml)</th> <th>% TPH</th> <th>BOD (mg/l)</th> <th>COD (mg/l)</th> </tr> </thead> <tbody> <tr> <td>8</td> <td>1</td> <td>32.5</td> <td>8</td> <td>20</td> <td>20</td> <td>53</td> <td>153</td> <td>226.8</td> </tr> <tr> <td>16</td> <td>2</td> <td>32.5</td> <td>11</td> <td>20</td> <td>12.5</td> <td>56</td> <td>189</td> <td>245.6</td> </tr> <tr> <td>12</td> <td>3</td> <td>40</td> <td>8</td> <td>12.5</td> <td>20</td> <td>52</td> <td>178</td> <td>286.5</td> </tr> <tr> <td>18</td> <td>4</td> <td>40</td> <td>8</td> <td>5</td> <td>12.5</td> <td>51.89</td> <td>196.5</td> <td>258</td> </tr> <tr> <td>23</td> <td>5</td> <td>32.5</td> <td>5</td> <td>12.5</td> <td>20</td> <td>68.5</td> <td>221</td> <td>356</td> </tr> <tr> <td>6</td> <td>6</td> <td>32.5</td> <td>8</td> <td>20</td> <td>5</td> <td>61.53</td> <td>216</td> <td>341</td> </tr> <tr> <td>11</td> <td>7</td> <td>25</td> <td>8</td> <td>12.5</td> <td>20</td> <td>67.84</td> <td>256</td> <td>382</td> </tr> <tr> <td>20</td> <td>8</td> <td>40</td> <td>8</td> <td>20</td> <td>12.5</td> <td>53.52</td> <td>188</td> <td>256</td> </tr> <tr> <td>14</td> <td>9</td> <td>32.5</td> <td>11</td> <td>5</td> <td>12.5</td> <td>65.23</td> <td>225</td> <td>358</td> </tr> <tr> <td>29</td> <td>10</td> <td>32.5</td> <td>8</td> <td>12.5</td> <td>12.5</td> <td>70.25</td> <td>241</td> <td>442</td> </tr> <tr> <td>25</td> <td>11</td> <td>32.5</td> <td>8</td> <td>12.5</td> <td>12.5</td> <td>69.05</td> <td>221</td> <td>395</td> </tr> <tr> <td>22</td> <td>12</td> <td>32.5</td> <td>11</td> <td>12.5</td> <td>5</td> <td>65.2</td> <td>198</td> <td>305</td> </tr> <tr> <td>21</td> <td>13</td> <td>32.5</td> <td>5</td> <td>12.5</td> <td>5</td> <td>62.59</td> <td>225.6</td> <td>298.5</td> </tr> <tr> <td>19</td> <td>14</td> <td>25</td> <td>8</td> <td>20</td> <td>12.5</td> <td>53</td> <td>184.2</td> <td>219.9</td> </tr> <tr> <td>1</td> <td>15</td> <td>25</td> <td>5</td> <td>12.5</td> <td>12.5</td> <td>61.52</td> <td>205</td> <td>298.5</td> </tr> <tr> <td>28</td> <td>16</td> <td>32.5</td> <td>8</td> <td>12.5</td> <td>12.5</td> <td>64.25</td> <td>236.5</td> <td>336.8</td> </tr> <tr> <td>13</td> <td>17</td> <td>32.5</td> <td>5</td> <td>5</td> <td>12.5</td> <td>68.54</td> <td>295.6</td> <td>358</td> </tr> <tr> <td>5</td> <td>18</td> <td>32.5</td> <td>8</td> <td>5</td> <td>5</td> <td>60.21</td> <td>258</td> <td>356.5</td> </tr> <tr> <td>10</td> <td>19</td> <td>40</td> <td>8</td> <td>12.5</td> <td>5</td> <td>58.5</td> <td>187.5</td> <td>265</td> </tr> <tr> <td>7</td> <td>20</td> <td>32.5</td> <td>8</td> <td>5</td> <td>20</td> <td>63.25</td> <td>225</td> <td>398.5</td> </tr> <tr> <td>4</td> <td>21</td> <td>40</td> <td>11</td> <td>12.5</td> <td>12.5</td> <td>55.26</td> <td>198</td> <td>268</td> </tr> <tr> <td>9</td> <td>22</td> <td>25</td> <td>8</td> <td>12.5</td> <td>5</td> <td>59.25</td> <td>189.65</td> <td>258</td> </tr> <tr> <td>27</td> <td>23</td> <td>32.5</td> <td>8</td> <td>12.5</td> <td>12.5</td> <td>64.58</td> <td>242.6</td> <td>421.2</td> </tr> <tr> <td>15</td> <td>24</td> <td>32.5</td> <td>5</td> <td>20</td> <td>12.5</td> <td>61.22</td> <td>215.8</td> <td>395.8</td> </tr> <tr> <td>3</td> <td>25</td> <td>25</td> <td>11</td> <td>12.5</td> <td>12.5</td> <td>60.25</td> <td>210.5</td> <td>298.5</td> </tr> <tr> <td>24</td> <td>26</td> <td>32.5</td> <td>11</td> <td>12.5</td> <td>20</td> <td>62.54</td> <td>235.6</td> <td>329.5</td> </tr> <tr> <td>2</td> <td>27</td> <td>40</td> <td>5</td> <td>12.5</td> <td>12.5</td> <td>58.21</td> <td>189.58</td> <td>298.5</td> </tr> <tr> <td>17</td> <td>28</td> <td>25</td> <td>8</td> <td>5</td> <td>12.5</td> <td>59.52</td> <td>178.5</td> <td>258.5</td> </tr> <tr> <td>26</td> <td>29</td> <td>32.5</td> <td>8</td> <td>12.5</td> <td>12.5</td> <td>55.68</td> <td>153.8</td> <td>221.35</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 4</span>  <p>Diagnostics case statistics, experimental design, and results of Box-Behnken Design for crude oil and PAHs degradation. </p>  <table> <thead> <tr> <th>Run order</th> <th>Actual value</th> <th>Predicted value</th> <th>Residual</th> <th>Leverage</th> <th>Internally studentized residual</th> <th>Externally studentized residual</th> <th>Cook's distance</th> <th>Influence on fitted value DFFITS</th> <th>Standard order</th> </tr> </thead> <tbody> <tr> <td>15</td> <td>61.52</td> <td>63.19</td> <td>−1.67</td> <td>0.583</td> <td>−0.694</td> <td>−0.681</td> <td>0.045</td> <td>−0.805</td> <td>1</td> </tr> <tr> <td>27</td> <td>58.21</td> <td>58.69</td> <td>−0.48</td> <td>0.583</td> <td>−0.201</td> <td>−0.194</td> <td>0.004</td> <td>−0.229</td> <td>2</td> </tr> <tr> <td>25</td> <td>60.25</td> <td>61.34</td> <td>−1.09</td> <td>0.583</td> <td>−0.455</td> <td>−0.442</td> <td>0.019</td> <td>−0.523</td> <td>3</td> </tr> <tr> <td>21</td> <td>55.26</td> <td>55.17</td> <td>0.092</td> <td>0.583</td> <td>0.038</td> <td>0.037</td> <td>0.000</td> <td>0.044</td> <td>4</td> </tr> <tr> <td>18</td> <td>60.21</td> <td>59.94</td> <td>0.27</td> <td>0.583</td> <td>0.115</td> <td>0.111</td> <td>0.001</td> <td>0.131</td> <td>5</td> </tr> <tr> <td>6</td> <td>61.53</td> <td>60.66</td> <td>0.87</td> <td>0.583</td> <td>0.363</td> <td>0.352</td> <td>0.012</td> <td>0.416</td> <td>6</td> </tr> <tr> <td>20</td> <td>63.25</td> <td>65.70</td> <td>−2.45</td> <td>0.583</td> <td>−1.019</td> <td>−1.021</td> <td>0.097</td> <td>−1.208</td> <td>7</td> </tr> <tr> <td>1</td> <td>53.00</td> <td>54.85</td> <td>−1.85</td> <td>0.583</td> <td>−0.770</td> <td>−0.759</td> <td>0.055</td> <td>−0.898</td> <td>8</td> </tr> <tr> <td>22</td> <td>59.25</td> <td>57.17</td> <td>2.08</td> <td>0.583</td> <td>0.866</td> <td>0.858</td> <td>0.070</td> <td>1.015</td> <td>9</td> </tr> <tr> <td>19</td> <td>58.50</td> <td>59.38</td> <td>−0.88</td> <td>0.583</td> <td>−0.368</td> <td>−0.357</td> <td>0.013</td> <td>−0.422</td> <td>10</td> </tr> <tr> <td>7</td> <td>67.84</td> <td>64.69</td> <td>3.15</td> <td>0.583</td> <td>1.312</td> <td>1.350</td> <td>0.161</td> <td>1.597</td> <td>11</td> </tr> <tr> <td>3</td> <td>52.00</td> <td>51.81</td> <td>0.19</td> <td>0.583</td> <td>0.078</td> <td>0.075</td> <td>0.001</td> <td>0.089</td> <td>12</td> </tr> <tr> <td>17</td> <td>68.54</td> <td>65.01</td> <td>3.53</td> <td>0.583</td> <td>1.471</td> <td>1.542</td> <td>0.202</td> <td>1.824</td> <td>13</td> </tr> <tr> <td>9</td> <td>65.23</td> <td>63.28</td> <td>1.95</td> <td>0.583</td> <td>0.812</td> <td>0.801</td> <td>0.062</td> <td>0.948</td> <td>14</td> </tr> <tr> <td>24</td> <td>61.22</td> <td>60.90</td> <td>0.32</td> <td>0.583</td> <td>0.132</td> <td>0.127</td> <td>0.002</td> <td>0.150</td> <td>15</td> </tr> <tr> <td>2</td> <td>56.00</td> <td>57.27</td> <td>−1.27</td> <td>0.583</td> <td>−0.527</td> <td>−0.513</td> <td>0.026</td> <td>−0.607</td> <td>16</td> </tr> <tr> <td>28</td> <td>59.52</td> <td>62.06</td> <td>−2.54</td> <td>0.583</td> <td>−1.060</td> <td>−1.065</td> <td>0.105</td> <td>−1.260</td> <td>17</td> </tr> <tr> <td>4</td> <td>51.89</td> <td>52.65</td> <td>−0.76</td> <td>0.583</td> <td>−0.319</td> <td>−0.308</td> <td>0.009</td> <td>−0.365</td> <td>18</td> </tr> <tr> <td>14</td> <td>53.00</td> <td>52.93</td> <td>0.074</td> <td>0.583</td> <td>0.031</td> <td>0.030</td> <td>0.000</td> <td>0.035</td> <td>19</td> </tr> <tr> <td>8</td> <td>53.52</td> <td>51.67</td> <td>1.85</td> <td>0.583</td> <td>0.772</td> <td>0.760</td> <td>0.056</td> <td>0.899</td> <td>20</td> </tr> <tr> <td>13</td> <td>62.59</td> <td>64.26</td> <td>−1.67</td> <td>0.583</td> <td>−0.698</td> <td>−0.685</td> <td>0.045</td> <td>−0.810</td> <td>21</td> </tr> <tr> <td>12</td> <td>65.20</td> <td>65.87</td> <td>−0.67</td> <td>0.583</td> <td>−0.278</td> <td>−0.268</td> <td>0.007</td> <td>−0.317</td> <td>22</td> </tr> <tr> <td>5</td> <td>68.50</td> <td>68.52</td> <td>−0.025</td> <td>0.583</td> <td>−0.010</td> <td>−0.010</td> <td>0.000</td> <td>−0.012</td> <td>23</td> </tr> <tr> <td>26</td> <td>62.54</td> <td>61.56</td> <td>0.98</td> <td>0.583</td> <td>0.410</td> <td>0.398</td> <td>0.016</td> <td>0.470</td> <td>24</td> </tr> <tr> <td>11</td> <td>69.05</td> <td>64.76</td> <td>4.29</td> <td>0.200</td> <td>1.290</td> <td>1.324</td> <td>0.028</td> <td>0.662</td> <td>25</td> </tr> <tr> <td>29</td> <td>55.68</td> <td>64.76</td> <td>−9.08</td> <td>0.200</td> <td>−2.732</td> <td>−3.852</td> <td>0.124</td> <td>−1.926</td> <td>26</td> </tr> <tr> <td>23</td> <td>64.58</td> <td>64.76</td> <td>−0.18</td> <td>0.200</td> <td>−0.055</td> <td>−0.053</td> <td>0.000</td> <td>−0.026</td> <td>27</td> </tr> <tr> <td>16</td> <td>64.25</td> <td>64.76</td> <td>−0.51</td> <td>0.200</td> <td>−0.154</td> <td>−0.149</td> <td>0.000</td> <td>−0.074</td> <td>28</td> </tr> <tr> <td>10</td> <td>70.25</td> <td>64.76</td> <td>5.49</td> <td>0.200</td> <td>1.651</td> <td>1.773</td> <td>0.045</td> <td>0.886</td> <td>29</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 5</span>  <p>Analysis of variances using ANOVA for response surface by quadratic model.</p>  <table> <thead> <tr> <th>Source</th> <th>Sum of squares</th> <th>df</th> <th>Mean square</th> <th> <i>F</i>-value </th> <th> <i>P</i> value, Prob &amp;gt; <i>F</i> </th> <th> </th> </tr> </thead> <tbody> <tr> <td>Model</td> <td>639.46</td> <td>14</td> <td>45.68</td> <td> <i>3.31</i> </td> <td> <i>0.0163</i> </td> <td> <i>Significant</i> </td> </tr> <tr> <td> <i>A</i>-temperature </td> <td>85.33</td> <td>1</td> <td>85.33</td> <td>6.18</td> <td>0.0262</td> <td> </td> </tr> <tr> <td> <i>B</i>-pH </td> <td>21.60</td> <td>1</td> <td>21.60</td> <td>1.56</td> <td>0.2316</td> <td> </td> </tr> <tr> <td> <i>C</i>-salinity </td> <td>76.86</td> <td>1</td> <td>76.86</td> <td>5.56</td> <td>0.0334</td> <td> </td> </tr> <tr> <td> <i>D</i>-inoculum Size </td> <td>1.875 <i>E</i> − 003 </td> <td>1</td> <td>1.875 <i>E</i> − 003 </td> <td>1.357 <i>E</i> − 004 </td> <td>0.9909</td> <td> </td> </tr> <tr> <td> <i>AB</i> </td> <td>0.71</td> <td>1</td> <td>0.71</td> <td>0.051</td> <td>0.8245</td> <td> </td> </tr> <tr> <td> <i>AC</i> </td> <td>16.61</td> <td>1</td> <td>16.61</td> <td>1.20</td> <td>0.2914</td> <td> </td> </tr> <tr> <td> <i>AD</i> </td> <td>56.93</td> <td>1</td> <td>56.93</td> <td>4.12</td> <td>0.0618</td> <td> </td> </tr> <tr> <td> <i>BC</i> </td> <td>0.91</td> <td>1</td> <td>0.91</td> <td>0.066</td> <td>0.8010</td> <td> </td> </tr> <tr> <td> <i>BD</i> </td> <td>18.36</td> <td>1</td> <td>18.36</td> <td>1.33</td> <td>0.2683</td> <td> </td> </tr> <tr> <td> <i>CD</i> </td> <td>33.47</td> <td>1</td> <td>33.47</td> <td>2.42</td> <td>0.1419</td> <td> </td> </tr> <tr> <td> <i>A</i> <sup>2</sup> </td> <td>231.68</td> <td>1</td> <td>231.68</td> <td>16.77</td> <td>0.0011</td> <td> </td> </tr> <tr> <td> <i>B</i> <sup>2</sup> </td> <td>4.27</td> <td>1</td> <td>4.27</td> <td>0.31</td> <td>0.5871</td> <td> </td> </tr> <tr> <td> <i>C</i> <sup>2</sup> </td> <td>101.60</td> <td>1</td> <td>101.60</td> <td>7.35</td> <td>0.0169</td> <td> </td> </tr> <tr> <td> <i>D</i> <sup>2</sup> </td> <td>1.76</td> <td>1</td> <td>1.76</td> <td>0.13</td> <td>0.7268</td> <td> </td> </tr> <tr> <td>Residual</td> <td>193.42</td> <td>14</td> <td>13.82</td> <td> </td> <td> </td> <td> </td> </tr> <tr> <td> Lack of fit</td> <td>62.13</td> <td>10</td> <td>6.21</td> <td>0.19</td> <td>0.9850</td> <td>Not significant</td> </tr> <tr> <td> Pure error</td> <td>131.28</td> <td>4</td> <td>32.82</td> <td> </td> <td> </td> <td> </td> </tr> <tr> <td>Cor total</td> <td>832.88</td> <td>28</td> <td> </td> <td> </td> <td> </td> <td> </td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 6</span>  <p>Coded values of the test variables.</p>  <table> <thead> <tr> <th>Factor</th> <th>Coefficient</th> <th>df</th> <th>Standard</th> <th>95% CI</th> <th>95% CI</th> <th>VIF</th> </tr> <tr> <th>estimate</th> <th>error</th> <th>Low</th> <th>High</th> </tr> </thead> <tbody> <tr> <td>Intercept</td> <td>64.76</td> <td>1</td> <td>1.66</td> <td>61.20</td> <td>68.33</td> <td> </td> </tr> <tr> <td> <i>A</i>-temperature </td> <td>−2.67</td> <td>1</td> <td>1.07</td> <td>−4.97</td> <td>−0.37</td> <td>1.00</td> </tr> <tr> <td> <i>B</i>-pH </td> <td>−1.34</td> <td>1</td> <td>1.07</td> <td>−3.64</td> <td>0.96</td> <td>1.00</td> </tr> <tr> <td> <i>C</i>-salinity </td> <td>−2.53</td> <td>1</td> <td>1.07</td> <td>−4.83</td> <td>−0.23</td> <td>1.00</td> </tr> <tr> <td> <i>D</i>-inoculum Size </td> <td>−0.013</td> <td>1</td> <td>1.07</td> <td>−2.31</td> <td>2.29</td> <td>1.00</td> </tr> <tr> <td> <i>AB</i> </td> <td>−0.42</td> <td>1</td> <td>1.86</td> <td>−4.41</td> <td>3.57</td> <td>1.00</td> </tr> <tr> <td> <i>AC</i> </td> <td>2.04</td> <td>1</td> <td>1.86</td> <td>−1.95</td> <td>6.02</td> <td>1.00</td> </tr> <tr> <td> <i>AD</i> </td> <td>−3.77</td> <td>1</td> <td>1.86</td> <td>−7.76</td> <td>0.21</td> <td>1.00</td> </tr> <tr> <td> <i>BC</i> </td> <td>−0.48</td> <td>1</td> <td>1.86</td> <td>−4.46</td> <td>3.51</td> <td>1.00</td> </tr> <tr> <td> <i>BD</i> </td> <td>−2.14</td> <td>1</td> <td>1.86</td> <td>−6.13</td> <td>1.84</td> <td>1.00</td> </tr> <tr> <td> <i>CD</i> </td> <td>−2.89</td> <td>1</td> <td>1.86</td> <td>−6.88</td> <td>1.09</td> <td>1.00</td> </tr> <tr> <td> <i>A</i> <sup>2</sup> </td> <td>−5.98</td> <td>1</td> <td>1.46</td> <td>−9.11</td> <td>−2.85</td> <td>1.08</td> </tr> <tr> <td> <i>B</i> <sup>2</sup> </td> <td>0.81</td> <td>1</td> <td>1.46</td> <td>−2.32</td> <td>3.94</td> <td>1.08</td> </tr> <tr> <td> <i>C</i> <sup>2</sup> </td> <td>−3.96</td> <td>1</td> <td>1.46</td> <td>−7.09</td> <td>−0.83</td> <td>1.08</td> </tr> <tr> <td> <i>D</i> <sup>2</sup> </td> <td>−0.52</td> <td>1</td> <td>1.46</td> <td>−3.65</td> <td>2.61</td> <td>1.08</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 7</span>  <p>Assessment of rate of degradation by measuring BOD, COD, % TPH, and TPC for the bacterial isolate <i>S. rhizophila (MP-1) </i>KX082814 for observed and predicted optimum conditions through RSM-BBD. </p>  <table> <thead> <tr> <th>Test parameter</th> <th>Observed optimum parameters through RSM-BBD</th> <th>Validation of predicted optimum condition through RSM-BBD</th> </tr> </thead> <tbody> <tr> <td>TPC (cfu/ml)</td> <td>4.8 × 10 <sup>−8</sup> </td> <td>6.7 × 10 <sup>−9</sup> </td> </tr> <tr> <td>TPH (%)</td> <td>71.82%</td> <td>79.53%</td> </tr> <tr> <td>BOD (mg/L)</td> <td>253.5</td> <td>325.2</td> </tr> <tr> <td>COD (mg/L)</td> <td>816</td> <td>835.4</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 8</span>  <p>Validation of crude oil degradation by GC/MS analysis after treatment with <i>S. rhizophila </i>(PM-1) KX082814 on 15th day. </p>  <table> <thead> <tr> <th>Major components of crude oil GC/MS</th> <th>Major residual components of crude oil after treatment</th> </tr> <tr> <th>Peak</th> <th>R. time</th> <th>Linear &amp;amp; branched</th> <th>% area</th> <th>Peak</th> <th>R. time</th> <th>Linear &amp;amp; branched</th> <th>% area</th> </tr> </thead> <tbody> <tr> <td>1</td> <td>7.447</td> <td>C10H22</td> <td>2.19</td> <td>1</td> <td>22.97</td> <td>C14H30</td> <td>6.4</td> </tr> <tr> <td>2</td> <td>10.113</td> <td>C11H24</td> <td>3.18</td> <td>2</td> <td>25.20</td> <td>C16H34</td> <td>7.7</td> </tr> <tr> <td>3</td> <td>11.625</td> <td>C16H33Cl</td> <td>0.63</td> <td>3</td> <td>29.35</td> <td>C16H34</td> <td>8.29</td> </tr> <tr> <td>4</td> <td>12.877</td> <td>C12H26</td> <td>4.96</td> <td>4</td> <td>33.13</td> <td>C16H34</td> <td>7.53</td> </tr> <tr> <td>5</td> <td>15.592</td> <td>C13H28</td> <td>7.4</td> <td>5</td> <td>36.59</td> <td>C16H34</td> <td>5.85</td> </tr> <tr> <td>6</td> <td>19.728</td> <td>C16H34</td> <td>2.41</td> <td>6</td> <td>38.21</td> <td>C16H34</td> <td>4.2</td> </tr> <tr> <td>7</td> <td>20.682</td> <td>C15H32</td> <td>8.66</td> <td>7</td> <td>31.28</td> <td>C21H44</td> <td>8.02</td> </tr> <tr> <td>8</td> <td>23.035</td> <td>C16H34</td> <td>8.56</td> <td>8</td> <td>17.55</td> <td>C8H17I</td> <td>0.69</td> </tr> <tr> <td>9</td> <td>25.273</td> <td>C20H42</td> <td>8.41</td> <td>9</td> <td>41.28</td> <td>C9H19I</td> <td>1.77</td> </tr> <tr> <td>10</td> <td>27.392</td> <td>C18H38</td> <td>7.6</td> <td>10</td> <td>29.20</td> <td>C11H24</td> <td>1.43</td> </tr> <tr> <td>11</td> <td>29.413</td> <td>C18H38</td> <td>7.03</td> <td>11</td> <td>18.15</td> <td>C12H26</td> <td>2.66</td> </tr> <tr> <td>12</td> <td>34.932</td> <td>C24H50</td> <td>3.68</td> <td>12</td> <td>27.52</td> <td>C12H26</td> <td>4.61</td> </tr> <tr> <td>13</td> <td>36.617</td> <td>C24H50</td> <td>2.61</td> <td>13</td> <td>39.77</td> <td>C13H28</td> <td>3.23</td> </tr> <tr> <td>14</td> <td>38.234</td> <td>C24H50</td> <td>1.74</td> <td>14</td> <td>25.32</td> <td>C20H42</td> <td>10.23</td> </tr> <tr> <td>15</td> <td>39.790</td> <td>C27H56</td> <td>1.17</td> <td>15</td> <td>27.33</td> <td>C20H42</td> <td>7.13</td> </tr> <tr> <td>16</td> <td>41.291</td> <td>C27H56</td> <td>0.67</td> <td>16</td> <td>34.89</td> <td>C20H42</td> <td>6.89</td> </tr> <tr> <td>17</td> <td>42.739</td> <td>C27H56</td> <td>0.55</td> <td>17</td> <td>42.73</td> <td>C6H14</td> <td>1.01</td> </tr> <tr> <td>18</td> <td>44.137</td> <td>C27H56</td> <td>0.34</td> <td>18</td> <td>26.53</td> <td>C11H24</td> <td>0.64</td> </tr> <tr> <td>19</td> <td>45.485</td> <td>C27H56</td> <td>0.25</td> <td>19</td> <td>21.89</td> <td>C12H26</td> <td>0.63</td> </tr> <tr> <td>20</td> <td>46.792</td> <td>C28H58</td> <td>0.15</td> <td>20</td> <td>30.72</td> <td>C12H26</td> <td>0.71</td> </tr> <tr> <td>21</td> <td>48.057</td> <td>C34H70</td> <td>0.14</td> <td> </td> <td> </td> <td> </td> <td> </td> </tr> <tr> <td>22</td> <td>8.046</td> <td>C11H24</td> <td>0.86</td> <td> </td> <td> </td> <td> </td> <td> </td> </tr> <tr> <td>23</td> <td>11.743</td> <td>C12H26</td> <td>0.18</td> <td> </td> <td> </td> <td> </td> <td> </td> </tr> <tr> <td>24</td> <td>11.859</td> <td>C12H26</td> <td>0.4</td> <td> </td> <td> </td> <td> </td> <td> </td> </tr> <tr> <td>25</td> <td>13.242</td> <td>C13H28</td> <td>0.87</td> <td> </td> <td> </td> <td> </td> <td> </td> </tr> <tr> <td>26</td> <td>14.287</td> <td>C13H28</td> <td>0.09</td> <td> </td> <td> </td> <td> </td> <td> </td> </tr> <tr> <td>27</td> <td>14.336</td> <td>C13H28</td> <td>0.02</td> <td> </td> <td> </td> <td> </td> <td> </td> </tr> <tr> <td>28</td> <td>24.106</td> <td>C18H38</td> <td>2.83</td> <td> </td> <td> </td> <td> </td> <td> </td> </tr> <tr> <td>29</td> <td>25.390</td> <td>C19H40</td> <td>6.38</td> <td> </td> <td> </td> <td> </td> <td> </td> </tr> <tr> <td>30</td> <td>27.561</td> <td>C20H42</td> <td>2.91</td> <td> </td> <td> </td> <td> </td> <td> </td> </tr> </tbody> </table> </div> </div>  </body></html>