<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Biomolecules</span> <span class="iso-abbrev" title="iso-abbrev">Biomolecules</span> <span class="publisher-id" title="publisher-id">biomolecules</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Biomolecules</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">2218-273X</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">MDPI</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6359376">6359376</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3390/biom9010007">10.3390/biom9010007</a> </span> <span class="publisher-id" title="publisher-id">biomolecules-09-00007</span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Total Fatty Acid Analysis of Human Blood Samples in One Minute by High-Resolution Mass Spectrometry </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gallego</span> <span class="given-names" tagx="given-names" title="given-names">Sandra F.</span> </span> <a href="#af1-biomolecules-09-00007">1</a> <a href="#fn1-biomolecules-09-00007">†</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hermansson</span> <span class="given-names" tagx="given-names" title="given-names">Martin</span> </span> <a href="#af1-biomolecules-09-00007">1</a> <a href="#fn1-biomolecules-09-00007">†</a> </span> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0003-4886-0811</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liebisch</span> <span class="given-names" tagx="given-names" title="given-names">Gerhard</span> </span> <a href="#af2-biomolecules-09-00007">2</a> </span> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-2648-6526</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hodson</span> <span class="given-names" tagx="given-names" title="given-names">Leanne</span> </span> <a href="#af3-biomolecules-09-00007">3</a> </span> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0003-4963-0276</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ejsing</span> <span class="given-names" tagx="given-names" title="given-names">Christer S.</span> </span> <a href="#af1-biomolecules-09-00007">1</a> <a href="#af4-biomolecules-09-00007">4</a> <a href="#c1-biomolecules-09-00007">*</a> </span> </div> <span class="citation_author_institution" id="af1-biomolecules-09-00007">[1], <span class="email" tagx="email" title="email">sandra.fnn@gmail.com</span> <span class="email" tagx="email" title="email">martinher@bmb.sdu.dk</span> </span> <span class="citation_author_institution" id="af2-biomolecules-09-00007">[2], <span class="email" tagx="email" title="email">gerhard.liebisch@klinik.uni-regensburg.de</span> </span> <span class="citation_author_institution" id="af3-biomolecules-09-00007">[3], <span class="email" tagx="email" title="email">leanne.hodson@ocdem.ox.ac.uk</span> </span> <span class="citation_author_institution" id="af4-biomolecules-09-00007">[4], </span> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"> <span class="label" tagx="label" title="label">*</span>Correspondence: <span class="email" tagx="email" title="email">cse@bmb.sdu.dk</span> </div> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label">†</span> <p>These authors contributed equally to this work.</p> </div> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-12-12</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2019-1</span> </span> <span class="volume" tagx="volume" title="volume">9</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">7</span> <span class="history" title="history"> <span class="received" title="received">received: 2018-11-24</span> <span class="accepted" title="accepted">accepted: 2018-12-17</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2018</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license ( <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>). </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Total fatty acid analysis is a routine method in many areas, including lipotyping of individuals in personalized medicine, analysis of foodstuffs, and optimization of oil production in biotechnology. This analysis is commonly done by converting fatty acyl (FA) chains of intact lipids into FA methyl esters (FAMEs) and monitoring these by gas-chromatography (GC)-based methods, typically requiring at least 15 min of analysis per sample. Here, we describe a novel method that supports fast, precise and accurate absolute quantification of total FA levels in human plasma and serum samples. The method uses acid-catalyzed transesterification with <sup>18</sup>O-enriched H <sub>2</sub>O (i.e., H <sub>2</sub> <sup>18</sup>O) to convert FA chains into <sup>18</sup>O-labeled free fatty acids. The resulting "mass-tagged" FA analytes can be specifically monitored with improved signal-to-background by 1 min of high resolution Fourier transform mass spectrometry (FTMS) on an Orbitrap-based mass spectrometer. By benchmarking to National Institute of Standards and Technology (NIST) certified standard reference materials we show that the performance of our method is comparable, and at times superior, to that of gold-standard GC-based methods. In addition, we demonstrate that the method supports the accurate quantification of FA differences in samples obtained in dietary intervention studies and also affords specific monitoring of ingested stable isotope-labeled fatty acids ( <sup>13</sup>C <sub>16</sub>-palmitate) in normoinsulinemic and hyperinsulinemic human subjects. Overall, our novel high-throughput method is generic and suitable for many application areas, spanning basic research to personalized medicine, and is particularly useful for laboratories equipped with high resolution mass spectrometers, but lacking access to GC-based instrumentation. </p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
total fatty acid analysis</div> <div class="kwd" title="kwd">
human blood plasma</div> <div class="kwd" title="kwd">
shotgun lipidomics</div> <div class="kwd" title="kwd">
high resolution mass spectrometry</div> <div class="kwd" title="kwd">
Orbitrap</div> </div> </div> </div> <div class="body" title="body"> <div class="1.introduction" title="sec"> <div class="title" tagx="title" title="title">
1. Introduction</div> <p>Total fatty acyl (FA) analysis of human blood samples (i.e., lipotyping) is important for the diagnosis of essential fatty acid deficiency [ <a href="#B1-biomolecules-09-00007">1</a>] and inborn errors of fatty acid metabolism [ <a href="#B2-biomolecules-09-00007">2</a>], for lipid-based risk stratification at the population-level, and for monitoring the metabolic health of individuals [ <a href="#B3-biomolecules-09-00007">3</a>, <a href="#B4-biomolecules-09-00007">4</a>, <a href="#B5-biomolecules-09-00007">5</a>, <a href="#B6-biomolecules-09-00007">6</a>]. Moreover, total FA analysis is also in widespread use in other fields of research, including optimization of oil production by microorganisms and plants [ <a href="#B7-biomolecules-09-00007">7</a>, <a href="#B8-biomolecules-09-00007">8</a>], and monitoring the composition of foods [ <a href="#B9-biomolecules-09-00007">9</a>]. These application areas typically require a high sample throughput, which inherently benefit from simple, high-throughput routines supporting accurate, precise, and absolute quantification of total FA levels. </p> <p>Total FA analysis is typically carried out using gas-chromatography (GC)-based methods. The underlying methodology is straightforward and involves using either acid- or base-catalyzed reactions to convert FA chains of intact lipids and non-esterified fatty acids into volatile FA methyl ester (FAME) species that can be detected using GC coupled to a flame ionization detector (GC-FID) [ <a href="#B10-biomolecules-09-00007">10</a>, <a href="#B11-biomolecules-09-00007">11</a>] or a mass spectrometer (GC-MS) [ <a href="#B10-biomolecules-09-00007">10</a>, <a href="#B11-biomolecules-09-00007">11</a>, <a href="#B12-biomolecules-09-00007">12</a>, <a href="#B13-biomolecules-09-00007">13</a>]. By using different chromatographic parameters (incl. column materials, gradients) and detection principles, it is possible to monitor FA analytes at different levels of structural resolution, which typically scales with increased analysis time and technical diligence of the operator [ <a href="#B13-biomolecules-09-00007">13</a>, <a href="#B14-biomolecules-09-00007">14</a>]. As such, total FA analysis ranges from the identification of the total number of acyl carbon atoms and double bonds (e.g., FA 18:1), requiring about 15 min of analysis per sample, to the separation of distinct FA isomers having different positions and configurations of double bonds (e.g., FA 18:1(9Z) and FA 18:1(11E)), requiring up to 125 min of analysis [ <a href="#B13-biomolecules-09-00007">13</a>]. Thus, the downside of increasing structural resolution is a significant increase in analysis time. Another potential downside of GC-based approaches is that accurate quantification requires multiple isotope-labeled internal standards (isotope dilution method) as well as generation of multiple calibration curves [ <a href="#B13-biomolecules-09-00007">13</a>, <a href="#B14-biomolecules-09-00007">14</a>]. Notably, in clinical settings, and in other application areas, one needs to strike a unique balance between the analysis time, the depth of structural resolution, and the required sample throughput. </p> <p>Total FA analysis can also be performed by electrospray ionization-based methods coupled to upfront liquid chromatography (i.e., LC-MS) or by direct infusion (shotgun) MS. Although GC-MS is considered the gold-standard method for total FA analysis, the detection capabilities and analysis time of LC-MS-based methods parallel those of GC-MS. Samples require chemical derivatization, or hydrolysis to free fatty acids, but the methodology can be highly sensitive [ <a href="#B15-biomolecules-09-00007">15</a>] and support metabolic flux analysis [ <a href="#B16-biomolecules-09-00007">16</a>]. More recently, a method for total FA analysis using multiplexed isobaric tagging and 40 min of LC-MS was described [ <a href="#B17-biomolecules-09-00007">17</a>]. Although the structural resolution provided by this method is no match to that of GC-MS (only identification of total number of carbon atoms and double bonds), the potential multiplexing of up to ten samples drives this approach towards high-throughput. </p> <p>FA analysis by direct infusion (shotgun) MS is also relatively simple and fast, and, in comparison to GC- and LC-based approaches, devoid of any sample carry-over when using automated chip-based sprayers for injecting individual samples [ <a href="#B18-biomolecules-09-00007">18</a>]. Various derivatization methods for analyzing non-esterified fatty acids by direct infusion MS have been developed [ <a href="#B19-biomolecules-09-00007">19</a>, <a href="#B20-biomolecules-09-00007">20</a>]. These methods, however, are not directly applicable for monitoring the composition of esterified FA analytes derived from intact lipid molecules. Nevertheless, a particular virtue of these methods is that they enable MS/MS-based structural characterization of double bond positions. We have recently used high resolution Fourier transform mass spectrometry (FTMS) in negative ion mode for the detection of non-esterified fatty acids as carboxylate anions and quantification of these in a larger cohort of plasma samples [ <a href="#B21-biomolecules-09-00007">21</a>]. This work suggested that negative ion mode FTMS analysis could also be used for total FA analysis, and potentially be faster and more sensitive than most high throughput-based GC and LC methods. </p> <p>Here, we describe a novel method for the quantification of total FA levels in human plasma and serum samples. The method uses acid-catalyzed hydrolysis of intact lipids in the presence of <sup>18</sup>O-enriched water (i.e., H <sub>2</sub> <sup>18</sup>O). This reaction yields <sup>18</sup>O-labeled free fatty acid species that can be specifically detected and quantified by automated high resolution FTMS analysis on an Orbitrap-based instrument coupled to a chip-based nanoelectrospray ionization source. Notably, the reaction can be carried out directly with diluted plasma and serum without prior lipid extraction, thereby bypassing potential biases deriving from sample extraction. We show that (i) negative ion mode analysis of deprotonated FA analytes is exceptionally sensitive, (ii) quantitative analysis can be performed in only 1 min, (iii) the method is accurate and precise, and (iv) that high resolution FTMS analysis with a resolving power above 400,000 (full width at half maximum, FWHM) affords baseline separation of <sup>13</sup>C, <sup>2</sup>H, and <sup>18</sup>O isotopologues, which offers a new analytical avenue for metabolic flux analysis. This method is generic and will be useful for laboratories equipped with high resolution mass spectrometers, but lacking access to GC-based instrumentation, and for quantifying total FA levels in a variety of biological sample matrices. </p> </div> <div class="2.materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
2. Materials and Methods</div> <div class="2.1.chemicalsandstandards" title="sec"> <div class="title" tagx="title" title="title">
2.1. Chemicals and Standards</div> <p>Acetonitrile, <i>n</i>-hexane, methanol, 2-propanol, and water were purchased from Biosolve BV (Valkenswaard, The Netherlands). Chloroform was from Rathburn Chemicals (Walkerburn, UK). Ammonium formate was from Sigma-Aldrich (Buchs, Switzerland). All solvents and chemicals were HPLC grade. Sulfuric acid and butylated hydroxytoluene (BHT) were from Sigma-Aldrich (Steinheim, Germany), <sup>18</sup>O-labeled water (i.e., H <sub>2</sub> <sup>18</sup>O) was from Medical Isotopes (Pelham, NH, USA). Lipid standards PE 15:0/18:1(+ <sup>2</sup>H <sub>7</sub>) (1-pentadecanoyl-2-oleoyl(d7)-sn-glycero-3-phosphoethanolamine) and LPC 16:0(+ <sup>2</sup>H <sub>3</sub>) (1-palmitoyl-d3-sn-glycero-3-phosphocholine) were obtained from Avanti Polar Lipids (Alabaster, AL, USA). A FAME standard mixture, with equal amounts of FAME 16:0, 18:0, 16:1, 18:1, 18:2, 18:3n−3, 18n−6, 20:3, 20:4, 20:5, 22:4, 22:5n−3, 22:5n−6 and 22:6, was obtained from Larodan Fine Chemicals (Solna, Sweden). </p> </div> <div class="2.2.humanplasmasamples" title="sec"> <div class="title" tagx="title" title="title">
2.2. Human Plasma Samples</div> <p>National Institute of Standards and Technology (NIST) Standard Reference Material (SRM) 1950 and 2378 were used for validation. In short, SRM 1950 was collected from 100 fasted individuals in the age range of 40–50 years and represents the plasma average composition of the US population as defined by race, sex, and health. SRM 2378 consists of 3 serum materials collected from: (1) donors who did not take either fish or flaxseed oil supplements for one month prior to collection, (2) donors who took flaxseed oil supplements for a minimum of one month prior to collection, and (3) donors who took fish oil supplements for a minimum of one month prior to collection. </p> <p>Plasma samples from five normoinsulinemic and five hyperinsulinemic subjects were obtained as previously described [ <a href="#B22-biomolecules-09-00007">22</a>]. In short, after an overnight fast the subjects had a cannula inserted into an antecubital vein, and baseline (0 h) blood plasma samples were taken. Subjects were then fed a mixed meal containing 200 mg of <sup>13</sup>C <sub>16</sub>-labeled palmitic acid (FA 16:0(+ <sup>13</sup>C <sub>16</sub>)). Another blood plasma sample was taken after 6 h, shown previously to yield the highest incorporation of FA 16:0(+ <sup>13</sup>C <sub>16</sub>) into plasma triacylglycerols [ <a href="#B22-biomolecules-09-00007">22</a>]. The study was approved by Portsmouth Clinical Research Ethics Committee (REC 12/SC/0267), and all subjects gave written informed consent. The study was performed in accordance with the Helsinki Declaration. </p> </div> <div class="2.3.samplepreparation" title="sec"> <div class="title" tagx="title" title="title">
2.3. Sample Preparation</div> <p>Plasma and serum, 15 µL, were mixed with 210 µL of 155 mM ammonium formate. Fifteen µL of this mixture (equivalent to 1 µL of undiluted blood sample) was transferred into a 1.1 mL glass vial (La-Pha-Pack GmbH) with 7 µL of 162.3 µM PE 15:0/18:1(+ <sup>2</sup>H <sub>7</sub>) added as internal standard, and vacuum evaporated. Next, the sample was dissolved in 80 µL of freshly prepared 0.75 M H <sub>2</sub>SO <sub>4</sub> in acetonitrile/H <sub>2</sub> <sup>18</sup>O (9:1, v/v) containing 50 µg/mL BHT. For hydrolysis, the sample was placed in a ThermoMixer (Eppendorf, Wesseling-Berzdorf, Germany) for 5 h at 90 °C and 750 rpm. After cooling, fatty acids were extracted with 180 µL of <i>n</i>-hexane by mixing for 10 min at 1400 rpm and centrifugation for 5 min at 1000 <i>g</i>. The fatty acid extraction with <i>n</i>-hexane was repeated, and the combined extracts were vacuum evaporated. </p> </div> <div class="2.4.massspectrometricanalysis" title="sec"> <div class="title" tagx="title" title="title">
2.4. Mass Spectrometric Analysis</div> <p>The total FA extracts were dissolved in 500 µL chloroform/methanol/2-propanol (1:2:4, v/v/v) containing 0.75 mM ammonium formate and loaded in a 96-well plate (Eppendorf, Hamburg, Germany). Ten µL of each fatty acid extract was infused using the robotic nanoflow ion source TriVersa NanoMate (Advion Biosciences, Ithaca, NY, USA) and analyzed in negative ion mode using an Orbitrap Fusion Tribrid mass spectrometer (Thermo Fisher Scientific, San Jose, CA, USA). Ionization voltage was −0.96 kV and back pressure was 1.25 psi. The temperature of the ion transfer tube was 275 °C. S-lens radio frequency level was set to 60%. Negative ion mode FTMS analysis was performed in m/z range 150–420. Each sample was analyzed for 1 min. All full scan FTMS data were acquired in profile mode, using a max injection time of 100 ms, automated gain control for an ion target of 10 <sup>5</sup>, three microscans (to reduce file size), and a target resolution setting of 500,000. </p> </div> <div class="2.5.lipididentificationandquantification" title="sec"> <div class="title" tagx="title" title="title">
2.5. Lipid Identification and Quantification</div> <p>Carboxylate anions of <sup>18</sup>O-labeled and unlabeled ( <sup>16</sup>O) FA analytes detected by FTMS analysis with a mass accuracy better than ±2.5 ppm were identified and quantified using ALEX <sup>123</sup> software [ <a href="#B23-biomolecules-09-00007">23</a>]. FA species were quantified by summing the intensities of doubly ( <sup>18</sup>O <sub>2</sub>) and singly ( <sup>16</sup>O <sup>18</sup>O) tagged FA analytes, normalizing to the sum intensity of doubly ( <sup>18</sup>O <sub>2</sub>) and singly ( <sup>16</sup>O <sup>18</sup>O) tagged internal standard FA 18:1(+ <sup>2</sup>H <sub>7</sub>), and multiplying by the spike amount of PE 15:0/18:1(+ <sup>2</sup>H <sub>7</sub>). To minimize bias from FA-contaminants present in reagents and other materials (see below), a blank correction was performed by subtracting the average amount of FA detected in reaction blanks from the amounts quantified in the samples. Statistical analysis, paired and unpaired <i>t</i>-tests, was performed using SAS 9.3 (SAS Institute, Cary, NC, USA). </p> </div> <div class="2.6.fameanalysis" title="sec"> <div class="title" tagx="title" title="title">
2.6. FAME Analysis</div> <p>Human plasma and a reaction blank were analyzed as previously described [ <a href="#B24-biomolecules-09-00007">24</a>]. Briefly, samples were derivatized using methanolic acetylchloride in <i>n</i>-hexane, and FAMEs were extracted using potassium carbonate in water. The total FAME extracts were analyzed by GC-MS using a total run time of 15 min. </p> </div> </div> <div class="3.resultsanddiscussion" title="sec"> <div class="title" tagx="title" title="title">
3. Results and Discussion</div> <div class="3.1.totalfattyacylanalysisbydirectinfusionftmsanalysis" title="sec"> <div class="title" tagx="title" title="title">
3.1. Total Fatty Acyl Analysis by Direct Infusion FTMS Analysis</div> <p>The aim of the current study was to develop a method that supports fast, precise, and accurate absolute quantification of total FA levels in human blood plasma and serum samples using direct infusion-high resolution FTMS analysis. To this end, we adapted the acid-catalyzed transesterification-based protocol commonly used for total FA analysis by GC-based approaches to generate free fatty acids instead of FAMEs. This was achieved by substituting methanol with H <sub>2</sub>O (see Materials and Methods for details). The rationale for this design was that negative ion mode analysis of deprotonated fatty acids (i.e., [M-H] <sup>−</sup>) is highly sensitive, and that quantitative analysis of intact lipids by direct infusion FTMS can be performed using only a few minutes of analysis per sample [ <a href="#B25-biomolecules-09-00007">25</a>, <a href="#B26-biomolecules-09-00007">26</a>, <a href="#B27-biomolecules-09-00007">27</a>]. Through meticulous method optimization we devised a method that uses only 1 µL of human plasma or serum, instead of 10–50 µL plasma as required by gold-standard GC-MS-based routines [ <a href="#B13-biomolecules-09-00007">13</a>, <a href="#B24-biomolecules-09-00007">24</a>]. For total FA analysis, the 1 µL of plasma or serum is spiked with a defined amount of internal standard (e.g., PE 15:0/18:1(+ <sup>2</sup>H <sub>7</sub>)), followed by evaporating the sample to dryness, hydrolyzing intact lipids using sulfuric acid (H <sub>2</sub>SO <sub>4</sub>) in acetonitrile/H <sub>2</sub>O (9:1, v/v), extracting the free fatty acids with hexane, and analyzing (2% of) the final total FA extract by 1 min of automated direct infusion-high resolution FTMS analysis. Using this approach, in conjunction with hydrolysis in regular <sup>16</sup>O-rich H <sub>2</sub>O, yields high resolution FTMS spectra with detection of endogenous FAs and internal standard-derived FAs as deprotonated carboxylate ions ( <a href="#biomolecules-09-00007-f001">Figure 1</a>A). We note that this approach (i.e., full scan FTMS analysis) only allows annotating detected FAs at the "lipid species-level", indicating the total number C atoms and double bonds in a particular FA analyte. </p> <p>Having established that our strategy in principle enables fast and simple monitoring of total FA levels, we proceeded to evaluate its performance in terms of analytical accuracy. In our initial assessments we found that especially FA 16:0 and FA 18:0 give rise to high background intensity levels ( <a href="#app1-biomolecules-09-00007">Figures S1 and S2</a>). This analytical caveat prompts poor signal-to-background values for the detection of endogenous FA 16:0 and FA 18:0, which in turn hampers their accurate and precise quantification. Importantly, this caveat is common to all total FA analysis methods, including GC-MS-based routines ( <a href="#app1-biomolecules-09-00007">Figure S3</a>), and is caused by FA contaminants present even in high-grade organic solvents, glassware and pipette tips used for sample preparation [ <a href="#B28-biomolecules-09-00007">28</a>]. Notably, we found that the signal-to-background for FTMS analysis of FA 16:0 and FA 18:0 in 1 µL of human plasma was 9.9 and 4.3, respectively ( <a href="#app1-biomolecules-09-00007">Figures S1 and S2</a>). In comparison, the signal-to-background for GC-MS-based analysis of FAME 16:0 and FAME 18:0 in 1 µL of human plasma is approximately 3.3 and 1.8, respectively ( <a href="#app1-biomolecules-09-00007">Figure S3</a>). Hence, the performance our novel method for monitoring of FA 16:0 and FA 18:0, based on lipid hydrolysis with regular <sup>16</sup>O-rich H <sub>2</sub>O and FTMS analysis, is comparable to that of GC-MS-based routines. </p> </div> <div class="3.2.hydrolysiswithh218oimprovessignal-to-background" title="sec"> <div class="title" tagx="title" title="title">
3.2. Hydrolysis with H <sub>2</sub> <sup>18</sup>O Improves Signal-to-Background </div> <p>To minimize, or potentially remove, the high background intensity from FA 16:0 and FA 18:0 we next explored the efficacy of hydrolyzing intact lipids in the presence of <sup>18</sup>O-enriched H <sub>2</sub>O (i.e., H <sub>2</sub> <sup>18</sup>O). We deemed that this procedure should reduce the contribution of background by incorporating a "mass tag" label into endogenous FAs with either one or two <sup>18</sup>O atoms, corresponding to inserting a specific mass offset of 2.0042 and 4.0085, respectively, compared to underivatized, background FA species having two <sup>16</sup>O atoms. Using this strategy, we first found that the unique mass signature of the <sup>18</sup>O atom, and using a resolving power in the order of 400,000 (FWHM at m/z 300) for FTMS analysis, afforded specific detection of FA species labeled with either one or two <sup>18</sup>O atoms with baseline separation from isobaric FA molecule having any configuration of <sup>13</sup>C, <sup>12</sup>C, <sup>2</sup>H, <sup>1</sup>H and <sup>16</sup>O atoms ( <a href="#biomolecules-09-00007-f001">Figure 1</a>B–F). Notably, baseline separation of isobaric FA analytes can be achieved using instrumentation with a mass resolution higher than ~95,000 (FWHM at m/z 300). Assessing the FA 16:0 and FA 18:0 background levels after hydrolysis with H <sub>2</sub> <sup>18</sup>O showed more than a 2.3-fold improvement in the signal-to-background for detection of endogenous FA 16:0 and FA 18:0, respectively ( <a href="#app1-biomolecules-09-00007">Figures S1 and S2</a>). As a safeguard for using H <sub>2</sub> <sup>18</sup>O we also investigated, over a period of 56 days (eight weeks), whether the incorporated <sup>18</sup>O atoms would scramble during sample storage at −20 °C. This assessment demonstrated that <sup>18</sup>O-labeled FA analytes are highly stable and do not incorporate <sup>16</sup>O atoms upon storage ( <a href="#app1-biomolecules-09-00007">Figure S4</a>). Based on these observations, we concluded that acid-catalyzed lipid hydrolysis with H <sub>2</sub> <sup>18</sup>O and high resolution FTMS analysis is a feasible and specific strategy for monitoring total FA levels in human plasma samples. </p> </div> <div class="3.3.instrumentresponseisindependentoffattyacylchainlengthanddoublebondnumber" title="sec"> <div class="title" tagx="title" title="title">
3.3. Instrument Response Is Independent of Fatty Acyl Chain Length and Double Bond Number </div> <p>The combined efficiency of ionization and detection of lipid molecules is an important parameter for accurate and absolute quantification. This efficiency is dependent on numerous parameters, including the chemistry of the lipid class, the FA chain length and number of double bonds, the ionization mechanism (i.e., ion source), the type of mass spectrometer and the acquisition method [ <a href="#B11-biomolecules-09-00007">11</a>, <a href="#B14-biomolecules-09-00007">14</a>]. </p> <p>To examine the instrument response for deprotonated FA analytes, we hydrolyzed and analyzed a FAME standard mixture with 14 different species in equal amounts (w/v) (where 18:3 and 22:5 are present as two isomers). This evaluation showed that all deprotonated FA analytes produced an identical instrument response ( <a href="#biomolecules-09-00007-f002">Figure 2</a>). Importantly, this demonstrates that the instrument response for detection of deprotonated FA species using nanoelectrospray ionization and FTMS analysis is practically independent of FA chain length and the number of double bonds, at least for FAs with chain lengths from 16 to 22. Moreover, this also justifies using the spike of only a single internal standard (e.g., FA 18:1(+ <sup>2</sup>H <sub>7</sub>) derived from PE 15:0/18:1(+ <sup>2</sup>H <sub>7</sub>)) to quantify the wide range of FA analytes present in human plasma and serum samples. </p> </div> <div class="3.4.dynamicquantificationrangeoftotalfattyacylanalysisbyftms" title="sec"> <div class="title" tagx="title" title="title">
3.4. Dynamic Quantification Range of Total Fatty Acyl Analysis by FTMS</div> <p>Next, we evaluated the dynamic quantification range of our method. To this end, we prepared a dilution series where the synthetic standard PE 15:0/18:1(+ <sup>2</sup>H <sub>7</sub>) was titrated relative to a constant amount of the synthetic standard LPC 16:0(+ <sup>2</sup>H <sub>3</sub>). This dilution series was spiked into human plasma samples, which were subjected to direct acid-catalyzed hydrolysis using H <sub>2</sub> <sup>18</sup>O. The resulting total FA extracts were analyzed by 1 min FTMS analysis. To evaluate the dynamic quantification range, we plotted the intensity ratio of FA 18:1(+ <sup>2</sup>H <sub>7</sub>) and FA 16:0(+ <sup>2</sup>H <sub>3</sub>) as a function of their molar ratio ( <a href="#biomolecules-09-00007-f003">Figure 3</a>). The response was linear, with a slope value of approximately one across four orders of magnitude and having a detection limit of approximately 0.9 nM in human plasma (corresponding to the molar ratio value of 0.0013 in <a href="#biomolecules-09-00007-f003">Figure 3</a>). Thus, our method is approximately two orders of magnitude more sensitive than typical GC-MS-based methods [ <a href="#B11-biomolecules-09-00007">11</a>]. Based on this result, we conclude that our novel method affords rapid and highly sensitive quantification of FAs in human plasma. We note that the instrument response for FA 15:0 was also linear, with a slope value of approximately one, but only across three orders of magnitude since human plasma contains low amounts of this odd-chain FA ( <a href="#app1-biomolecules-09-00007">Figure S5</a>). </p> </div> <div class="3.5.accuratefattyacylquantificationbyhighresolutionftmsanalysis" title="sec"> <div class="title" tagx="title" title="title">
3.5. Accurate Fatty Acyl Quantification by High Resolution FTMS Analysis</div> <p>To assess the accuracy of our method, we analyzed the NIST human plasma standard reference material (SRM) 1950. This reference material has previously been used for compiling consensus values on the absolute levels of FAs in human plasma and also for comparing the performance of several laboratories using GC- and LC-based approaches for total FA analysis [ <a href="#B29-biomolecules-09-00007">29</a>]. To benchmark the performance of our method, we subjected the SRM 1950 plasma to acid-catalyzed hydrolysis with H <sub>2</sub> <sup>18</sup>O and analyzed the total FA extracts by direct infusion FTMS analysis. Overall, we found that estimates of total FA levels by our method were in good agreement with the SRM 1950 consensus values, and also that the accuracy of our method was in several instances superior to that of certain laboratories using GC- or LC-based methods for analysis ( <a href="#biomolecules-09-00007-f004">Figure 4</a>). In the comparison to consensus values, we found that the most abundant FA 18:2 had a subtle difference of −7%. For the most abundant monounsaturated FA 16:1 and FA 18:1 the accuracy of quantification differed by 8% and 13%, respectively. For polyunsaturated FAs, such as FA 20:4, 22:5 and 22:6, the accuracy of quantification differed by −7%, 2% and −22%, respectively. For FA 16:0 and FA 18:0, having the highest background intensity (see above), we found that the estimated concentration differed by −37% and −13%. We note that this discrepancy is related to the difference in signal-to-background mentioned above ( <a href="#app1-biomolecules-09-00007">Figures S1 and S3</a>) and the higher accuracy of estimating the FA background level by our approach. As such, our method seems to provide a more accurate estimate of the relatively high FA 16:0 background level than GC-based routines, and using this higher background amount to estimate the endogenous FA 16:0 level yields, by background subtraction, a lower endogenous FA 16:0 concentration than listed in the SRM 1950 Certificate of Analysis. Overall, based on our detailed assessment of the quantification accuracy, we conclude that our novel method affords accurate absolute quantification of total FA levels in human plasma, and its performance is comparable, or at times superior, to that of numerous laboratories using gold-standard GC- and LC-based routines for total FA analysis. </p> </div> <div class="3.6.totalfattyacylanalysisbyftmsispreciseandapplicableforroutineanalysis" title="sec"> <div class="title" tagx="title" title="title">
3.6. Total Fatty Acyl Analysis by FTMS Is Precise and Applicable for Routine Analysis</div> <p>Next, we determined the analytical precision of our novel method for total FA quantification in human plasma samples. To this end, we used two types of plasma samples: i) a pooled plasma sample from fasted normoinsulinemic subjects having a total FA concentration of 11.5 mM (normal lipid level), and ii) a pooled plasma sample from 6 h postprandial hyperinsulinemic subjects having a total FA concentration of 24.8 mM (high lipid level). To determine the intra-day and inter-day precision, we independently prepared and analyzed five total FA extracts of these two types of plasma samples. This sample preparation and analysis was repeated on three independent days. The precision on individual days (intra-day precision) showed a coefficient of variation (CV) always below 8.6% and 11.3% for the normal and high lipid level samples, respectively ( <a href="#biomolecules-09-00007-t001">Table 1</a>). The inter-day (day-to-day) precision showed CV values always below 11.1%. These performance characteristics demonstrate that our novel method for total FA analysis, based on acid-catalyzed hydrolysis with H <sub>2</sub> <sup>18</sup>O and direct FTMS analysis, is similar to that of gold-standard approaches using FAME derivatization and GC-based analysis [ <a href="#B11-biomolecules-09-00007">11</a>], and also to that of routine methods used for quantification of other clinically-relevant lipid molecules [ <a href="#B25-biomolecules-09-00007">25</a>, <a href="#B30-biomolecules-09-00007">30</a>, <a href="#B31-biomolecules-09-00007">31</a>]. </p> </div> <div class="3.7.lipotypingofhumanserumsamplesbytotalfattyacylanalysis" title="sec"> <div class="title" tagx="title" title="title">
3.7. Lipotyping of Human Serum Samples by Total Fatty Acyl Analysis</div> <p>Next, we evaluated the efficacy of our method for determining differences in total FA levels in samples from humans subjected to dietary interventions. For this purpose, we used the NIST SRM 2378 [ <a href="#B32-biomolecules-09-00007">32</a>], a reference material of pooled human serum from three groups of healthy individuals: (i) subjects supplemented with fish oil (enriched in eicosapentaenoic acid, FA 20:5, and docosahexaenoic acid, FA 22:6), (ii) subjects supplemented with flaxseed oil (enriched in linoleic acid, FA 18:2), (iii) and control subjects (no dietary intervention). Total FA analysis of these serum samples demonstrated again that our method is accurate and capable of reproducing expected total FA levels listed in the Certificate of Analysis ( <a href="#app1-biomolecules-09-00007">Table S1</a>). Here the main differences were found for FA 16:0 and FA 20:4, which were in averaged under-estimated by 40% and over-estimated by 42%, respectively. Furthermore, our results also showed that serum from subjects supplemented with fish oil had increased levels of polyunsaturated FAs, including up to 5-fold and 2-fold higher FA 20:5 and FA 22:6 concentrations, respectively, compared to control serum ( <a href="#biomolecules-09-00007-f005">Figure 5</a>). In addition, we also observed a 1.4-fold higher concentration of FA 18:1 in individuals supplemented with flaxseed oil. Taken together, these results consolidate that our novel total FA analysis method supports accurate monitoring of differences in FA composition in fasted human blood samples and is applicable for personalized lipotyping similar to GC- and LC-based approaches. </p> </div> <div class="3.8.quantitativeanalysisofstableisotope-labeledfattyacylsinhumanplasma" title="sec"> <div class="title" tagx="title" title="title">
3.8. Quantitative Analysis of Stable Isotope-Labeled Fatty Acyls in Human Plasma</div> <p>Based on the analytical capability of our high resolution Orbitrap-based mass spectrometer to baseline-separate individual FAs having distinct composition of isotopologues (e.g., <sup>12</sup>C, <sup>13</sup>C, <sup>1</sup>H, <sup>2</sup>H, <sup>16</sup>O and <sup>18</sup>O atoms), we next decided to explore whether our method would also be applicable for metabolic studies in humans. Such studies are commonly done by feeding human subjects with stable isotope-labeled fatty acids, sampling blood plasma at defined time points after intake, and performing time series analysis of tracer incorporation using GC-MS [ <a href="#B22-biomolecules-09-00007">22</a>, <a href="#B33-biomolecules-09-00007">33</a>]. To evaluate the performance of our platform for such studies, we analyzed plasma samples from ten individuals, classified as normoinsulinemic ( <i>n</i> = 5) and hyperinsulinemic ( <i>n</i> = 5). The plasma samples were collected at baseline (time point 0 h) and 6 h after the individuals had consumed a mixed meal containing 200 mg of <sup>13</sup>C <sub>16</sub>-labeled palmitic acid (i.e., FA 16:0(+ <sup>13</sup>C <sub>16</sub>)) [ <a href="#B22-biomolecules-09-00007">22</a>]. Notably, previous analysis has shown that after 6 h the plasma pool of FA 16:0(+ <sup>13</sup>C <sub>16</sub>) is primarily present as a non-esterified fatty acid (~1 µM) and incorporated into triacylglycerol (~2 µM) [ <a href="#B22-biomolecules-09-00007">22</a>]. </p> <p>Total FA analysis using acid-catalyzed H <sub>2</sub> <sup>18</sup>O hydrolysis and 1 min of FTMS analysis showed that deprotonated FA 16:0(+ <sup>13</sup>C <sub>16</sub>) labeled with one or two <sup>18</sup>O atoms were detected in all plasma samples collected at the 6 h time point and not at baseline ( <a href="#biomolecules-09-00007-f006">Figure 6</a>). This shows that our method is specific and effectively free of noise for monitoring the uptake of FA 16:0(+ <sup>13</sup>C <sub>16</sub>) into blood plasma. We also searched the mass spectral data for whether any metabolic products of FA 16:0(+ <sup>13</sup>C <sub>16</sub>) could be detected, including FA 16:1(+ <sup>13</sup>C <sub>16</sub>), FA 18:0(+ <sup>13</sup>C <sub>16</sub>) and FA 18:1(+ <sup>13</sup>C <sub>16</sub>). However, none of these metabolites could be detected, potentially due to the limit of detection of our method. The total FA 16:0(+ <sup>13</sup>C <sub>16</sub>) concentration in plasma from the normoinsulinemic and hyperinsulinemic subjects was estimated to be 4.9 ± 2.4 µM and 5.4 ± 3.1 µM, respectively. Statistical analysis showed no significant difference in the total FA 16:0(+ <sup>13</sup>C <sub>16</sub>) concentration (unpaired, two-sided <i>t</i>-test, <i>p</i>-value = 0.7). Importantly, this result corroborates the previous findings by Pramfalk et al. (2016) reporting a similar total concentration of FA 16:0(+ <sup>13</sup>C <sub>16</sub>) in the plasma samples and no difference between the normoinsulinemic and hyperinsulinemic subjects at the 6 h time point. However, the previous study did find a significant difference in the temporal profile of non-esterified FA 16:0(+ <sup>13</sup>C <sub>16</sub>) level between the two groups of subjects. </p> <p>Next, we investigated whether the plasma concentration of other (unlabeled) FA species showed any major differences across the two time points and between the two groups of subjects. Statistical analysis of time-dependent changes in absolute FA concentrations within each group showed that only three FAs changed significantly; FA 14:0 (in both groups), FA 18:1 (only in normoinsulinemic subjects) and FA 16:0(+ <sup>13</sup>C <sub>16</sub>) (in both groups) (paired, two-sided <i>t</i>-test, <i>p</i>-value &amp;lt; 0.05). No significant differences were found between the two groups (unpaired, two-sided <i>t</i>-test, <i>p</i>-value &amp;lt; 0.05). Notably, this relatively low frequency of significant differences correlates to a relatively high variance in total FA concentration among all the samples ( <a href="#app1-biomolecules-09-00007">Figure S6</a>). </p> <p>As an alternative approach to pinpoint differences, we transformed the concentration data to relative mol%, thereby reducing the influence of variations in the total FA concentration. Repeating the statistical analysis revealed that the relative proportions of 18 out of 31 monitored FA species change significantly in either of the two groups ( <a href="#biomolecules-09-00007-f007">Figure 7</a>) (paired, two-sided <i>t</i>-test, <i>p</i>-value &amp;lt; 0.05). The three common and most significantly changing FAs were FA 16:1, FA 14:0 and FA 18:3, all of which were reduced 6 h after food intake. The analysis also pinpointed a specific subset of polyunsaturated FAs that are significantly changing only in the normoinsulinemic subjects (i.e., FA 22:4, FA 20:3, FA 22:5, FA 18:2, and FA 16:2), and four distinct FA species, primarily saturated and monounsaturated very long chain FA species, FA 24:1, FA 24:0, FA 22:0, and FA 12:0, that are significantly changing only in the hyperinsulinemic subjects. Finally, the statistical analysis of differences between the normoinsulinemic and hyperinsulinemic subjects revealed that only the relative change of FA 16:1 is statistically significant (unpaired, two-sided <i>t</i>-test, <i>p</i>-value = 0.04). Here, the temporal reduction in the FA 16:1 level is 1.5-fold higher in normoinsulinemic as compared to hyperinsulinemic subjects. Taken together, these results highlight that our new method lends itself to also lipid metabolic studies in humans, and other mammals, with the ability to reproducibly and specifically monitor incorporation of stable isotope-labeled FA tracers as well as changes in steady-state FA levels following food intake. </p> </div> </div> <div class="4.conclusions" title="sec"> <div class="title" tagx="title" title="title">
4. Conclusions</div> <p>Here, we described the development and validation of a novel method that supports fast, precise, and accurate absolute quantification of total FA levels in human plasma and serum samples. The method uses acid-catalyzed hydrolysis of intact lipids in the presence of <sup>18</sup>O-labeled H <sub>2</sub>O (i.e., H <sub>2</sub> <sup>18</sup>O) and therethrough the liberation of FA chains as <sup>18</sup>O-labeled analytes. These "mass-tagged" FA analytes can be specifically detected, with improved signal-to-background, using automated chip-based nanoelectrospray ionization and only 1 min of high resolution FTMS analysis on an Orbitrap Fusion mass spectrometer. We demonstrate that our novel method has similar, or improved, analytical performance in terms of accuracy and precision compared to numerous laboratories using gold-standard GC-based methods for total FA analysis ( <a href="#biomolecules-09-00007-f004">Figure 4</a>). We deem that our novel method should prove highly useful for laboratories equipped with high resolution mass spectrometers, but lacking access to GC-based instrumentation. </p> <p>We also demonstrate that our method is applicable for specific tracking of stable isotope-labeled FA tracers in humans. In particular, our method demonstrated that the total concentration of <sup>13</sup>C <sub>16</sub>-labled FA 16:0 in plasma 6 h after intake was similar in normoinsulinemic and hyperinsulinemic subjects. In contrast, the analysis of steady-state FA levels showed that a subset of polyunsaturated FAs, including FA 18:2, FA 20:3, FA 22:4 and FA 22:5, were significantly reduced only in normoinsulinemic subjects and not changed in hyperinsulinemic subjects. Conversely, we found a subset of very long chain FA species, including FA 24:1, FA 24:0 and FA 22:0, that were significantly reduced only in the hyperinsulinemic subjects. These changes corroborate the notions that polyunsaturated FAs have a positive impact on metabolic health [ <a href="#B34-biomolecules-09-00007">34</a>] and that very long chain FAs, especially when incorporated into sphingolipids, are associated with increased cardiovascular risk [ <a href="#B35-biomolecules-09-00007">35</a>, <a href="#B36-biomolecules-09-00007">36</a>]. We note, however, that these results have only been obtained by analysis of a relatively low number of subjects ( <i>n</i> = 5), and that analysis of a larger cohort of subjects, and with more time points, should be performed in order to increase the statistical power of our results. It is tempting to propose that such studies could in future be performed using not only a single FA tracer, but instead a mixture of stable isotope-labeled FA tracers, e.g., FA 18:2n−6, FA 18:3n−3 and FA 16:0, which will simultaneously provide insights into the metabolic trajectories of a wider range of FA species under normoinsulinemic and hyperinsulinemic conditions. </p> <p>We note that our method does not provide the same level of structural resolution as some of the more time-consuming GC-MS-based approaches. However, it does strike a unique balance between analytical speed and FA molecule coverage. As such, gold-standard GC-MS-based approaches are able to monitor around 44 FAME species using an analysis time of 15 min, i.e., effectively detecting 2.9 FAMEs per minute of analysis [ <a href="#B24-biomolecules-09-00007">24</a>]. In comparison, our novel approach supports monitoring of at least 31 FA species by 1 min of analysis. We note that, if needed, this analysis time can be reduced to 0.5 min per sample without jeopardizing the FA coverage and the analytical performance. Furthermore, our method can also be adapted, through increased analysis time, to record MS <sup>2</sup> scans with diagnostic radical FA fragment ions that identify double bond-positional isomers, at least for monounsaturated FA species (e.g., FA 18:1(9) and FA 18:1(11)) [ <a href="#B37-biomolecules-09-00007">37</a>]. Further developments of this orthogonal approach, for example by combining specialized gas-phase dissociation mechanisms such as OzID [ <a href="#B38-biomolecules-09-00007">38</a>], should enable the specific detection and quantification of many more FA double bond positional isomers. It is tempting to propose that such technology, if successfully implemented, could potentially be the beginning of making GC-based platforms for routine analysis of total FA levels obsolete. </p> <p>Finally, our novel method is not only restricted to total FA analysis of human blood samples. We note that our generic, high-throughput method can also be used in many other fields of research. These include routine analysis for assessing food quality, biotechnological application aimed at optimizing oil production in plants and microorganisms, and for biochemical assays of lipid enzymatic activities (e.g., fatty acid synthases, desaturases and elongases). We also note that the use of H <sub>2</sub> <sup>18</sup>O can be replaced with regular <sup>16</sup>O-rich H <sub>2</sub>O to reduce costs, although this will lead to less accurate estimates of total FA 16:0 and 18:0 levels (1 mL H <sub>2</sub> <sup>18</sup>O for 125 samples costs approx. 88 Euro at the present). Finally, we note that our novel method can also serve as a quality control routine to accurately quantify and verify the concentrations of commercially available lipid standards used for absolute quantification in global lipidomics [ <a href="#B21-biomolecules-09-00007">21</a>, <a href="#B26-biomolecules-09-00007">26</a>, <a href="#B35-biomolecules-09-00007">35</a>, <a href="#B39-biomolecules-09-00007">39</a>, <a href="#B40-biomolecules-09-00007">40</a>, <a href="#B41-biomolecules-09-00007">41</a>], instead of using cumbersome colorimetric assays that require high amounts of material to quantify, for example, glycerophospholipids. </p> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>We thank John Bowden for NIST standard reference materials, and members of the Ejsing laboratory for constructive comments. </p> </div> <div class="app-group" title="app-group"> <div class="app" title="app"> <div class="title" tagx="title" title="title">
Supplementary Materials</div> <p>The following are available online at <span class="uri" title="uri"> <a href="http://www.mdpi.com/2218-273X/9/1/7/s1">http://www.mdpi.com/2218-273X/9/1/7/s1</a> </span>, Figure S1, Figure S2, Figure S3, Figure S4, Figure S5, Figure S6, Table S1 and Data file S1. </p> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="biomolecules-09-00007-s001.zip">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>M.H. and C.S.E. designed the study; S.F.G., G.L., L.H. and C.S.E. performed the experiments and all authors participated in data analysis; M.H. and C.S.E. wrote the manuscript; all coauthors read this article and agreed to the submission. </p> <div class="title" tagx="title" title="title">
Funding</div> <p>This work was funded by the Danish Council for Strategic Research (11-116196, C.S.E.), the University of Southern Denmark (SDU2020, C.S.E.) and the VILLUM Center for Bioanalytical Sciences (VKR023179). L.H. is a BHF Senior Research Fellow in Basic Science. </p> <div class="title" tagx="title" title="title">
Conflicts of Interest</div> <p>The authors declare no conflict of interest.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="B1-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">1.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Siguel</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Diagnosing essential fatty acid deficiency</span> <span class="source" tagx="source" title="source">Circulation</span> <span class="year" tagx="year" title="year">1998</span> <span class="volume" tagx="volume" title="volume">97</span> <span class="fpage" tagx="fpage" title="fpage">2580</span> <span class="lpage" tagx="lpage" title="lpage">2583</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1161/01.CIR.97.25.2580">10.1161/01.CIR.97.25.2580</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9657481">9657481</a> </span> </span> </li> <li tag="ref"> <a name="B2-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">2.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moser</span> <span class="given-names" tagx="given-names" title="given-names">A.B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kreiter</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bezman</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lu</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raymond</span> <span class="given-names" tagx="given-names" title="given-names">G.V.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Naidu</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moser</span> <span class="given-names" tagx="given-names" title="given-names">H.W.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Plasma very long chain fatty acids in 3,000 peroxisome disease patients and 29,000 controls </span> <span class="source" tagx="source" title="source">Ann. Neurol.</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">45</span> <span class="fpage" tagx="fpage" title="fpage">100</span> <span class="lpage" tagx="lpage" title="lpage">110</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/1531-8249(199901)45:1&lt;100::AID-ART16&gt;3.0.CO;2-U">10.1002/1531-8249(199901)45:1&amp;lt;100::AID-ART16&amp;gt;3.0.CO;2-U</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9894883">9894883</a> </span> </span> </li> <li tag="ref"> <a name="B3-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">3.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mozaffarian</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Katan</span> <span class="given-names" tagx="given-names" title="given-names">M.B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ascherio</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stampfer</span> <span class="given-names" tagx="given-names" title="given-names">M.J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Willett</span> <span class="given-names" tagx="given-names" title="given-names">W.C.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Trans fatty acids and cardiovascular disease</span> <span class="source" tagx="source" title="source">N. Engl. J. Med.</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">354</span> <span class="fpage" tagx="fpage" title="fpage">1601</span> <span class="lpage" tagx="lpage" title="lpage">1613</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1056/NEJMra054035">10.1056/NEJMra054035</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16611951">16611951</a> </span> </span> </li> <li tag="ref"> <a name="B4-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">4.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fung</span> <span class="given-names" tagx="given-names" title="given-names">T.T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rimm</span> <span class="given-names" tagx="given-names" title="given-names">E.B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Spiegelman</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rifai</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tofler</span> <span class="given-names" tagx="given-names" title="given-names">G.H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Willett</span> <span class="given-names" tagx="given-names" title="given-names">W.C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hu</span> <span class="given-names" tagx="given-names" title="given-names">F.B.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Association between dietary patterns and plasma biomarkers of obesity and cardiovascular disease risk </span> <span class="source" tagx="source" title="source">Am. J. Clin. Nutr.</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">73</span> <span class="fpage" tagx="fpage" title="fpage">61</span> <span class="lpage" tagx="lpage" title="lpage">67</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/ajcn/73.1.61">10.1093/ajcn/73.1.61</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11124751">11124751</a> </span> </span> </li> <li tag="ref"> <a name="B5-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">5.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Forouhi</span> <span class="given-names" tagx="given-names" title="given-names">N.G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koulman</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharp</span> <span class="given-names" tagx="given-names" title="given-names">S.J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Imamura</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kroger</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schulze</span> <span class="given-names" tagx="given-names" title="given-names">M.B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Crowe</span> <span class="given-names" tagx="given-names" title="given-names">F.L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huerta</span> <span class="given-names" tagx="given-names" title="given-names">J.M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guevara</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beulens</span> <span class="given-names" tagx="given-names" title="given-names">J.W.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Differences in the prospective association between individual plasma phospholipid saturated fatty acids and incident type 2 diabetes: The EPIC-InterAct case-cohort study </span> <span class="source" tagx="source" title="source">Lancet Diabetes Endocrinol.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="fpage" tagx="fpage" title="fpage">810</span> <span class="lpage" tagx="lpage" title="lpage">818</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S2213-8587(14)70146-9">10.1016/S2213-8587(14)70146-9</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25107467">25107467</a> </span> </span> </li> <li tag="ref"> <a name="B6-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">6.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Loomba</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Quehenberger</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Armando</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dennis</span> <span class="given-names" tagx="given-names" title="given-names">E.A.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Polyunsaturated fatty acid metabolites as novel lipidomic biomarkers for noninvasive diagnosis of nonalcoholic steatohepatitis </span> <span class="source" tagx="source" title="source">J. Lipid Res.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">56</span> <span class="fpage" tagx="fpage" title="fpage">185</span> <span class="lpage" tagx="lpage" title="lpage">192</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1194/jlr.P055640">10.1194/jlr.P055640</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25404585">25404585</a> </span> </span> </li> <li tag="ref"> <a name="B7-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">7.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peng</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moghaddam</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brinin</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Williams</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mundree</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Haritos</span> <span class="given-names" tagx="given-names" title="given-names">V.S.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Functional assessment of plant and microalgal lipid pathway genes in yeast to enhance microbial industrial oil production </span> <span class="source" tagx="source" title="source">Biotechnol. Appl. Biochem.</span> <span class="year" tagx="year" title="year">2018</span> <span class="volume" tagx="volume" title="volume">65</span> <span class="fpage" tagx="fpage" title="fpage">138</span> <span class="lpage" tagx="lpage" title="lpage">144</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/bab.1573">10.1002/bab.1573</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28649761">28649761</a> </span> </span> </li> <li tag="ref"> <a name="B8-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">8.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liang</span> <span class="given-names" tagx="given-names" title="given-names">M.H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jiang</span> <span class="given-names" tagx="given-names" title="given-names">J.G.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Advancing oleaginous microorganisms to produce lipid via metabolic engineering technology</span> <span class="source" tagx="source" title="source">Prog. Lipid Res.</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">52</span> <span class="fpage" tagx="fpage" title="fpage">395</span> <span class="lpage" tagx="lpage" title="lpage">408</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.plipres.2013.05.002">10.1016/j.plipres.2013.05.002</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23685199">23685199</a> </span> </span> </li> <li tag="ref"> <a name="B9-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">9.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mansson</span> <span class="given-names" tagx="given-names" title="given-names">H.L.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Fatty acids in bovine milk fat</span> <span class="source" tagx="source" title="source">Food Nutr. Res.</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">52</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3402/fnr.v52i0.1821">10.3402/fnr.v52i0.1821</a> </span> </span> </li> <li tag="ref"> <a name="B10-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">10.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Roberts</span> <span class="given-names" tagx="given-names" title="given-names">L.D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McCombie</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Titman</span> <span class="given-names" tagx="given-names" title="given-names">C.M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Griffin</span> <span class="given-names" tagx="given-names" title="given-names">J.L.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A matter of fat: An introduction to lipidomic profiling methods</span> <span class="source" tagx="source" title="source">J. Chromatogr. B Anal. Technol. Biomed. Life Sci.</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">871</span> <span class="fpage" tagx="fpage" title="fpage">174</span> <span class="lpage" tagx="lpage" title="lpage">181</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.jchromb.2008.04.002">10.1016/j.jchromb.2008.04.002</a> </span> </span> </li> <li tag="ref"> <a name="B11-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">11.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dodds</span> <span class="given-names" tagx="given-names" title="given-names">E.D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McCoy</span> <span class="given-names" tagx="given-names" title="given-names">M.R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rea</span> <span class="given-names" tagx="given-names" title="given-names">L.D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kennish</span> <span class="given-names" tagx="given-names" title="given-names">J.M.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Gas chromatographic quantification of fatty acid methyl esters: Flame ionization detection vs. electron impact mass spectrometry </span> <span class="source" tagx="source" title="source">Lipids</span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">40</span> <span class="fpage" tagx="fpage" title="fpage">419</span> <span class="lpage" tagx="lpage" title="lpage">428</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11745-006-1399-8">10.1007/s11745-006-1399-8</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16028722">16028722</a> </span> </span> </li> <li tag="ref"> <a name="B12-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">12.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ruttkay-Nedecky</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Subik</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The OGD1 gene, affecting 2-oxoglutarate dehydrogenase in <i>S. cerevisiae</i>, is closely linked to HIS5 on chromosome IX </span> <span class="source" tagx="source" title="source">Curr. Genet.</span> <span class="year" tagx="year" title="year">1990</span> <span class="volume" tagx="volume" title="volume">17</span> <span class="fpage" tagx="fpage" title="fpage">85</span> <span class="lpage" tagx="lpage" title="lpage">88</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/BF00313254">10.1007/BF00313254</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2178788">2178788</a> </span> </span> </li> <li tag="ref"> <a name="B13-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">13.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kuiper</span> <span class="given-names" tagx="given-names" title="given-names">H.C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wei</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McGunigale</span> <span class="given-names" tagx="given-names" title="given-names">S.L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vesper</span> <span class="given-names" tagx="given-names" title="given-names">H.W.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Quantitation of trans-fatty acids in human blood via isotope dilution-gas chromatography-negative chemical ionization-mass spectrometry </span> <span class="source" tagx="source" title="source">J. Chromatogr. B Anal. Technol. Biomed. Life Sci.</span> <span class="year" tagx="year" title="year">2018</span> <span class="volume" tagx="volume" title="volume">1076</span> <span class="fpage" tagx="fpage" title="fpage">35</span> <span class="lpage" tagx="lpage" title="lpage">43</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.jchromb.2017.12.038">10.1016/j.jchromb.2017.12.038</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29396366">29396366</a> </span> </span> </li> <li tag="ref"> <a name="B14-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">14.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Quehenberger</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Armando</span> <span class="given-names" tagx="given-names" title="given-names">A.M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dennis</span> <span class="given-names" tagx="given-names" title="given-names">E.A.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">High sensitivity quantitative lipidomics analysis of fatty acids in biological samples by gas chromatography-mass spectrometry </span> <span class="source" tagx="source" title="source">Biochim. Biophys. Acta</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">1811</span> <span class="fpage" tagx="fpage" title="fpage">648</span> <span class="lpage" tagx="lpage" title="lpage">656</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.bbalip.2011.07.006">10.1016/j.bbalip.2011.07.006</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21787881">21787881</a> </span> </span> </li> <li tag="ref"> <a name="B15-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">15.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pettinella</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">S.H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cipollone</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blair</span> <span class="given-names" tagx="given-names" title="given-names">I.A.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Targeted quantitative analysis of fatty acids in atherosclerotic plaques by high sensitivity liquid chromatography/tandem mass spectrometry </span> <span class="source" tagx="source" title="source">J. Chromatogr. B Anal. Technol. Biomed. Life Sci.</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">850</span> <span class="fpage" tagx="fpage" title="fpage">168</span> <span class="lpage" tagx="lpage" title="lpage">176</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.jchromb.2006.11.023">10.1016/j.jchromb.2006.11.023</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17174160">17174160</a> </span> </span> </li> <li tag="ref"> <a name="B16-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">16.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kamphorst</span> <span class="given-names" tagx="given-names" title="given-names">J.J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fan</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lu</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">White</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rabinowitz</span> <span class="given-names" tagx="given-names" title="given-names">J.D.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Liquid chromatography-high resolution mass spectrometry analysis of fatty acid metabolism</span> <span class="source" tagx="source" title="source">Anal. Chem.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">83</span> <span class="fpage" tagx="fpage" title="fpage">9114</span> <span class="lpage" tagx="lpage" title="lpage">9122</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/ac202220b">10.1021/ac202220b</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22004349">22004349</a> </span> </span> </li> <li tag="ref"> <a name="B17-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">17.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Choi</span> <span class="given-names" tagx="given-names" title="given-names">A.A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">systematic analysis of fatty acids in human cells with a multiplexed isobaric tag (TMT)-based method </span> <span class="source" tagx="source" title="source">J. Proteome Res.</span> <span class="year" tagx="year" title="year">2018</span> <span class="volume" tagx="volume" title="volume">17</span> <span class="fpage" tagx="fpage" title="fpage">1606</span> <span class="lpage" tagx="lpage" title="lpage">1614</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/acs.jproteome.7b00896">10.1021/acs.jproteome.7b00896</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29521506">29521506</a> </span> </span> </li> <li tag="ref"> <a name="B18-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">18.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stahlman</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ejsing</span> <span class="given-names" tagx="given-names" title="given-names">C.S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tarasov</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Perman</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boren</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ekroos</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">High-throughput shotgun lipidomics by quadrupole time-of-flight mass spectrometry</span> <span class="source" tagx="source" title="source">J. Chromatogr. B Anal. Technol. Biomed. Life Sci.</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">877</span> <span class="fpage" tagx="fpage" title="fpage">2664</span> <span class="lpage" tagx="lpage" title="lpage">2672</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.jchromb.2009.02.037">10.1016/j.jchromb.2009.02.037</a> </span> </span> </li> <li tag="ref"> <a name="B19-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">19.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheng</span> <span class="given-names" tagx="given-names" title="given-names">J.X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ouyang</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xia</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Photochemical tagging for quantitation of unsaturated fatty acids by mass spectrometry</span> <span class="source" tagx="source" title="source">Anal. Chem.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">88</span> <span class="fpage" tagx="fpage" title="fpage">8931</span> <span class="lpage" tagx="lpage" title="lpage">8935</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/acs.analchem.6b02834">10.1021/acs.analchem.6b02834</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27560604">27560604</a> </span> </span> </li> <li tag="ref"> <a name="B20-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">20.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Han</span> <span class="given-names" tagx="given-names" title="given-names">R.H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Han</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Fatty acidomics: Global analysis of lipid species containing a carboxyl group with a charge-remote fragmentation-assisted approach </span> <span class="source" tagx="source" title="source">Anal. Chem.</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">85</span> <span class="fpage" tagx="fpage" title="fpage">9312</span> <span class="lpage" tagx="lpage" title="lpage">9320</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/ac402078p">10.1021/ac402078p</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23971716">23971716</a> </span> </span> </li> <li tag="ref"> <a name="B21-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">21.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gallego</span> <span class="given-names" tagx="given-names" title="given-names">S.F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sprenger</span> <span class="given-names" tagx="given-names" title="given-names">R.R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Neess</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pauling</span> <span class="given-names" tagx="given-names" title="given-names">J.K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Faergeman</span> <span class="given-names" tagx="given-names" title="given-names">N.J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ejsing</span> <span class="given-names" tagx="given-names" title="given-names">C.S.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Quantitative lipidomics reveals age-dependent perturbations of whole-body lipid metabolism in ACBP deficient mice </span> <span class="source" tagx="source" title="source">Biochim. Biophys. Acta</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">1862</span> <span class="fpage" tagx="fpage" title="fpage">145</span> <span class="lpage" tagx="lpage" title="lpage">155</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.bbalip.2016.10.012">10.1016/j.bbalip.2016.10.012</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26581172">26581172</a> </span> </span> </li> <li tag="ref"> <a name="B22-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">22.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pramfalk</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pavlides</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Banerjee</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McNeil</span> <span class="given-names" tagx="given-names" title="given-names">C.A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Neubauer</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Karpe</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hodson</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">fasting plasma insulin concentrations are associated with changes in hepatic fatty acid synthesis and partitioning prior to changes in liver fat content in healthy adults </span> <span class="source" tagx="source" title="source">Diabetes</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">65</span> <span class="fpage" tagx="fpage" title="fpage">1858</span> <span class="lpage" tagx="lpage" title="lpage">1867</span> <span class="pub-id"> <a href="https://dx.doi.org/10.2337/db16-0236">10.2337/db16-0236</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27207513">27207513</a> </span> </span> </li> <li tag="ref"> <a name="B23-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">23.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ellis</span> <span class="given-names" tagx="given-names" title="given-names">S.R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paine</span> <span class="given-names" tagx="given-names" title="given-names">M.R.L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eijkel</span> <span class="given-names" tagx="given-names" title="given-names">G.B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pauling</span> <span class="given-names" tagx="given-names" title="given-names">J.K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Husen</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jervelund</span> <span class="given-names" tagx="given-names" title="given-names">M.W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hermansson</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ejsing</span> <span class="given-names" tagx="given-names" title="given-names">C.S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Heeren</span> <span class="given-names" tagx="given-names" title="given-names">R.M.A.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Automated, parallel mass spectrometry imaging and structural identification of lipids</span> <span class="source" tagx="source" title="source">Nat. Methods</span> <span class="year" tagx="year" title="year">2018</span> <span class="volume" tagx="volume" title="volume">15</span> <span class="fpage" tagx="fpage" title="fpage">515</span> <span class="lpage" tagx="lpage" title="lpage">518</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/s41592-018-0010-6">10.1038/s41592-018-0010-6</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29786091">29786091</a> </span> </span> </li> <li tag="ref"> <a name="B24-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">24.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ecker</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Scherer</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmitz</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liebisch</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A rapid GC-MS method for quantification of positional and geometric isomers of fatty acid methyl esters </span> <span class="source" tagx="source" title="source">J. Chromatogr. B Anal. Technol. Biomed. Life Sci.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">897</span> <span class="fpage" tagx="fpage" title="fpage">98</span> <span class="lpage" tagx="lpage" title="lpage">104</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.jchromb.2012.04.015">10.1016/j.jchromb.2012.04.015</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22542399">22542399</a> </span> </span> </li> <li tag="ref"> <a name="B25-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">25.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gallego</span> <span class="given-names" tagx="given-names" title="given-names">S.F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hojlund</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ejsing</span> <span class="given-names" tagx="given-names" title="given-names">C.S.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Easy, fast, and reproducible quantification of cholesterol and other lipids in human plasma by combined high resolution MSX and FTMS analysis </span> <span class="source" tagx="source" title="source">J. Am. Soc. Mass Spectrom.</span> <span class="year" tagx="year" title="year">2018</span> <span class="volume" tagx="volume" title="volume">29</span> <span class="fpage" tagx="fpage" title="fpage">34</span> <span class="lpage" tagx="lpage" title="lpage">41</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s13361-017-1829-2">10.1007/s13361-017-1829-2</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29063477">29063477</a> </span> </span> </li> <li tag="ref"> <a name="B26-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">26.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Casanovas</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sprenger</span> <span class="given-names" tagx="given-names" title="given-names">R.R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tarasov</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ruckerbauer</span> <span class="given-names" tagx="given-names" title="given-names">D.E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hannibal-Bach</span> <span class="given-names" tagx="given-names" title="given-names">H.K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zanghellini</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jensen</span> <span class="given-names" tagx="given-names" title="given-names">O.N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ejsing</span> <span class="given-names" tagx="given-names" title="given-names">C.S.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Quantitative analysis of proteome and lipidome dynamics reveals functional regulation of global lipid metabolism </span> <span class="source" tagx="source" title="source">Chem. Biol.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">22</span> <span class="fpage" tagx="fpage" title="fpage">412</span> <span class="lpage" tagx="lpage" title="lpage">425</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.chembiol.2015.02.007">10.1016/j.chembiol.2015.02.007</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25794437">25794437</a> </span> </span> </li> <li tag="ref"> <a name="B27-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">27.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Graessler</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schwudke</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schwarz</span> <span class="given-names" tagx="given-names" title="given-names">P.E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Herzog</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shevchenko</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bornstein</span> <span class="given-names" tagx="given-names" title="given-names">S.R.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Top-down lipidomics reveals ether lipid deficiency in blood plasma of hypertensive patients </span> <span class="source" tagx="source" title="source">PLoS ONE</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">e6261</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0006261">10.1371/journal.pone.0006261</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19603071">19603071</a> </span> </span> </li> <li tag="ref"> <a name="B28-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">28.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yao</span> <span class="given-names" tagx="given-names" title="given-names">C.H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">G.Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gross</span> <span class="given-names" tagx="given-names" title="given-names">R.W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Patti</span> <span class="given-names" tagx="given-names" title="given-names">G.J.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Inaccurate quantitation of palmitate in metabolomics and isotope tracer studies due to plastics </span> <span class="source" tagx="source" title="source">Metabolomics</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">12</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11306-016-1081-y">10.1007/s11306-016-1081-y</a> </span> </span> </li> <li tag="ref"> <a name="B29-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">29.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schantz</span> <span class="given-names" tagx="given-names" title="given-names">M.M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Powers</span> <span class="given-names" tagx="given-names" title="given-names">C.D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schleicher</span> <span class="given-names" tagx="given-names" title="given-names">R.L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Betz</span> <span class="given-names" tagx="given-names" title="given-names">J.M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wise</span> <span class="given-names" tagx="given-names" title="given-names">S.A.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Interlaboratory analytical comparison of fatty acid concentrations in serum or plasma</span> <span class="source" tagx="source" title="source">Clin. Chim. Acta</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">462</span> <span class="fpage" tagx="fpage" title="fpage">148</span> <span class="lpage" tagx="lpage" title="lpage">152</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.cca.2016.09.013">10.1016/j.cca.2016.09.013</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27662814">27662814</a> </span> </span> </li> <li tag="ref"> <a name="B30-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">30.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liebisch</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lieser</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rathenberg</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Drobnik</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmitz</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">High-throughput quantification of phosphatidylcholine and sphingomyelin by electrospray ionization tandem mass spectrometry coupled with isotope correction algorithm </span> <span class="source" tagx="source" title="source">Biochim. Biophys. Acta-Mol. Cell Biol. Lipids</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">1686</span> <span class="fpage" tagx="fpage" title="fpage">108</span> <span class="lpage" tagx="lpage" title="lpage">117</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.bbalip.2004.09.003">10.1016/j.bbalip.2004.09.003</a> </span> </span> </li> <li tag="ref"> <a name="B31-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">31.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liebisch</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Binder</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schifferer</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Langmann</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schulz</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmitz</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">High throughput quantification of cholesterol and cholesteryl ester by electrospray ionization tandem mass spectrometry (ESI-MS/MS) </span> <span class="source" tagx="source" title="source">Biochim. Biophys. Acta</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">176</span> <span class="fpage" tagx="fpage" title="fpage">121</span> <span class="lpage" tagx="lpage" title="lpage">128</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.bbalip.2005.12.007">10.1016/j.bbalip.2005.12.007</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16458590">16458590</a> </span> </span> </li> <li tag="ref"> <a name="B32-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">32.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Benner</span> <span class="given-names" tagx="given-names" title="given-names">B.A.</span> <span class="suffix" tagx="suffix" title="suffix">Jr.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schantz</span> <span class="given-names" tagx="given-names" title="given-names">M.M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Powers</span> <span class="given-names" tagx="given-names" title="given-names">C.D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schleicher</span> <span class="given-names" tagx="given-names" title="given-names">R.L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Camara</span> <span class="given-names" tagx="given-names" title="given-names">J.E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharpless</span> <span class="given-names" tagx="given-names" title="given-names">K.E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yen</span> <span class="given-names" tagx="given-names" title="given-names">J.H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sniegoski</span> <span class="given-names" tagx="given-names" title="given-names">L.T.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Standard Reference Material (SRM) 2378 fatty acids in frozen human serum. Certification of a clinical SRM based on endogenous supplementation of polyunsaturated fatty acids </span> <span class="source" tagx="source" title="source">Anal. Bioanal. Chem.</span> <span class="year" tagx="year" title="year">2018</span> <span class="volume" tagx="volume" title="volume">410</span> <span class="fpage" tagx="fpage" title="fpage">2321</span> <span class="lpage" tagx="lpage" title="lpage">2329</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00216-017-0841-5">10.1007/s00216-017-0841-5</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29435636">29435636</a> </span> </span> </li> <li tag="ref"> <a name="B33-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">33.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bickerton</span> <span class="given-names" tagx="given-names" title="given-names">A.S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Roberts</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fielding</span> <span class="given-names" tagx="given-names" title="given-names">B.A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hodson</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blaak</span> <span class="given-names" tagx="given-names" title="given-names">E.E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wagenmakers</span> <span class="given-names" tagx="given-names" title="given-names">A.J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gilbert</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Karpe</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Frayn</span> <span class="given-names" tagx="given-names" title="given-names">K.N.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Preferential uptake of dietary fatty acids in adipose tissue and muscle in the postprandial period </span> <span class="source" tagx="source" title="source">Diabetes</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">56</span> <span class="fpage" tagx="fpage" title="fpage">168</span> <span class="lpage" tagx="lpage" title="lpage">176</span> <span class="pub-id"> <a href="https://dx.doi.org/10.2337/db06-0822">10.2337/db06-0822</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17192479">17192479</a> </span> </span> </li> <li tag="ref"> <a name="B34-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">34.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martinez-Fernandez</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Laiglesia</span> <span class="given-names" tagx="given-names" title="given-names">L.M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huerta</span> <span class="given-names" tagx="given-names" title="given-names">A.E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martinez</span> <span class="given-names" tagx="given-names" title="given-names">J.A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moreno-Aliaga</span> <span class="given-names" tagx="given-names" title="given-names">M.J.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Omega-3 fatty acids and adipose tissue function in obesity and metabolic syndrome</span> <span class="source" tagx="source" title="source">Prostag. Other Lipid Mediat.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">121</span> <span class="fpage" tagx="fpage" title="fpage">24</span> <span class="lpage" tagx="lpage" title="lpage">41</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.prostaglandins.2015.07.003">10.1016/j.prostaglandins.2015.07.003</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26219838">26219838</a> </span> </span> </li> <li tag="ref"> <a name="B35-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">35.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tarasov</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ekroos</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Suoniemi</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kauhanen</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sylvanne</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hurme</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gouni-Berthold</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Berthold</span> <span class="given-names" tagx="given-names" title="given-names">H.K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kleber</span> <span class="given-names" tagx="given-names" title="given-names">M.E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Laaksonen</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Molecular lipids identify cardiovascular risk and are efficiently lowered by simvastatin and <i>PCSK9</i> deficiency </span> <span class="source" tagx="source" title="source">J. Clin. Endocrinol. Metab.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">99</span> <span class="fpage" tagx="fpage" title="fpage">E45</span> <span class="lpage" tagx="lpage" title="lpage">E52</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1210/jc.2013-2559">10.1210/jc.2013-2559</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24243630">24243630</a> </span> </span> </li> <li tag="ref"> <a name="B36-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">36.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Laaksonen</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ekroos</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sysi-Aho</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hilvo</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vihervaara</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kauhanen</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Suoniemi</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hurme</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marz</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Scharnagl</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Plasma ceramides predict cardiovascular death in patients with stable coronary artery disease and acute coronary syndromes beyond LDL-cholesterol </span> <span class="source" tagx="source" title="source">Eur. Heart J.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">37</span> <span class="fpage" tagx="fpage" title="fpage">1967</span> <span class="lpage" tagx="lpage" title="lpage">1976</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/eurheartj/ehw148">10.1093/eurheartj/ehw148</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27125947">27125947</a> </span> </span> </li> <li tag="ref"> <a name="B37-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">37.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pauling</span> <span class="given-names" tagx="given-names" title="given-names">J.K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hermansson</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hartler</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Christiansen</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gallego</span> <span class="given-names" tagx="given-names" title="given-names">S.F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peng</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ahrends</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ejsing</span> <span class="given-names" tagx="given-names" title="given-names">C.S.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Proposal for a common nomenclature for fragment ions in mass spectra of lipids</span> <span class="source" tagx="source" title="source">PLoS ONE</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">12</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">e0188394</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0188394">10.1371/journal.pone.0188394</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29161304">29161304</a> </span> </span> </li> <li tag="ref"> <a name="B38-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">38.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mitchell</span> <span class="given-names" tagx="given-names" title="given-names">T.W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pham</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thomas</span> <span class="given-names" tagx="given-names" title="given-names">M.C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blanksby</span> <span class="given-names" tagx="given-names" title="given-names">S.J.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Identification of double bond position in lipids: From GC to OzID</span> <span class="source" tagx="source" title="source">J. Chromatogr. B Anal. Technol. Biomed. Life Sci.</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">877</span> <span class="fpage" tagx="fpage" title="fpage">2722</span> <span class="lpage" tagx="lpage" title="lpage">2735</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.jchromb.2009.01.017">10.1016/j.jchromb.2009.01.017</a> </span> </span> </li> <li tag="ref"> <a name="B39-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">39.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Surma</span> <span class="given-names" tagx="given-names" title="given-names">M.A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Herzog</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vasilj</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Klose</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Christinat</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morin-Rivron</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Simons</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Masoodi</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sampaio</span> <span class="given-names" tagx="given-names" title="given-names">J.L.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">An automated shotgun lipidomics platform for high throughput, comprehensive, and quantitative analysis of blood plasma intact lipids </span> <span class="source" tagx="source" title="source">Eur. J. Lipid Sci. Technol.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">117</span> <span class="fpage" tagx="fpage" title="fpage">1540</span> <span class="lpage" tagx="lpage" title="lpage">1549</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/ejlt.201500145">10.1002/ejlt.201500145</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26494980">26494980</a> </span> </span> </li> <li tag="ref"> <a name="B40-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">40.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sigruener</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kleber</span> <span class="given-names" tagx="given-names" title="given-names">M.E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Heimerl</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liebisch</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmitz</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maerz</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Glycerophospholipid and sphingolipid species and mortality: The Ludwigshafen Risk and Cardiovascular Health (LURIC) study </span> <span class="source" tagx="source" title="source">PLoS ONE</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">e85724</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0085724">10.1371/journal.pone.0085724</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24465667">24465667</a> </span> </span> </li> <li tag="ref"> <a name="B41-biomolecules-09-00007" /> <span class="label" tagx="label" title="label">41.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sales</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Graessler</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ciucci</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Al-Atrib</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vihervaara</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schuhmann</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kauhanen</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sysi-Aho</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bornstein</span> <span class="given-names" tagx="given-names" title="given-names">S.R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bickle</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Gender, Contraceptives and individual metabolic predisposition shape a healthy plasma lipidome </span> <span class="source" tagx="source" title="source">Sci. Rep.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">27710</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/srep27710">10.1038/srep27710</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27295977">27295977</a> </span> </span> </li> </ul> </div> </div> <div class="floats-group" title="floats-group"> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <p>Detection of deprotonated fatty acyl (FA) analytes by 1 min of high resolution Fourier transform mass spectrometry (FTMS). ( <b>A</b>) Negative ion mode FTMS spectrum of a total FA extract of human plasma using regular <sup>16</sup>O-rich H <sub>2</sub>O for acid-catalyzed hydrolysis. ( <b>B</b>) Negative ion mode FTMS spectrum of a total FA extract of human plasma using <sup>18</sup>O-enriched H <sub>2</sub>O for acid-catalyzed hydrolysis. ( <b>C</b>– <b>F</b>) Selected m/z intervals of the FTMS spectrum shown in B. These m/z intervals show baseline separation of deprotonated FA species with different numbers of double bonds, <sup>18</sup>O, <sup>16</sup>O, <sup>13</sup>C, <sup>12</sup>C, <sup>2</sup>H and <sup>1</sup>H atoms. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <p>Instrument response for detection of deprotonated fatty acyl (FA) analytes is independent of chain length and number of double bonds. A standard mixture with 14 different NEFA species present in equal amounts (where FA 18:3 and FA 22:5 are present as two isomers, 18:3n−3 and 18:3n−6, and 22:5n−3 and 22:5n−6, respectively), was subjected to acid-catalyzed hydrolysis using H <sub>2</sub> <sup>18</sup>O and Fourier transform mass spectrometry (FTMS). Data represent average ± SD ( <i>n</i> = 3). </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <p>Dynamic quantification range of total fatty acyl (FA) analysis. PE 15:0/18:1(+ <sup>2</sup>H <sub>7</sub>) was titrated relative to a constant amount of LPC 16:0(+ <sup>2</sup>H <sub>3</sub>), spiked into 1 µL human plasma and subjected to acid-catalyzed hydrolysis using H <sub>2</sub> <sup>18</sup>O. The total FA extracts were analyzed by direct infusion Fourier transform mass spectrometry (FTMS). The x-axis shows the concentration of FA 18:1(+ <sup>2</sup>H <sub>7</sub>) relative to the concentration of FA 16:0(+ <sup>2</sup>H <sub>3</sub>) (i.e., molar ratio). The y-axis shows the intensity of deprotonated FA 18:1(+ <sup>2</sup>H <sub>7</sub>) relative to the intensity of FA 16:0(+ <sup>2</sup>H <sub>3</sub>). Depicted values derive from two replicate analyses. The line indicates the linear function with slope 1. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 4</span> </a> </div>   <p>Total fatty acyl (FA) levels in human plasma standard reference material (SRM) 1950. Blue bars represent certified and reference values ± expanded uncertainties ( <i>U</i> <sub>95%</sub>) as listed in the certificate of analysis for SRM 1950. Grey bars represent medians of average values measured by different laboratories. The concentrations measured by the different laboratories are shown as individual data points. Green bars represent the average concentration ± SD ( <i>n</i> = 4) measured by Fourier transform mass spectrometry (FTMS) of plasma samples hydrolyzed using sulfuric acid in acetonitrile/H <sub>2</sub> <sup>18</sup>O. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 5</span> </a> </div>   <p>Total fatty acyl (FA) levels in serum from human subjects supplemented with fish oil or flaxseed oil, and subjects receiving no dietary intervention (SRM 2378). Serum lipids were subjected to acid-catalyzed hydrolyzed with H <sub>2</sub> <sup>18</sup>O and analyzed by high resolution Fourier transform mass spectrometry (FTMS). Data represent average ± SD ( <i>n</i> = 4, independently prepared and analyzed samples). * <i>p</i> &amp;lt; 0.001 (unpaired, two-sided Student’s <i>t</i>-test with comparison to "No supplement"). </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 6</span> </a> </div>   <p>Specific monitoring of fatty acyl (FA) 16:0(+ <sup>13</sup>C <sub>16</sub>) in human plasma by high resolution Fourier transform mass spectrometry (FTMS). Representative FTMS spectra of plasma from a normoinsulinemic subject at 0 h (baseline) and 6 h after intake of FA 16:0(+ <sup>13</sup>C <sub>16</sub>). Plasma samples were subjected to acid-catalyzed hydrolysis with H <sub>2</sub> <sup>18</sup>O and analyzed by high resolution FTMS analysis. Indicated FA analytes are detected as deprotonated carboxylate anions. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 7</span> </a> </div>   <p>Total fatty acyl (FA) profiles of normoinsulinemic (NI) and hyperinsulinemic (HI) subjects at 0 h (baseline) and 6 h after intake of FA 16:0(+ <sup>13</sup>C <sub>16</sub>). Plasma lipids were subjected to acid-catalyzed hydrolysis with H <sub>2</sub> <sup>18</sup>O and analyzed by high resolution Fourier transform mass spectrometry (FTMS). Interconnected data points in box plots represent repeated measurements from five individuals within each group. <i>p</i>-values are based on paired, two-sided <i>t</i>-tests of differences within each subject group. <i>q</i>-values are based on unpaired, two-sided <i>t</i>-tests of time-dependent differences between the two groups. <i>p</i>- and <i>q</i>-values below 0.05 are considered statistically significant. </p>   </div> <div class="table-wrap_UNKNOWN">
biomolecules-09-00007-t001_Table 1 <span class="label" tagx="label" title="label">Table 1</span>  <p>Intra-day and inter-day precision of total FA analysis in human plasma samples.</p>  <table> <thead> <tr> <th /> <th /> <th>Intra-Day</th> <th>Inter-Day</th> </tr> <tr> <th /> <th /> <th>Day 1 ( <i>n</i> = 5) </th> <th>Day 2 ( <i>n</i> = 5) </th> <th>Day 3 ( <i>n</i> = 5) </th> <th>( <i>n</i>= 3 × 5) </th> </tr> <tr> <th>Analyte</th> <th>Insulin</th> <th>µM ± SD</th> <th>CV (%)</th> <th>µM ± SD</th> <th>CV (%)</th> <th>µM ± SD</th> <th>CV (%)</th> <th>µM ± SD</th> <th>CV (%)</th> </tr> </thead> <tbody> <tr> <td>FA 16:0</td> <td>Normal</td> <td>1852 ± 103</td> <td>5.5</td> <td>2085 ± 179</td> <td>8.6</td> <td>2028 ± 105</td> <td>5.2</td> <td>1988 ± 161</td> <td>8.1</td> </tr> <tr> <td>High</td> <td>4239 ± 294</td> <td>6.9</td> <td>3798 ± 288</td> <td>7.6</td> <td>4373 ± 218</td> <td>5.0</td> <td>4137 ± 356</td> <td>8.6</td> </tr> <tr> <td>FA 18:1</td> <td>Normal</td> <td>2898 ± 54</td> <td>1.9</td> <td>3132 ± 156</td> <td>5.0</td> <td>3130 ± 183</td> <td>5.8</td> <td>3053 ± 174</td> <td>5.7</td> </tr> <tr> <td>High</td> <td>10859 ± 534</td> <td>4.9</td> <td>9618 ± 279</td> <td>2.9</td> <td>11114 ± 508</td> <td>4.6</td> <td>10531 ± 797</td> <td>7.6</td> </tr> <tr> <td>FA 18:2</td> <td>Normal</td> <td>3404 ± 133</td> <td>3.9</td> <td>3734 ± 192</td> <td>5.1</td> <td>3765 ± 162</td> <td>4.3</td> <td>3634 ± 227</td> <td>6.2</td> </tr> <tr> <td>High</td> <td>3442 ± 275</td> <td>8.0</td> <td>* 3031 ± 102</td> <td>3.4</td> <td>3640 ± 412</td> <td>11.3</td> <td>3396 ± 378</td> <td>11.1</td> </tr> <tr> <td>FA 20:4</td> <td>Normal</td> <td>920 ± 24</td> <td>2.6</td> <td>948 ± 26</td> <td>2.8</td> <td>998 ± 80</td> <td>8.0</td> <td>955 ± 58</td> <td>6.0</td> </tr> <tr> <td>High</td> <td>1728 ± 95</td> <td>5.5</td> <td>1504 ± 56</td> <td>3.7</td> <td>1750 ± 111</td> <td>6.4</td> <td>1661 ± 142</td> <td>8.6</td> </tr> <tr> <td>FA 20:5</td> <td>Normal</td> <td>108 ± 3.6</td> <td>3.3</td> <td>115 ± 3.1</td> <td>2.7</td> <td>121 ± 10</td> <td>8.6</td> <td>115 ± 8.2</td> <td>7.1</td> </tr> <tr> <td>High</td> <td>188 ± 8.9</td> <td>4.7</td> <td>165 ± 6.5</td> <td>3.9</td> <td>194 ± 14</td> <td>7.0</td> <td>182 ± 16</td> <td>8.8</td> </tr> <tr> <td>FA 22:5</td> <td>Normal</td> <td>73 ± 3.0</td> <td>4.1</td> <td>76 ± 1.8</td> <td>2.4</td> <td>82 ± 5.9</td> <td>7.2</td> <td>77 ± 5.3</td> <td>6.8</td> </tr> <tr> <td>High</td> <td>146 ± 8.5</td> <td>5.8</td> <td>123 ± 4.4</td> <td>3.6</td> <td>147 ± 9.9</td> <td>6.7</td> <td>139 ± 13</td> <td>9.6</td> </tr> <tr> <td>FA 22:6</td> <td>Normal</td> <td>197 ± 5.2</td> <td>2.7</td> <td>205 ± 5.3</td> <td>2.6</td> <td>216 ± 16</td> <td>7.4</td> <td>206 ± 12</td> <td>6.0</td> </tr> <tr> <td>High</td> <td>163 ± 8.5</td> <td>5.2</td> <td>141 ± 6.4</td> <td>4.5</td> <td>167 ± 14</td> <td>8.6</td> <td>157 ± 15</td> <td>9.7</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>Values are average fatty acyl (FA) concentrations (µM) in human plasma from a fasted normoinsulinemic subject (representing normal total lipid level) and a 6 h postprandial hyperinsulinemic subject (representing high total lipid level). Plasma samples were hydrolyzed in the presence of H <sub>2</sub> <sup>18</sup>O to assess intra-day precision. Acid-catalyzed hydrolysis of normal and high lipid level samples was done on three consecutive days to assess inter-day precision. * one value out of five for "FA 18:2 Day 2 High insulin" has been removed after outlier correction. CV: coefficient of variation </p> </div> </div> </div> </div>  </body></html>