<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-journal-id" title="nlm-journal-id">0410462</span> <span class="pubmed-jr-id" title="pubmed-jr-id">6011</span> <span class="nlm-ta" title="nlm-ta">Nature</span> <span class="iso-abbrev" title="iso-abbrev">Nature</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Nature</span> </div> <span class="issn-ppub" tagx="issn" title="issn-ppub">0028-0836</span> <span class="issn-epub" tagx="issn" title="issn-epub">1476-4687</span> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6423962">6423962</a> </span> <span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/30046109">30046109</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41586-018-0360-3">10.1038/s41586-018-0360-3</a> </span> <span class="manuscript" title="manuscript">nihpa965312</span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Accumulation of 8,9-unsaturated sterols drives oligodendrocyte formation and remyelination</div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hubler</span> <span class="given-names" tagx="given-names" title="given-names">Zita</span> </span> <a href="#FN1">*</a> <a href="#A1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Allimuthu</span> <span class="given-names" tagx="given-names" title="given-names">Dharmaraja</span> </span> <a href="#FN1">*</a> <a href="#A1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bederman</span> <span class="given-names" tagx="given-names" title="given-names">Ilya</span> </span> <a href="#A2">2</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Elitt</span> <span class="given-names" tagx="given-names" title="given-names">Matthew S.</span> </span> <a href="#A1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Madhavan</span> <span class="given-names" tagx="given-names" title="given-names">Mayur</span> </span> <a href="#A1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Allan</span> <span class="given-names" tagx="given-names" title="given-names">Kevin C.</span> </span> <a href="#A1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shick</span> <span class="given-names" tagx="given-names" title="given-names">H. Elizabeth</span> </span> <a href="#A1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garrison</span> <span class="given-names" tagx="given-names" title="given-names">Eric</span> </span> <a href="#A3">3</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Karl</span> <span class="given-names" tagx="given-names" title="given-names">Molly</span> </span> <a href="#A3">3</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Factor</span> <span class="given-names" tagx="given-names" title="given-names">Daniel C.</span> </span> <a href="#A1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nevin</span> <span class="given-names" tagx="given-names" title="given-names">Zachary S.</span> </span> <a href="#A1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sax</span> <span class="given-names" tagx="given-names" title="given-names">Joel L.</span> </span> <a href="#A1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thompson</span> <span class="given-names" tagx="given-names" title="given-names">Matthew A.</span> </span> <a href="#A1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fedorov</span> <span class="given-names" tagx="given-names" title="given-names">Yuriy</span> </span> <a href="#A4">4</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jin</span> <span class="given-names" tagx="given-names" title="given-names">Jing</span> </span> <a href="#A5">5</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wilson</span> <span class="given-names" tagx="given-names" title="given-names">William K.</span> </span> <a href="#A5">5</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Giera</span> <span class="given-names" tagx="given-names" title="given-names">Martin</span> </span> <a href="#A6">6</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bracher</span> <span class="given-names" tagx="given-names" title="given-names">Franz</span> </span> <a href="#A7">7</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miller</span> <span class="given-names" tagx="given-names" title="given-names">Robert H.</span> </span> <a href="#A3">3</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tesar</span> <span class="given-names" tagx="given-names" title="given-names">Paul J.</span> </span> <a href="#A1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Adams</span> <span class="given-names" tagx="given-names" title="given-names">Drew J.</span> </span> <a href="#A1">1</a> </span> </div> <span class="citation_author_institution" id="A1">[1], </span> <span class="citation_author_institution" id="A2">[2], </span> <span class="citation_author_institution" id="A3">[3], </span> <span class="citation_author_institution" id="A4">[4], </span> <span class="citation_author_institution" id="A5">[5], </span> <span class="citation_author_institution" id="A6">[6], </span> <span class="citation_author_institution" id="A7">[7], </span> <div class="author-notes" title="author-notes"> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label">*</span> <p>These authors contributed equally to this work.</p> </div> <div class="fn-type-con" title="con"> <p>Author Contributions</p> <p>Z.H., D.A., M.S.E., M.M., Z.S.N., K.A., H.E.S., M.T., and D.J.A. evaluated the effects of small molecules and genetic manipulations on oligodendrocyte formation <i>in vitro</i>. Z.H., D.A., I.B., M.A.T., F.B., and D.J.A. performed and analyzed sterol profiling experiments in OPCs <i>in vitro</i>. D.C.F., Y.F., P.J.T., and D.J.A. performed high-throughput screening. Z.H., I.B., H.E.S., E.G., M.M., M.K., R.H.M, P.J.T., and D.J.A. evaluated the <i>in vivo</i> efficacy of small molecules on remyelination and sterol levels. Z.H. and J.L.S. profiled nuclear hormone receptors. Z.H., M.M., and Z.S.N. performed experiments on human cortical spheroids. J.J., W.J.W, M.G., and F.B. synthesized and purified sterol reagents. Z.H., D.A., P.J.T. and D.J.A. analysed all data and wrote the manuscript. All authors provided intellectual input, edited and approved the final manuscript. </p> </div> <div class="corresp" title="corresp"> <b>Contact:</b> Drew Adams, PhD, Department of Genetics and Genome Sciences, Case Western Reserve University School of Medicine, 10900 Euclid Ave., Cleveland, Ohio 44106, USA, Phone: (216) 368-4922, <span class="email" tagx="email" title="email">drew.adams@case.edu</span> </div> </div> <span class="pub-date-nihms-submitted" title="pub-date-nihms-submitted">nihms-submitted: <span>2018-6-6</span> </span> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-7-7</span> </span> <span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2018-8</span> </span> <span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2019-3-3</span> </span> <span class="volume" tagx="volume" title="volume">560</span> <span class="issue" tagx="issue" title="issue">7718</span> <span class="fpage" tagx="fpage" title="fpage">372</span> <span class="lpage" tagx="lpage" title="lpage">376</span> <div class="permissions"> <span class="copyright" title="copyright">(C) , </span> <span class="license" title="license"> <span class="license-p" title="license-p">Users may view, print, copy, and download text and data-mine the content in such documents, for the purposes of academic research, subject always to the full Conditions of use: <span class="uri" title="uri"> <a href="http://www.nature.com/authors/editorial_policies/license.html#terms">http://www.nature.com/authors/editorial_policies/license.html#terms</a> </span> </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Regeneration of myelin is mediated by oligodendrocyte progenitor cells (OPCs), an abundant stem cell population in the CNS and the principal source of new myelinating oligodendrocytes. Loss of myelin-producing oligodendrocytes in the central nervous system (CNS) underlies a number of neurological diseases, including multiple sclerosis (MS) and diverse genetic diseases <sup> <a href="#R1">1</a>– <a href="#R3">3</a> </sup>. Using high throughput chemical screening approaches, we and others have identified small molecules that stimulate oligodendrocyte formation from OPCs and functionally enhance remyelination <i>in vivo</i> <sup> <a href="#R4">4</a>– <a href="#R10">10</a> </sup>. Here we show a broad range of these pro-myelinating small molecules function not through their canonical targets but by directly inhibiting CYP51 (cytochrome P450, family 51), TM7SF2, or EBP (emopamil binding protein), a narrow range of enzymes within the cholesterol biosynthesis pathway. Subsequent accumulation of the 8,9-unsaturated sterol substrates of these enzymes is a key mechanistic node that promotes oligodendrocyte formation, as 8,9-unsaturated sterols are effective when supplied to OPCs in purified form while analogous sterols lacking this structural feature have no effect. Collectively, our results define a unifying sterol-based mechanism-of-action for most known small-molecule enhancers of oligodendrocyte formation and highlight specific targets to propel the development of optimal remyelinating therapeutics. </p> </div> </div> </div> <div class="body" title="body"> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p>Previously we identified structurally-diverse imidazole antifungal drugs as a robust class of small molecules that stimulate the generation of new mouse and human oligodendrocytes and enhance remyelination in mouse disease models <sup> <a href="#R4">4</a> </sup>. Imidazole antifungals mediate their effects in yeast by inhibiting CYP51, an essential enzyme for sterol biosynthesis in both fungal and mammalian cells (for a detailed cholesterol biosynthesis diagram, see <a href="#F5">Extended Data Figure 1</a>). We noted that across a panel of nine azole-containing molecules, the ability to inhibit CYP51 <i>in vitro</i> and in OPCs predicted enhanced formation of myelin basic protein-positive (MBP+) oligodendrocytes from mouse epiblast stem cell-derived OPCs ( <a href="#F1">Fig. 1a-d</a>, <a href="#F6">Extended Data Fig. 2a-c</a>). To measure CYP51 inhibition in OPCs, we used gas chromatography / mass spectrometry (GCMS) to quantitate increased levels of CYP51’s substrate (lanosterol) and decreased cholesterol levels ( <a href="#F1">Fig. 1b</a>, <a href="#F6">Extended Data Fig. 2c-e</a>). <sup> <a href="#R11">11</a> <a href="#R12">12</a> <a href="#R13">13</a> </sup> For ketoconazole, the dose-response for accumulation of lanosterol closely resembled the dose-response for enhanced oligodendrocyte formation ( <a href="#F6">Extended Data Fig. 2f,g</a>). Notably, we confirmed all effects of small molecules on oligodendrocyte formation and sterol levels using a second, independently isolated batch of OPCs, and key results were also validated using mouse primary OPCs ( <a href="#F6">Extended Data Fig. 2b-i</a>; see <a href="#S2">Methods</a> for details of OPC derivations). Additionally, the effects of azole molecules were confirmed using an orthogonal image quantitation approach, a second oligodendrocyte marker, and an LC/MS method for detecting cellular sterols ( <a href="#F6">Extended Data Fig. 2j-l</a>). </p> <p>We next used RNA interference and metabolite supplementation to independently confirm the role of CYP51 in oligodendrocyte formation. Cell-permeable siRNA reagents depleted CYP51 transcript levels in OPCs by 80% <sup> <a href="#R14">14</a> </sup>, led to significant accumulation of lanosterol, and enhanced formation of MBP+ oligodendrocytes ( <a href="#F1">Fig. 1e-f</a>, <a href="#F6">Extended Data Fig. 2m-o</a>). Additionally, we treated OPCs directly with purified lanosterol and observed enhanced formation of MBP+ oligodendrocytes in a dose-responsive fashion ( <a href="#F1">Fig. 1g-h</a>, <a href="#F6">Extended Data Fig. 2p-q</a>). These findings support CYP51 as the functional target of imidazole antifungals in OPCs and suggest that accumulation of sterol intermediates may play a direct role in enhancing oligodendrocyte formation. </p> <p>Since CYP51 inhibition was sufficient to induce the formation of oligodendrocytes, we used a chemical genetics approach to test whether modulation of other steps in cholesterol biosynthesis has a similar effect. ( <a href="#F2">Fig. 2a</a>, <a href="#F5">Extended Data Fig. 1</a>). We used GC/MS-based sterol profiling in OPCs to validate a panel of eight small molecules as selectively inhibiting their known enzyme targets within the cholesterol biosynthesis pathway. ( <a href="#F7">Extended Data Fig. 3a-d</a>; see Source Data for abundance of all quantitated metabolites in all GCMS-based sterol profiling experiments). Only molecules targeting CYP51 (ketoconazole), TM7SF2 (amorolfine <sup> <a href="#R15">15</a> </sup>), and EBP (TASIN-1 <sup> <a href="#R16">16</a> </sup>) enhanced formation of MBP+ oligodendrocytes, whereas inhibitors of the five other pathway enzymes were ineffective ( <a href="#F2">Fig. 2b</a>, <a href="#F7">Extended Data Fig. 3e-h</a>). Treatments had little effect on cell number ( <a href="#F7">Extended Data Fig. 3e</a>). Concentrations of amorolfine and TASIN-1 that enhanced oligodendrocyte formation also led to accumulation of 14-dehydrozymostenol and zymostenol, ( <a href="#F7">Extended Data Fig. 3i-j</a>). Moreover, distinct structural classes of inhibitors of CYP51, TM7SF2, and EBP comparably enhanced oligodendrocyte formation, including at picomolar doses ( <a href="#F8">Extended Data Fig. 4a-h</a>) <sup> <a href="#R17">17</a> </sup>. </p> <p>We also used CRISPR/Cas9 targeting to evaluate the effects of genetic suppression of EBP. OPCs expressing Cas9 and guide RNA targeting EBP demonstrated reduced EBP transcript levels, robust accumulation of the expected intermediate zymostenol, and enhanced formation of oligodendrocytes under differentiation-permissive conditions ( <a href="#F2">Fig. 2c,d</a>, <a href="#F8">Extended Data Fig. 4k</a>). Two independent guide RNA sequences provided comparable results ( <a href="#F8">Extended Data Fig. 4i-l</a>). In total, this genetic and chemical genetic analysis suggests that inhibition of the cholesterol biosynthesis pathway within a limited window of enzymes between CYP51 and EBP is sufficient for enhancing the formation of oligodendrocytes. </p> <p>The efficacy of these small molecules and genetic perturbations is not mediated by simple reduction of sterol levels, as statin drugs and methyl β-cyclodextrin treatment depleted cholesterol levels comparably without enhancing oligodendrocyte formation ( <a href="#F2">Fig. 2b</a>, <a href="#F7">Extended Data Fig. 3a,b</a>, <a href="#F9">5a,b</a>). Because treatment of OPCs with CYP51’s substrate lanosterol enhanced oligodendrocyte formation, we examined the effects of other purified sterols. Treatment of OPCs with 8,9-unsaturated sterols, including 14-dehydrozymostenol (accumulates following TM7SF2 inhibition) and zymostenol (accumulates following EBP inhibition), enhanced the formation of MBP+ oligodendrocytes. By contrast, sterols lacking the 8,9 unsaturation, including cholesterol itself, <sup> <a href="#R18">18</a> </sup> were ineffective ( <a href="#F2">Fig. 2e,h</a>, <a href="#F9">Extended Data Fig. 5c</a>). A total of nine natural and unnatural 8,9-unsaturated sterols enhanced oligodendrocyte formation from OPCs, with 2,2-Dimethyl-zymosterol optimally potent among the sterols evaluated to date ( <a href="#F2">Fig. 2f</a>; <a href="#F9">Extended Data Fig. 5d-l, o</a>). Conversely, preventing the accumulation of 8,9-unsaturated sterols in OPCs by inhibition of lanosterol synthase with Ro 48–8071 abrogated the enhanced oligodendrocyte formation observed from the CYP51 inhibitor ketoconazole. ( <a href="#F9">Extended Data Fig. 5m,n,p</a>). Additionally, analogs of either zymostenol or 8-dehydrocholesterol that lack the 8,9-unsaturation were inactive, highlighting the 8,9-unsaturation as a crucial structural feature for activity in OPCs ( <a href="#F2">Fig. 2g</a>; <a href="#F9">Extended Data Fig. 5k,l</a>). Finally, co-treating OPCs with ketoconazole and MAS-412 provided no further benefit over ketoconazole alone, confirming a redundant mechanism ( <a href="#F9">Extended Data Fig. 5q-r</a>). Together these findings indicate that the accumulation of 8,9-unsaturated sterols in OPCs is a central mechanism for enhancing oligodendrocyte formation whether these sterols arise from small-molecule inhibition of cholesterol biosynthesis enzymes or are supplied to OPCs in purified form. </p> <p>Most of the 8,9-unsaturated sterols shown here to enhance oligodendrocyte formation have previously been shown to function as signaling lipids in oocytes by inducing the resumption of meiosis <sup> <a href="#R19">19</a>, <a href="#R20">20</a> </sup>. While the direct cellular targets of 8,9-unsaturated ‘meiosis-activating sterols’ remain poorly understood, past work suggested nuclear hormone receptors (NHRs) may play a role <sup> <a href="#R19">19</a> </sup>. We evaluated 2,2-dimethylzymosterol and the pathway inhibitors ketoconazole and TASIN-1 in cell-based reporter assays for 20 NHRs, but no molecule showed significant activity in any assay ( <a href="#F9">Extended Data Fig. 5s-u</a>). Additional experiments discounted a role for SREBP2, which transcriptionally regulates cholesterol homeostasis, suggesting these sterols act by mechanisms beyond NHRs or SREBP2 ( <a href="#F9">Extended Data Fig. 5v</a>). Together these studies suggest a novel role for the ‘meiosis-activating sterols’ in promoting oligodendrocyte formation. </p> <p>In parallel, we executed a screen of over 3,000 bioactive small molecules and approved drugs at a uniform dose of 2 μM ( <a href="#F10">Extended Data Fig. 6a</a>). In addition to molecules previously annotated as enhancing OPC differentiation <sup> <a href="#R5">5</a>, <a href="#R6">6</a>, <a href="#R9">9</a> </sup>,we also identified many confirmed hits whose known targets did not cluster into easily discernible categories ( <a href="#SD2">Supporting Table 1</a>). Among the top 10 novel enhancers of oligodendrocyte formation, we noted four molecules had previously been shown to inhibit TM7SF2 or EBP in CNS-derived cells <sup> <a href="#R11">11</a>, <a href="#R21">21</a> </sup>. In fact, GCMS-based sterol profiling revealed that all 10 top hits led to 8,9-unsaturated sterol accumulation at the screening dose, while randomly selected library members had no effect on sterol levels or oligodendrocyte formation. ( <a href="#F3">Fig. 3a</a>, <a href="#F10">Extended Data Fig. 6b-f</a>). </p> <p>Given the frequency of cholesterol pathway modulators within our top screening hits, we assessed whether any previously-reported enhancers of remyelination identified by drug screening might also induce sterol intermediate accumulation. At concentrations that promoted oligodendrocyte formation, benztropine, clemastine, tamoxifen, and U50488 induced accumulation of zymostenol and zymosterol and decreased basal sterol levels, indicative of inhibition of EBP in OPCs ( <a href="#F3">Fig. 3b</a>, <a href="#F10">Extended Data Fig. 6g-l</a>). Tamoxifen has been shown previously to directly inhibit EBP enzymatic activity <sup> <a href="#R22">22</a> <a href="#R11">11</a> <a href="#R23">23</a> </sup>, and we confirmed that benztropine, clemastine, tamoxifen, U50488, and several HTS hits all inhibited EBP directly in a biochemical assay ( <a href="#F3">Fig. 3c</a>). <sup> <a href="#R22">22</a> </sup> By contrast, liothyronine and bexarotene showed minimal effects on sterol levels in OPCs ( <a href="#F3">Fig. 3b</a>, <a href="#F10">Extended Data 6g</a>), consistent with their known functions as modulators of transcription factor function and confirming that many, but not all, treatments that enhance oligodendrocyte formation cause 8,9-unsaturated sterol accumulation. </p> <p>While each of these bioactive small molecules has a previously annotated ‘canonical’ target, extensive structure-activity relationship data reveal that the ability to inhibit EBP, rather than the canonical target, predicts enhanced oligodendrocyte formation. For example, we validated a panel of six muscarinic receptor antagonists that all showed near-completed inhibition of the M1, M3, and M5 muscarinic receptor subtypes at the HTS dose of 2 μM ( <a href="#F10">Extended Data Fig. 6m,p</a>). Among these molecules, only clemastine and benztropine inhibited EBP in OPCs, and only clemastine and benztropine enhanced oligodendrocyte formation ( <a href="#F10">Extended Data Fig. 6j,k,m-r</a>). Likewise, among selective estrogen receptor modulators, toremifine and ospemifine are structurally near-identical and show comparable cellular antiestrogen activity. However, only toremifine inhibited EBP in OPCs, and only toremifine enhanced oligodendrocyte formation ( <a href="#F11">Extended Data Fig. 7a-g</a>). Conversely, while 4-hydroxy-tamoxifen as expected showed 100-fold enhanced cellular antiestrogen activity relative to tamoxifen, both molecules have comparable potency for inhibition of EBP and comparable potency for enhancing oligodendrocyte formation ( <a href="#F11">Extended Data Fig. 7h-j</a>). Finally, the leading novel hit from our HTS, EPZ005687, was annotated as an inhibitor of the histone methyltransferase EZH2. However, analysis of three additional structurally-related EZH2 inhibitors revealed that EPZ005687 uniquely inhibited EBP and enhanced oligodendrocyte formation ( <a href="#F11">Extended Data Fig. 7k-r</a>). Across various antimuscarinics, SERMs, and EZH2 inhibitors, the ability to inhibit EBP, rather than each molecule’s canonical activity, predicts enhanced oligodendrocyte formation. </p> <p>We next tested the potential for combinations of small molecules to show additive or non-additive effects. Combining the thyroid hormone agonist liothyronine with a range of sterol-modulating OPC differentiation-inducing treatments produced additive effects on oligodendrocyte formation, indicating that these molecules likely function by mechanisms other than thyroid hormone receptor signaling to enhance oligodendrocyte generation ( <a href="#F12">Extended Data Fig. 8a-b</a>). In contrast, combinations of ketoconazole at a maximally effective dose with benztropine, clemastine, tamoxifen, or U50488 did not enhance differentiation above levels seen for ketoconazole alone ( <a href="#F12">Extended Data Fig. 8c-e</a>), consistent with these molecules sharing 8,9-unsaturated sterol accumulation as a common mechanism for induction of oligodendrocyte formation. </p> <p>Since our <i>in vitro</i> OPC assays only model the initial differentiation event into oligodendrocytes, we next tested whether sterol pathway modulation also enhanced subsequent oligodendrocyte maturation and myelination <i>in vitro</i> and <i>in vivo</i>. First, we cultured OPCs on electrospun microfibers to assess the effects of sterol pathway modulators on oligodendrocytes’ ability to track and wrap along axon-like substrates <sup> <a href="#R24">24</a>, <a href="#R25">25</a> </sup>. Ketoconazole (CYP51), amorolfine (TM7SF2), and TASIN-1 (EBP) all significantly enhanced MBP+ oligodendrocyte tracking along and wrapping of the microfibers. By contrast, inhibition of other enzymes up- or downstream in the pathway had little effect on oligodendrocyte maturation and ensheathment of microfibers ( <a href="#F12">Extended Data Fig. 8f-k</a>). </p> <p>Previously we established that the imidazole antifungal miconazole, which targets CYP51, penetrates the mouse blood brain barrier and enhances remyelination in mouse models of demyelination <sup> <a href="#R4">4</a> </sup>. Here we evaluated brain-penetrant molecules with affinity for TM7SF2 (ifenprodil) and EBP (tamoxifen) using a well-established mouse model where injection of lysolecithin is used to create focal lesions of demyelination in the dorsal column white matter of the adult spinal cord <sup> <a href="#R26">26</a> </sup>. In vehicle treated animals, profiles of sparsely distributed remyelinating axons characterized by thin myelin sheaths were detected mainly at the periphery of the lesion, while ultrastructural analyses revealed unmyelinated axons or axons with a single wrap of myelin ( <a href="#F4">Fig. 4a,b</a>). By contrast, following 8 days of treatment with ifenprodil or tamoxifen, remyelination was widespread throughout the lesion ( <a href="#F4">Fig. 4a,b</a>, <a href="#F13">Extended Data Fig. 9a</a>), consistent with a recent report regarding tamoxifen <sup> <a href="#R9">9</a> </sup>. Critically, we used GCMS-based sterol profiling of brain tissue from mice treated with miconazole, ifenprodil, and tamoxifen to demonstrate that these therapeutic dosing regimens all led to significant accumulation of 8,9-unsaturated sterols within the mouse brain, indicating inhibition of CYP51, TM7SF2, and EBP ( <a href="#F4">Fig. 4c</a>). Collectively, these data show that small molecule inhibitors of CYP51, TM7SF2, and EBP can engage their sterol pathway targets and significantly enhance remyelination in mice. </p> <p>Finally, the oligodendrocyte-enhancing and sterol-modulating activities of leading pathway inhibitors extend to human cells and tissue. Various small molecules caused accumulation of the expected 8,9-unsaturated sterol intermediates both in a human glioma cell line and in human pluripotent stem cell-derived cortical spheroids <sup> <a href="#R27">27</a> </sup>, confirming that these molecules similarly engage the sterol synthesis pathway in mouse and human cells and CNS tissue ( <a href="#F13">Extended Data Fig. 9b-c</a>). Critically, miconazole and ifenprodil also significantly enhanced the generation human oligodendrocytes in a three-dimensional human pluripotent stem cell-derived cortical spheroid model, indicating conservation of function across species ( <a href="#F4">Fig. 4d-e</a>). </p> <p>Here we have defined a dominant mechanism shared by many small molecule enhancers of remyelination: elevation of 8,9-unsaturated sterol intermediate levels by inhibition of a narrow range of cholesterol biosynthesis enzymes between CYP51 and EBP. In all we have characterized twenty-seven small molecules as both enhancing myelination and elevating 8,9-unsaturated sterol intermediate levels <sup> <a href="#R11">11</a>, <a href="#R21">21</a>, <a href="#R23">23</a> </sup>. Mechanistically, several lines of evidence support a central signaling role for 8,9-unsaturated sterols in the observed enhanced oligodendrocyte formation, including the ability of nine independent 8,9-unsaturated sterols to enhance the formation of oligodendrocytes when supplied to OPCs ( <a href="#F14">Extended Data Fig. 10</a>). </p> <p>Myelin is cholesterol-enriched, and past work has established that genetic or pharmacological treatments that inhibit early enzymes in cholesterol biosynthesis lead to hypomyelination in vivo <sup> <a href="#R28">28</a>– <a href="#R30">30</a> </sup>. Our work supports these observations, as inhibition of HMGCoA reductase and squalene synthesis have neutral-to-negative effects on oligodendrocyte formation in our assays ( <a href="#F2">Fig. 2b</a>, <a href="#F7">Extended Data Fig. 3</a>). These enzymes catalyze steps prior to the synthesis of the first sterol intermediate, so their inhibition prevents the synthesis of all cellular sterols. Our findings establish an alternate paradigm in which the cholesterol biosynthesis pathway can be leveraged to enhance the formation of new oligodendrocytes by targeting later steps whose inhibition does not cause net depletion of cellular sterols. Instead, acute inhibition of CYP51, TM7SF2, or EBP during OPC differentiation induces a ‘sterol shift’ in which a minority of cellular cholesterol is diverted to 8,9-unsaturated sterol intermediates that signal to enhance oligodendrocyte formation. Importantly, we and others have independently shown that multiple molecules now annotated by us as enhancing 8,9-unsaturated sterol intermediate levels can regenerate functional myelin <i>in vivo</i>, as evidenced by reversal of paralysis in mice with MS-like disease <sup> <a href="#R4">4</a> <a href="#R6">6</a> <a href="#R5">5</a> </sup>. Ultimately, our work demonstrates that modulating the sterol landscape in OPCs can enhance the formation of oligodendrocytes and points to new therapeutic targets, potent inhibitors for these targets, and metabolite-based biomarkers to accelerate the development of optimal remyelinating therapeutics. </p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
METHODS</div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
Data Availability Statement</div> <p>The authors declare that data supporting the findings of this study are available within the paper [and its <a href="#SD1">supplementary information files</a>] or are available from the corresponding author upon request. Source data for all GCMS-based sterol profiling experiments and animal experiments are provided with the paper. </p> </div> <div class="statisticsandreproducibility:" title="sec"> <div class="title" tagx="title" title="title">
Statistics and reproducibility:</div> <p>No statistical methods were used to predetermine sample size. Data were expressed as mean ± standard deviation and <i>P</i> values were calculated using an unpaired two-tailed Student’s <i>t</i>-test for pairwise comparison of variables with a 95% confidence interval and n-2 degrees of freedom, where n is the total number of samples, in all figures except <a href="#F4">Figure 4b</a>. In <a href="#F4">Figure 4b</a>, <i>P</i> values were calculated using an unpaired two-tailed Mann-Whitney test with at 95% confidence interval and the data plotted as a Tukey box and whisker plot. Boxes indicate the interquartile range, and the horizontal line represents the median. Biological replicates: <a href="#F1">Figure 1</a>: <b>c</b>) n = 4 wells per condition, except DMSO, n = 24; <b>f</b>) n = 17 wells for DMSO, n = 7 for siControl &amp;amp; siCYP51; <b>g</b>) n = 8 wells for DMSO and n = 4 for lanosterol; <a href="#F2">Figure 2</a>: <b>b)</b> n = 4 wells per condition, except DMSO, n = 24. <b>c)</b> n = 3 wells for sgControl and n = 4 for sgEBP. <b>e, f, g)</b> n = 4 wells per condition, except n = 8 for DMSO and n = 7 ketoconazole in <b>e</b>, n = 12 for DMSO in <b>f</b>, n = 16 for DMSO and ketoconazole, n = 8 for cholesterol in <b>g</b>. Independent experiments: <a href="#F2">Figures 2b and 2f</a> are representative of three and 2c, e, g are two independent experiments using OPC-5 cells; for validation in an independent derivation of OPCs, see <a href="#F7">Extended Data Fig. 3</a>, <a href="#F8">4</a> and <a href="#F9">5</a>. </p> </div> <div class="smallmolecules:" title="sec"> <div class="title" tagx="title" title="title">
Small molecules:</div> <p>The identity and purity of small molecules were authenticated by LC/MS before use ( <a href="#SD2">Supporting Table 2</a>). The following compounds were purchased from Sigma-Aldrich as a solid: Ketoconazole, Miconazole, Clotrimazole, Fluconazole, Fulvestrant, Ifenprodil, Benztropine, Bexarotene, Tamoxifen, 4-Hydroxytamoxifen, Medroxyprogesterone acetate, Ospemifene, GSK343, Trans-U50488, Methyl-β-cyclodextrin, 5α−Cholestan-3β-ol, and Cholesterol. The following compounds were purchased from Cayman Chemicals as a solid: Liothyronine, Clemastine, AY9944, YM53601 and Ro-48–8071. The following compounds were obtained from Janssen Pharmaceuticals as a solid: 2-Methyl-ketoconazole, <i>R</i>-trans-Ketoconazole, and <i>S</i>-trans-Ketoconazole. Mevastatin was purchased as a solid from Selleck Chemicals. The following compounds were purchased from Selleck Chemicals as a 10 mM DMSO solution: Bifonazole, Butoconazole, Amorolfine, Toremifene, EPZ005687, EPZ6438, UNC1999, Hydroxyzine, Ziprasidone, p-Fluorohexahydro-sila-difenidol (abbreviated in figures as Sigma H127), Vesamicol, Raloxifene, L-745,870, TMB-8, Pramoxine, Varespladib, Tanshinone-I, Levofloxacin, Nateglinide, Abiraterone, Allopurinol, Detomidine, Rivastigmine, Beta carotene, BEZ-235, Scopolamine, and Homatropine. Pirenzepine and Telenzepine were purchased from Sigma-Aldrich as a 10 mM DMSO solution. Cholesterol biosynthetic intermediates were purchased from Avanti Polar Lipids as a solid: Lanosterol, Zymosterol, Zymostenol, Lathosterol, Desmosterol, 7-dehydrodesmosterol, FF-MAS (4,4-dimethyl-5α-cholesta-8,14,24-trien-3β-ol), 8,9-dehydrocholesterol, and 2,2-dimethylzymosterol (2,2-dimethyl-5α-cholesta-8,24-dien-3β-ol). 14-dehydrozymostenol (5α-cholesta-8,14-dien-3β-ol), MAS-412 (4,4-dimethyl-5α-cholesta-8,14-dien-3β-ol), and MAS-414 (4,4-dimethyl-5α-cholesta-8-en-3β-ol) were provided by Franz Bracher, Ludwig-Maximilians University of Munich. Imidazole 124 <sup> <a href="#R31">31</a> </sup>, TASIN-1 <sup> <a href="#R16">16</a> </sup>, TASIN-449 <sup> <a href="#R17">17</a> </sup>, and MGI39 <sup> <a href="#R32">32</a> </sup> were synthesized as reported. T-MAS (4,4-dimethyl-5α-cholesta-8,24-dien-3β-ol) from HPLC purification of yeast extracts was provided by Jing Jin and William K. Wilson, Rice University. </p> </div> <div class="mouseopcpreparation:" title="sec"> <div class="title" tagx="title" title="title">
Mouse OPC preparation:</div> <p>To rigorously assess effects of small molecule and genetic treatments on OPCs, all treatments were assayed in two batches of epiblast stem cell-derived OPCs, and key results were confirmed using mouse primary OPCs. OPCs were generated from two separate EpiSC lines, EpiSC5 (giving rise to OPC-5 OPCs) and 129O1 (giving rise to OPC-1 OPCs). Unless otherwise noted, results in OPC-5 cells are presented in Figures <a href="#F1">1</a>– <a href="#F4">4</a> while results in OPC-1 are presented in the Extended Data. </p> <p>EpiSC-derived OPCs were obtained using <i>in vitro</i> differentiation protocols and culture conditions described previously <sup> <a href="#R33">33</a> </sup>. To ensure uniformity throughout all <i>in vitro</i> screening experiments, EpiSC-derived OPCs were sorted to purity by fluorescence activated cell sorting at passage five with conjugated CD 140a-APC (eBioscience, 17–1401; 1:80) and NG2-AF488 (Millipore, AB5320A4; 1:100) antibodies. Sorted batches of OPCs were expanded and frozen down in aliquots. OPCs were thawed into growth conditions for one passage before use in further assays. Cultures were regularly tested and shown to be mycoplasma free. </p> <p>To obtain mouse primary OPCs, whole brain was removed from post-natal day 2 pups anesthetized on ice. Brains were placed in cold DMEM/F12, and the cortices were isolated and the meninges were removed. The cortices were manually chopped and processed with the Tumor Dissociation Kit (Miltenyi) and incubated at 37 °C for 10 minutes. The cell suspension was filtered through a 70 μM filter and centrifuged at 200×g for 4 minutes at room temperature. The cells were washed in DMEM/F12, re-centrifuged and plated in poly-Ornithine and Laminin-treated flasks containing DMEM/F12 supplemented with N2 Max, B27 (ThermoFisher), 20ng/mL FGF, and 20ng/mL PDGF. OPCs were passaged once prior to treatment. Media was changed every 48 hours. </p> </div> <div class="invitrophenotypicscreeningofopcs:" title="sec"> <div class="title" tagx="title" title="title">
In vitro phenotypic screening of OPCs:</div> <p>EpiSC-derived OPCs were grown and expanded in poly-ornithine (PO) and laminin-coated flasks with growth medium (DMEM/F12 supplemented with N2-MAX (R&amp;amp;D Systems), B-27 (ThermoFisher), GlutaMax (Gibco), FGF2 (10 μg/mL, R&amp;amp;D systems, 233-FB-025) and PDGF-AA (10 μg/mL, R&amp;amp;D systems, 233-AA-050) before harvesting for plating. The cells were seeded onto poly-D-lysine 96-well CellCarrier or CellCarrierUltra plates (PerkinElmer) coated with laminin (Sigma, L2020; 15 μg/ml) using multi-channel pipet. For the experiment, 800,000 cells/mL stock in differentiation medium (DMEM/F12 supplemented with N2-MAX and B-27) was prepared and stored on ice for 2 h. Then, 40,000 cells were seeded per well in differentiation medium and allowed to attach for 30 min before addition of drug. For dose–response testing of all molecules except sterols, a 1000x compound stock in dimethyl sulphoxide (DMSO) was added to assay plates with 0.1 μL solid pin multi-blot replicators (V &amp;amp; P Scientific; VP 409), resulting in a final primary screening concentration of 1x. Sterols were added to cells as an ethanol solution (0.2% final ethanol concentration). Positive control wells (ketoconazole, 2.5 μM) and DMSO vehicle controls were included in each assay plate. Cells were incubated under standard conditions (37 °C, 5% CO <sub>2</sub>) for 3 days and fixed with 4% paraformaldehyde (PFA) in phosphate buffered saline (PBS) for 20 min. Fixed plates were washed with PBS (200 μL per well) twice, permeabilized with 0.1% Triton X-100 and blocked with 10% donkey serum (v/v) in PBS for 40 min. Then, cells were labelled with antibodies recognizing MBP (Abcam, ab7349; 1:200) or PLP1 (1:1,000, clone AA3, generously provided by B. Trapp, Cleveland Clinic) for 16 h at 4° C followed by detection with Alexa Fluor conjugated secondary antibodies (1:500) for 45 min. Nuclei were visualized by DAPI staining (Sigma; 1 μg/ml). During washing steps, PBS was added using a multi-channel pipet and aspiration was performed using Biotek EL406 washer dispenser (Biotek) equipped with a 96-well aspiration manifold. </p> </div> <div class="high-contentimagingandanalysis:" title="sec"> <div class="title" tagx="title" title="title">
High-content imaging and analysis:</div> <p>Plates were imaged on the Operetta High Content Imaging and Analysis system (PerkinElmer) and a set of 6 fields captured from each well resulting in an average of 1200 cells being scored per well. Analysis (PerkinElmer Harmony and Columbus software) began by identifying intact nuclei stained by DAPI; that is, those traced nuclei that were larger than 300 μm <sup>2</sup> in surface area. Each traced nucleus region was then expanded by 50% and cross-referenced with the mature myelin protein (MBP) stain to identify oligodendrocyte nuclei, and from this the percentage of oligodendrocytes was calculated. In some experiments, PLP1 staining was performed instead of MBP, or the total process length of MBP+ oligodendrocytes was calculated as previously described. <sup> <a href="#R4">4</a> </sup> </p> </div> <div class="opcsdifferentiationandsterolprofilingaftermethyl-β-cyclodextrintreatment:" title="sec"> <div class="title" tagx="title" title="title">
OPCs differentiation and sterol profiling after Methyl-β-cyclodextrin treatment:</div> <p>EpiSCs derived OPCs harvested from culture flasks were resuspended in 10 mL of differentiation medium to a final cell density of 500,000 cells/mL, to this cell-culture grade water or Methyl-β-cyclodextrin (1 mM) was added and incubated at 37 C. After 30 min the cells were washed twice with differentiation medium (5 mL), and spilt into two portions for differentiation and sterol profiling. The 1000,000 cells/condition were directly processed as described in GC/MS-based sterol profiling to measure the endogenous in sterol levels. For differentiation, the cells were resuspended in differentiation medium to a final cell density of 800,000 cells/mL and plated in a PDL/laminin coated 96-well CellCarrierUltra plate. After 72 h, the cells were fixed, stained, imaged and quantified as described above. </p> </div> <div class="high-throughputscreeningof3,000bioactivesmallmolecules:" title="sec"> <div class="title" tagx="title" title="title">
High-throughput screening of 3,000 bioactive small molecules:</div> <p>EpiSC-derived OPCs were grown and expanded in poly-ornithine and laminin-coated flasks before harvesting for plating. Cells were dispensed in differentiation media supplemented with Noggin (R&amp;amp;D Systems; 100 ng/ml), Neurotrophin 3 (R&amp;amp;D Systems; 10 ng/ml), cAMP (Sigma; 50 μM), and IGF-1 (R&amp;amp;D Systems; 100 ng/ml)) using a Biotek EL406 Microplate Washer Dispenser (Biotek) equipped with 5 μL dispense cassette (Biotek), into poly-D-lysine/ laminin (Sigma, L2020; 4 μg/ml)-coated sterile, 384-well, CellCarrier ultra plates (PerkinElmer), to a final density of 12,500 cells per well and allowed to attach for 45 min before addition of drug. A 3 mM stock of bioactive compound library in dimethylsulphoxide (DMSO) were prepared in an Abgene storage 384-well plate (ThermoFisher Scientific; AB1055). These were added to assay plates using a 50 nL solid pin tool attached to Janus automated workstation (Perkin Elmer), resulting in a final screening concentration of 2 μM. Cells were incubated at 37 °C for 1 hour and then T3 (Sigma; 40 ng/ml) was added to all wells except negative controls, to which FGF (20 ng/ml) was added instead. Negative controls and T3-alone were included in each assay plate. After incubation at 37° C for 72 h, cells were fixed, washed and stained similar to 96-well OPC assay protocol, although all the washing steps were performed using a Biotek EL406 Microplate Washer Dispenser (Biotek) equipped with a 96-well aspiration manifold. Cells were stained with DAPI (Sigma; 1 μg/ml) and MBP antibody (Abcam, ab7349; 1:100). Plates were imaged on the Operetta High Content Imaging and Analysis system (PerkinElmer) and a set of 4 fields captured from each well resulting in an average of 700 cells being scored per well. Analysis was performed as in High-Content Imaging and Analysis, above. All plates for the primary screen were processed and analyzed simultaneously to minimize variability. Molecules causing more than 20% reduction in nuclear count relative to DMSO control wells were removed from consideration, and hits were called on the basis of largest fold-increase in percentage of MBP+ oligodendrocytes relative to DMSO controls within the same plate. When selecting the leading hits for further experiments, molecules obtained in previous screens were omitted, including imidazole antifungals and clemastine. </p> </div> <div class="gc/ms-basedsterolprofiling:" title="sec"> <div class="title" tagx="title" title="title">
GC/MS-based sterol profiling:</div> <p>EpiSC-derived OPCs were plated at 0.5 million cells per ml in PDL- and laminin-coated six or twelve well plate with differentiation media. After 24 hours, cells were dissociated with Accutase, rinsed with saline, and cell pellets were frozen. For sterol analyses, cells were lysed in methanol (Sigma-Aldrich) with agitation for 30 minutes and cell debris removed by centrifugation at 10,000 rpm for 15 min. Cholesterol-d7 standard (25,26,26,26,27,27,27- <sup>2</sup>H <sub>7</sub>-cholesterol, Cambridge Isotope Laboratories) was added before drying under nitrogen stream and derivatization with 55 μl of bis(trimethylsilyl)trifluoroacetamide/trimethylchlorosilane to form trimethylsilyl derivatives. Following derivatization at 60 °C for 20 minutes, 1 μl was analyzed by gas chromatography/mass spectrometry using an Agilent 5973 Network Mass Selective Detector equipped with a 6890 gas chromatograph system and a HP-5MS capillary column (60 m × 0.25 mm × 0.25 μm). Samples were injected in splitless mode and analyzed using electron impact ionization. Ion fragment peaks were integrated to calculate sterol abundance, and quantitation was relative to cholesterol-d7. The following m/z ion fragments were used to quantitate each metabolite: cholesterol-d7 (465), FF-Mas (482), cholesterol (368), zymostenol (458), zymosterol (456), desmosterol (456, 343), 7-dehydrocholesterol (456, 325), lanosterol (393), lathosterol (458), 14-dehydrozymostenol (456). Calibration curves were generated by injecting varying concentrations of sterol standards and maintaining a fixed amount of cholesterol-D7. The human glioma cell line GBM528 was a gift of Jeremy Rich (Cleveland Clinic). These cells were validated as unique by STR profiling (Miller et al., <i>Nature</i>, <b>2017</b> <i>547</i> 355–359). </p> </div> <div class="lc/ms-basedsterolprofiling:" title="sec"> <div class="title" tagx="title" title="title">
LC/MS-based sterol profiling:</div> <p>Sterols were extracted after treatment of OPC-5 OPCs with ketoconazole as described in GC/MS-based sterol profiling above. Picolinate derivatization, chromatographic separation, and mass spectrometric detection were performed as reported previously. <sup> <a href="#R34">34</a> </sup> Peaks from selective reaction monitoring were integrated to calculate sterol abundance, and quantitation was relative to cholesterol- <i>d</i> <sub>7</sub>. </p> </div> <div class="humancorticalspheroids:" title="sec"> <div class="title" tagx="title" title="title">
Human cortical spheroids:</div> <p>Human cortical spheroids were generated as described previously <sup> <a href="#R27">27</a> </sup> with modifications to enable the inclusion and differentiation of OPCs. In brief, spheroids were treated with miconazole or ifenprodil (2 μM) from days 62–72 and assayed on day 93 for MyRF+ oligodendrocytes (Rabbit anti-MyRF antibody was generously provided by M. Wegner and used at 1:1000). [Reference to be updated when spheroid manuscript publishes]. </p> </div> <div class="cyp51enzymaticassay:" title="sec"> <div class="title" tagx="title" title="title">
CYP51 enzymatic assay:</div> <p>CYP51 enzymatic activity was measured using a reported method with slight modifications <sup> <a href="#R35">35</a> </sup>: rat CYP51 (Cypex, Inc.) was used as enzyme; reaction volume was 500 μl; reaction time was 30 minutes; lanosterol concentration was 50 μM; and reactions were quenched with 500 μl isopropanol. Finally, 15 μl of each reaction/isopropanol mixture was injected onto a SCIEX Triple Quad 6500 LC-MS/MS system using an APCI ion source in positive ion mode with a Shimadzu UFLC-20AD HPLC and a Phenomenex Kinetix C18XB 50×2.1×2.6 column at 40 °C. </p> </div> <div class="ebpenzymaticassay:" title="sec"> <div class="title" tagx="title" title="title">
EBP enzymatic assay:</div> <p>EBP enzymatic activity was measured using a reported method with slight modifications <sup> <a href="#R22">22</a> </sup>: active EBP was obtained from mouse microsomes, inhibitors were added, zymostenol was added at a final concentration of 25 μM in a final reaction volume of 500 μl, and the reaction incubated at 37 °C for 2 h. Sterols were extracted using 3 × 1 ml hexanes, cholesterol-d7 was added to enable quantitation, and the pooled organics were dried (Na <sub>2</sub>SO <sub>4</sub>) and evaporated under nitrogen gas. Samples were then silylated and analyzed using GC/MS as described above. </p> </div> <div class="sirnatreatments:" title="sec"> <div class="title" tagx="title" title="title">
siRNA treatments:</div> <p>Cell-permeable siRNAs were obtained as pools of 4 individual siRNAs targeting mouse CYP51, or a non-targeting control (Accell siRNAs, Dharmacon. Pooled CYP51 siRNA sequence: GUCUGUUUUGAGAUUAGU; CGACUAUGCUUCGUUUAUA; CGCUGCUCUUCAAUAGUAA; CUAUUAAGUUAUUGUGAAC. Non-targeting control siRNA: UGGUUUACAUGUCGACUAA). For differentiation analysis, cells were plated in 96-well plate (as detailed above) and treated with 1 μM pooled siRNA suspended in RNAse free water diluted in differentiation media (as detailed above). For sterol analysis cells were plated in a six-well plate at 300,000 cells per well in standard differentiation media supplemented with PDGF (R&amp;amp;D Systems, 20 ng/ml), neurotrophin 3 (R&amp;amp;D Systems; 10 ng/ml), cAMP (Sigma; 50 μM), IGF-1 (R&amp;amp;D Systems; 100 ng/ml), noggin (R&amp;amp;D Systems; 100 ng/ml). At 24 hours, 1 μM siRNA was added to the media. Cells were grown for three more days in siRNA containing media, with growth factor supplementation every 48h, before harvesting and processing for GC/MS analysis as detailed above. </p> </div> <div class="crispr/cas9-mediatedtargetingofebp:" title="sec"> <div class="title" tagx="title" title="title">
CRISPR/Cas9-mediated targeting of EBP:</div> <p>Guide RNA sequences were obtained using the Broad Brie library and manufactured by IDT. Nucleotide sequences (sgRNA sequence: GAAACGCAATCACTACCCAT (sgEBP); GGGGCCTAATTGTGATCACG (sgEBP2)) were prepared and inserted into the LentiCRISPRv2 plasmid (Addgene, 52961) using the instructions from GeckoLibrary preparation: briefly, Fastdigest BsbmB1 (fermentas) was used for plasmid digestion, T4 PNK (NEB M0201S) for nucleotide annealing, and Quick Ligase (NEB M2200S) for sgRNA insertion. Insertion was confirmed by sanger sequencing. Hek293T cells were transfected using Lenti-x shots as per manufacturer’s protocol (Clontech). After 24 hours the media was changed to OPC media for collection of virus. 48 hours later the media was collected, supplemented with FGF, PDGF, and protamine sulfate (Sigma, 8ug/mL), and used to transduce OPCs. 24 hours later the media was changed to non-virus containing media for 48 hours. Cells underwent two 48 hour stretches of puromycin selection (Invitrogen). After 24 hours of recovery in non-selection media, cells were plated for differentiation, GCMS, and qPCR as described above. </p> </div> <div class="focaldemyelination,drugtreatmentandhistologicalanalysis:" title="sec"> <div class="title" tagx="title" title="title">
Focal demyelination, drug treatment and histological analysis:</div> <p>Focal demyelination in the dorsal column of the spinal cord was induced by the injection of 1% LPC solution. 12 week old C57BL/6 female mice were anesthetized using isoflurane and T10 laminectomies were performed. 1μl of 1% LPC was infused into the dorsal column at a rate of 15 μl/hour. At day 4, animals were randomized into treatment groups prior to treatment (2 animals were excluded due to surgical complications). Between days 4 and 11 post laminectomy, animals received daily injections of either vehicle or drug intraperitoneally. Drugs were dissolved in DMSO or corn oil and then diluted with sterile saline for injections such that final doses were 2 mg/kg for Tamoxifen and 10 mg/kg for Ifenprodil. This experiment was done in a blinded manner: compounds were coded to ensure the researchers performing the experiments were unaware of the treatment being administered to each animal. All animals were euthanized 12 days post laminectomy (n= 4–6 per group). Mice were anesthetized using ketamine/xylazine rodent cocktail and then euthanized by transcardial perfusion with 4% PFA, 2% glutaraldehyde, and 0.1 M sodium cacodylate. Samples were osmicated, stained <i>en bloc</i> with uranyl acetate and embedded in EMbed 812, an Epon-812 substitute (EMS). 1 μm sections were cut and stained with toluidine blue and visualized on a light microscope (Leica DM5500B). The number of myelinated axons per unit area was counted from sections obtained from the middle of each lesion and then averaged over each treatment group. All sections within the lesion area were scored (Vehicle, 10 sections; Tamoxifen, 11 sections; Ifenprodil, 28 sections). A Mann-Whitney statistical analysis was performed to assess statistical significance. </p> </div> <div class="analysisofmousebrainsterollevels:" title="sec"> <div class="title" tagx="title" title="title">
Analysis of mouse brain sterol levels:</div> <p>Ten to twelve week old male C57BL/6 mice were injected with 2mg/kg tamoxifen, 10 mg/kg ifenprodil, or 10 mg/kg miconazole dissolved in corn oil (tamoxifen) or DMSO (ifenprodil, miconazole) in sterile saline daily for three days. Mice were anaesthetized with isoflurane and perfused with phosphate buffered saline to remove blood from the brain. Brains were collected and flash frozen using liquid nitrogen. The samples were pulverized and 50–100 milligrams of tissue were collected for further processing. A modified Folch protocol was used for extraction of sterols. <sup> <a href="#R36">36</a> </sup> Briefly, samples were resuspended in a 2:1 chloroform/methanol mixture and homogenized. Cell debris was removed by centrifugation at 4000 g for 10 min. The solution was dried under air and resuspended in hexane with a cholesterol-D7 standard and dried again. Lipids were derivatized with 70 μl of bis(trimethylsilyl)trifluoroacetamide; 2 μls were injected and analyzed by GC/MS as described above. </p> </div> <div class="estrogen-dependentcellproliferationassay:" title="sec"> <div class="title" tagx="title" title="title">
Estrogen-dependent cell proliferation assay:</div> <p>Estrogen-dependent cell proliferation was measured as previously described with minor modifications <sup> <a href="#R37">37</a> </sup>. After growth in estrogen-free media (Phenol red-free RPMI supplemented with 10% charcoal stripped fetal bovine serum) for 5 days, cells were seeded at 2,500 cells/well into 96 well plates. The following day 3x drug containing media was added to triplicate wells and cells were allowed to grow for an additional 5 days at 37 ˚C in standard a 5% CO <sub>2</sub> humidified incubator. Total DNA per well was measured using an adaptation of the method of Labarca and Paigen <sup> <a href="#R38">38</a> </sup>. At this time media was removed, cells were washed one time with 0.25x PBS and 100 μl of distilled water was added. Plates were frozen and thawed to enhance cell lysis and 200 μl of 10 μg/ml Hoechst 33258 (Sigma-Aldrich, St. Louis, MO.) in 2M NaCl, 1mM EDTA, 10mM Tris-HCl pH 7.4 was added. After incubation at room temperature for 2 hours, plates were read in a SpectraMax i3 fluorescent plate reader (Molecular Devices, Sunnyvale, CA) with excitation at 360nm and emission at 460nm. All values were converted to microgram DNA per well using a standard curve derived from purified salmon testes DNA. T47D cells were provided by the Translational Research Shared Resource of the Case Comprehensive Cancer Center and used without further authentication beyond the observed estrogen-dependent cell proliferation. </p> </div> <div class="oligodendrocyteformationandimagingonelectrospunmicrofibers:" title="sec"> <div class="title" tagx="title" title="title">
Oligodendrocyte formation and imaging on electrospun microfibers:</div> <p>A 12-well plate containing Mimetex aligned scaffold (microfiber plate, AMSBIO, AMS.TECL-006–1X, Electrospun poly-L-lactide Scaffold, 2 μM fibre diameter cell crown inserts) was prepared as previously described <sup> <a href="#R24">24</a> </sup>. Briefly, fiber inserts were sterilized with 70% ethanol and washed with PBS before being coated with polyornithine and laminin. After laminin coating, 100,000 cells/mL of EpiSC-derived OPCs (1.5 mL/well) were plated in differentiation medium. After 24 h the media was replaced with fresh media containing small molecule treatments. Every 48 h the media was replaced with fresh compound containing media for a total of 14 days. Plates were fixed with 4% PFA, permeabilized with 0.1% Triton X-100, and blocked with 10% donkey serum (v/v) in PBS for 60 min. Plates were stained for MBP (Abcam, ab7349; 1:100) and DAPI staining (Sigma; 5 μg/ml). After staining, the inserts were moved into new 12-well plate and covered with 2 mL of PBS before imaging in Operetta high content Imaging and analysis system. Plates were imaged on the Operetta High Content Imaging and Analysis system (PerkinElmer) and a set of 8 fields captured from each well resulting in an average of 45,000 cells being scored per well. Analysis (PerkinElmer Harmony and Columbus software) identified intact nuclei stained by DAPI and calculated the MBP signal intensity per cell per well. Microfiber insert tracking images were taken using a Leica DMi8 with a 20x Dry/NA 0.40 objective. Microfiber plate inserts were mounted using Flouromount-G (SouthernBiotech) and allowed to partially harden before coverslips were added and the insert ring was removed. Confocal images were obtained on a Leica SP8 confocal scanning microscope, with 40x oil/NA 1.30 objective. Confocal stacks of 0.336 μm z-steps were taken at 1024×1024. Each fluorophore was excited sequentially and all contrast and brightness changes were applied consistently between images. </p> <p>A separate analysis approach was performed on an independent experiment performed as above except the small-molecule treatment was limited to the first 4 days of the 14 day culture period. After staining, the fiber inserts were mounted on a glass slide (Fisherbrand Superfrost Plus Microscope Slides) using Fluormount-G (Southern Biotech) with a cover glass (Fisherbrand Microscope Cover Glass) and dried at RT in dark for 36 h. The mounted inserts were imaged on the Operetta High Content Imaging and Analysis system (PerkinElmer) and a set of 22 fields captured from each condition resulting in an average of 2000 cells being scored per well. The total microfiber area was calculated using bright filed imaging and a spot-finding function (area larger than 2 px <sup>2</sup>). The MBP+ pixel area within the defined microfiber area was then defined and the percentage of the total microfiber area calculated. </p> </div> <div class="cyp51qpcr:" title="sec"> <div class="title" tagx="title" title="title">
CYP51 qPCR:</div> <p>Cells were plated at 500,000 cells per well in a six-well plate and were grown in standard differentiation media supplemented with PDGF, neurotrophin 3, cAMP, IGF-1, and noggin for four days as described above. At 24 hours, cells were treated with 1 μM siRNA. Growth factors were added every 48 hours. After three days of siRNA treatment, RNA was isolated with the RNeasy Mini Kit (Qiagen), and cDNA was made using High-Capacity RNA-to-cDNA™ Kit (Applied Biosystems). Exon spanning primers for ActinB (Thermo-Fisher, Taqman, Mm02619580_g1) and CYP51 (Thermo-Fisher, Taqman, Mm00490968_m1) were used for detection of relative RNA levels by quantitative real time PCR (Applied Biosystems, 7300 Realtime PCR system). Cycle time and outliers were calculated using Applied Biosystems’ 7300 System Sequence Detection Software version 1.4. </p> </div> <div class="ebpqpcr:" title="sec"> <div class="title" tagx="title" title="title">
EBP qPCR:</div> <p>OPCs were accutased and 1 million cells per cell line was spun down and RNA was isolated with the RNeasy Mini Kit (Qiagen), DNA was removed using the DNA-free RNA removal kit (Invitrogen), and cDNA was made using High-Capacity RNA-to-cDNA™ Kit (Applied Biosystems). Primers for exon 5 of EBP (Forward primer: TGTGCGAGGAGGAAGAAGAT, Reverse Primer: GATAGGCCACCCCGTTTATT) and GAPDH (Forward primer: AGGTCGGTGTGAACGGATTTG; Reverse primer: GGGGTCGTTGATGGCAACA) were manufactured by IDT and gene expression was asses using Power SYBR® Green Master Mix (Applied Biosystems) were used for detection of relative RNA levels by quantitative real time PCR (Quantstudio 7 flex system). Cycle time and outliers were calculated using QuantStudio™ Software V1.3. </p> </div> <div class="muscarinicreceptorantagonismassay:" title="sec"> <div class="title" tagx="title" title="title">
Muscarinic Receptor Antagonism Assay:</div> <p>GeneBLAzer M1-NFAT- <i>bla</i> CHO-K1 cells (or M3- or M5-NFAT- <i>bla</i> CHO-K1 cells)(ThermoFisher) were thawed into Assay Media (DMEM, 10% dialyzed FBS, 25 mM HEPES pH 7.3, 0.1 mM NEAA). 10,000 cells/well were added to a 384-well TC treated assay plate and incubated 16–24 h at 37 ˚C. 4 μl of a 10x stock of antimuscarinic molecules was added to the plate and incubated 30 min. 4 μl of 10x control agonist Carbachol at the pre-determined EC80 concentration was added to wells containing antimuscarinic molecules. The plate was incubated 5 h and 8 μl of 1 μM Substrate + Solution D Loading Solution was added to each well and the plate was incubated 2 h at room temperature before reading on a fluorescence plate reader. </p> </div> <div class="srebpqpcr:" title="sec"> <div class="title" tagx="title" title="title">
SREBP qPCR:</div> <p>Cells were plated at 1 million cells per well in a six-well plate and were grown in standard differentiation media supplemented with with DMSO, Mevastatin (2.5 μM), Ro 48–8071 (500 nM), Ketoconazole (2.5 μM), Tasin-1 (100 nM), or Amorolfine (100 nM). At 24 hours, RNA was isolated with the RNeasy Mini Kit (Qiagen), and cDNA was made using High-Capacity RNA-to-cDNA™ Kit (Applied Biosystems). Exon spanning primers ActinB (Thermo-Fisher, Taqman, Mm02619580_g1), LSS (Thermo-Fisher, Taqman, Mm00461312_m1), LDLR (Thermo-Fisher, Taqman, Mm01177349_m1), and DHCR7 (Thermo-Fisher, Taqman, Mm00514571_m1) were used for detection of relative RNA levels by quantitative real time PCR (Applied Biosystems, 7300 Realtime PCR system). Cycle time and outliers were calculated using Applied Biosystems’ 7300 System Sequence Detection Software version 1.4. </p> </div> <div class="nr2c2andnr2f1luciferaseassays:" title="sec"> <div class="title" tagx="title" title="title">
NR2C2 and NR2F1 Luciferase assays:</div> <p>Forty-eight hours prior to transfection, 100,000 Hek293T cells were plated per well in a 24 well plate. HEK293T cells were chosen because they were used previously in this assay. <sup> <a href="#R39">39</a> </sup> NR2C2 (Origene, MR221079) or NR2F1 (gift from Dr. Christian Schaaf) and NGFI promoter reporter plasmid (generous gift of C. Schaaf) were transfected using Lipofectamine 2000 (Thermo-Fisher, 11668027) as per manufacturer’s protocol. After 16 hours, Hek293 cells were treated with the compounds (2,2-dimethyl-zymosterol 5 μM, FF-MAS 10 μM, Ketoconazole 2.5 μM, Tasin-1 100 nM, Mevastatin 2.5 μM, Liothyronine 3 μM, and all-trans retinoic acid 5 μM). 32 hours later cells were lysed using a firefly Luciferase assay system (Promega, E1500) and readout using Synergy Neo2 High Performance plate reader. </p> </div> <div class="nuclearreceptorprofiling:" title="sec"> <div class="title" tagx="title" title="title">
Nuclear receptor profiling:</div> <p>Luciferase reporter assays performed by Indigo Biosciences were used to assess interaction of 2,2-dimethylzymosterol (5 μM), Ketoconazole (2.5 μM), and Tasin-1 (250 nM) with human ERα, GR, LXRβ, NFκB, NRF2, PGR, PPARα, PPARγ, RARα, RARγ, RXRα, RXRβ, TRα, TRβ and VDR in <i>agonist</i> mode and ERRα, RORα and RORγ in <i>inverse-agonist</i> mode. The reporter for these assays is firefly luciferase linked with either the genetic response elements (GRE) or the Gal4 upstream activation sequence (UAS). These cells also express either the native receptor or a receptor in which the native N-terminal DNA binding domain (DBD) has been replaced with that of the yeast Gal4 DBD. The specifics of each assay are shown in the table below. In brief, a suspension of Reporter Cells was prepared in Cell Recovery Medium (CRM; containing 5% (RORγ) or 10% charcoal stripped FBS for others). 100 μl of the Reporter Cell suspension was dispensed into wells of a white 96-well assay plate. Test compound, reference compounds, and the respective vehicle were diluted into INDIGO’s Compound Screening Medium (CSM; containing 5% (RORγ) or 10% charcoal stripped FBS for others). 100 μl of each treatment medium was dispensed into duplicate assay wells pre-dispensed with Reporter Cells. Assay plates were incubated at 37 <sup>◦</sup>C for 24 hr. Following the incubation period, for <i>agonist and inverse-agonist</i> assays, treatment media were discarded and 100 μl/well of Luciferase Detection Reagent was added. RLUs were quantified from each assay well to determine <i>agonist</i> or <i>inverse-agonist</i> activity using the following assay designs: </p> <p> <b>ERα (NR3A1);</b>Native Receptor;ER GRE-Luciferase </p> <p> <b>ERRα (NR3B1);</b>Gal4 DBD hybrid Receptor;Gal4 UAS-Luciferase </p> <p> <b>GR (NR3C1);</b>Native Receptor;GR GRE-Luciferase </p> <p> <b>LXRβ (NR1H2);</b>Gal4 DBD hybrid Receptor;Gal4 UAS-Luciferase </p> <p> <b>PGR (NR3C3);</b>Native Receptor;PGR GRE-Luciferase </p> <p> <b>PPARδ (NR1C2);</b>Gal4 DBD hybrid Receptor;Gal4 UAS-Luciferase </p> <p> <b>PPARγ (NR1C3);</b>Gal4 DBD hybrid Receptor;Gal4 UAS-Luciferase </p> <p> <b>RARα (NR1B1);</b>Gal4 DBD hybrid Receptor;Gal4 UAS-Luciferase </p> <p> <b>RARγ (NR1B3);</b>Gal4 DBD hybrid Receptor;Gal4 UAS-Luciferase </p> <p> <b>RORα (NR1F1);</b>Gal4 DBD hybrid Receptor;Gal4 UAS-Luciferase </p> <p> <b>RORγ (NR1F3);</b>Gal4 DBD hybrid Receptor;Gal4 UAS-Luciferase </p> <p> <b>RXRα (NR2B1);</b>Gal4 DBD hybrid Receptor;Gal4 UAS-Luciferase </p> <p> <b>RXRβ (NR2B2);</b>Gal4 DBD hybrid Receptor;Gal4 UAS-Luciferase </p> <p> <b>TRα (NR1A1);</b>Gal4 DBD hybrid Receptor;Gal4 UAS-Luciferase </p> <p> <b>TRβ (NR1A2);</b>Gal4 DBD hybrid Receptor;Gal4 UAS-Luciferase </p> <p> <b>VDR (NR1I1);</b>Gal4 DBD hybrid Receptor;Gal4 UAS-Luciferase </p> <p> <b>NF-κB;</b>Native NF-κB;NF-κB GRE-Luciferase </p> <p> <b>NRF2;</b>Native Receptor;ARE-Luciferase </p> </div> <div class="animalwelfare:" title="sec"> <div class="title" tagx="title" title="title">
Animal Welfare:</div> <p>All animal experiments were performed in accordance with protocols approved by the Case Western Reserve University and George Washington University Institutional Animal Care and Use Committees. </p> </div> </div> <div class="extended-data" title="extended-data"> <div class="title" tagx="title" title="title">
Extended Data</div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Extended Data Figure 1.</span> </a> </div>   <div class="title" tagx="title" title="title">
Expanded cholesterol synthesis pathway diagram.</div> <p>The cascade cyclization of squalene epoxide, catalyzed by lanosterol synthase (LSS), provides the first sterol, lanosterol. Processing of lanosterol to cholesterol can proceed via the Kandutsch-Russell and/or Bloch pathways, which use the same enzymes and process substrates that vary only in the presence or absence of the C24 double bond. Intermediates in blue have been confirmed in our GC/MS-based sterol profiling assay using authentic standards. Sterol 14-reductase activity in mouse is shared by two genes, TM7SF2 and LBR. Consistent with past reports (ref. 21), inhibition of sterol 14-reductase activity can lead to accumulation of the expected upstream intermediate (FF-MAS) or 14-dehydrozymostenol, also known as cholesta-8,14-dien-3-β-ol. Green indicates enzyme targets and small molecules whose inhibition promotes oligodendrocyte formation. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Extended Data Figure 2.</span> </a> </div>   <div class="title" tagx="title" title="title">
CYP51 is the functional target by which imidazole antifungals enhance oligodendrocyte formation. </div> <p> <b>a)</b> Azole molecules with varying degrees of potency for mammalian CYP51 inhibition. Throughout, green labels indicate molecules considered active, while red labels indicate inactive molecules. <b>b)</b> Percentage of MBP <sup>+</sup> oligodendrocytes generated from a second, independent derivation of OPCs (OPC-1) at 72 h following treatment with the indicated concentrations of azoles. n = 4 wells per condition except DMSO (n = 24), with &amp;gt; 1,000 cells analyzed per well. <b>c)</b> GC/MS-based quantitation of lanosterol levels in a second derivation of OPCs (OPC-1) treated 24 h with the indicated azoles at 2.5 μM. n = 2 wells per condition. <b>d, e)</b> GC/MS-based quantitation of cholesterol levels in OPCs (OPC-5 and OPC-1) treated 24 h with the indicated azoles at 2.5 μM. n = 2 wells per condition <b>. f, g)</b> GC/MS-based quantitation of lanosterol levels in OPCs (OPC-5, OPC-1) treated 24 h with the indicated doses of ketoconazole. n = 2 wells per condition. Concentrations shown in panels <b>f</b> and <b>g</b> mirror those shown in <a href="#F1">Fig. 1c</a> and panel <b>b</b>. <b>h)</b> Percentage of MBP <sup>+</sup> oligodendrocytes generated from mouse primary OPCs at 72 h following treatment with the indicated imidazole antifungals at 3 μM. n = 4 wells per condition, with &amp;gt; 1,000 cells analyzed per well. <b>i)</b> GC/MS-based quantitation of lanosterol levels in mouse primary OPCs treated 24 h with the indicated imidazole antifungals at 3 μM. n = 2 wells per condition. <b>j)</b> Assessment of oligodendrocyte formation using an alternative image quantitation metric, fold increase in total neurite length. Panel is a re-analysis of data shown in <a href="#F1">Fig. 1c</a>. n = 4 wells per condition except DMSO (n = 24), with &amp;gt; 1,000 cells analyzed per well. <b>k)</b> Percentage of oligodendrocytes generated from OPCs at 72 h following treatment with ketoconazole (2.5 μM) as measured by PLP1 immunostaining. Left, OPC-5; right, OPC-1. n = 8 wells per condition, with &amp;gt; 1,000 cells analyzed per well. <b>l)</b> LC/MS-based quantitation of lanosterol levels in OPC-5 treated 24 h with ketoconazole at 2.5 μM. n = 2 wells per condition. <b>m)</b> CYP51 mRNA levels measured by RT-qPCR following 96-h treatment with non-targeting or CYP51-targeting pools of cell-permeable siRNAs. n = 2 wells per condition. <b>n</b>) GC/MS-based quantitation of lanosterol levels in OPC-1 treated 96 h with the indicated pooled siRNA reagents. n = 2 wells per condition. <b>o)</b> Percentage of MBP <sup>+</sup> oligodendrocytes generated from a second, independent batch of OPCs (OPC-1) at 72 h following treatment with the indicated reagents. n = 3 wells per condition, with &amp;gt; 1,000 cells analyzed per well. <b>p)</b> Percentage of MBP <sup>+</sup> oligodendrocytes generated from an independent derivation of OPCs at 72 h following treatment with exogenous lanosterol. n = 4 wells per condition except DMSO and Ketoconazole (n = 8), with &amp;gt; 1,000 cells analyzed per well. <b>q)</b> Representative images of OPC-5 cells treated 72 h with the indicated siRNA reagents and lanosterol. Nuclei are labeled with DAPI (blue), and oligodendrocytes are indicated by immunostaining for myelin basic protein (green). Scale bar, 100 μm. All bar graphs indicate mean ± standard deviation, panels <b>b</b>, <b>d, h</b>, <b>i</b>, <b>k</b>, <b>l</b>, <b>o</b> and <b>p</b> are representative of two independent experiments, and all findings have been confirmed in a second independent derivation of OPCs ( <a href="#F1">Fig. 1</a>). </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Extended Data Figure 3.</span> </a> </div>   <div class="title" tagx="title" title="title">
Effect of small-molecule inhibition of the cholesterol biosynthesis pathway on enhancing oligodendrocyte formation. </div> <p> <b>a)</b> GC/MS-based quantitation of sterol levels in OPCs (OPC-5) treated 24 h with the indicated inhibitors of cholesterol biosynthesis. Left, cholesterol; right, desmosterol. n = 2 wells per condition. Inhibitors were used at the following doses unless otherwise noted: mevastatin, ketoconazole, MGI-39, 2.5 μM; YM53601, 2 μM; Ro 48–8071, amorolfine, TASIN-1, 100 nM; AY9944, 200 nM. <b>b)</b> GC/MS-based quantitation of sterol levels in a second derivation of OPCs (OPC-1). Left, cholesterol; right, desmosterol. n = 2 wells per condition. <b>c)</b> GC/MS-based quantitation of the sterol intermediates expected to accumulate following treatment of OPCs with the indicated inhibitors of cholesterol biosynthesis for 24 h. n = 2 wells per condition. <b>d)</b> GC/MS-based quantitation of the sterol intermediates expected to accumulate following treatment of a second derivation of OPCs (OPC-1) with the indicated inhibitors of cholesterol biosynthesis for 24 h. n = 2 wells per condition. In <b>c</b> and <b>d</b>, no accumulation of other sterol intermediates indicative of off-target effects within the cholesterol pathway were observed (see Source Data). <b>e)</b> Representative images of OPC-5 cells treated 72 h with the indicated small molecules. All treatments are at the highest concentration shown in <a href="#F2">Fig. 2b</a>. Scale bar, 100 μm. <b>f)</b> Percentage of MBP <sup>+</sup> oligodendrocytes generated from a second batch of OPCs (OPC-1) at 72 h following treatment with the indicated cholesterol pathway inhibitors. n = 4 wells per condition, except DMSO, n = 24, with &amp;gt; 1,000 cells analyzed per well. <b>g)</b> Percentage of MBP <sup>+</sup> oligodendrocytes generated from mouse primary OPCs at 72 h following treatment with the indicated cholesterol pathway inhibitors at 300 nM. n = 4 wells per condition, except DMSO, n = 8, with &amp;gt; 1,000 cells analyzed per well. <b>h)</b> GC/MS-based quantitation of sterol intermediate levels in mouse primary OPCs treated 24 h with the indicated inhibitors of cholesterol biosynthesis at 300 nM. Left, 14-dehydrozymostenol levels following treatment with amorolfine; Right, zymostenol levels following treatment with TASIN-1. n = 2 wells per condition. <b>i, j)</b> GC/MS-based quantitation of sterol intermediate levels in OPC-5 ( <b>i</b>) and OPC-1 ( <b>j</b>) treated 24 h with the indicated doses of inhibitors of cholesterol biosynthesis. Left, 14-dehydrozymostenol levels following treatment with amorolfine; Right, zymostenol levels following treatment with TASIN-1. n = 2 wells per condition. Concentrations shown in panel i mirror those shown in panel f. All bar graphs indicate mean +/− standard deviation, and panels <b>a</b>, <b>c</b>, <b>e-h</b> are representative of two independent experiments. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Extended Data Figure 4.</span> </a> </div>   <div class="title" tagx="title" title="title">
Effect of independent chemical-genetic and genetic modulators of CYP51, sterol 14 reductase and EBP on oligodendrocyte formation and cholesterol biosynthesis. </div> <p> <b>a, d, g)</b> Percentage of MBP <sup>+</sup> oligodendrocytes generated from two independent derivation of OPCs at 72 h following treatment with the indicated concentrations of medroxyprogesterone acetate ( <b>a</b>), 2-methyl ketoconazole ( <b>d</b>) or TASIN-449 ( <b>g</b>). n = 4 wells per condition, except DMSO, n = 12 in <b>a</b> and <b>d</b>. In <b>g</b>, for OPC-5, n = 4 except DMSO, n = 7; for OPC-1, n = 3 except DMSO, n = 6. <b>b, e, h)</b> GC/MS-based quantitation of sterol levels in two independent derivations of OPCs treated 24 h with medroxyprogesterone acetate at 10 μM ( <b>a</b>), 2-methyl ketoconazole at 2.5 μM ( <b>e</b>) and TASIN-449 at the indicated concentrations ( <b>h</b>). n = 2 wells per condition. <b>c, f)</b> Rat CYP51 enzymatic activity following treatment with varying concentrations of medroxyprogesterone acetate ( <b>c</b>) and 2-methyl ketoconazole ( <b>f</b>) as measured by LC/MS-based quantitation of the CYP51 product FF-MAS. n = 2 independent enzymatic assays. <b>i)</b> Percentage of MBP <sup>+</sup> oligodendrocytes generated from OPCs (OPC-5) infected with lentivirus expressing Cas9 and an independent guide RNA targeting EBP (cf. <a href="#F2">Fig. 2c</a>). 8 wells per condition, with &amp;gt; 1,000 cells analyzed per well. Two-tailed Student’s <i>t</i>-test, * <i>P</i> = 0.0009. <b>j)</b> Functional validation of CRISPR-based targeting of EBP with a second sgRNA using GC/MS-based quantitation of zymostenol levels. n = 2 wells per condition. <b>k)</b> EBP mRNA levels measured by RT-qPCR in OPCs (OPC-5) infected with lentivirus expressing Cas9 and either of two guide RNAs targeting EBP. One well per condition, with results validated in an independent experiment. <b>l</b>) Representative images of the oligodendrocyte formation assay shown in Main <a href="#F2">Figure 2c</a>. Nuclei are labeled with DAPI (blue), and oligodendrocytes are indicated by immunostaining for myelin basic protein (green). Scale bar, 100 μm. All bar graphs indicate mean +/− standard deviation, and panels <b>a, d, g</b>, <b>i and k</b> are representative of two independent experiments. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Extended Data Figure 5.</span> </a> </div>   <div class="title" tagx="title" title="title">
Effect of 8,9-unsaturated sterols on oligodendrocyte formation.</div> <p> <b>a)</b> Percentage of MBP <sup>+</sup> oligodendrocytes generated from OPCs (OPC-5) at 72 h following treatment with methyl β-cyclodextrin (1 mM) for 30 min at 37 °C. n = 8 wells per condition, with &amp;gt; 1,000 cells analyzed per well. <b>b)</b> GC/MS-based quantitation of cholesterol (left) and desmosterol (right) levels in OPCs (OPC-5) treated with methyl β-cyclodextrin (Me-β-CD) at 1 mM or ketoconazole at 2.5 μM. n = 2 wells per condition. <b>c, d)</b> Percentage of MBP <sup>+</sup> oligodendrocytes generated from OPC-1 ( <b>c</b>) and OPC-5 cells ( <b>d</b>) at 72 h following treatment with the indicated purified sterol intermediates. n = 4 wells per condition, except n = 8 for DMSO and ketoconazole, with &amp;gt; 1,000 cells analyzed per well. Green text highlights metabolites that accumulate after treatments that enhance oligodendrocyte formation ( <a href="#F2">Fig. 2e</a>, <a href="#F7">Extended Data Fig. 3c</a>). <b>e)</b> Percentage of MBP+ oligodendrocytes generated from OPC­1 at 72 h following treatment with MAS-412 and MAS-414. n = 4 wells per condition, with &amp;gt; 1,000 cells analyzed per well. <b>f)</b> Representative images of OPC­5 cells treated 72 h with DMSO, MAS-412, or MAS-414 (3 μM). Nuclei are labeled with DAPI (blue), and oligodendrocytes are indicated by immunostaining for myelin basic protein (green). Scale bar, 100 μm. <b>g)</b> Percentage of MBP <sup>+</sup> oligodendrocytes generated from OPC-1 at 72 h following treatment with 2,2-dimethyl-zymosterol. n = 4 wells per condition except DMSO (n = 12), with &amp;gt; 1,000 cells analyzed per well. <b>h)</b> Representative images of OPC-5 cells treated 72 h with vehicle and 2,2-dimethyl-zymosterol (2.5 μM). Nuclei are labeled with DAPI (blue), and oligodendrocytes are indicated by immunostaining for myelin basic protein (green). Scale bar, 100 μm. <b>i)</b> Percentage of MBP+ oligodendrocytes generated from OPC-5 (left) and OPC-1 (right) at 72 h following treatment with FF-MAS or T-MAS. n = 4 wells per condition except DMSO and Ketoconazole (n = 8), with &amp;gt; 1,000 cells analyzed per well. <b>j)</b> Percentage of MBP <sup>+</sup> oligodendrocytes generated from OPC-5 and OPC-1 OPCs at 72 h following treatment with the indicated concentrations of cholesterol. n = 8 wells per condition, with &amp;gt; 1,000 cells analyzed per well. <b>k, l)</b> Percentage of MBP <sup>+</sup> oligodendrocytes generated from OPC-5 and OPC-1 at 72 h following treatment with the indicated concentrations of sterols that are structurally identical aside from the presence or absence of the 8,9 double bond (structures in panel <b>o</b>). n ≥ 3 wells per condition (see dot plots as replicate values vary by condition), with &amp;gt; 1,000 cells analyzed per well. <b>m)</b> Percentage of MBP <sup>+</sup> oligodendrocytes generated from OPCs (OPC-5) at 72 h following treatment with the indicated small molecules or combinations of small molecules (ketoconazole, 2.5 μM; Ro 48–8071, 11 nM; liothyronine, 3 μM). n = 3 wells per condition, except DMSO n = 11, ketoconazole n = 13, liothyronine n = 8 &amp;amp; liothyronine + Ro 48–8071 n = 4, with &amp;gt; 1,000 cells analyzed per well. <b>n)</b> GC/MS-based quantitation of lanosterol levels in OPCs (OPC-5) treated 24 h with the indicated small molecules or combinations of small molecules at concentrations stated in m. n = 2 wells per condition. <b>o)</b> Structures of zymostenol, 8,9-dehydrocholesterol, 5α-cholestanol, and cholesterol. <b>p)</b> Total cell number as measured by counting of DAPI+ nuclei in the experiment presented in panel <b>m</b>. <b>q, r)</b> Percentage of MBP+ oligodendrocytes generated from OPCs (OPC­5 and OPC-1) at 72 h following treatment with the indicated small molecules or combinations of small molecules in two independent batches of OPCs (ketoconazole, 2.5 μM; MAS­412, 5 μM). In <b>q</b>, n = 16 for DMSO, 8 for Ketoconazole, and 4 for remaining bars. In <b>r</b>, n = 8 wells per condition. <b>s)</b> Luciferase reporter assays were used to assess if 2,2-Dimethylzymosterol (5 μM), Ketoconazole (2.5 μM), and TASIN-1 (250 nM) modulate human ERα, GR, LXRβ, NFkB, NRF2, PGR, PPARδ, PPARγ, RARα, RARγ, RXRα, RXRβ, TRα, TRβ and VDR transcriptional activity in <i>agonist</i> mode and ERRα, RORα and RORγ in <i>inverse-agonist</i> mode. n = 2 wells per condition and n = 3 wells per positive control conditions. <b>t)</b> Effects of sterols (2,2-dimethylzymosterol 5 μM, FF-MAS 10 μM) and small molecules (Ketoconazole 2.5 μM, TASIN-1 100 nM) on the NR2F1-mediated activation of a NGFI-A promoter driven luciferase reporter. n = 2 wells per condition. <b>u)</b> Effects of 2,2-dimethylzymosterol (5 μM) on NR2C2-mediated activation of a NGFI-A promoter driven luciferase reporter in comparison to cells transfected with reporter only, untreated, or treated with a previously reported positive control (all-trans retinoic acid, ATRA, 5 μM). n = 2 wells per condition. <b>v)</b> LSS, DHCR7, LDLR mRNA levels measured by RT-qPCR following 24 h treatment with DMSO, Mevastatin (2.5 μM), Ro 48–8071 (500 nM), Ketoconazole (2.5 μM), TASIN-1 (100 nM), or Amorolfine (100 nM). n = 2 wells. All bar graphs indicate mean +/− standard deviation, and panels <b>a-n</b>, and <b>t-v</b> are representative of two independent experiments. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Extended Data Figure 6</span> </a> </div>   <div class="title" tagx="title" title="title">
Inhibiting CYP51, TM7SF2 and EBP is a unifying mechanism for many small-molecule enhancers of oligodendrocyte formation identified by high-throughput screening. </div> <p> <b>a)</b> Percentage of MBP <sup>+</sup> oligodendrocytes (relative to DMSO control wells) generated from OPCs (OPC-1 derivation) at 72 h following treatment with a library of 3,000 bioactive small molecules, each at 2 μM. Each dot represents the result for one small molecule in the library. Red, imidazole antifungals; blue, clemastine; green, EPZ005687, the top novel hit molecule which is discussed in <a href="#F11">Extended Data Fig. 7</a> below. <b>b, c)</b> Percentage of MBP <sup>+</sup> oligodendrocytes generated from OPCs (Left: OPC-5; Right: OPC-1) at 72 h following treatment with ketoconazole, nine top molecules identified by bioactives screening (green), and nine randomly chosen library members (red) at a uniform dose of 5 μM. n = 4 wells per condition except DMSO and Ketoconazole, n = 12 wells, with &amp;gt; 1,000 cells analyzed per well. <b>d)</b> GC/MS-based quantitation of zymosterol, zymostenol, and 14-dehydrozymostenol levels in a second batch of OPCs treated 24 h with the indicated screening hits and randomly chosen library members at 2 μM. n = 1; for validation in a second derivation of OPCs, see <a href="#F3">Fig. 3a</a>. Molecules are clustered by enzyme targeted (top labels). <b>e)</b> Percentage of MBP <sup>+</sup> oligodendrocytes generated from OPCs at 72 h following treatment with the indicated doses of fulvestrant, one of the top 10 HTS hits. n = 4 wells per condition except DMSO, n = 12), with &amp;gt; 1,000 cells analyzed per well. <b>f)</b> GC/MS-based quantitation of lanosterol levels in OPCs treated 24 h with fulvestrant at 2 μM. n = 2 wells per condition. <b>g, h, i)</b> GC/MS-based quantitation of metabolite levels in OPCs treated 24 h with the indicated previously-reported enhancers of oligodendrocyte formation at the following doses: Benztropine, 2 μM; Clemastine, 1 μM; Tamoxifen, 100 nM; U50488, 5 μM; bexarotene, 1 μM; liothyronine, 3 μM. n = 2 wells per condition. <b>j, k)</b> Percentage of MBP <sup>+</sup> oligodendrocytes generated from OPCs (OPC-5 left, OPC-1 right) at 72 h following treatment with the indicated previously-reported enhancers of oligodendrocyte formation. n = 4 wells per condition, except DMSO n = 20 for OPC-5 and n = 12 for OPC-1, with &amp;gt; 1,000 cells analyzed per well. All doses are μM. <b>l)</b> Representative images of OPCs treated 72 h with the indicated small molecules. All treatments in <b>l</b> are at the highest concentration shown in panel <b>j</b>. Scale bar, 100 μm. <b>m)</b> Structures of muscarinic receptor antagonists used in this study. <b>n, q)</b> Percentage of MBP <sup>+</sup> oligodendrocytes generated from OPCs (OPC-5: top, OPC-1: bottom) at 72 h following treatment with ketoconazole or the indicated muscarinic receptor modulators at 2 μM, the concentration used during screening. n = 4 wells per condition except DMSO and Ketoconazole, n = 8, with &amp;gt; 1,000 cells analyzed per well. <b>o)</b> GC/MS-based quantitation of three metabolite levels in OPC-5 OPCs treated 24 h with U50488 (5 μM) or the indicated muscarinic receptor modulators (2 μM). Left, zymostenol; center, cholesterol; right, desmosterol. n = 2 wells per condition. <b>p)</b> Heatmap indicating inhibition of muscarinic receptor isoforms M1, M3, and M5 by the indicated small molecules (2 μM) assayed using GeneBLAzer NFAT- <i>bla</i> CHO-K1 cells. n = 2 wells per condition. <b>r)</b> GC/MS-based quantitation of three metabolite levels in OPC-1 OPCs treated 24 h with clemastine (1 μM) or the indicated muscarinic receptor modulators at 2 μM. n = 2 wells per condition. Left, zymostenol; center, zymosterol; right, cholesterol. <i>p</i>-Fluorohexahydro-sila-difenidol is abbreviated as Sigma H127. All bar graphs indicate mean +/− standard deviation, and panels <b>b</b>, <b>c</b>, <b>e, i, j, k, n</b> and <b>q</b> are representative of two independent experiments. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Extended Data Figure 7.</span> </a> </div>   <div class="title" tagx="title" title="title">
Effect of selective estrogen receptor modulators and EZH2 inhibitors on cellular EBP function and oligodendrocyte formation. </div> <p> <b>a)</b> Structures of selective estrogen receptor modulators used in this study. <b>b)</b> Effects of ospemifene and toremifene on the estrogen-dependent growth of T47D cells. n = 3 wells per condition. <b>c, d)</b> Percentage of MBP <sup>+</sup> oligodendrocytes generated from two independent batches of OPCs at 72 h following treatment with ospemifene and toremifene. n = 4 wells per condition except DMSO and Ketoconazole, n = 8, with &amp;gt; 1,000 cells analyzed per well. <b>e)</b> Representative images of OPCs treated 72 h with the indicated small molecules. All molecules were treated at 300 nM. Scale bar, 100 μm. <b>f, g)</b> GC/MS-based quantitation of two metabolite levels in OPCs treated 24 h with ospemifene and toremifene at 300 nM. Left, zymostenol; right, cholesterol. n = 2 wells per condition. <b>h)</b> Percentage of MBP <sup>+</sup> oligodendrocytes generated from two independent batches of OPCs at 72 h following treatment with tamoxifen and 4-hydroxytamoxifen. Left, OPC-5; right, OPC-1. n = 4 wells per condition, except DMSO, n = 6 for OPC-1 (right). <b>i)</b> Effects of tamoxifen and 4-hydroxytamoxifen on the estrogen-dependent growth of T47D cells. n = 3 wells per condition. <b>j)</b> GC/MS-based quantitation of zymostenol (left axis) and zymosterol levels (right axis) in OPC-5 and OPC-1 treated 24 h with tamoxifen and 4-hydroxytamoxifen at the indicated concentrations. n = 2 wells per condition. Keto = Ketoconazole. <b>k)</b> Percentage of MBP <sup>+</sup> oligodendrocytes generated from OPCs at 72 h following treatment with the indicated structurally analogous EZH2 inhibitors. n = 4 wells per condition, except DMSO, n = 12, with &amp;gt; 1,000 cells analyzed per well. <b>l)</b> Percentage of MBP <sup>+</sup> oligodendrocytes generated from a second batch of OPCs at 72 h following treatment with the indicated structurally analogous EZH2 inhibitors. n = 4 wells per condition, except DMSO, n = 12, with &amp;gt; 1,000 cells analyzed per well. <b>m)</b> Percentage of MBP <sup>+</sup> oligodendrocytes generated from mouse primary OPCs at 72 h following treatment with EPZ005687. n = 4 wells per condition, except DMSO, n = 12, with &amp;gt; 1,000 cells analyzed per well. <b>n)</b> Structure of EPZ005687 and structurally analogous EZH2 inhibitors. <b>o)</b> Representative images of OPCs treated 72 h with the indicated EZH2 inhibitors. All treatments are at 2 μM. Scale bar, 100 μm. <b>p)</b> GC/MS-based quantitation of two sterol intermediates following treatment of OPCs with the indicated EZH2 inhibitors at 1 μM for 24 h. Left, zymostenol; right, zymosterol. n = 2 wells per condition. <b>q)</b> GC/MS-based quantitation of two sterol intermediates following treatment of a second derivation of OPCs with the indicated EZH2 inhibitors at 1 μM for 24 h. Left, zymostenol; right, zymosterol. n = 2 wells per condition. <b>r)</b> GC/MS-based quantitation of two sterol intermediates following treatment of mouse primary OPCs with EPZ005687 at 2 μM for 24 h. Left, zymostenol; right, zymosterol. n = 2 wells per condition. All bar graphs indicate mean +/− standard deviation, and panels <b>c</b>, <b>d</b>, <b>h</b>, <b>k-o</b>, and <b>r</b> are representative of two independent experiments. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Extended Data Figure 8.</span> </a> </div>   <div class="title" tagx="title" title="title">
Effect of combinations of small-molecule treatments on oligodendrocyte formation, and ability of oligodendrocytes to track along and wrap electrospun microfibers after single small molecule treatments. </div> <p> <b>a, b)</b> Percentage of MBP <sup>+</sup> oligodendrocytes generated from OPCs (OPC-1: Left, OPC-5: Right) at 72 h following treatment with the indicated combinations of liothyronine and enhancers of oligodendrocyte formation. Unless noted, the following concentrations were used: ketoconazole, 2.5 μM; benztropine, 2 μM; clemastine 2 μM; tamoxifen 200 nM; liothyronine, 3 μM. n = 4 wells per treatment condition, with &amp;gt; 1,000 cells analyzed per well. Lio = liothyronine. <b>c, d)</b> Percentage of MBP <sup>+</sup> oligodendrocytes generated from OPCs at 72 h following treatment with the indicated combinations of ketoconazole and enhancers of oligodendrocyte formation. n = 4 wells per treatment condition, with &amp;gt; 1,000 cells analyzed per well. Keto = ketoconazole. <b>e)</b> Representative images of OPCs treated 72 h with the indicated small molecules. Small-molecule concentrations are as in panel <b>a</b>. Scale bar, 100 μm. <b>f)</b> Fold-increase of MBP+ oligodendrocytes following plating of OPCs (OPC-5) onto microfibers and treatment for 14 days with the indicated pathway modulators. n = 2 wells per condition, except DMSO, n = 4. <b>g)</b> In an independent experiment, OPCs (OPC-5) were plated onto microfibers, treated with small molecules for 4 days, and fixed and stained after 14 days. The extent to which MBP+ oligodendrocytes tracked along the microfiber substrate was measured. n = 2 wells per condition. <b>h)</b> Total DAPI+ cell number in the experiment described in <b>g</b>. <b>i)</b> Representative images highlighting tracking along the microfiber substrate. Each image is a montage of 4 separate images within the same well. Green, MBP. Scale bar, 100 μm. <b>j)</b> High-resolution images of MBP+ oligodendrocytes tracking along microfibers. Green, MBP; Blue, DAPI. Ketoconazole, 2.5 μM. Scale bar, 50 μm. <b>k)</b> Confocal imaging of OPCs seeded onto aligned microfibers and treated 14 days with ketoconazole (2.5 μM). The plane of the cross-section is highlighted in yellow and the cross-section, in which green fluorescence appears to encircle several microfibers, is shown in the bottom panel. Green, MBP; Blue, DAPI. All bar graphs indicate mean +/− standard deviation, and panels <b>a-d</b> are representative of two independent experiments. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Extended Data Figure 9.</span> </a> </div>   <div class="title" tagx="title" title="title">
Effect of oligodendrocyte-enhancing small molecules on sterol levels in human cells and human cortical spheroids. </div> <p> <b>a)</b> Representative images of toluidine blue stained sections of LPC-lesioned dorsal spinal cord from mice treated 8 days with ifenprodil (10 mg per kg) or tamoxifen (2 mg per kg). Scale bar, 20 μm. <b>b)</b> GC/MS-based quantitation of three metabolite levels in human glioma cells (GBM528) treated 24 h with the indicated small molecules at the following concentrations: Tamoxifen, 100 nM; Clemastine, 2 μM; Ifenprodil, 2 μM; Ketoconazole, 2.5 μM; Amorolfine, 100 nM. Left, lanosterol; center, zymostenol; right, 14-dehydrozymostenol. n = 2 wells per condition. <b>c)</b> GC/MS-based quantitation of three metabolite levels in two independent batches of human cortical spheroids treated 24 h with the indicated small molecules at 2 μM. Left, lanosterol; center, zymostenol; right, zymosterol. n = 3 spheroids per condition; representative of two independent experiments. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Extended Data Figure 10.</span> </a> </div>   <div class="title" tagx="title" title="title">
27 small molecules and nine purified 8,9-unsaturated sterols shown here to enhance the formation of oligodendrocytes. </div> <p> <b>a)</b> Schematic showing the proposed mechanism of action for enhancing oligodendrocyte formation by diverse small molecules which enhance oligodendrocyte formation. <b>b)</b> Molecules that enhance oligodendrocyte formation are grouped by enzyme inhibited (GCMS analysis in OPCs): CYP51, top; sterol 14-reductase, center; EBP, bottom. <b>c</b>) Purified 8,9-unsaturated sterols that enhance oligodendrocyte formation. </p>   </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">1</span> <div class="media" title="media"> <a href="NIHMS965312-supplement-1.docx">LINK</a> </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">2</span> <div class="media" title="media"> <a href="NIHMS965312-supplement-2.pdf">LINK</a> </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">3</span> <div class="media" title="media"> <a href="NIHMS965312-supplement-3.pdf">LINK</a> </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>This work was supported by National Institutes of Health grant NS095280 (R.H.M., P.J.T), Conrad N. Hilton Foundation Pilot Innovator in MS Award (D.J.A.), Mallinckrodt Foundation Grant Award (D.J.A), philanthropic support from the Peterson, Fakhouri, Long, Goodman, Geller, Judge, and Weidenthal families, and unrestricted support from the CWRU School of Medicine. Additional support was provided by the Small-Molecule Drug Development, Proteomics, and Translational Research Shared Resources of the Case Comprehensive Cancer Center (P30 CA043703). We gratefully acknowledge use of the Leica SP8 confocal microscope in the Light Microscopy Imaging Facility at CWRU made available through the Office of Research Infrastructure (NIH-ORIP) Shared Instrumentation Grant (S10 OD016164). The authors thank M. Drumm, T. Miller, B. Karl, O. Iyoha-Bello, J. Pink., P. Conrad, R. Lee, X. Li, D. Schlatzer, K. Polak, Janssen Pharmaceuticals, CXR Biosciences, ThermoFisher, Avanti Polar Lipids, and the P. Scacheri Lab for technical assistance and discussion. </p> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-COI-statement" title="COI-statement"> <p> <b>Competing Interests.</b> The authors acknowledge competing interests. See online edition for details (currently in Methods). </p> </div> <div class="fn-type-COI-statement" title="COI-statement"> <p> <b>Competing Interests:</b> The authors declare the following competing interests: D.J.A, P.J.T, Z.H., D.A., M.S.E. and R.H.M. are inventors on patents and patent applications that relate to this work and have been licensed to Convelo Therapeutics, Inc., which seeks to develop remyelinating therapeutics. D.J.A. and P.J.T. hold equity in Convelo Therapeutics, Inc. and receive consulting income from Convelo Therapeutics, Inc. After resubmission of this work, D.C.F. became an employee of Convelo Therapeutics, Inc. </p> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
MAIN REFERENCES</div> <li tag="ref"> <a name="R1" /> <span class="label" tagx="label" title="label">1</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goldman</span> <span class="given-names" tagx="given-names" title="given-names">SA</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nedergaard</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Windrem</span> <span class="given-names" tagx="given-names" title="given-names">MS</span> </span> <span class="mixed-article-title" title="mixed-article-title">Glial progenitor cell-based treatment and modeling of neurological disease</span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">338</span>, <span class="fpage" tagx="fpage" title="fpage">491</span>– <span class="lpage" tagx="lpage" title="lpage">495</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.1218071">10.1126/science.1218071</a> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23112326">23112326</a> </span> </span> </li> <li tag="ref"> <a name="R2" /> <span class="label" tagx="label" title="label">2</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fancy</span> <span class="given-names" tagx="given-names" title="given-names">SP</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Overcoming remyelination failure in multiple sclerosis and other myelin disorders</span>. <span class="source" tagx="source" title="source">Exp Neurol</span> <span class="volume" tagx="volume" title="volume">225</span>, <span class="fpage" tagx="fpage" title="fpage">18</span>– <span class="lpage" tagx="lpage" title="lpage">23</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.expneurol.2009.12.020">10.1016/j.expneurol.2009.12.020</a> </span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20044992">20044992</a> </span> </span> </li> <li tag="ref"> <a name="R3" /> <span class="label" tagx="label" title="label">3</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Franklin</span> <span class="given-names" tagx="given-names" title="given-names">RJ</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ffrench-Constant</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="mixed-article-title" title="mixed-article-title">Remyelination in the CNS: from biology to therapy</span>. <span class="source" tagx="source" title="source">Nat Rev Neurosci</span> <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">839</span>– <span class="lpage" tagx="lpage" title="lpage">855</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nrn2480">10.1038/nrn2480</a> </span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18931697">18931697</a> </span> </span> </li> <li tag="ref"> <a name="R4" /> <span class="label" tagx="label" title="label">4</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Najm</span> <span class="given-names" tagx="given-names" title="given-names">FJ</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Drug-based modulation of endogenous stem cells promotes functional remyelination in vivo </span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">522</span>, <span class="fpage" tagx="fpage" title="fpage">216</span>– <span class="lpage" tagx="lpage" title="lpage">220</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nature14335">10.1038/nature14335</a> </span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25896324">25896324</a> </span> </span> </li> <li tag="ref"> <a name="R5" /> <span class="label" tagx="label" title="label">5</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deshmukh</span> <span class="given-names" tagx="given-names" title="given-names">VA</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">A regenerative approach to the treatment of multiple sclerosis</span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">502</span>, <span class="fpage" tagx="fpage" title="fpage">327</span>– <span class="lpage" tagx="lpage" title="lpage">332</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nature12647">10.1038/nature12647</a> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24107995">24107995</a> </span> </span> </li> <li tag="ref"> <a name="R6" /> <span class="label" tagx="label" title="label">6</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mei</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Micropillar arrays as a high-throughput screening platform for therapeutics in multiple sclerosis </span>. <span class="source" tagx="source" title="source">Nat Med</span> <span class="volume" tagx="volume" title="volume">20</span>, <span class="fpage" tagx="fpage" title="fpage">954</span>– <span class="lpage" tagx="lpage" title="lpage">960</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nm.3618">10.1038/nm.3618</a> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24997607">24997607</a> </span> </span> </li> <li tag="ref"> <a name="R7" /> <span class="label" tagx="label" title="label">7</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mei</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Identification of the Kappa-Opioid Receptor as a Therapeutic Target for Oligodendrocyte Remyelination </span>. <span class="source" tagx="source" title="source">J Neurosci</span> <span class="volume" tagx="volume" title="volume">36</span>, <span class="fpage" tagx="fpage" title="fpage">7925</span>– <span class="lpage" tagx="lpage" title="lpage">7935</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1523/JNEUROSCI.1493-16.2016">10.1523/JNEUROSCI.1493-16.2016</a> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27466337">27466337</a> </span> </span> </li> <li tag="ref"> <a name="R8" /> <span class="label" tagx="label" title="label">8</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">JK</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Retinoid X receptor gamma signaling accelerates CNS remyelination</span>. <span class="source" tagx="source" title="source">Nat Neurosci</span> <span class="volume" tagx="volume" title="volume">14</span>, <span class="fpage" tagx="fpage" title="fpage">45</span>– <span class="lpage" tagx="lpage" title="lpage">53</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nn.2702">10.1038/nn.2702</a> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21131950">21131950</a> </span> </span> </li> <li tag="ref"> <a name="R9" /> <span class="label" tagx="label" title="label">9</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gonzalez</span> <span class="given-names" tagx="given-names" title="given-names">GA</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Tamoxifen accelerates the repair of demyelinated lesions in the central nervous system</span>. <span class="source" tagx="source" title="source">Sci Rep</span> <span class="volume" tagx="volume" title="volume">6</span>, <span class="fpage" tagx="fpage" title="fpage">31599</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1038/srep31599">10.1038/srep31599</a> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27554391">27554391</a> </span> </span> </li> <li tag="ref"> <a name="R10" /> <span class="label" tagx="label" title="label">10</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lariosa-Willingham</span> <span class="given-names" tagx="given-names" title="given-names">KD</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">A high throughput drug screening assay to identify compounds that promote oligodendrocyte differentiation using acutely dissociated and purified oligodendrocyte precursor cells </span>. <span class="source" tagx="source" title="source">BMC Res Notes</span> <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">419</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1186/s13104-016-2220-2">10.1186/s13104-016-2220-2</a> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27592856">27592856</a> </span> </span> </li> <li tag="ref"> <a name="R11" /> <span class="label" tagx="label" title="label">11</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Korade</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">The Effect of Small Molecules on Sterol Homeostasis: Measuring 7-Dehydrocholesterol in Dhcr7-Deficient Neuro2a Cells and Human Fibroblasts </span>. <span class="source" tagx="source" title="source">J Med Chem</span> <span class="volume" tagx="volume" title="volume">59</span>, <span class="fpage" tagx="fpage" title="fpage">1102</span>– <span class="lpage" tagx="lpage" title="lpage">1115</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1021/acs.jmedchem.5b01696">10.1021/acs.jmedchem.5b01696</a> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26789657">26789657</a> </span> </span> </li> <li tag="ref"> <a name="R12" /> <span class="label" tagx="label" title="label">12</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Giera</span> <span class="given-names" tagx="given-names" title="given-names">MM</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">C.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bracher</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="mixed-article-title" title="mixed-article-title">Analysis and Experimental Inhibition of Distal Cholesterol Biosynthesis</span>. <span class="source" tagx="source" title="source">Chromatographia</span> <span class="volume" tagx="volume" title="volume">78</span>, <span class="fpage" tagx="fpage" title="fpage">343</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s10337-014-2796-4">10.1007/s10337-014-2796-4</a> </span> ( <span class="year" tagx="year" title="year">2015</span>). </span> </li> <li tag="ref"> <a name="R13" /> <span class="label" tagx="label" title="label">13</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Giera</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Plossl</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bracher</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="mixed-article-title" title="mixed-article-title">Fast and easy in vitro screening assay for cholesterol biosynthesis inhibitors in the post-squalene pathway </span>. <span class="source" tagx="source" title="source">Steroids</span> <span class="volume" tagx="volume" title="volume">72</span>, <span class="fpage" tagx="fpage" title="fpage">633</span>– <span class="lpage" tagx="lpage" title="lpage">642</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.steroids.2007.04.005">10.1016/j.steroids.2007.04.005</a> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17583759">17583759</a> </span> </span> </li> <li tag="ref"> <a name="R14" /> <span class="label" tagx="label" title="label">14</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mir</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Le Breton</span> <span class="given-names" tagx="given-names" title="given-names">GC</span> </span> <span class="mixed-article-title" title="mixed-article-title">A novel nuclear signaling pathway for thromboxane A2 receptors in oligodendrocytes: evidence for signaling compartmentalization during differentiation </span>. <span class="source" tagx="source" title="source">Mol Cell Biol</span> <span class="volume" tagx="volume" title="volume">28</span>, <span class="fpage" tagx="fpage" title="fpage">6329</span>– <span class="lpage" tagx="lpage" title="lpage">6341</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1128/MCB.00482-08">10.1128/MCB.00482-08</a> </span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18710937">18710937</a> </span> </span> </li> <li tag="ref"> <a name="R15" /> <span class="label" tagx="label" title="label">15</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jachak</span> <span class="given-names" tagx="given-names" title="given-names">GR</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Silicon Incorporated Morpholine Antifungals: Design, Synthesis, and Biological Evaluation</span>. <span class="source" tagx="source" title="source">ACS Med Chem Lett</span> <span class="volume" tagx="volume" title="volume">6</span>, <span class="fpage" tagx="fpage" title="fpage">1111</span>– <span class="lpage" tagx="lpage" title="lpage">1116</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1021/acsmedchemlett.5b00245">10.1021/acsmedchemlett.5b00245</a> </span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26617963">26617963</a> </span> </span> </li> <li tag="ref"> <a name="R16" /> <span class="label" tagx="label" title="label">16</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Selective targeting of mutant adenomatous polyposis coli (APC) in colorectal cancer</span>. <span class="source" tagx="source" title="source">Sci Transl Med</span> <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">361ra140</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1126/scitranslmed.aaf8127">10.1126/scitranslmed.aaf8127</a> </span> ( <span class="year" tagx="year" title="year">2016</span>). </span> </li> <li tag="ref"> <a name="R17" /> <span class="label" tagx="label" title="label">17</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jef DeBrabander</span> <span class="given-names" tagx="given-names" title="given-names">JWS</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Wentian</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nijhawan</span> <span class="given-names" tagx="given-names" title="given-names">Deepak</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Theodoropoulos</span> <span class="given-names" tagx="given-names" title="given-names">Pano</span> </span>. <span class="source" tagx="source" title="source">Targeting Emopamil Binding Protein with Small Molecules that Induce an Abnormal Feedbak Response by Lowering Endogenous Cholesterol Biosynthesis </span>. ( <span class="year" tagx="year" title="year">2016</span>). </span> </li> <li tag="ref"> <a name="R18" /> <span class="label" tagx="label" title="label">18</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saher</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Therapy of Pelizaeus-Merzbacher disease in mice by feeding a cholesterol-enriched diet </span>. <span class="source" tagx="source" title="source">Nat Med</span> <span class="volume" tagx="volume" title="volume">18</span>, <span class="fpage" tagx="fpage" title="fpage">1130</span>– <span class="lpage" tagx="lpage" title="lpage">1135</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nm.2833">10.1038/nm.2833</a> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22706386">22706386</a> </span> </span> </li> <li tag="ref"> <a name="R19" /> <span class="label" tagx="label" title="label">19</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Byskov</span> <span class="given-names" tagx="given-names" title="given-names">AG</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Andersen</span> <span class="given-names" tagx="given-names" title="given-names">CY</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leonardsen</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="mixed-article-title" title="mixed-article-title">Role of meiosis activating sterols, MAS, in induced oocyte maturation</span>. <span class="source" tagx="source" title="source">Mol Cell Endocrinol</span> <span class="volume" tagx="volume" title="volume">187</span>, <span class="fpage" tagx="fpage" title="fpage">189</span>– <span class="lpage" tagx="lpage" title="lpage">196</span> ( <span class="year" tagx="year" title="year">2002</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11988327">11988327</a> </span> </span> </li> <li tag="ref"> <a name="R20" /> <span class="label" tagx="label" title="label">20</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grondahl</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="mixed-article-title" title="mixed-article-title">Oocyte maturation. Basic and clinical aspects of in vitro maturation (IVM) with special emphasis of the role of FF-MAS </span>. <span class="source" tagx="source" title="source">Dan Med Bull</span> <span class="volume" tagx="volume" title="volume">55</span>, <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">16</span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18321442">18321442</a> </span> </span> </li> <li tag="ref"> <a name="R21" /> <span class="label" tagx="label" title="label">21</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Canfran-Duque</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Atypical antipsychotics alter cholesterol and fatty acid metabolism in vitro</span>. <span class="source" tagx="source" title="source">J Lipid Res</span> <span class="volume" tagx="volume" title="volume">54</span>, <span class="fpage" tagx="fpage" title="fpage">310</span>– <span class="lpage" tagx="lpage" title="lpage">324</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1194/jlr.M026948">10.1194/jlr.M026948</a> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23175778">23175778</a> </span> </span> </li> <li tag="ref"> <a name="R22" /> <span class="label" tagx="label" title="label">22</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moebius</span> <span class="given-names" tagx="given-names" title="given-names">FF</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Pharmacological analysis of sterol delta8-delta7 isomerase proteins with [3H]ifenprodil</span>. <span class="source" tagx="source" title="source">Mol Pharmacol</span> <span class="volume" tagx="volume" title="volume">54</span>, <span class="fpage" tagx="fpage" title="fpage">591</span>– <span class="lpage" tagx="lpage" title="lpage">598</span> ( <span class="year" tagx="year" title="year">1998</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9730919">9730919</a> </span> </span> </li> <li tag="ref"> <a name="R23" /> <span class="label" tagx="label" title="label">23</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gylling</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Tamoxifen and toremifene lower serum cholesterol by inhibition of delta 8-cholesterol conversion to lathosterol in women with breast cancer </span>. <span class="source" tagx="source" title="source">J Clin Oncol</span> <span class="volume" tagx="volume" title="volume">13</span>, <span class="fpage" tagx="fpage" title="fpage">2900</span>– <span class="lpage" tagx="lpage" title="lpage">2905</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1200/JCO.1995.13.12.2900">10.1200/JCO.1995.13.12.2900</a> </span> ( <span class="year" tagx="year" title="year">1995</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8523053">8523053</a> </span> </span> </li> <li tag="ref"> <a name="R24" /> <span class="label" tagx="label" title="label">24</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bechler</span> <span class="given-names" tagx="given-names" title="given-names">ME</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Byrne</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ffrench-Constant</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="mixed-article-title" title="mixed-article-title">CNS Myelin Sheath Lengths Are an Intrinsic Property of Oligodendrocytes</span>. <span class="source" tagx="source" title="source">Curr Biol</span> <span class="volume" tagx="volume" title="volume">25</span>, <span class="fpage" tagx="fpage" title="fpage">2411</span>– <span class="lpage" tagx="lpage" title="lpage">2416</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.cub.2015.07.056">10.1016/j.cub.2015.07.056</a> </span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26320951">26320951</a> </span> </span> </li> <li tag="ref"> <a name="R25" /> <span class="label" tagx="label" title="label">25</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">A culture system to study oligodendrocyte myelination processes using engineered nanofibers</span>. <span class="source" tagx="source" title="source">Nat Methods</span> <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">917</span>– <span class="lpage" tagx="lpage" title="lpage">922</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nmeth.2105">10.1038/nmeth.2105</a> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22796663">22796663</a> </span> </span> </li> <li tag="ref"> <a name="R26" /> <span class="label" tagx="label" title="label">26</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mi</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Promotion of central nervous system remyelination by induced differentiation of oligodendrocyte precursor cells </span>. <span class="source" tagx="source" title="source">Ann Neurol</span> <span class="volume" tagx="volume" title="volume">65</span>, <span class="fpage" tagx="fpage" title="fpage">304</span>– <span class="lpage" tagx="lpage" title="lpage">315</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1002/ana.21581">10.1002/ana.21581</a> </span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19334062">19334062</a> </span> </span> </li> <li tag="ref"> <a name="R27" /> <span class="label" tagx="label" title="label">27</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pasca</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Functional cortical neurons and astrocytes from human pluripotent stem cells in 3D culture </span>. <span class="source" tagx="source" title="source">Nat Methods</span> <span class="volume" tagx="volume" title="volume">12</span>, <span class="fpage" tagx="fpage" title="fpage">671</span>– <span class="lpage" tagx="lpage" title="lpage">678</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nmeth.3415">10.1038/nmeth.3415</a> </span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26005811">26005811</a> </span> </span> </li> <li tag="ref"> <a name="R28" /> <span class="label" tagx="label" title="label">28</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miron</span> <span class="given-names" tagx="given-names" title="given-names">VE</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Statin therapy inhibits remyelination in the central nervous system</span>. <span class="source" tagx="source" title="source">Am J Pathol</span> <span class="volume" tagx="volume" title="volume">174</span>, <span class="fpage" tagx="fpage" title="fpage">1880</span>– <span class="lpage" tagx="lpage" title="lpage">1890</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.2353/ajpath.2009.080947">10.2353/ajpath.2009.080947</a> </span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19349355">19349355</a> </span> </span> </li> <li tag="ref"> <a name="R29" /> <span class="label" tagx="label" title="label">29</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Klopfleisch</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Negative impact of statins on oligodendrocytes and myelin formation in vitro and in vivo </span>. <span class="source" tagx="source" title="source">J Neurosci</span> <span class="volume" tagx="volume" title="volume">28</span>, <span class="fpage" tagx="fpage" title="fpage">13609</span>– <span class="lpage" tagx="lpage" title="lpage">13614</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1523/JNEUROSCI.2765-08.2008">10.1523/JNEUROSCI.2765-08.2008</a> </span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19074034">19074034</a> </span> </span> </li> <li tag="ref"> <a name="R30" /> <span class="label" tagx="label" title="label">30</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saher</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">High cholesterol level is essential for myelin membrane growth</span>. <span class="source" tagx="source" title="source">Nat Neurosci</span> <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">468</span>– <span class="lpage" tagx="lpage" title="lpage">475</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nn1426">10.1038/nn1426</a> </span> ( <span class="year" tagx="year" title="year">2005</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15793579">15793579</a> </span> </span> </li> </ul> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
METHODS REFERENCES</div> <li tag="ref"> <a name="R31" /> <span class="label" tagx="label" title="label">31</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Godefroi</span> <span class="given-names" tagx="given-names" title="given-names">EF</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Heeres</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van Cutsem</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Janssen</span> <span class="given-names" tagx="given-names" title="given-names">PA</span> </span> <span class="mixed-article-title" title="mixed-article-title">The preparation and antimycotic properties of derivatives of 1-phenethylimidazole</span>. <span class="source" tagx="source" title="source">J Med Chem</span> <span class="volume" tagx="volume" title="volume">12</span>, <span class="fpage" tagx="fpage" title="fpage">784</span>– <span class="lpage" tagx="lpage" title="lpage">791</span> ( <span class="year" tagx="year" title="year">1969</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/4897900">4897900</a> </span> </span> </li> <li tag="ref"> <a name="R32" /> <span class="label" tagx="label" title="label">32</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Giera</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Renard</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Plossl</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bracher</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="mixed-article-title" title="mixed-article-title">Lathosterol side chain amides: a new class of human lathosterol oxidase inhibitors</span>. <span class="source" tagx="source" title="source">Steroids</span> <span class="volume" tagx="volume" title="volume">73</span>, <span class="fpage" tagx="fpage" title="fpage">299</span>– <span class="lpage" tagx="lpage" title="lpage">308</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.steroids.2007.10.015">10.1016/j.steroids.2007.10.015</a> </span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18164739">18164739</a> </span> </span> </li> <li tag="ref"> <a name="R33" /> <span class="label" tagx="label" title="label">33</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Najm</span> <span class="given-names" tagx="given-names" title="given-names">FJ</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Rapid and robust generation of functional oligodendrocyte progenitor cells from epiblast stem cells </span>. <span class="source" tagx="source" title="source">Nat Methods</span> <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">957</span>– <span class="lpage" tagx="lpage" title="lpage">962</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nmeth.1712">10.1038/nmeth.1712</a> </span> [ <span class="comment">nmeth.1712 pii</span>] ( <span class="year" tagx="year" title="year">2011</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21946668">21946668</a> </span> </span> </li> <li tag="ref"> <a name="R34" /> <span class="label" tagx="label" title="label">34</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Honda</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">Highly sensitive quantification of key regulatory oxysterols in biological samples by LC-ESI-MS/MS </span>. <span class="source" tagx="source" title="source">J Lipid Res</span> <span class="volume" tagx="volume" title="volume">50</span>, <span class="fpage" tagx="fpage" title="fpage">350</span>– <span class="lpage" tagx="lpage" title="lpage">357</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1194/jlr.D800040-JLR200">10.1194/jlr.D800040-JLR200</a> </span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18815436">18815436</a> </span> </span> </li> <li tag="ref"> <a name="R35" /> <span class="label" tagx="label" title="label">35</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Warrilow</span> <span class="given-names" tagx="given-names" title="given-names">AG</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Parker</span> <span class="given-names" tagx="given-names" title="given-names">JE</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kelly</span> <span class="given-names" tagx="given-names" title="given-names">DE</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kelly</span> <span class="given-names" tagx="given-names" title="given-names">SL</span> </span> <span class="mixed-article-title" title="mixed-article-title">Azole affinity of sterol 14alpha-demethylase (CYP51) enzymes from Candida albicans and Homo sapiens </span>. <span class="source" tagx="source" title="source">Antimicrob Agents Chemother</span> <span class="volume" tagx="volume" title="volume">57</span>, <span class="fpage" tagx="fpage" title="fpage">1352</span>– <span class="lpage" tagx="lpage" title="lpage">1360</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1128/AAC.02067-12">10.1128/AAC.02067-12</a> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23274672">23274672</a> </span> </span> </li> <li tag="ref"> <a name="R36" /> <span class="label" tagx="label" title="label">36</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Folch</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lees</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sloane Stanley</span> <span class="given-names" tagx="given-names" title="given-names">GH</span> </span> <span class="mixed-article-title" title="mixed-article-title">A simple method for the isolation and purification of total lipides from animal tissues</span>. <span class="source" tagx="source" title="source">J Biol Chem</span> <span class="volume" tagx="volume" title="volume">226</span>, <span class="fpage" tagx="fpage" title="fpage">497</span>– <span class="lpage" tagx="lpage" title="lpage">509</span> ( <span class="year" tagx="year" title="year">1957</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/13428781">13428781</a> </span> </span> </li> <li tag="ref"> <a name="R37" /> <span class="label" tagx="label" title="label">37</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pink</span> <span class="given-names" tagx="given-names" title="given-names">JJ</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jordan</span> <span class="given-names" tagx="given-names" title="given-names">VC</span> </span> <span class="mixed-article-title" title="mixed-article-title">Models of estrogen receptor regulation by estrogens and antiestrogens in breast cancer cell lines </span>. <span class="source" tagx="source" title="source">Cancer Res</span> <span class="volume" tagx="volume" title="volume">56</span>, <span class="fpage" tagx="fpage" title="fpage">2321</span>– <span class="lpage" tagx="lpage" title="lpage">2330</span> ( <span class="year" tagx="year" title="year">1996</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8625307">8625307</a> </span> </span> </li> <li tag="ref"> <a name="R38" /> <span class="label" tagx="label" title="label">38</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Labarca</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paigen</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="mixed-article-title" title="mixed-article-title">A simple, rapid, and sensitive DNA assay procedure</span>. <span class="source" tagx="source" title="source">Anal Biochem</span> <span class="volume" tagx="volume" title="volume">102</span>, <span class="fpage" tagx="fpage" title="fpage">344</span>– <span class="lpage" tagx="lpage" title="lpage">352</span> ( <span class="year" tagx="year" title="year">1980</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/6158890">6158890</a> </span> </span> </li> <li tag="ref"> <a name="R39" /> <span class="label" tagx="label" title="label">39</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bosch</span> <span class="given-names" tagx="given-names" title="given-names">DG</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> <span class="mixed-article-title" title="mixed-article-title">NR2F1 mutations cause optic atrophy with intellectual disability</span>. <span class="source" tagx="source" title="source">Am J Hum Genet</span> <span class="volume" tagx="volume" title="volume">94</span>, <span class="fpage" tagx="fpage" title="fpage">303</span>– <span class="lpage" tagx="lpage" title="lpage">309</span>, doi: <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ajhg.2014.01.002">10.1016/j.ajhg.2014.01.002</a> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24462372">24462372</a> </span> </span> </li> </ul> </div> </div> <div class="floats-group" title="floats-group"> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1.</span> </a> </div>   <div class="title" tagx="title" title="title">
Imidazoles inhibit CYP51 to enhance oligodendrocyte formation.</div> <p> <b>a)</b> Rat CYP51 enzymatic activity following treatment with azoles. n = 2 independent enzymatic assays. <b>b)</b> GC/MS-based quantitation of lanosterol levels in OPCs treated with the indicated azoles at 2.5 μM. n = 2 wells per condition. <b>c</b>, <b>f</b>, <b>g)</b> Percentage of MBP <sup>+</sup> oligodendrocytes generated from OPCs following treatment with azoles ( <b>c</b>), cell permeable siRNA reagents ( <b>f</b>), and lanosterol ( <b>g</b>). n ≥ 4 wells per condition; for exact well counts in all figures, see Statistics and Reproducibility. In <b>f</b>, *, <i>P</i> = 0.0005, two-tailed Student’s t-test. <b>d)</b> Representative images of OPCs treated with the indicated azoles. Nuclei are labeled with DAPI (blue), and oligodendrocytes are indicated by immunostaining for myelin basic protein (green). Scale bar, 100 μm. <b>e)</b> GC/MS-based quantitation of lanosterol levels in OPCs treated with the indicated reagents. n = 2 wells per condition. <b>h</b>) Structure of lanosterol. All bar graphs indicate mean ± standard deviation. Experiments in c, d, and g are representative of three independent experiments, while b, e, and f are representative of two independent experiments using OPC-5 cells; for validation in an independent derivation of OPCs, see <a href="#F6">Extended Data Fig. 2</a>. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2.</span> </a> </div>   <div class="title" tagx="title" title="title">
Small-molecule inhibition of CYP51, TM7SF2, or EBP enhances oligodendrocyte formation via accumulation of 8,9-unsaturated sterols. </div> <p> <b>a)</b> Abbreviated cholesterol biosynthesis pathway. For greater detail, see <a href="#F5">Extended Data Figure 1</a>. <b>b)</b> Percentage of MBP <sup>+</sup> oligodendrocytes generated from OPCs treated with the indicated pathway inhibitors. n ≥ 4 wells per condition. <b>c)</b> Percentage of MBP <sup>+</sup> oligodendrocytes generated from OPCs expressing Cas9 and guide RNA targeting EBP. n ≥ 3 wells per condition. <b>d)</b> Functional validation of Cas9-based targeting of EBP using GC/MS-based quantitation of zymostenol levels. n = 2 wells per condition. <b>e, f, g)</b> Percentage of MBP <sup>+</sup> oligodendrocytes generated from OPCs with the indicated purified sterols. n ≥ 4 wells per condition. <b>h)</b> Structures of various sterols. All bar graphs indicate mean ± standard deviation. See Statistics and Reproducibility for exact well counts. Experiments in b-g are representative of two or more independent experiments using OPC-5 cells; for validation in an independent derivation of OPCs, see <a href="#F7">Extended Data Fig. 3</a>– <a href="#F9">5</a>. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3.</span> </a> </div>   <div class="title" tagx="title" title="title">
Inhibition of TM7SF2 and EBP is a unifying mechanism for many small-molecule enhancers of oligodendrocyte formation. </div> <p> <b>a)</b> Quantitation of sterol levels in OPCs treated with the indicated molecules at 2 μM (one well per condition; for validation in a second derivation of OPCs, see <a href="#F10">Extended Data Fig. 6</a>). ( <b>b</b>) Quantitation of sterol levels in OPCs treated with the indicated previously-reported enhancers of oligodendrocyte formation (n = 2 wells per condition except DMSO, n = 6). Representative of two independent experiments. <b>c)</b> Quantitation of EBP enzymatic activity in a biochemical assay. All treatments 10 μM. n = 3 independent enzymatic assays, except DMSO, n = 5. Bars indicate mean; error bars indicate standard deviation. <i>p</i>-Fluorohexahydro-sila-difenidol abbreviated as Sigma H127. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 4.</span> </a> </div>   <div class="title" tagx="title" title="title">
Accumulation of 8,9-unsaturated sterols enhances remyelination <i>in vivo</i> and in human brain spheroids. </div> <p> <b>a)</b> Representative electron microscopy images of LPC-lesioned dorsal spinal cord from mice treated with ifenprodil or tamoxifen. Scale bar, 5 μm. <b>b)</b> Tukey plot showing quantitation of remyelinated axons in LPC-lesioned spinal cord from mice in panel <b>a</b>. n = 6 animals per group except vehicle, n = 4. **, <i>P</i> = 0.0004; *, <i>P</i> = 0.048, two-tailed Mann-Whitney test. <b>c)</b> Quantitation of brain sterol levels in mice treated with miconazole, ifenprodil, or tamoxifen. n = 4 animals per group. <i>P</i> = 0.0007 for miconazole, <i>P</i> = 0.0003 for ifenprodil, <i>P</i> = 0.0006 for tamoxifen, two-tailed Student’s <i>t</i>-test. <b>d)</b> Quantitation of MyRF+ oligodendrocytes within human myelinating cortical spheroids following treatment with miconazole (2 μM) or ifenprodil (2 μM). n = 4 spheroids per treatment condition. <i>P</i> = 0.0009 for miconazole and <i>P</i> = 0.0009 for ifenprodil, two-tailed Student’s <i>t</i>-test. <b>e)</b> Representative images of spheroids. DAPI+ nucelei (blue) and MyRF+ oligodendrocytes (red) are labeled. Scale bar, 100 μm. In c and d, bar graphs indicate mean and error bars indicate standard deviation. </p>   </div> </div>  </body></html>