<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Nat Commun</span> <span class="iso-abbrev" title="iso-abbrev">Nat Commun</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Nature Communications</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">2041-1723</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6707142">6707142</a> </span> <span class="publisher-id" title="publisher-id">11290</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41467-019-11290-x">10.1038/s41467-019-11290-x</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Orthogonal monoterpenoid biosynthesis in yeast constructed on an isomeric substrate</div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-6304-9413</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ignea</span> <span class="given-names" tagx="given-names" title="given-names">Codruta</span> </span> <a href="#Aff1">1</a> <a href="#Aff5">5</a> </span> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-6796-0079</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raadam</span> <span class="given-names" tagx="given-names" title="given-names">Morten H.</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-5582-9463</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Motawia</span> <span class="given-names" tagx="given-names" title="given-names">Mohammed S.</span> </span> <a href="#Aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Makris</span> <span class="given-names" tagx="given-names" title="given-names">Antonios M.</span> </span> <a href="#Aff2">2</a> </span> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-0792-050X</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vickers</span> <span class="given-names" tagx="given-names" title="given-names">Claudia E.</span> </span> <a href="#Aff3">3</a> <a href="#Aff4">4</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kampranis</span> <span class="given-names" tagx="given-names" title="given-names">Sotirios C.</span> </span> <span class="address" title="address"> <span class="email" tagx="email" title="email">soka@plen.ku.dk</span> </span> <a href="#Aff1">1</a> </span> <span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0001 0674 042X</span> <span class="institution-id" title="institution-id">grid.5254.6</span> <span class="institution" title="institution">Biochemical Engineering Group, Plant Biochemistry Section, Department of Plant and Environmental Sciences, </span> <span class="institution" title="institution">University of Copenhagen, </span> </div> </span> <span class="citation_author_institution" id="Aff2">[2], </span> <span class="citation_author_institution" id="Aff3">[3], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0000 9320 7537</span> <span class="institution-id" title="institution-id">grid.1003.2</span> <span class="institution" title="institution">Australian Institute for Bioengineering and Nanotechnology (AIBN), </span> <span class="institution" title="institution">University of Queensland, </span> </div> </span> <span class="citation_author_institution" id="Aff4">[4], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">grid.1016.6</span> <span class="institution" title="institution">CSIRO Synthetic Biology Future Science Platform, </span> <span class="institution" title="institution">Commonwealth Scientific and Industrial Research Organisation (CSIRO), </span> </div> </span> <span class="citation_author_institution" id="Aff5">[5], <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">0000 0001 0674 042X</span> <span class="institution-id" title="institution-id">grid.5254.6</span> <span class="institution" title="institution">Present Address: Department of Chemistry, </span> <span class="institution" title="institution">University of Copenhagen, </span> </div> </span> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-8-8</span> </span> <span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2019-8-8</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2019</span> </span> <span class="volume" tagx="volume" title="volume">10</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">3799</span> <span class="history" title="history"> <span class="received" title="received">received: 2018-9-23</span> <span class="accepted" title="accepted">accepted: 2019-7-3</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , </span> <span class="license" title="license"> <span class="license-p" title="license-p"> <b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Synthetic biology efforts for the production of valuable chemicals are frequently hindered by the structure and regulation of the native metabolic pathways of the chassis. This is particularly evident in the case of monoterpenoid production in <i>Saccharomyces cerevisiae</i>, where the canonical terpene precursor geranyl diphosphate is tightly coupled to the biosynthesis of isoprenoid compounds essential for yeast viability. Here, we establish a synthetic orthogonal monoterpenoid pathway based on an alternative precursor, neryl diphosphate. We identify structural determinants of isomeric substrate selectivity in monoterpene synthases and engineer five different enzymes to accept the alternative substrate with improved efficiency and specificity. We combine the engineered enzymes with dynamic regulation of metabolic flux to harness the potential of the orthogonal substrate and improve the production of industrially-relevant monoterpenes by several-fold compared to the canonical pathway. This approach highlights the introduction of synthetic metabolism as an effective strategy for high-value compound production. </p> </div> <div class="abstract" title="abstract"> <p>Titers of monoterpenoids production in yeast are low due to the fact that the geranyl diphosphate (GPP)-based pathway can redirect metabolic fluxes to growth. Here, the authors build an orthogonal pathway by selecting the <i>cis</i> isomer of GPP as an alternative precursor and achieve high titer monoterpene production. </p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Subject terms</div> <div class="kwd" title="kwd">
Metabolic engineering</div> <div class="kwd" title="kwd">
Metabolic engineering</div> <div class="kwd" title="kwd">
Synthetic biology</div> <div class="kwd" title="kwd">
Metabolic pathways</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">https://doi.org/10.13039/501100009708</span> <span class="institution" title="institution">Novo Nordisk Fonden (Novo Nordisk Foundation)</span> </div> </div> <span class="award-id" tagx="award-id" title="award-id">NNF16OC0019554</span> <span class="award-id" tagx="award-id" title="award-id">NNF16OC0021760</span> <span class="principal-award-recipient" title="principal-award-recipient"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ignea</span> <span class="given-names" tagx="given-names" title="given-names">Codruta</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kampranis</span> <span class="given-names" tagx="given-names" title="given-names">Sotirios C.</span> </span> </span> </div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">https://doi.org/10.13039/501100003448</span> <span class="institution" title="institution">General Secretariat for Research and Technology (GSRT)</span> </div> </div> <span class="award-id" tagx="award-id" title="award-id">09ΣΥΝ-23-879</span> <span class="award-id" tagx="award-id" title="award-id">09ΣΥΝ-23-879</span> <span class="principal-award-recipient" title="principal-award-recipient"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Makris</span> <span class="given-names" tagx="given-names" title="given-names">Antonios M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kampranis</span> <span class="given-names" tagx="given-names" title="given-names">Sotirios C.</span> </span> </span> </div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"> <span class="institution-id" title="institution-id">https://doi.org/10.13039/501100008920</span> <span class="institution" title="institution">State of Queensland | Department of Science, Information Technology and Innovation, Queensland Government (Department of Science, Information Technology and Innovation) </span> </div> </div> </div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2019" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Synthetic biology approaches for the production of valuable chemicals intervene in the native metabolic pathways of the chassis to balance synthesis of the desired chemical with the production of biomass. However, the structure and regulation of native metabolism has been optimized through evolution to suit the specific needs of the host organism, frequently making it particularly challenging to redirect metabolic fluxes toward the heterologous pathway introduced to synthesize the desirable product. A solution to overcome this obstacle would be to make the heterologous pathway less connected to the chassis’ native metabolism. Ideally, a heterologous pathway must be orthogonal to the host pathways. The concept of orthogonality describes the ability of a system component to be varied without affecting the performance of other components of the same system (e.g., an orthogonal ribosome specifically translates an orthogonal mRNA that is not a substrate for cellular ribosomes) <sup> <a href="#CR1">1</a> </sup>. Although, when used broadly, the term "orthogonal" denotes independence, when applied to biosynthetic pathways, it denotes minimum dependence <sup> <a href="#CR2">2</a> </sup>. This is because the orthogonal pathway still requires precursor influx from the chassis. Thus, for a pathway to be orthogonal, it must have only a single branch point from basic metabolism <sup> <a href="#CR3">3</a> </sup>. Establishing an orthogonal route is advantageous for improved pathway control and performance. The unique branching point can be exploited to create a metabolic valve to redirect fluxes to the orthogonal route and enable dynamic regulation in response to the metabolic status or growth phase <sup> <a href="#CR4">4</a>– <a href="#CR9">9</a> </sup>. </p> <p>A characteristic case, where the establishment of an orthogonal metabolic pathway could be beneficial, is the synthesis of monoterpenoids in <i>Saccharomyces cerevisiae</i>. Monoterpenoids are widely used as flavors, antibacterials, and insecticides <sup> <a href="#CR10">10</a>, <a href="#CR11">11</a> </sup>, and recently, their applications have expanded to include high-density fuels, renewable polymers, and green plastics <sup> <a href="#CR12">12</a> </sup>. However, extraction of monoterpenoids from natural sources cannot meet the increasing demand, prompting efforts for their biotechnological production in microorganisms <sup> <a href="#CR13">13</a>– <a href="#CR17">17</a> </sup>. Monoterpene scaffolds are synthesized by the conversion of the canonical 10-carbon terpene precursor geranyl diphosphate (GPP) by monoterpene synthases, which catalyze the rearrangement of the substrate to generate a plethora of structures <sup> <a href="#CR18">18</a> </sup> (Fig. <a href="#Fig1">1</a>). GPP itself is formed by the fusion of two C <sub>5</sub> units, dimethylallyl diphosphate (DMAPP) and isopentenyl diphosphate (IPP). Further addition of IPP to GPP generates the 15-carbon farnesyl diphosphate (FPP), a precursor of the sesquiterpenes (C <sub>15</sub>), diterpenes (C <sub>20</sub>), polyprenols, ubiquinone, and sterols. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 1</span> </a> </div>   <p>Engineering an orthogonal monoterpene biosynthetic pathway in yeast. The native yeast isoprenoid biosynthesis includes pathways responsible for the synthesis of sterols, dolichols, ubiquinone, as well as protein prenylation. Production of monoterpenoids in yeast has so far been based on utilizing the GPP synthesized as an intermediate in primary isoprenoid metabolism. This GPP-based non-orthogonal pathway (shown in gray) connects back to primary metabolism through the Erg20p-catalyzed conversion of GPP to FPP. Expression of <i>Sl</i>NPPS1 in yeast cells enables the synthesis of the <i>cis</i>-isomer of GPP, NPP, which cannot be converted to FPP and links back to the biosynthesis of yeast isoprenoids. NPP can be utilized by engineered monoterpene synthases to produce different monoterpenes. The NPP-based pathway, depicted in green, is orthogonal to the yeast metabolism, because it branches out from the main pathway only at one point, following the synthesis of IPP and DMAPP. This branch point creates a metabolic valve, depicted here using the engineering symbol for a three-point valve, which can be regulated to direct fluxes to the desired products. This valve can be controlled dynamically to regulate fluxes between the two branches. We installed dynamic regulation by introducing the ergosterol-repressed promoter of the <i>ERG1</i> gene upstream of the <i>ERG20</i> gene. This valve diverts fluxes to the orthogonal branch when sterol levels increase. TPS terpene synthase, CYPs cytochrome P450 enzymes, ADHs alcohol dehydrogenases, GGPP geranylgeranyl diphosphate, P <sub> <i>ERG1</i> </sub> ERG1 promoter </p>   </div> <p /> <p>The yeast <i>S. cerevisiae</i> is an attractive host for terpene bioproduction, due to its robustness, applicability to industrial bioprocesses, and the possibility for terpene scaffold decoration by the functional expression of cytochrome P450 enzymes. Metabolic engineering efforts in yeast for the production of FPP-derived compounds, such as the sesquiterpenes artemisinin and farnesene, have achieved industrial-scale levels <sup> <a href="#CR19">19</a>– <a href="#CR22">22</a> </sup>. However, production of GPP-derived compounds has so far been considerably less efficient <sup> <a href="#CR14">14</a>, <a href="#CR23">23</a>, <a href="#CR24">24</a> </sup>. One of the factors contributing to this lower efficiency may be the structure of the isoprenoid biosynthesis pathway in yeast, which is optimized for the production of FPP-derived molecules that are essential for growth and viability. A key feature of the pathway that highlights this metabolic optimization is the sequential nature of FPP synthesis by the native yeast prenyltransferase, Erg20p. Erg20p is a bifunctional enzyme that catalyzes two successive steps of the pathway; initially, it condenses DMAPP and IPP to form GPP, and subsequently it catalyzes the condensation of GPP with one more IPP molecule to generate FPP <sup> <a href="#CR25">25</a> </sup> (Fig. <a href="#Fig1">1</a>). This facilitates the metabolic economy of the cell <sup> <a href="#CR26">26</a> </sup>, as yeast does not naturally produce other GPP-derived compounds. However, the sequential reaction catalyzed by Erg20p appears to limit the GPP pool, hindering monoterpenoid production <sup> <a href="#CR14">14</a> </sup>. Overexpression of a GPP synthase to increase GPP availability resulted in only a moderate monoterpene increase, because the increased GPP flux was efficiently channeled by the endogenous Erg20p to produce sterols and other essential isoprenoids <sup> <a href="#CR14">14</a> </sup>. Further efforts, in which Erg20p was engineered to be inefficient only in the FPP-synthesizing step, increased the GPP pool without abolishing sterol synthesis <sup> <a href="#CR14">14</a> </sup>. However, despite additional improvements through N-degron-dependent protein degradation of the native Erg20p to minimize competition with the introduced monoterpene synthase <sup> <a href="#CR23">23</a> </sup>, the obtained monoterpene titers were orders of magnitude lower than the titers of sesquiterpene production by other engineered yeast platforms <sup> <a href="#CR21">21</a>, <a href="#CR22">22</a>, <a href="#CR24">24</a>, <a href="#CR27">27</a> </sup>. These observations suggest that there is still considerable potential for improving the monoterpene biosynthesis in yeast. </p> <p>To overcome limitations in monoterpene production that may be imposed by the structure and regulation of the yeast native metabolism, we set out to establish an orthogonal pathway. To achieve orthogonality, the new pathway must have only one branching point from yeast metabolism. To meet this requirement, the synthetic pathway must not involve GPP because Erg20p connects GPP back to basic yeast metabolism via FPP (Fig. <a href="#Fig1">1</a>). This requirement creates a considerable challenge, as GPP is the precursor of monoterpenes. Thus, we examined the possibility of using alternative precursors for monoterpene biosynthesis. Although it has been shown that GPP is the canonical substrate of monoterpene synthases <sup> <a href="#CR18">18</a> </sup>, recent studies in different tomato species have revealed the existence of a small group of enzymes that deviate from this rule and preferentially convert the <i>cis</i>-isomer of GPP, neryl diphosphate (NPP), to a limited number of monoterpene hydrocarbons <sup> <a href="#CR28">28</a>– <a href="#CR30">30</a> </sup>. A dedicated <i>cis-</i>prenyltransferase, neryl diphosphate synthase ( <i>Sl</i>NPPS1), was found to generate the required NPP substrate by condensing DMAPP and IPP in the <i>cis</i>-configuration <sup> <a href="#CR28">28</a> </sup> (Fig. <a href="#Fig1">1</a>). NPP and NPP-derived monoterpene biosynthesis has only been observed in these few plant species and is entirely absent in yeast. </p> <p>Here, we use NPP as an alternative substrate to establish an orthogonal pathway for monoterpene scaffold production. We identify a single residue dictating isomeric substrate selectivity in a monoterpene synthase and engineer canonical terpene synthases to specifically accept the alternative substrate. As a result, we establish a complete synthetic pathway that extends up to the step of monoterpene oxidation and achieve marked improvements in the production of several industrially important monoterpenes. </p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="justifyingtheconstructionofanorthogonalmonoterpenepathway" title="sec"> <div class="title" tagx="title" title="title">
Justifying the construction of an orthogonal monoterpene pathway</div> <p>To justify the need for constructing an orthogonal pathway, we first confirmed that the low titers of monoterpene observed in previous efforts <sup> <a href="#CR14">14</a>, <a href="#CR23">23</a> </sup> were not due to product toxicity, and that additional factors, such as pathway structure, may play an important role. We evaluated the growth of yeast cells engineered to provide high levels of isoprenoid precursors (strain AM94; Supplementary Table <a href="#MOESM1">1</a>) under conditions that recapitulate production of specific titers of monoterpenes. In agreement with previous studies <sup> <a href="#CR31">31</a>, <a href="#CR32">32</a> </sup>, yeast cells grew equally well when supplemented with increasing monoterpene concentrations up to 500 mg L <sup>−1</sup> (Supplementary Fig. <a href="#MOESM1">1</a>), which was more than seven times higher than the current best-production titers <sup> <a href="#CR23">23</a> </sup>. Subsequently, we introduced the <i>Citrus limon</i> limonene synthase ( <i>Cl</i>LimS) <sup> <a href="#CR33">33</a> </sup> into strain AM94 and confirmed that limonene productivity was not reduced, even when the culture was supplemented with a limonene concentration that was tenfold higher than the titer produced by the strain (Supplementary Fig. <a href="#MOESM1">2</a>). </p> <p>To further corroborate previous evidence suggesting that the sequential reaction of Erg20p may be a contributing factor to low monoterpene productivity <sup> <a href="#CR14">14</a> </sup>, we supplemented the cell extract of <i>Cl</i>LimS-expressing AM94 cells with isoprenoid precursors and measured the formation of limonene. Addition of GPP, in the presence of IPP and DMAPP, resulted in the production of low amounts of limonene and a significant larger amount of FPP (Supplementary Fig. <a href="#MOESM1">3</a>). This suggested that most of the GPP precursor was used by Erg20p to synthesize FPP, and only a small amount of the GPP added was converted to limonene by <i>Cl</i>LimS (Supplementary Fig. <a href="#MOESM1">3</a>). Further experiments using <sup>13</sup>C-GPP confirmed that &amp;lt;10% of the <sup>13</sup>C-GPP was converted to limonene under these conditions, and most of it was channeled to FPP. Therefore, Erg20p is very efficient in utilizing GPP, reducing the GPP pool and hindering monoterpene production. Thus, we set out to engineer an orthogonal pathway that does not depend on GPP. </p> </div> <div class="establishingefficientnppbiosynthesisinyeastcells" title="sec"> <div class="title" tagx="title" title="title">
Establishing efficient NPP biosynthesis in yeast cells</div> <p>A cDNA fragment encoding <i>Sl</i>NPPS1 was cloned into the yeast vector pHTDHmyc under the P <sub> <i>TDH3</i> </sub> constitutive promoter and expressed in strain AM94. To monitor NPP synthesis, we also introduced yeast codon-optimized versions of three different NPP-specific monoterpene synthases, <i>Solanum lycopersicum</i> phellandrene synthase ( <i>Sl</i>PHS1) <sup> <a href="#CR28">28</a> </sup>, <i>Solanum habrochaites</i> limonene synthase ( <i>Sh</i>LimS) <sup> <a href="#CR30">30</a> </sup>, and <i>S. habrochaites</i> pinene synthase ( <i>Sh</i>PinS) <sup> <a href="#CR30">30</a> </sup>. When expressed in yeast in the absence of <i>Sl</i>NPPS1, no new products were detected, as none of the three monoterpene synthases were able to utilize the available GPP. However, upon <i>Sl</i>NPPS1 co-expression, all three synthases produced their reported NPP-derived products. Specifically, <i>Sl</i>PHS1 produced <i>β</i>-phellandrene (48.6%), δ-2-carene (36.6%), α-phellandrene (8.3%), and α-terpinene (6.5%). <i>Sh</i>LimS produced limonene as a single product, and <i>Sh</i>PinS produced α-pinene (84%) and limonene (16%) (Fig. <a href="#Fig2">2a</a>). Product titers in all cases were very low, ranging from 0.009 mg L <sup>−1</sup> α-pinene to 0.16 mg L <sup>−1</sup> <i>β</i>-phellandrene (Supplementary Table <a href="#MOESM1">2</a>). Furthermore, we observed that AM94 cells engineered to express <i>Sl</i>NPPS1 in the absence of any monoterpene synthase produced detectable levels of nerol, as confirmed by comparison of mass spectra and retention time with an authentic standard (Fig. <a href="#Fig2">2b</a>). By analogy to the formation of geraniol via the hydrolysis of GPP by intracellular phosphatases like Lpp1p and Dpp1p <sup> <a href="#CR14">14</a>, <a href="#CR34">34</a> </sup>, we attributed the production of nerol to the hydrolysis of NPP that accumulated in the absence of a suitable terpene synthase. Taken together, these results confirmed that the alternative substrate, NPP, is synthesized in the engineered yeast strain and it can be used to produce monoterpenes. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 2</span> </a> </div>   <p>Engineering NPP production in yeast. <b>a</b> Three NPP-specific monoterpene synthases were co-expressed in yeast with <i>Sl</i>NPPS1. <i>Sl</i>PHS1 (brown) produced a mixture of monoterpenes, including β-phellandrene, δ-2-carene, α-phellandrene, and α-terpinene. <i>Sh</i>LimS (green) produced exclusively limonene, while <i>Sh</i>PinS (blue) synthesized α-pinene as the main product, together with low amounts of limonene. <b>b</b> Production of nerol by yeast cells (brown), presumably resulting from the hydrolysis of excess NPP. The peak marked with a star (*) corresponds to a non-terpene yeast-derived compound. The nerol standard is shown in green. <b>c</b> Stepwise engineering of NPP production in yeast was monitored by measurement of the monoterpene products synthesized by the three NPP-specific synthases. <b>d</b> Production of the nonspecific monoterpene product nerol by the engineered cells increased as monoterpene yields improved, suggesting that the selected NPP-specific synthases could not fully harvest the NPP flux. Error bars correspond to the mean absolute deviation (MAD) around the mean ( <i>n</i> = 3, corresponding to three separate yeast transformations processed and analyzed independently). Source data of <b>c</b> and <b>d</b> are provided as a Source Data file </p>   </div> <p /> <p>To ensure that NPP can form the basis for an orthogonal pathway, we confirmed that it did not interfere with the rest of the yeast isoprenoid metabolism. With a combination of in vitro and in vivo experiments, we confirmed that NPP was not taken up by Erg20p to produce larger prenyl diphosphates. We expressed Erg20p in <i>Escherichia coli</i> and used the purified protein (Supplementary Fig. <a href="#MOESM1">4</a>) in in vitro assays with GPP and NPP. Although the recombinant enzyme was able to produce FPP from IPP and GPP, it was not able to catalyze elongation of NPP. We were also unable to detect any NPP-derived products of Erg20p in yeast cells or in yeast cell extracts supplemented with NPP and IPP (Supplementary Fig. <a href="#MOESM1">5</a>). We also confirmed that the growth properties of <i>Sl</i>NPPS1-expressing yeast cells were very similar to their empty vector-containing counterparts (Supplementary Fig. <a href="#MOESM1">6</a>). </p> </div> <div class="redirectingfluxtotheorthogonalpathway" title="sec"> <div class="title" tagx="title" title="title">
Redirecting flux to the orthogonal pathway</div> <p>The new branching point established for monoterpene biosynthesis could function as a valve to direct the flux of substrates away from FPP synthesis and to increase the monoterpene yield (Fig. <a href="#Fig1">1</a>). First, we aimed to decrease the competition for IPP by Erg20p. To achieve this, we introduced Erg20p(N127W), a dominant-negative Erg20p variant that inhibits the FPP synthesis step of the endogenous wild-type enzyme <sup> <a href="#CR14">14</a>, <a href="#CR35">35</a> </sup>. This resulted in a 3–4-fold increase (depending on the enzyme) in the total monoterpene yield of all three NPP-specific monoterpene synthases (Fig. <a href="#Fig2">2c</a> and Supplementary Table <a href="#MOESM1">2</a>). </p> <p>Heterozygous gene deletions in yeast have been shown to decrease the level of the corresponding protein to 50% <sup> <a href="#CR36">36</a> </sup>. Thus, to further reduce the flux through the FPP-synthesizing branch, we downregulated Erg20p by shifting to the yeast strain MIC2, which carries a deletion in one of the two <i>ERG20</i> alleles <sup> <a href="#CR14">14</a> </sup>. This resulted in a further 1.9–2.8-fold increase in monoterpene titer (Fig. <a href="#Fig2">2c</a> and Supplementary Table <a href="#MOESM1">2</a>). </p> <p>Previous studies had shown that fusion of a heterologous enzyme to Erg20p can be beneficial for the enzyme’s stability <sup> <a href="#CR37">37</a>– <a href="#CR39">39</a> </sup>. We fused <i>Sl</i>NPPS1 to the C terminus of Erg20p(N127W) and confirmed that the fused protein reached higher intracellular levels compared with the non-fused <i>Sl</i>NPPS1 (Supplementary Fig. <a href="#MOESM1">7</a>). We subsequently introduced the Erg20p(N127W)– <i>Sl</i>NPPS1 variant into strain MIC2 to achieve an additional 1.4–2-fold improvement in production. The combination of these interventions resulted in an overall 12–18-fold increase, reaching 2.7 mg L <sup>−1</sup> β-phellandrene, 2.5 mg L <sup>−1</sup> limonene, and 0.11 mg L <sup>−1</sup> α-pinene (Fig. <a href="#Fig2">2c</a> and Supplementary Table <a href="#MOESM1">2</a>). Following these improvements, a significant amount of nerol could be detected in the NPP-producing cells overexpressing <i>Sh</i>LimS, <i>Sh</i>PinS, or <i>Sl</i>PHS1 (Fig. <a href="#Fig2">2d</a> and Supplementary Table <a href="#MOESM1">3</a>). This suggested that the combination of these interventions achieved efficient synthesis of NPP, but utilization of the alternative substrate has reached a bottleneck at the level of the monoterpene synthases that resulted in its accumulation and eventual conversion to nerol. </p> </div> <div class="canonicalmonoterpenesynthasesacceptnppasasubstrate" title="sec"> <div class="title" tagx="title" title="title">
Canonical monoterpene synthases accept NPP as a substrate</div> <p>The accumulation of nerol indicated that the three NPP-specific synthases tested here could not harvest the full potential of the orthogonal pathway, and more efficient biocatalysts were required. However, only few NPP-specific synthases have been reported to date <sup> <a href="#CR28">28</a>, <a href="#CR30">30</a>, <a href="#CR40">40</a> </sup>, limiting the possibility that a search to identify enzymes more active than the synthases used here would be successful. Furthermore, the product range of the known NPP-specific enzymes was limited to only four main products, β-phellandrene <sup> <a href="#CR28">28</a> </sup>, limonene <sup> <a href="#CR30">30</a> </sup>, α-pinene <sup> <a href="#CR30">30</a> </sup>, and nerol <sup> <a href="#CR40">40</a> </sup>. To identify enzymes that are more active and to enable the production of a broad range of different monoterpene scaffolds, we turned into the rich resource of canonical, GPP-converting, plant monoterpene synthases. We selected five synthases with different product profiles, including <i>Cl</i>LimS <sup> <a href="#CR33">33</a> </sup>, <i>Salvia fruticosa</i> 1,8-cineole synthase ( <i>Sf</i>CinS1) <sup> <a href="#CR41">41</a> </sup>, <i>Salvia pomifera</i> sabinene synthase ( <i>Sp</i>SabS) <sup> <a href="#CR41">41</a> </sup>, <i>Pinus taeda α-</i>pinene synthase ( <i>Pt</i>PinS) <sup> <a href="#CR42">42</a> </sup>, and <i>Solanum elaeagnifolium</i> camphene synthase ( <i>Se</i>CamS) <sup> <a href="#CR43">43</a> </sup>, and tested whether they can accept NPP as a substrate. When introduced to strain MIC2 expressing Erg20p(N127W), i.e., a GPP-synthesizing strain, all five enzymes produced their expected monoterpene products (Fig. <a href="#Fig3">3</a>). When Erg20p(N127W) was replaced by the Erg20p(N127W)– <i>Sl</i>NPPS1 fusion to establish the production of NPP, an overall increase ranging between 1.3- and 2.8-fold (depending on the enzyme) in monoterpene formation compared with the GPP-only producing strain was achieved with all canonical monoterpene synthases (Fig. <a href="#Fig3">3</a>). This suggested that the canonical enzymes could accept both substrates, resulting in overall higher yields, and that the different enzymes had varying capacity to accept the isomeric substrate. We confirmed that the selected monoterpene synthases were able to utilize NPP in in vitro assays with bacterially expressed enzymes (Supplementary Figs. <a href="#MOESM1">4</a> and <a href="#MOESM1">8a</a>). We also examined the product profile of the five enzymes with NPP in vitro and found that four of them synthesized almost the same blend of monoterpenes as with GPP. Only the profile of <i>Se</i>CamS shifted from camphene toward limonene with NPP in vitro (Fig. <a href="#Fig3">3e</a>; Supplementary Fig. <a href="#MOESM1">8a</a>), which was reflected in the product profile of NPP-producing yeast cells expressing <i>Se</i>CamS (Supplementary Fig. <a href="#MOESM1">8b</a>). There was no new monoterpene product with any of the synthases and NPP. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 3</span> </a> </div>   <p>Monoterpene production by canonical terpene synthases. Five different monoterpene synthases were overexpressed in yeast cells producing solely GPP (brown), or also producing NPP as an alternative substrate (green). The monoterpene product titers increased markedly when NPP was also available with all synthases tested: <i>Salvia fruticosa</i> 1,8-cineole synthase ( <i>Sf</i>CinS1) ( <b>a</b>), <i>Pinus taeda</i> α-pinene synthase ( <i>Pt</i>PinS) ( <b>b</b>), <i>Citrus limon</i> limonene synthase ( <i>Cl</i>LimS) ( <b>c</b>), <i>S. pomifera</i> sabinene synthase ( <i>Sp</i>SabS) ( <b>d</b>), and <i>S. elaeagnifolium</i> camphene synthase ( <i>Se</i>CamS) ( <b>e</b>). A significant shift from camphene to limonene was observed in the product profile of <i>Se</i>CamS with NPP. Yeast cells producing NPP in the absence of any monoterpene synthase are shown as control (blue). Samples were analyzed in triplicate ( <i>n</i> = 3 biologically independent samples). Error bars for each sample correspond to the mean absolute deviation (MAD) around the mean. Source data of <b>a</b>– <b>e</b> are provided as a Source Data file </p>   </div> <p /> <p>Kinetic analysis of the reaction with each isomeric substrate revealed that the canonical synthases exhibited higher affinity and overall catalytic efficiency ( <i>k</i> <sub>cat</sub> <sup>app</sup>/ <i>K</i> <sub>M</sub> <sup>app</sup>) with GPP than with NPP ( <i>Sf</i>CinS1: 0.213 min <sup>−1</sup> μM <sup>−1</sup> with GPP vs. 0.137 min <sup>−1</sup> μM <sup>−1</sup> with NPP; <i>Cl</i>LimS: 0.735 min <sup>−1</sup> μM <sup>−1</sup> with GPP vs. 0.456 min <sup>−1</sup> μM <sup>−1</sup> with NPP; <i>Se</i>CamS: 0.21 min <sup>−1</sup> μM <sup>−1</sup> with GPP vs. 0.044 min <sup>−1</sup> μM <sup>−1</sup> with NPP; Table <a href="#Tab1">1</a>). Yet, the observation that yeast productivity with <i>Cl</i>LimS and <i>Pt</i>PinS when both the GPP and the NPP pathways were present was more than double, compared with when only the GPP pathway was active (Fig. <a href="#Fig3">3</a>), provided two main conclusions. First, it indicated that these terpene synthases were not the limiting step of the pathway. Second, the ability of <i>Cl</i>LimS and <i>Pt</i>PinS to support equal or higher titers with NPP, despite this not being their preferred substrate, suggested that the established NPP-based pathway was efficient and had the potential to exceed the GPP pathway in productivity. To further evaluate the performance of the NPP-based pathway, we examined the kinetic characteristics of <i>Sl</i>NPPS1 using in vitro assays with enzymes expressed and purified in <i>E. coli</i> (Supplementary Fig. <a href="#MOESM1">4</a>). The determined kinetic parameters revealed that <i>Sl</i>NPPS1 is significantly less efficient than Erg20p in utilizing DMAPP and IPP (Supplementary Table <a href="#MOESM1">4</a>; refs. <sup> <a href="#CR14">14</a>, </sup> <sup> <a href="#CR28">28</a> </sup>). Combined with the preference of canonical terpene synthases for GPP rather than NPP (Table <a href="#Tab1">1</a>), we concluded that the observed increase in productivity is not due to favorable kinetics of the <i>Sl</i>NPPS1 branch. <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  </div></p><p>Kinetic parameters of terpene synthases and variants</p>  <table> <thead> <tr> <th>Enzyme variant</th> <th>Substrate</th> <th> <i>k</i> <sub>cat</sub> <sup>app</sup> (min <sup>−1</sup>) </th> <th> <i>K</i> <sub>M</sub> <sup>app</sup> (μM) </th> <th> <i>k</i> <sub>cat</sub> <sup>app</sup>/ <i>K</i> <sub>M</sub> <sup>app</sup> (min <sup>−1</sup> μM <sup>−1</sup>) </th> </tr> </thead> <tbody> <tr> <td> <i>Sf</i>CinS1 wild type </td> <td>GPP</td> <td>5.56 ± 0.38</td> <td>26.12 ± 2.79</td> <td>0.213</td> </tr> <tr> <td> <i>Sf</i>CinS1 wild type </td> <td>NPP</td> <td>12.2 ± 2.67</td> <td>89.46 ± 4.99</td> <td>0.137</td> </tr> <tr> <td> <i>Sf</i>CinS1(F571Y) </td> <td>GPP</td> <td>0.37 ± 0.03</td> <td>29.93 ± 6.14</td> <td>0.012</td> </tr> <tr> <td> <i>Sf</i>CinS1(F571Y) </td> <td>NPP</td> <td>0.75 ± 0.12</td> <td>37.48 ± 14.26</td> <td>0.020</td> </tr> <tr> <td> <i>Sf</i>CinS1(F571H) </td> <td>GPP</td> <td>0.86 ± 0.09</td> <td>18.05 ± 5.07</td> <td>0.048</td> </tr> <tr> <td> <i>Sf</i>CinS1(F571H) </td> <td>NPP</td> <td>0.74 ± 0.08</td> <td>238.1 ± 34.35</td> <td>0.003</td> </tr> <tr> <td> <i>Sf</i>CinS1(F571V) </td> <td>GPP</td> <td>0.32 ± 0.08</td> <td>271.5 ± 95.30</td> <td>0.001</td> </tr> <tr> <td> <i>Sf</i>CinS1(F571V) </td> <td>NPP</td> <td>0.45 ± 0.04</td> <td>67.24 ± 13.30</td> <td>0.007</td> </tr> <tr> <td> <i>Sf</i>CinS1(F571L) </td> <td>GPP</td> <td>0.44 ± 0.07</td> <td>55.72 ± 9.22</td> <td>0.008</td> </tr> <tr> <td> <i>Sf</i>CinS1(F571L) </td> <td>NPP</td> <td>0.64 ± 0.08</td> <td>71.82 ± 17.63</td> <td>0.009</td> </tr> <tr> <td> <i>Cl</i>LimS wild type </td> <td>GPP</td> <td>18.34 ± 0.88</td> <td>24.94 ± 3.76</td> <td>0.735</td> </tr> <tr> <td> <i>Cl</i>LimS wild type </td> <td>NPP</td> <td>37.16 ± 1.76</td> <td>81.55 ± 7.33</td> <td>0.456</td> </tr> <tr> <td> <i>Cl</i>LimS(H570F) </td> <td>GPP</td> <td>14.64 ± 2.50</td> <td>12.86 ± 7.20</td> <td>1.138</td> </tr> <tr> <td> <i>Cl</i>LimS(H570F) </td> <td>NPP</td> <td>29.70 ± 1.82</td> <td>31.94 ± 5.14</td> <td>0.930</td> </tr> <tr> <td> <i>Cl</i>LimS(H570Y) </td> <td>GPP</td> <td>8.92 ± 0.43</td> <td>11.76 ± 4.82</td> <td>0.759</td> </tr> <tr> <td> <i>Cl</i>LimS(H570Y) </td> <td>NPP</td> <td>21.86 ± 1.66</td> <td>23.98 ± 6.41</td> <td>0.911</td> </tr> <tr> <td> <i>Cl</i>LimS(H570L) </td> <td>GPP</td> <td>0.24 ± 0.07</td> <td>16.89 ± 6.72</td> <td>0.014</td> </tr> <tr> <td> <i>Cl</i>LimS(H570L) </td> <td>NPP</td> <td>4.58 ± 0.66</td> <td>110.5 ± 19.30</td> <td>0.041</td> </tr> <tr> <td> <i>Cl</i>LimS(H570V) </td> <td>GPP</td> <td>1.16 ± 0.08</td> <td>56.70 ± 12.42</td> <td>0.021</td> </tr> <tr> <td> <i>Cl</i>LimS(H570V) </td> <td>NPP</td> <td>1.24 ± 0.10</td> <td>32.44 ± 14.14</td> <td>0.038</td> </tr> <tr> <td> <i>Cl</i>LimS(H570I) </td> <td>GPP</td> <td>0.21 ± 0.02</td> <td>24.43 ± 6.44</td> <td>0.009</td> </tr> <tr> <td> <i>Cl</i>LimS(H570I) </td> <td>NPP</td> <td>1.90 ± 0.13</td> <td>27.06 ± 4.72</td> <td>0.070</td> </tr> <tr> <td> <i>Se</i>CamS wild type </td> <td>GPP</td> <td>3.91 ± 0.22</td> <td>18.58 ± 2.77</td> <td>0.210</td> </tr> <tr> <td> <i>Se</i>CamS wild type </td> <td>NPP</td> <td>4.82 ± 0.39</td> <td>107.49 ± 12.48</td> <td>0.044</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>Errors represent standard error of the fitted parameters. Source data are provided in a Source Data file </p> </div>  <p /> <p>The canonical monoterpene synthases convert GPP and NPP to the same blend of products (Supplementary Fig. <a href="#MOESM1">8</a>), not allowing to determine the relative contribution of the two substrates simply by examining the product profile. To resolve this, we synthesized <sup>13</sup>C-labeled NPP (Supplementary Fig. <a href="#MOESM1">9</a>), so as to distinguish the products of each substrate when mixtures of the labeled compound with unlabeled GPP were used. We added precursors to extracts of yeast cells expressing <i>Cl</i>LimS and determined the isotopic composition of the produced limonene using GC–APCI–QqToF analysis. In these experiments, <sup>13</sup>C-NPP was initially shown to be channeled efficiently to limonene synthesis, while GPP was mostly consumed to produce FPP (Supplementary Figs. <a href="#MOESM1">10</a> and <a href="#MOESM1">11</a>). In subsequent competition experiments between <sup>13</sup>C-labeled NPP and GPP, when the two isomeric substrates were added at equal concentrations, over 90% of the limonene produced originated from <sup>13</sup>C-NPP. The contribution of the two substrates to limonene production was comparable, only when GPP was in tenfold excess (Supplementary Fig. <a href="#MOESM1">12a</a>). </p> <p>Comparison of the overall production titers revealed that <i>Cl</i>LimS and <i>Pt</i>PinS were 24 and 224 times more efficient, respectively, in limonene and α-pinene synthesis in NPP-producing yeast cells than their NPP-specific counterparts, <i>Sh</i>LimS and <i>Sh</i>PinS (Supplementary Table <a href="#MOESM1">5</a>). Taken together, these findings confirmed that canonical monoterpene synthases can serve as efficient surrogate synthases in this synthetic pathway and suggested that canonical enzymes could offer a starting point for protein engineering efforts to improve NPP utilization. </p> </div> <div class="engineeringnpp-specificmonoterpenesynthases" title="sec"> <div class="title" tagx="title" title="title">
Engineering NPP-specific monoterpene synthases</div> <p>To fully exploit the potential of the orthogonal pathway, we set out to engineer the selected canonical monoterpene synthases, with the aim to increase their efficiency and specificity for NPP. Initially, we focused on <i>Sf</i>CinS1 due to the availability of structural information and our previous experience in engineering this enzyme to alter its substrate and product specificity <sup> <a href="#CR41">41</a> </sup>. In these efforts, we developed a library of 19 variants at different positions in the active site <sup> <a href="#CR39">39</a>, <a href="#CR41">41</a> </sup>. Here, we differentially screened this library in GPP- and NPP-producing yeast strains to identify sites that potentially influence substrate specificity. As shown in Fig. <a href="#Fig4">4a</a>, one site, F571, stood out as playing a critical role in selecting between the two isomeric substrates, as several variants at this position were able to markedly improve the performance of the enzyme with NPP. Variants <i>Sf</i>CinS1(F571V), <i>Sf</i>CinS1(F571Y), and <i>Sf</i>CinS1(F571I) were 18, 9, and 7 times more efficient, respectively, when the NPP pathway was present, compared with their performance with only GPP as a substrate (Fig. <a href="#Fig4">4a</a>). Although the catalytic efficiency of the variants was lower than that of the wild-type enzyme (Supplementary Table <a href="#MOESM1">6</a>), these findings suggested that F571 may be acting as a switch that differentiates substrate preference between NPP and GPP. In addition to F571, certain substitutions in a second residue, N338, also resulted in shifts in specificity, albeit in a less efficient or consistent manner. Variants <i>Sf</i>CinS1(N338A) and <i>Sf</i>CinS1(N338S) exhibited an opposing substrate selectivity. <i>Sf</i>CinS1(N338A) became a strictly GPP-utilizing enzyme, while <i>Sf</i>CinS1(N338S) was 6.7 times more efficient when the NPP pathway was present (Fig. <a href="#Fig4">4a</a>). To test for potential synergistic effects, we also constructed several variants combining F571 substitutions with substitutions in the other positions (Supplementary Table <a href="#MOESM1">7</a>). However, no further improvements in substrate specificity were observed (Fig. <a href="#Fig4">4a</a>). For further studies, we focused exclusively on the role of F571, because of the extent and consistency of specificity changes. </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 4</span> </a> </div>   <p>Engineering SfCinS1 to become specific for NPP. <b>a</b> A library of 19 different <i>Sf</i>CinS variants was evaluated in yeast cells producing either only GPP, or both GPP and NPP. To identify variants with improved specificity for NPP, the ratio of monoterpene product titers obtained with and without the NPP pathway was determined and shown here in logarithmic scale. Samples were analyzed in triplicate originating from independent yeast transformations ( <i>n</i> = 3 biologically independent samples). Errors correspond to the mean absolute deviation (MAD) around the mean. <b>b</b> Proposed GPP and NPP cyclization mechanism leading to the synthesis of 1,8-cineole by <i>Sf</i>CinS1. Initially, GPP binds at the extended conformation. Following syn migration of the diphosphate, the transoid extended conformation of LPP transitions first to the cisoid form and then to the catalytically competent cisoid closed conformation <sup> <a href="#CR44">44</a> </sup>. In NPP, the C-2,3 bond is already in the <i>cis-</i>conformation, bypassing the requirement for the transoid to cisoid transition. The insets show models of the active site of <i>Sf</i>CinS1 with two proposed conformers of 2-fluorolinalyl diphosphate (2F-LPP) superimposed. The two conformations have been obtained by soaking crystals of mint limonene synthase with 2F-LPP or 2-fluoro-geranyl diphosphate (2F-GPP). In the crystal, 2F-GPP is converted to the transoid 2F-LPP form, while 2F-LPP assumes a conformation that resembles the extended cisoid form of LPP. For the GPP reaction to proceed, extensive conformational changes of the substrate are needed for the conversion from transoid to cisoid. F571 is positioned in the center of this transition. Red arrows indicate chemical bond rotation. Graphics produced with UCSF Chimera <sup> <a href="#CR68">68</a> </sup>. Source data of <b>a</b> are provided as a Source Data file </p>   </div> <p /> <p>Analysis of the structural information available for <i>Sf</i>CinS1 <sup> <a href="#CR41">41</a> </sup> in the context of biochemical and structural studies on the enzymatic mechanism of monoterpene synthases <sup> <a href="#CR44">44</a> </sup> provided insight into the possible role of F571. As shown in Fig. <a href="#Fig4">4b</a>, the early steps of the canonical monoterpene synthase mechanism involve the initial binding of GPP in an extended conformation, followed by the ionization of the substrate and the syn migration of the diphosphate moiety to C-3. The resulting linalyl diphosphate (LPP) maintains the transoid conformation of GPP (Fig. <a href="#Fig4">4b</a>). Thus, for cyclization to take place, the C-2,3 bond must rotate to convert LPP from the transoid to the cisoid conformer. Subsequent conformational change of LPP to the closed configuration brings C-6 and C-1 to a position competent for cyclization. Following cyclization, the resulting carbocations rearrange to finally yield different hydrocarbons (Fig. <a href="#Fig4">4b</a>). Due to the cisoid conformation of NPP, its cyclization does not require the same transition from the transoid to the cisoid form (Fig. <a href="#Fig4">4b</a>). As shown in Fig. <a href="#Fig4">4b</a>, in the structure of <i>Sf</i>CinS1, F571 is positioned at a short distance from C-1 of LPP. This analysis suggested two possible explanations for the ability of F571 to control isomeric substrate selectivity. Considering the extensive conformational changes that are required for the substrate to transition from the transoid to the cisoid form (compare the structure of the two intermediate analogs in Fig. <a href="#Fig4">4b</a>), one scenario would be that F571 plays a direct role in this process. In this case, substitutions in F571 that impair the transoid to cisoid transition would have a negative impact in the GPP-based reaction, but would not interfere with the transition-independent NPP reaction. Alternatively, F571 may not be directly involved in the transition process; instead, the size and orientation of the side chain at position 571 may alter the geometry of the active site to favor binding of one substrate versus the other. To distinguish between these two mechanisms, we conducted a detailed kinetic analysis of <i>Sf</i>CinS1 mutants. <i>Sf</i>CinS1(F571V), <i>Sf</i>CinS1(F571Y), and <i>Sf</i>CinS1(F571L) showed shifts in <i>K</i> <sub>M</sub> <sup>app</sup> either favoring NPP binding or disfavoring binding of GPP (Table <a href="#Tab1">1</a>), suggesting that the residue at position 571 exerts stereochemical control. In agreement, the reverse behavior of <i>Sf</i>CinS1(F571H), which showed preference for GPP in vivo, was the result of a significant increase in <i>K</i> <sub>M</sub> <sup>app</sup> for NPP. </p> <p>We explored whether residues analogous to F571 in the other canonical monoterpene synthases play a similarly critical role. Based on amino acid sequence alignment (Supplementary Fig. <a href="#MOESM1">13</a>), candidate residues in the other monoterpene synthases were identified and substituted with aromatic or aliphatic side chains of different size using site-directed mutagenesis (Supplementary Table <a href="#MOESM1">7</a>). As shown in Fig. <a href="#Fig5">5</a>, in all four enzymes tested, we were able to identify variants at this position that shift substrate selectivity to NPP. Specifically, in <i>Cl</i>LimS, all H570 variants showed higher preference for NPP than the wild-type enzyme in the yeast system (Fig. <a href="#Fig5">5a</a>, left panel). Among them, <i>Cl</i>LimS(H570Y) and <i>Cl</i>LimS(H570F) also achieved improved performance in NPP-based monoterpene production in yeast (Fig. <a href="#Fig5">5a</a>, right panel). In <i>Cl</i>LimS(H570Y), this was the result of a twofold improvement on the catalytic efficiency ( <i>k</i> <sub>cat</sub> <sup>app</sup>/ <i>K</i> <sub>M</sub> <sup>app</sup>) with NPP, which was mostly due to a lower <i>K</i> <sub>M</sub> <sup>app</sup> with NPP and a decreased <i>k</i> <sub>cat</sub> <sup>app</sup> with GPP (Table <a href="#Tab1">1</a>). In the case of <i>Cl</i>LimS(H570F), the higher overall productivity in the yeast system was the result of improved binding of both substrates. <i>Cl</i>LimS(H570I), <i>Cl</i>LimS(H570L), and <i>Cl</i>LimS(H570V) were even more specific for NPP (Fig. <a href="#Fig5">5a</a>, left panel), as manifested by an eightfold higher <i>k</i> <sub>cat</sub> <sup>app</sup>/ <i>K</i> <sub>M</sub> <sup>app</sup> in <i>Cl</i>LimS(H570I) and a threefold higher <i>k</i> <sub>cat</sub> <sup>app</sup>/ <i>K</i> <sub>M</sub> <sup>app</sup> in <i>Cl</i>LimS(H570L) (Table <a href="#Tab1">1</a>). However, the overall efficiency of these variants was low and could not sustain high-product titers (Fig. <a href="#Fig5">5a</a>, right panel; Table <a href="#Tab1">1</a>). We evaluated <i>Cl</i>LimS(H570Y) in competition experiments using yeast extracts and <sup>13</sup>C-NPP. <i>Cl</i>LimS(H570Y) was markedly more efficient than wild-type <i>Cl</i>LimS in harvesting NPP fluxes, and even when GPP was present at a tenfold excess over <sup>13</sup>C-NPP, over 75% of the products were derived from <sup>13</sup>C-NPP (Supplementary Fig. <a href="#MOESM1">11b</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Fig. 5</span> </a> </div>   <p>Monoterpene profile of the terpene synthase variants. Variants of <i>Cl</i>LimS ( <b>a</b>), <i>Sp</i>SabS ( <b>b</b>), <i>Se</i>CamS ( <b>c</b>), and <i>Pt</i>PinS ( <b>d</b>), carrying different substitutions in the residue homologous to <i>Sf</i>CinS1(F571), were analyzed in yeast cells producing NPP versus non-NPP-producing cells. On the left-hand side of panels <b>a</b>– <b>d</b>, the bar charts show the ratio of the total monoterpene production titer when both the NPP and GPP pathways are present divided by the titer of monoterpene production by the same variant when only the GPP pathway is present. The bar charts on the right-hand side of panels <b>a</b>– <b>d</b> show the total monoterpene titer obtained by each mutant synthase when the NPP pathway was present in comparison to the corresponding wild-type enzyme titer (normalized to 1), indicating the overall efficiency of each enzyme variant. <b>e</b> <i>Se</i>CamS variants H583F and H583V switch specificity from a camphene synthase to a highly efficient limonene synthase. <b>f</b> The overall improvements in pathway performance enable the introduction of an additional downstream modification step. Yeast cells co-expressing <i>Sp</i>SabS and CYP750B1 produce detectable amounts of <i>trans</i>-sabin-3-ol only when the NPP pathway is present. Replacement of wild-type <i>Sp</i>SabS by the NPP-specific variant <i>Sp</i>SabS(H561F) improved trans-sabin-3-ol synthesis even further. <b>g</b> Introduction of P <sub> <i>ERG1</i> </sub> upstream of <i>ERG20</i> in strain MIC3 installs dynamic control of fluxes through Erg20p. In combination with the NPP-specific variants <i>Cl</i>LimS(H570Y) and <i>Sp</i>SabS(H561F), this leads to further improvement in limonene and sabinene titers, respectively. Strains denoted by GPP express Erg20p(N127W) and produce only GPP, while strains denoted by GPP + NPP express Erg20p(N127W)- <i>Sl</i>NPPS1 and produce both GPP and NPP. Samples were prepared independently from three different yeast transformations, processed and analyzed separately ( <i>n</i> = 3 biologically independent samples). The average yield of each compound between the samples was calculated, and the mentioned ratios and fold improvements in the product profile were deduced. Error bars in panels <b>a</b>– <b>d</b> correspond to the geometric mean of the original two relative MAD values. Source data of <b>a</b>– <b>d</b> and <b>g</b> are provided as a Source Data file </p>   </div> <p /> <p>In <i>Sp</i>SabS, substitution of H561 with F resulted in a highly efficient biocatalyst, with an almost threefold improved apparent specificity for NPP in yeast compared with the wild-type enzyme (Fig. <a href="#Fig5">5b</a>, left panel) and an overall 2.5-fold improvement in performance when the NPP pathway was present (Fig. <a href="#Fig5">5b</a>; right panel). We were not able to obtain active recombinant <i>Sp</i>SabS(H561F) to determine the kinetic parameters underlying these improvements. In the case of <i>Se</i>CamS, substitution of H583 resulted in drastic changes in specificity and activity. Variants <i>Se</i>CamS(H583F) and <i>Se</i>CamS(H583V) became limonene-synthesizing enzymes that exhibited three -times higher apparent selectivity for NPP compared to wild-type <i>Se</i>CamS (Fig. <a href="#Fig5">5c</a>, left panel). These two variants improved overall NPP-based production in yeast by 11 and 9 times, respectively (Fig. <a href="#Fig5">5c</a>, right panel and Fig. <a href="#Fig5">5e</a>). Finally, in <i>Pt</i>PinS, four out of five variants of F607 switched specificity to NPP in the yeast system (Fig. <a href="#Fig5">5d</a>), nevertheless, with lower activity than the parental enzyme. Overall, these results revealed a conserved role for F571 in <i>Sf</i>CinS1, and the corresponding residues in the other synthases, in dictating substrate selectivity. The identification of such a key residue in the terpene synthase active site reveals fundamental mechanistic details of these enzymes and will be invaluable for future protein engineering efforts aiming to expand the synthetic pathway developed here with additional synthases. </p> </div> <div class="improvingtitersandenablingdownstreamoxidationsteps" title="sec"> <div class="title" tagx="title" title="title">
Improving titers and enabling downstream oxidation steps</div> <p>Using the NPP-specific synthases developed here, we were able to harvest the orthogonal pathway established to produce monoterpenes with considerably higher efficiency than only with the GPP-dependent pathway. In shake-flask batch cultivation of strain MIC2 expressing the Erg20p(N127W)- <i>Sl</i>NPPS1 fusion, the <i>Cl</i>LimS(H570Y)-supported limonene titer reached 130 mg L <sup>−1</sup>, while <i>Sp</i>SabS(H561F) enabled the production of 72.7 mg L <sup>−1</sup> sabinene (Supplementary Table <a href="#MOESM1">8</a>). Furthermore, the engineered synthases were quite efficient in harvesting NPP, leading to a significant reduction in nerol production from excess NPP (Supplementary Table <a href="#MOESM1">8</a>). Engineering efficient monoterpene scaffold production in yeast is of particular interest because yeast cells offer an advantage over prokaryotic hosts for the functional expression of cytochrome P450 enzymes, which are required for the synthesis of more complex monoterpenoids. So far, low monoterpene titers have hindered efficient P450 catalyzed oxidation of monoterpenes by a yeast cell factory. We evaluated whether the overall improvements obtained here were sufficient to enable downstream oxidation steps by introducing the sabinene hydroxylase CYP750B1 <sup> <a href="#CR45">45</a> </sup> from <i>T. plicata</i>, together with a compatible cytochrome P450 oxidoreductase <sup> <a href="#CR38">38</a>, <a href="#CR46">46</a> </sup>, to establish production of <i>trans</i>-sabin-3-ol. While production of <i>trans</i>-sabin-3-ol was not detectable when only the GPP pathway was used, we were able to synthesize detectable amounts of <i>trans</i>-sabin-3-ol in the NPP-producing MIC2 strain using wild-type <i>Sp</i>SabS. The levels of <i>trans</i>-sabin-3-ol increased three times when the NPP-specific <i>Sp</i>SabS(H561F) variant was used instead of <i>Sp</i>SabS (Fig. <a href="#Fig5">5f</a>; Supplementary Fig. <a href="#MOESM1">14</a>). </p> </div> <div class="dynamicregulationoftheorthogonalpathway" title="sec"> <div class="title" tagx="title" title="title">
Dynamic regulation of the orthogonal pathway</div> <p>The metabolic valve created at the point of DMAPP and IPP condensation can be dynamically controlled to divert fluxes towards NPP in response to internal metabolite levels. A key metabolite in our system is ergosterol, the main yeast sterol. To minimize flux to the sterol branch when adequate levels of ergosterol were synthesized, we implemented a dynamic design based on the promoter of the <i>ERG1</i> gene. Expression of <i>ERG1</i> is negatively regulated by ergosterol via an ergosterol-responsive regulatory element in its promoter (P <sub> <i>ERG1</i> </sub>) <sup> <a href="#CR47">47</a> </sup>. A similar design was previously exploited to improve production of the sesquiterpene amorpha-4,11-diene by controlling <i>ERG9</i> <sup> <a href="#CR48">48</a> </sup>, and of the diterpene casbene by controlling both the <i>ERG9</i> and <i>ERG20</i> genes <sup> <a href="#CR49">49</a> </sup>. To dynamically regulate FPP synthesis in response to ergosterol levels in our system, we replaced the promoter of <i>ERG20</i> in strain MIC2 with an 807 bp region of P <sub> <i>ERG1</i> </sub> that contains the ergosterol-responsive element to give rise to strain MIC3. Using the engineered NPP-specific synthases and strain MIC3, we were able to produce monoterpenes with considerably higher efficiency than with the native GPP-based pathway. In semi-batch shake-flask cultivation, <i>Cl</i>LimS(H570Y)-supported limonene titer in MIC3 strain reached 166 mg L <sup>−1</sup>, which is 6.8-times higher than the titer obtained in GPP-only producing MIC2 cells with wild-type <i>Cl</i>LimS (Fig. <a href="#Fig5">5e</a> and Table <a href="#Tab2">2</a>). Similarly, with <i>Sp</i>SabS(H561F), we obtained 113 mg L <sup>−1</sup> sabinene under dynamic regulation (MIC3), which was 7.1 times higher than the highest titer obtained in MIC2 by the native pathway and <i>Sp</i>SabS (Fig. <a href="#Fig5">5e</a> and Table <a href="#Tab3">3</a>). <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 2</span>  </div></p><p>Production of limonene in semi-batch conditions</p>  <table> <thead> <tr> <th>Protein expressed</th> <th>Limonene (mg L <sup>−1</sup>) in MIC2 </th> <th>Nerol (mg L <sup>−1</sup>) in MIC2 </th> <th>Limonene (mg L <sup>−1</sup>) in MIC3 </th> <th>Nerol (mg L <sup>−1</sup>) in MIC3 </th> </tr> </thead> <tbody> <tr> <td>Erg20p(N127W) +  <i>Cl</i>LimS wt </td> <td>23.31 ± 5.97</td> <td>–</td> <td>41.49 ± 3.95</td> <td>–</td> </tr> <tr> <td>Erg20p(N127W) +  <i>Cl</i>LimS(H570Y) </td> <td>27.51 ± 5.04</td> <td>–</td> <td>86.30 ± 10.76</td> <td>–</td> </tr> <tr> <td>Erg20p(N127W)- <i>Sl</i>NPPS1 +  <i>Cl</i>LimS wt </td> <td>62.47 ± 15.32</td> <td>0.542</td> <td>71.84 ± 13.98</td> <td>0.042</td> </tr> <tr> <td>Erg20p(N127W)- <i>Sl</i>NPPS1 +  <i>Cl</i>LimS(H570Y) </td> <td>126.19 ± 18.97</td> <td>0.023</td> <td>167.38 ± 20.88</td> <td>0.015</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>Errors correspond to the mean absolute deviation (MAD) around the mean ( <i>n</i>  <i>=</i> 3 biologically independent samples). Source data are provided in a Source Data file </p> </div>  <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 3</span>  <p>Production of sabinene in semi-batch conditions</p>  <table> <thead> <tr> <th>Protein expressed</th> <th>Sabinene (mg L <sup>−1</sup>) in MIC2 </th> <th>Nerol (mg L <sup>−1</sup>) in MIC2 </th> <th>Sabinene (mg L <sup>−1</sup>) in MIC3 </th> <th>Nerol (mg L <sup>−1</sup>) in MIC3 </th> </tr> </thead> <tbody> <tr> <td>Erg20p(N127W) +  <i>Sp</i>SabS wt </td> <td>23.31 ± 5.97</td> <td>–</td> <td>41.49 ± 3.95</td> <td>–</td> </tr> <tr> <td>Erg20p(N127W) +  <i>Sp</i>SabS(H561F) </td> <td>27.51 ± 5.04</td> <td>–</td> <td>86.30 ± 10.76</td> <td>–</td> </tr> <tr> <td>Erg20p(N127W)- <i>Sl</i>NPPS1 +  <i>Sp</i>SabS wt </td> <td>62.47 ± 15.32</td> <td>0.542</td> <td>71.84 ± 13.98</td> <td>0.042</td> </tr> <tr> <td>Erg20p(N127W)- <i>Sl</i>NPPS1 +  <i>Sp</i>SabS(H561F) </td> <td>126.19 ± 18.97</td> <td>0.023</td> <td>167.38 ± 20.88</td> <td>0.015</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>Errors correspond to the mean absolute deviation (MAD) around the mean ( <i>n</i>  <i>=</i> 3 biologically independent samples). Source data are provided in a Source Data file </p> </div> </div> <p /> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p>We established a synthetic pathway for monoterpenoid synthesis based on an alternative terpene synthase substrate and engineered terpene synthases that preferentially utilize the isomeric substrate. This orthogonal pathway was more efficient than the native GPP-based pathway. Using kinetic studies and competition experiments with <sup>13</sup>C-labelled substrates, we confirmed that the basis for the improved performance of the NPP-based pathway is that NPP, unlike GPP, is not consumed by Erg20p to produce sterols. Construction of the orthogonal pathway established a metabolic valve at the point of DMAPP and IPP condensation, enabling its dynamic control <sup> <a href="#CR4">4</a>– <a href="#CR8">8</a>, <a href="#CR47">47</a> </sup>. In future efforts, additional titer improvements can be achieved by incorporating more elaborate control mechanisms, such as engineered degradation of Erg20p <sup> <a href="#CR23">23</a> </sup> or metabolic modeling to identify additional targets for dynamic regulation <sup> <a href="#CR9">9</a> </sup>. This approach has wider implications for the engineering of other high-value compounds where isomers can provide orthogonality, providing a blueprint for applying synthetic pathway design <sup> <a href="#CR50">50</a> </sup> and orthogonality considerations <sup> <a href="#CR3">3</a> </sup> in the engineering of biological production systems. </p> <p>Until now, efforts in monoterpene bioproduction have been more efficient when using bacterial hosts <sup> <a href="#CR51">51</a>– <a href="#CR54">54</a> </sup>. However, engineering the downstream steps of monoterpenoid biosynthesis in a bacterial chassis is challenging because several of these steps require the functional expression of cytochrome P450 enzymes, which is not readily achieved in bacteria <sup> <a href="#CR55">55</a> </sup>. In addition, several monoterpenoids exhibit significant toxicity to bacteria by interfering with the cell membrane, compromising its integrity and impacting the function of membrane-bound enzymes <sup> <a href="#CR13">13</a>, <a href="#CR56">56</a>– <a href="#CR58">58</a> </sup>. Furthermore, accumulation of terpenoid pathway precursors, such as IPP, has been found to have a negative impact on bacterial growth <sup> <a href="#CR59">59</a> </sup>. For these reasons, engineering efficient production of monoterpene scaffolds in yeast can be beneficial. This is highlighted here by the markedly improved production of <i>trans</i>-sabin-3-ol using the orthogonal pathway. </p> <p>These findings also make an important contribution to our understanding of the enzymatic mechanism of monoterpene synthases. The identification of a single residue with a consistent function in dictating isomeric substrate selectivity in the active site of all monoterpene synthases tested points to an evolutionarily conserved role and provide important knowledge for the engineering of NPP-specific monoterpene synthases. This, in turn, will help expand the platform presented here with additional highly efficient synthases for numerous other monoterpene scaffolds. Furthermore, the residue corresponding to <i>Sf</i>CinS1(F571) appears to play a multifaceted role in monoterpene synthases, as it has also been found to enable enzymes to differentiate between GPP and 2-methyl-GPP <sup> <a href="#CR39">39</a> </sup>. </p> <p>The vast majority of known terpenoids are synthesized from substrates with <i>a trans</i> configuration. Although the <i>cis</i> forms of the universal canonical diterpene precursors Z,Z-FPP and nerylneryl diphosphate (NNPP) have been reported in some organisms, very few <i>cis</i>-prenyl diphosphate-derived compounds have been reported so far <sup> <a href="#CR28">28</a>, <a href="#CR60">60</a>– <a href="#CR63">63</a> </sup>. It is unclear what has led to this stereochemical specialization through evolution, but further expansion of the platform developed here to produce <i>Z,Z</i>-FPP and NNPP combined with protein engineering of canonical terpene synthases, could provide access to sesquiterpene and diterpenoid compounds that are rare or not found in nature. </p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="chemicalsandenzymes" title="sec"> <div class="title" tagx="title" title="title">
Chemicals and enzymes</div> <p>Standards used include <i>cis</i>-3,7-dimethyl-2,6-octadien-1-ol (Aldrich, 268909-5ML), α-pinene (Aldrich, P-7408), β-myrcene (Sigma, M-0382), 1,8-cineole (Aldrich, C8,060-1), γ-terpinene (Aldrich, T2134), limonene (Sigma-Aldrich, 62118), camphene (Sigma-Aldrich, 456055), alpha-phellandrene (Sigma-Aldrich, W285611), 2-carene (Sigma-Aldrich, 232386), beta-phellandrene (Carbosynth, FB158830), linalool (Aldrich, L2602), and a 70% sabinene solution (kindly donated by VIORYL S.A., Athens, Greece). Phusion High-Fidelity DNA Polymerase (New England BioLabs, M0530S) and MyTaq DNA polymerase (BIO-21105, Bioline) were used in PCR amplifications. The QIAquick Gel Extraction Kit (#28704, Qiagen) was used for gel extraction and DNA purification. The NucleoSpin Plasmid Kit (740588.250, Macherey-Nagel) was used for plasmid DNA purification. Restriction enzymes used were from New England BioLabs. </p> </div> <div class="yeastmedia" title="sec"> <div class="title" tagx="title" title="title">
Yeast media</div> <p>D-(+)-glucose monohydrate (16301, Sigma); D-(+)-galactose (MG05201, CarboSynth); D(+)-raffinose pentahydrate (OR06197, CarboSynth); Yeast Nitrogen Base w/o AA (Y2025, US Biologicals); Complete Minimal (CM) medium was composed of 0.13% (w/v) dropout powder (all essential amino acids), 0.67% (w/v) yeast nitrogen base w/o AA, 2% glucose; For galactose- based medium, glucose was substituted with 2% galactose and 1% raffinose. </p> </div> <div class="constructionofmic3yeaststrain" title="sec"> <div class="title" tagx="title" title="title">
Construction of MIC3 yeast strain</div> <p>The promoter region of <i>ERG1</i> was amplified from genomic DNA of BY4741 yeast strain using primers SalI-Perg1-FP and Perg1-HindIII-RP to introduce the restriction sites <i>Sal</i>I at the 5′-end and <i>Hind</i>III at the 3′-end. The PCR product was inserted into a pCR2.1-TOPO vector by TOPO TA Cloning (450641, Thermo Fisher Scientific) and confirmed by sequencing. Subsequently, the pTOPO-P <sub>ERG1</sub> plasmid was digested with <i>Sal</i>I/ <i>Hind</i>III, and the DNA fragment corresponding to the <i>ERG1</i> promoter was gel purified and inserted into pCOD7 <sup> <a href="#CR24">24</a> </sup> linearized with <i>Sal</i>I/ <i>Hind</i>III to generate plasmid COD71 (P <sub>ERG1</sub>-CYC1t, LoxP-HIS5-LoxP). The COD71/P <sub>ERG1</sub> cassette was PCR amplified using primers COD71-ERG20-FP and COD71-ERG20-RP, adding flanking regions complementary to the 5′ and 3′ end the native promoter of <i>ERG20</i> gene. Following transformation into strain MIC2, correct integration was confirmed by PCR using ERG20-gseq-FP and ERG20-gseq-RP. </p> </div> <div class="genecloningandexpressioninyeast" title="sec"> <div class="title" tagx="title" title="title">
Gene cloning and expression in yeast</div> <p>Supplementary Table <a href="#MOESM1">7</a> contains all primers used in cloning procedures described below. </p> <p>For cloning of <i>Sl</i>NPPS1, the mature form of <i>Sl</i>NPPS1 lacking the presumed N-terminal transit sequence (1–44 a.a.) (NM_001247704), was amplified by PCR using primers SlNPPS1-S45BamHI and SlNPPS1-3XhoI. The amplicon was cloned into a bacterial plasmid using the TOPO TA kit (Invitrogen) and validated by Sanger sequencing. <i>Sl</i>NPPS1 was subcloned into the yeast vector pHTDH3myc and in the bacterial vector pRSETa using the <i>Bam</i>HI and <i>Xho</i>I sites. </p> <p>Fusion of the <i>Sl</i>NPPS1 with ERG20(N127W) was initiated by digestion of pYESmyc/ERG20(N127W)-5XGS <sup> <a href="#CR14">14</a> </sup> with <i>Eco</i>RI and <i>Xho</i>I to generate a linearized fragment. The <i>Sl</i>NPPS1 gene was amplified with SlNPPS1-5EcoRI and SlNPPS1-3XhoI and introduced into pCRII-TOPO vector by TOPO cloning. Subsequently, the <i>Sl</i>NPPS1 fragment was released from the above generated construct through restriction digestion with <i>Eco</i>RI and <i>Xho</i>I. The obtained fragments of pYESmyc/ERG20(N127W)-GS and <i>Sl</i>NPPS1 were ligated to generate pYESmyc/ERG20(N127W)-5XGS- <i>Sl</i>NPPS1 construct. </p> <p>Cloning of the TPSs in compatible yeast expression proceeded through the following steps A yeast codon-optimized version of <i>Citrus limon</i> (+)-S-limonene synthase ( <i>Cl</i>LimS) (GenBank AF514287.1) was PCR amplified from pCEV-G2-Ph/ <i>Cl</i>LimS <sup> <a href="#CR64">64</a> </sup> using primers LimS-BGL and LimS-XHOstop to introduce <i>Bgl</i>II and <i>Xho</i>I restriction sites at the 5′ and 3′ ends, respectively. Initially a TOPO cloning reaction was performed to clone the PCR product into vector pCRII-TOPO. Subsequently, the pCRII-TOPO/ <i>Cl</i>LimS construct was digested using <i>Bgl</i>II and <i>Xho</i>I restriction enzymes and the <i>Cl</i>LimS fragment was excised and subcloned into vector pWTHD3myc <sup> <a href="#CR27">27</a> </sup> linearized with <i>Bam</i>HI and <i>Xho</i>I. In this construct, <i>Cl</i>LimS was expressed under the control of the P <sub> <i>TDH3</i> </sub> promoter. Yeast codon-optimized versions of <i>S. lycopersicum</i> phellandrene synthase ( <i>Sl</i>PHS1) (FJ797957.1) <sup> <a href="#CR28">28</a> </sup>, <i>S. habrochaites</i> limonene synthase ( <i>Sh</i>LimS) (AFJ67826.1) <sup> <a href="#CR30">30</a> </sup>, <i>S. habrochaites</i> pinene synthase ( <i>Sh</i>PinS) (AFJ67816.1) <sup> <a href="#CR30">30</a> </sup>, and <i>P. taeda</i> α-pinene synthase ( <i>Pt</i>PinS) (GenBank Q84KL3.1) <sup> <a href="#CR42">42</a> </sup>, lacking the transit peptide, were obtained by gene synthesis. In addition, the synthetic fragments bear flanking regions containing specific restriction sites and a generic sequence compatible for USER cloning. The above genes were amplified and cloned by USER cloning into a generic backbone using primers USER-Gen-FP and USER-Gen-RP. <i>Bam</i>HI site and <i>Sal</i>I sites were introduced at the 5′ and 3′ end, respectively. The constructs were confirmed by sequencing. Subsequently, the genes of interest were excised from the generic USER vector by <i>Bam</i>HI and <i>Sal</i>I digestion and ligated under the P <sub>GAL1</sub> promoter into the pESC-TRP vector (Agilent Technologies, Cat. #217453) restricted with <i>Bam</i>HI and <i>Sal</i>I enzymes. This approach resulted in the construction of the following plasmids: pESC-TRP/ <i>Sl</i>PHS1, pESC-TRP/ <i>Sh</i>LimS, pESC-TRP/ <i>Sh</i>PinS, pESC-TRP/ <i>Pt</i>PinS. Constructs of <i>S. fruticosa</i> 1,8-cineole synthase ( <i>Sf</i>CinS1) (GenBank ABH07677.1), <i>S. pomifera</i> sabinene synthase ( <i>Sp</i>SabS) (GenBank DQ785794.1) and <i>Se</i>CamS in appropriate yeast vectors were available from previous work <sup> <a href="#CR14">14</a>, <a href="#CR41">41</a>, <a href="#CR43">43</a>, <a href="#CR65">65</a> </sup>. </p> <p>A yeast codon-optimized version of <i>T. plicata</i> CYP750B1 (KP004988) <sup> <a href="#CR45">45</a> </sup> was obtained by gene synthesis, bearing flanking regions containing specific restriction sites and a generic sequence compatible for USER cloning. The corresponding DNA fragment was amplified and cloned into a generic backbone by USER technology using the primers: USER-Gen-FP and USER-Gen-RP (Supplementary Table <a href="#MOESM1">7</a>). Subsequently, the gene encoding CYP750B1 was excised from the generic USER vector by <i>Bam</i>HI and <i>Sal</i>I digestion and ligated under the P <sub> <i>GAL1</i> </sub> promoter into pESC-HIS vector (Agilent Technologies, Cat. #217453) linearized with <i>Bam</i>HI and <i>Sal</i>I enzymes. The generated plasmid, pESC-HIS/ <i>Tp</i>CYP750B1, was confirmed by sequencing. A construct (pYX143/ <i>Pt</i>CPR2) containing the gene encoding NADPH-cytochrome P450 reductase from <i>Populus trichocarpa (Pt</i>CPR2) <sup> <a href="#CR46">46</a> </sup> was already available <sup> <a href="#CR38">38</a> </sup>. </p> </div> <div class="expressioninyeastofcyp750b1" title="sec"> <div class="title" tagx="title" title="title">
Expression in yeast of CYP750B1</div> <p>Subsequently, MIC2 yeast cells were co-transformed with four expression vectors bearing different auxotrophic markers: pYES2myc for expression of Erg20p(N127W) or Erg20p(N127W)- <i>Sl</i>NPPS1 using uracil selection, pJG4-4 for expression of <i>Sp</i>SabS wt and mutants (H561F or H561Y) under tryptophan selection, pYX143 for expression of <i>Pt</i>CPR2 under leucine selection, and pESC-HIS for expression of <i>Tp</i>CYP750B1 using histidine selection. <i>Pt</i>CPR2 was expressed from pYX143 (P <sub>TPI1</sub>, <i>LEU2</i>, cen) to achieve low level protein expression because this was found to have a positive effect on P450 activity <sup> <a href="#CR38">38</a> </sup>. All other proteins (Erg20p, <i>Sl</i>NPPS1, <i>Sp</i>SabS, and <i>Tp</i>CYP750B1) were expressed under the strong inducible promoter P <sub> <i>GAL1</i> </sub> and from high copy number (2μ) vectors. Sabinene-producing transformants expressing <i>Tp</i>CYP750B1/ <i>Pt</i>CPR2 were cultured and analyzed by SPME using GC–MS. </p> </div> <div class="terpenequantificationandextractionfromyeastcells" title="sec"> <div class="title" tagx="title" title="title">
Terpene quantification and extraction from yeast cells</div> <p>Selected <i>S. cerevisiae</i> strains were grown overnight at 30 °C and 150 rpm and subsequently induced in 2-mL liquid media using 20-mL glass vials with magnetic screw cap (Mikrolab Aarhus A/S). Solid phase microextraction (SPME) was applied for measuring the volatile terpenes produced in yeast cells after 48 h of culturing, using a 2 cm-50/30 µm DVB/Carboxen™/PDMS StableFlex™ fiber followed by GC–MS analysis (described below) <sup> <a href="#CR65">65</a> </sup>. For quantification, terpene extraction was performed using 1% (w/v) Diaion HP20 (Supelco, Bellefonte, PA) as adsorbent resin or overlay of the yeast culture with 10% dodecane <sup> <a href="#CR14">14</a>, <a href="#CR66">66</a>, <a href="#CR67">67</a> </sup> and GC-FID analysis. Prior to addition to the yeast culture, the HP20 resin was activated in 100% methanol. Following culturing, the beads were collected and washed with H <sub>2</sub>O to remove the yeast cells. Subsequently, the beads were eluted three times with ethanol, followed by three pentane elutions including 20–30 min incubation between each step. The pentane phase was initially extracted with equal volume of water and then concentrated to a 100-μL final volume and analyzed. Samples obtained from three separate yeast transformations were processed and analyzed independently. </p> </div> <div class="fedbatchcultures" title="sec"> <div class="title" tagx="title" title="title">
Fed batch cultures</div> <p>Saturated overnight yeast cultures were used to inoculate 10 mL of glucose-based CM selective medium to an OD <sub>600</sub> = 0.1, which were incubated at 30 °C at 150 rpm shaking until an OD <sub>600</sub> = 1. The pellets were washed twice with sterile ddH <sub>2</sub>O and resuspended in 10 mL of 2× galactose/raffinose-based CM-selective medium. The cultures were overlaid with 2 mL of dodecane containing 1 ppm isopropyl myristate as internal standard, grown at 30 °C at 150 rpm shaking speed and fed with 1 mL of 20× galactose/raffinose-based CM selective media after 24, 48, 72, and 96 h. Cultures were stopped after 6 days of cultivation and the dodecane phase was collected and diluted 1:100 in hexane containing 1 ppm squalene internal standard. Following GC-FID analysis, samples were quantified by comparison of retention time and peak area to that of standard curves of limonene and sabinene authentic standard. </p> </div> <div class="enzymeexpressionandpurification" title="sec"> <div class="title" tagx="title" title="title">
Enzyme expression and purification</div> <p>Enzymatic activity of terpene synthases, <i>Sl</i>NPPS1 and Erg20p with GPP and NPP was evaluated with <i>E. coli</i>-produced and purified recombinant enzyme and with yeast cell extracts. Wild-type and mutant 6×His-tagged TPSs <sup> <a href="#CR39">39</a>, <a href="#CR41">41</a> </sup>, <i>Sl</i>NPPS1 <sup> <a href="#CR28">28</a> </sup>, and Erg20p <sup> <a href="#CR14">14</a>, <a href="#CR37">37</a> </sup> were purified by Ni <sup>2+</sup>-NTA affinity chromatography from 200 mL cultures of <i>E. coli</i> BL21 DE3 growing in LB media at 19 °C <sup> <a href="#CR14">14</a>, <a href="#CR37">37</a>, <a href="#CR39">39</a> </sup>. Protein expression was induced by addition of 0.1 mM IPTG the cells were grown overnight at 19 °C. Subsequently, protein purification was performed by affinity chromatography using Ni-NTA agarose resin (Qiagen). Briefly, cells were disrupted by sonication in lysis buffer (60 mM Tris, pH 8, 20 mM <i>β</i>-mercaptoethanol, 0.35 M NaCl, 10 mM imidazole, 10% (w/v) glycerol, 1 mM PMSF, 1 mg/mL lysozyme) <sup> <a href="#CR41">41</a> </sup>. Following removal of the cell debris via centrifugation 2400 ×  <i>g</i> and 4 °C, the protein lysate was incubated with Ni-NTA agarose resin for 1 h at 4 °C with mild shaking. The protein was washed twice and eluted from the resin using 350 mM imidazole. </p> </div> <div class="enzymaticassays" title="sec"> <div class="title" tagx="title" title="title">
Enzymatic assays</div> <p>Terpene synthase activity was assayed in a 0.5-mL reaction containing 20 mM MOPS (pH 7.0), 20 mM MgCl <sub>2</sub>, 0.02 mM MnCl <sub>2</sub>, 1 mM DTT, 0.1 mg mL <sup>−1</sup> BSA, 50 ng enzyme and varying concentrations (0–150 µM) of substrate (GPP or NPP). The reaction was overlaid with an equal volume of hexane containing 1 ppm nonane and 1 ppm dodecane as internal standards and incubated for 1 h at 30 °C. The hexane phase was recovered and 2 μL were analyzed by GC–MS <sup> <a href="#CR9">9</a>, <a href="#CR41">41</a> </sup>. Prenyl diphosphate synthase activity was assayed in a 0.2-mL reaction containing 10 mM MOPS (pH 7.0), 5 mM MgCl <sub>2</sub>, 1 mM DTT, 0.1 mg mL <sup>−1</sup> BSA, 0.02 mM IPP, and 50 ng of recombinant Erg20p. GPP or NPP substrates were added at 0.02 mM. Reactions were terminated by addition of an equal volume (0.2 mL) of 2 N HCl in 83% ethanol and 0.2 mL of hexane. Following a 20 min incubation at 37 °C to allow diphosphate hydrolysis, reactions were neutralized with 0.2 mL of 10% NaOH. A 2-μL aliquot of the hexane phase of an in vitro enzymatic assay was analyzed by GC–MS. <i>Sl</i>NPPS1 enzymatic activity was assayed in the same conditions as for Erg20p <sup> <a href="#CR11">11</a> </sup>, using 50 ng of purified enzyme, 0.05 mM of IPP or DMAPP, and varying concentrations (0–0.2 mM) of DMAPP or IPP, respectively. Reactions were incubated for 1 h at 30 °C and stopped by acid treatment. Kinetic analysis was performed using WinCurveFit 1.1.8 (Kevin Raner software). Yeast cells obtained from a 10-mL culture were resuspended in 10 mM MOPS (pH 7.0), 5 mM MgCl <sub>2</sub>, and 1 mM DTT and disrupted by glass beads. The resulted yeast extracts were used in enzymatic assays as described above for bacterial purified terpene synthases and prenyl diphosphate synthases. </p> </div> <div class="competitionassays" title="sec"> <div class="title" tagx="title" title="title">
Competition assays</div> <p>Yeast extracts obtained as described above from cells expressing <i>Sl</i>NPPS1, Erg20p, <i>Cl</i>LimS wild-type or <i>Cl</i>LimS(H570Y) were used in 0.5-mL reaction containing 20 mM MOPS, 20 mM MgCl <sub>2</sub>, 0.02 mM MnCl <sub>2</sub>, 1 mM DTT, 0.01% (v/v) BSA, and varying concentrations of IPP, DMAPP, GPP, NPP, <sup>13</sup>C-GPP, or <sup>13</sup>C-NPP. The reactions were overlaid with hexane containing 1 ppm nonane and 1 ppm dodecane as internal standards and incubated for 1 h at 30 °C. Parallel reactions for hexane extraction and acid treatment following hexane extraction were performed <sup> <a href="#CR14">14</a>, <a href="#CR37">37</a> </sup>. The hexane phase (2 μL) was analyzed by GC-FID for product quantification and by GC–qTOF for detection of the <sup>13</sup>C isotope incorporated in reaction products. </p> </div> <div class="yeastgrowthassay" title="sec"> <div class="title" tagx="title" title="title">
Yeast growth assay</div> <p>AM94 cells harboring the pTDH3myc/ <i>Sl</i>NPPS1 construct or the empty pTDH3myc vector were grown in glucose-based media at 30 °C and 150 rpm for 20 h. Yeast growth was monitored by measuring the optical density at 600 nm at regular intervals. No difference in the growth curve was observed in yeast cells expressing <i>Sl</i>NPPS1 under the P <sub> <i>TDH3</i> </sub> constitutive promoter in comparison to cells carrying the empty vector. </p> </div> <div class="toxicityassays" title="sec"> <div class="title" tagx="title" title="title">
Toxicity assays</div> <p>Two different assays were considered to evaluate the limitation imposed on monoterpene production in yeast cells due to possible toxicity exhibited by these molecules to <i>S. cerevisiae</i>. The selected monoterpenes were limonene and sabinene. To assess the effect of monoterpene on yeast growth, AM94 yeast cells were used to inoculate 20 mL media to OD <sub>600</sub> ~0.05 (considered time 0). The cultures were overlaid with dodecane treated with varying concentrations of limonene or sabinene, respectively, to a final concentration of 0, 30, 100, and 500 mg L <sup>−1</sup> and incubated at 30 °C with 150 rpm shaking. The yeast cellular growth was monitored by OD <sub>600</sub> measurements taken every 2 h over a period of time of 34 h. For OD <sub>600</sub> values higher than 1, samples were diluted accordingly. </p> <p>To evaluate the effect of limonene accumulation, yeast cells expressing <i>Cl</i>LimS (100 µL) were used to inoculate 10 mL of galactose/raffinose-based CM-selective media overlaid with 10% dodecane containing varying concentration of limonene (0 mg L <sup>−1</sup>, 300 mg L <sup>−1</sup>, and 1 g L <sup>−1</sup>) and incubated for 48 h at 30 °C with 150 rpm shaking. Subsequently, the dodecane phase was recovered as described above and analyzed by GC-FID. The limonene exceeding the starting concentration was considered to be synthesized by yeast cells. </p> </div> <div class="gc–msanalysis" title="sec"> <div class="title" tagx="title" title="title">
GC–MS analysis</div> <p>GC–MS analysis was carried out using a ZB-5ms column and helium as a carrier gas with a constant velocity of 37 cm/s. Samples resulting from incubation of the SPME fiber for 30 min over the head space of the yeast cultures were analyzed using the following temperature program: initial temperature 60 °C, ramp to 135 °C with a rate of 3 °C min <sup>−1</sup>, ramp to 240 °C with a rate of 20 °C min <sup>−1</sup>, and hold for 5 min. To detect terpenols derived by the acid treatment of prenyl diphosphates, the temperature program of GC–MS analysis was modified as follows: initial temperature 60 °C, hold for 3 min, ramp to 80 °C with a rate of 3 °C min <sup>−1</sup>, ramp to 110 °C with a rate of 30 °C min <sup>−1</sup>, ramp to 130 °C with a rate of 3 °C min <sup>−1</sup>, ramp to 280 °C with a rate of 30 °C min <sup>−1</sup> and hold for 3 min. </p> </div> <div class="gc-fidanalysis" title="sec"> <div class="title" tagx="title" title="title">
GC-FID analysis</div> <p>GC-FID was carried on a HP-5MS UI capillary column (Agilent Technologies) with a (5%-Phenyl)-methylpolysiloxane (Ultra Inert) stationary phase. Column dimensions were 30 m length × 0.25 mm internal diameter × 0.25 film thickness. Helium was used as a carrier gas with a constant velocity of 50 cm/s. The instrument was equipped with a PTV injector. Hexane samples were analyzed using the following temperature program: hold initial temperature at 40 °C for 3 min, ramp to 80 °C with a rate of 3 °C min <sup>−1</sup>, followed by a ramp to 300 °C with a rate of 30 °C min <sup>−1</sup>, and a hold for 10 min. </p> </div> <div class="gc–apci–qqtofanalysis" title="sec"> <div class="title" tagx="title" title="title">
GC–APCI–QqToF analysis</div> <p>GC–APCI–QqToF analyses were performed using a Scion 456 GC system (Bruker Daltonics) coupled to a micrOTOF-Q II (QqToF) mass spectrometer (Bruker Daltonics) via an atmospheric pressure chemical ionization (APCI) source. Samples were analyzed by injecting 1 µL aliquots in splitless injection mode (injector temperature 250 °C) onto a BR-5ms capillary column (30 m × 0.25 mm ID × 0.25 µm film thickness; Bruker Daltonics). The samples were eluted using helium as the carrier gas with a constant flow of 1.0 mL min <sup>−1</sup>. The temperature gradient of the GC oven was programmed as follows: initial temperature 50 °C was held for 1 min, before a ramp to 280 °C at a rate of 15 °C min <sup>−1</sup>, and subsequent hold for 3.67 min, giving a total method duration of 20 min. The transfer line between the GC and QqToF was maintained at 280 °C throughout the analysis. The head temperature of the APCI source was set to 280 °C, and the APCI heater to 200 °C, with a charging voltage of 2000 V. The QqToF mass spectrometer was operated in full scan positive ion mode with the following instrument settings: <i>m/z</i> 80–310: nebulizer gas (nitrogen), 3.5 bar; drying gas (nitrogen), 2.5 L min <sup>−1</sup>; drying gas temperature, 240 °C; capillary voltage, 3000 V; spectra acquisition rate, 5 Hz. Every chromatogram was calibrated for accurate mass using automated infusion of perfluorotributylamine vapor (PFTBA; Sigma-Aldrich) for a short period at the beginning of each chromatographic run. All data acquisition was automated using a combination of Compass CDS (Version 3.0.1; Bruker Daltonics), Compass oTOF Control (Version 3.4, Bruker Daltonics) and Hystar (Version 3.2 SR4, Bruker Daltonics) software. For data analysis, Compass DataAnalysis software (Version 4.3, Bruker Daltonics) was used. </p> </div> <div class="calculationoflabeled:unlabeledlimoneneratio" title="sec"> <div class="title" tagx="title" title="title">
Calculation of labeled:unlabeled limonene ratio</div> <p>The relative contribution of GPP and NPP to limonene synthesis was estimated in competition assays where either GPP or NPP was labeled with two <sup>13</sup>C, causing a +2 mass shift in the proportion of the limonene mass spectra stemming from the labeled substrate. The limonene mass spectra contained a large amount of −2H ions for each major ion, causing the <sup>13</sup>C-labeled limonene to have significant overlap with the major ions of unlabeled limonene. The ratio between unlabeled and labeled limonene could, therefore, not be accurately calculated directly from the + 2 shift in major mass ions. It was instead estimated by comparing four ion pairs of unlabeled −2H and labeled +2 <sup>13</sup>C of <i>m/z</i> 135/139, 166/170, 191/195 and 205/209. The ratios for each pair were calculated based on the known relative ratio between signal intensities of each ion pair in the mass spectra between pure unlabeled and labeled limonene at equal concentrations. The weighted average and standard errors were calculated using the signal to noise for each pair as weighting. Details with exact numbers and calculations are provided in the Data Source file (sheets named Supplementary Fig. <a href="#MOESM1">12a</a> and Supplementary Fig. <a href="#MOESM1">12b</a>). </p> </div> <div class="moleculargraphics" title="sec"> <div class="title" tagx="title" title="title">
Molecular graphics</div> <p>Molecular graphics were produced using the UCSF Chimera package. Chimera is developed by the Resource for Biocomputing, Visualization, and Informatics at the University of California, San Francisco (supported by NIGMS P41-GM103311) <sup> <a href="#CR68">68</a> </sup>. </p> </div> <div class="compoundidentification" title="sec"> <div class="title" tagx="title" title="title">
Compound identification</div> <p>Identification of monoterpene compounds produced by yeast cells was based on GC–MS analysis and comparison of retention times and mass spectra with authentic standards (listed earlier in this section under Chemicals and enzymes). </p> </div> <div class="mutagenesis" title="sec"> <div class="title" tagx="title" title="title">
Mutagenesis</div> <p>Site-directed mutagenesis using the Quickchange method (Agilent) was performed for <i>Sf</i>CinS1, <i>Sp</i>SabS, and <i>Sel</i>CamS, while USER mutagenesis <sup> <a href="#CR69">69</a> </sup> was performed for <i>Pt</i>PinS and <i>Cl</i>LimS. The primers used for mutagenesis purposes are listed in Supplementary Table <a href="#MOESM1">7</a>. </p> </div> <div class="constructionofthesfcinsmodel" title="sec"> <div class="title" tagx="title" title="title">
Construction of the <i>Sf</i>CinS model </div> <p>A structural model of <i>Sf</i>CinS1 was constructed using the SWISS-MODEL server <sup> <a href="#CR70">70</a> </sup> using mint limonene synthase (pdb id: 2ong) as template. The 2-fluorolinalyl diphosphate and 2-fluorogeranyl diphosphate ligands were then superimposed on the model, based on their position in the mint limonene synthase structure. </p> </div> <div class="generalchemicalsynthesisprocedures" title="sec"> <div class="title" tagx="title" title="title">
General chemical synthesis procedures</div> <p>All reactions were monitored by TLC on aluminum sheets coated with silica gel 60F254 (0.2-mm thickness, Merck) and the components present were detected by charring with 10% H <sub>2</sub>SO <sub>4</sub> in MeOH. Column chromatography was carried out using silica gel 60 (particle size 0.040–0.063 mm, 230–400 mesh ASTM, Merck). Solvent extracts were dried with anhydrous MgSO <sub>4</sub> unless otherwise specified. The <sup>1</sup>H and <sup>13</sup>C NMR spectra were recorded on a Bruker Avance 400 spectrometer at 400 and 101 MHz, respectively. CDCl <sub>3</sub> was used as solvent (unless otherwise indicated), δH values are relative to internal TMS and δC values are referenced to the solvent [δC (CDCl <sub>3</sub>) = 77.0]. </p> </div> <div class="synthesisof13c-labeledneryl-andgpp" title="sec"> <div class="title" tagx="title" title="title">
Synthesis of <sup>13</sup>C-labeled neryl- and GPP </div> <p>Chemical synthesis of (Z)-[1,2- <sup>13</sup>C <sub>2</sub>]-neryl pyrophosphate <b>5a</b> and its isomer (E)-[1,2- <sup>13</sup>C <sub>2</sub>]-geranyl pyrophosphate <b>5b</b> was performed according to the following procedure: </p> <p>To synthesize ethyl ( <i>Z</i>)-[1,2- <sup>13</sup>C <sub>2</sub>]-3,7-dimethylocta-2,6-dienoate ( <b>3a</b>) and ethyl ( <i>E</i>)-[1,2- <sup>13</sup>C <sub>2</sub>]-3,7-dimethylocta-2,6-dienoate ( <b>3b</b>), a neat mixture of LiCl (0.25 g, 5.9 mmol), triethyl phosphono- <sup>13</sup>C <sub>2-</sub>acetate ( <b>1</b>) (1.0 g, 4.4 mmol) and DBU (1,8-diazabicyclo[5,4,0]undec-7-ene) (0.72 mL, 4.9 mmol) was stirred at rt for 1.0 h under Ar followed by the addition of 6-methyl-5-hepten-2-one ( <b>2</b>) (2.3 mL, 15.6 mmol). Stirring was continued at rt for 24 h. The reaction was quenched with water (50 mL) and the reaction mixture was extracted with Et <sub>2</sub>O (200 mL). The extract was washed with brine and dried (MgSO <sub>4</sub>). The solvent was evaporated and chromatographed on silica (100 g) with 0–10% Et <sub>2</sub>O in n-pentane to afford the separation of the pure ( <i>Z</i>)-isomer <b>3a</b> as colorless oil (87.0 mg, 10%), <i>E</i>/ <i>Z</i>-mixture (560.0 mg, 68%) and pure ( <i>E</i>)-isomer <b>3b</b> (175.0 mg, 20%) (829.0 mg in total, 95%). </p> <p>To synthesize ( <i>Z</i>)-[1,2- <sup>13</sup>C <sub>2</sub>]-3,7-dimethylocta-2,6-diene-1-ol ( <b>4a</b>), a stirred solution of ethyl ( <i>Z</i>)-[1,2- <sup>13</sup>C <sub>2</sub>]-3,7-dimethylocta-2,6-dienoate ( <b>3a</b>) (60.0 mg, 0.30 mmol) in diethyl ether (5.0 mL) and cooled to −78 °C. DIBAL-H (1.0 M in toluene) (0.73 mL, 0.73 mmol) was added dropwise and the reaction mixture was stirred at −78 <b> <i>°</i> </b>C for 1 h and then quenched by addition of MeOH (0.5 mL). The reaction mixture was warmed gradually to 0 °C and diluted with Et <sub>2</sub>O (10 mL) followed by the addition of saturated aqueous solution of Rochelle salt (10 mL), water (10 mL) and vigorously stirred. Once the Et <sub>2</sub>O layer became clear, it was separated, dried over MgSO <sub>4</sub>, filtered, and concentrated on a rotary evaporator under reduced pressure. Chromatographic purification of the resulting oil on silica (14 g) with 20% Et <sub>2</sub>O in n-pentane afforded the separation of the pure <b>4a</b> colorless oil (45.0 mg, colorless oil, 96%). </p> <p>( <i>E</i>)-[1,2- <sup>13</sup>C <sub>2</sub>]-3,7-dimethylocta-2,6-diene-1-ol ( <b>4b</b>) was synthesized exactly as described for the alcohol <b>4a</b>. </p> <p>The structure of ( <i>Z</i>)- [1,2- <sup>13</sup>C <sub>2</sub>]-3,7-Dimethylocta-2,6-diene-1-ol ( <b>4a</b>) was confirmed by NMR and HRMS analysis. <sup>1</sup>H NMR (600 MHz, CDCl <sub>3</sub>): δ1.61 (s, 3H), 1.69 (d, <sup>4</sup> <i>J</i> <sub>H,H</sub>  <i>=</i> 0.9 Hz,3H), 1.75 (dd, <sup>3</sup> <i>J</i> <sub>CH3,C-1</sub>  <i>=</i> 6.0 Hz, <sup>4</sup> <i>J</i> <sub>H,H</sub>  <i>=</i> 0.9 Hz, 3H), 2.10 (m, 4H,–C <i>H</i> <sub> <i>2</i> </sub>C <i>H</i> <sub> <i>2</i> </sub>–), 3.42 (d, <sup>2</sup> <i>J</i> <sub>OH,C-1</sub> = 6.5 Hz,1H, OH), 4.09 (dddd, <sup>1</sup> <i>J</i> <sub>H-1,C-1</sub>  <i> =</i> 141.9 Hz, <sup>2</sup> <i>J</i> <sub>H-1,H-1</sub>  <i>=</i> 12.3 Hz, <sup>3</sup> <i>J</i> <sub>H-1,H-2</sub>  <i>=</i> 7.2 Hz, <sup>2</sup> <i>J</i> <sub>H-1,C-2</sub>  <i>=</i> 4.0 Hz, <sup>4</sup> <i>J</i> <sub>H-1,H-4</sub>  <i>=</i> 0.8 Hz, 2H, – <sup>13</sup>C <i>H</i> <sub> <i>2</i> </sub>–OH), 5.10 (m, 1H, H-6), 5.45 (dtm, <sup>1</sup> <i>J</i> <sub>H-2,C-2</sub>  <i>=</i> 152.4 Hz, <sup>3</sup> <i>J</i>  <i>=</i> 7.2 Hz, 1H); <sup>13</sup>C NMR (151 MHz, CDCl <sub>3</sub>): δ 59.2 (d, <sup>1</sup> <i>J</i> <sub>C–C</sub> = 47.5 Hz, <sup>13</sup>C-1), 124.4 (d, <sup>1</sup> <i>J</i> <sub>C–C</sub> = 47.5 Hz, <sup>13</sup>C-−2), (only peaks for <sup>13</sup>C-labeled carbons reported). HRMS (m/z): [M + H–H <sub>2</sub>0] + calcd. For <sup>13</sup>C <sub>2</sub>C <sub>8</sub>H <sub>18</sub>O, 139.1397; found, 139.1392. </p> <p>The structure of ( <i>E</i>)-[1,2- <sup>13</sup>C <sub>2</sub>]-3,7-Dimethylocta-2,6-diene-1-ol ( <b>4b</b>) was confirmed by NMR and HRMS analysis. H NMR (600 MHz, CDCl <sub>3</sub>): δ1.61 (s, 3H), 1.68 (s, 3H), 1.75 (s, 3H), 2.03 and 2.11 (2m, 4H,–C <i>H</i> <sub> <i>2</i> </sub>C <i>H</i> <sub> <i>2</i> </sub>–), 3.41 (d, <sup>2</sup> <i>J</i> <sub>OH,C−1</sub> = 6.5 Hz,1H, OH), 4.16 (dddd, <sup>1</sup> <i>J</i> <sub>H-1,C-1</sub>  <i>=</i> 141.8 Hz, <sup>2</sup> <i>J</i> <sub>H-1,H-1</sub>  <i>=</i> 11.5 Hz, <sup>3</sup> <i>J</i> <sub>H-1,H-2</sub>  <i>=</i> 7.0 Hz, <sup>2</sup> <i>J</i> <sub>H-1,C-2</sub>  <i>=</i> 4.0 Hz, <sup>4</sup> <i>J</i> <sub>H-1,H-4</sub>  <i>=</i> 0.6 Hz, 2 H, – <sup>13</sup>C <i>H</i> <sub> <i>2</i> </sub>–OH), 5.10 (tm, <i>J</i>  <i>=</i> 7.0 Hz,1H, H−6), 5.45 (dtm, <sup>1</sup> <i>J</i> <sub>H−2,C−2</sub>  <i>=</i> 152.4 Hz, <sup>3</sup> <i>J</i>  <i>=</i> 7.0 Hz, 1H). <sup>13</sup>C NMR (151 MHz, CDCl <sub>3</sub>): δ59.6 (d, <sup>1</sup> <i>J</i> <sub>C–C</sub> = 47.5 Hz, <sup>13</sup>C-1), 124.4 (d, <sup>1</sup> <i>J</i> <sub>C–C</sub> = 47.5 Hz, <sup>13</sup>C-2), (only peaks for <sup>13</sup>C-labeled carbons reported). HRMS ( <i>m</i>/ <i>z</i>): [M + H–H <sub>2</sub>0] + calcd. For <sup>13</sup>C <sub>2</sub>C <sub>8</sub>H <sub>18</sub>O, 139.1397; found, 139.1387. </p> <p>To synthesize ( <i>Z</i>)-[1,2- <sup>13</sup>C <sub>2</sub>]-neryl pyrophosphate ( <b>5a)</b>, a stirred solution of (Z)-[1,2- <sup>13</sup>C <sub>2</sub>]-3,7-dimethylocta-2,6-diene-1-ol ( <b>4a</b>) (15.6 mg, 0.10 mmol) in dry benzene (2.0 mL) and dry CH <sub>2</sub>Cl <sub>2</sub> (0.2 mL) was added CBr <sub>4</sub> (68.0 mg, 0.21 mmol) and Ph <sub>3</sub>P (54.0 mg, 0.21 mmol) at 0 °C under a nitrogen atmosphere. After the solution was stirred at the same temperature for 2 h, n-pentane (2.0 mL) was added to the reaction mixture. The reaction mixture was filtered to remove triphenylphosphine oxide. The filtrate was concentrated under reduced pressure to afford the crude ( <i>Z</i>)-neranyl bromide (22.0 mg, quant.) as yellow oil which was immediately subjected to the following pyrophosphorylation reaction without further purification. A solution of the crude bromide in dry CH <sub>3</sub>CN (2 mL) was added dropwise to a stirred solution of [(n-Bu <sub>4</sub>)N] <sub>3</sub>HP <sub>2</sub>O <sub>7</sub> (135.0 mg, 0.15 mmol, 1.5 eq) in dry CH <sub>3</sub>CN (2.0 mL) at 0 °C under Ar. The mixture was stirred at rt for 24 h and concentrated on a rotary evaporator below 35 °C. The obtained residue was transferred to a centrifuge tube with 5.0 mL of acetone and concentrated NH <sub>4</sub>OH (0.5–1 mL) was added. The precipitated ammonium salts, isolated by centrifugation (1500 ×  <i>g</i> for 10 min), washed twice by resuspension in 5.0 of acetone containing 0.01 N NH <sub>4</sub>OH. The combined supernatants of the acetone containing NH <sub>4</sub>OH were roto-evaporate at below 35 °C. The resulting crude ( <i>Z</i>)-[1,2- <sup>13</sup>C <sub>2</sub>]-neryl pyrophosphate <b>5a</b> as its ammonium salt was pure enough to be used for the enzymatic studies. However, a pure sample of <b>5a</b> was accomplished by preparative TLC-plates using the eluent system MeOH/CH <sub>2</sub>Cl <sub>2</sub>/0.01 N NH <sub>4</sub>OH (1: 1.2: 0.3 v/v). </p> <p>( <i>E</i>)-[1,2- <sup>13</sup>C <sub>2</sub>]-geranyl pyrophosphate <b>5b</b> was synthesized exactly as described for <b>5a</b>. When synthesized organic pyrophosphates <b>5a</b> and <b>5b</b> were subjected to acid hydrolysis, the recovered alcohol was compared with authentic samples by TLC and GC–HRMS. In each case, the recovered alcohol was essentially identical with the corresponding alcohols <b>4a</b> and <b>4b</b>, respectively. </p> </div> <div class="reportingsummary" title="sec"> <div class="title" tagx="title" title="title">
Reporting summary</div> <p>Further information on research design is available in the <a href="#MOESM3">Nature Research Reporting Summary</a> linked to this article. </p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary information</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"> <a href="41467_2019_11290_MOESM1_ESM.pdf">LINK</a>  <p>Supplementary Info</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="41467_2019_11290_MOESM2_ESM.pdf">LINK</a>  <p>Peer Review</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="41467_2019_11290_MOESM3_ESM.pdf">LINK</a>  <p>Reporting Summary</p>  </div> </div> <p /> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"> <a href="41467_2019_11290_MOESM4_ESM.xlsx">LINK</a>  <p>Source Data</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p> <b>Peer review information:</b> <i>Nature Communications</i> thanks Radhakrishnan Mahadevan, Annie Tsong and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. </p> </div> <div class="fn-type-" title=""> <p> <b>Publisher’s note:</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. </p> </div> </div> <div class="supplementaryinformation" title="sec"> <div class="title" tagx="title" title="title">
Supplementary information</div> <p> <b>Supplementary Information</b> accompanies this paper at 10.1038/s41467-019-11290-x. </p> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We thank Dr Violetta Aru (Chemometrics &amp;amp; Analytical Technology, Department of Food Science, University of Copenhagen, Denmark) for performing the NMR experiments, Dr David Ian Pattison (Metabolomics Platform, Department of Plant and Environmental Sciences, University of Copenhagen, Denmark) for technical support in metabolite analysis and description of the GC–APCI–QqToF parameters and method, Eleni Lazaridi and Stephanie Petersen for technical assistance with the GC instruments (Metabolomics Platform, Department of Plant and Environmental Sciences, University of Copenhagen, Denmark), and Karel Miettinen (University of Copenhagen, Denmark) for critical reading of the manuscript. This work was supported by the Novo Nordisk Foundation (grants NNF16OC0019554 to C.I. and NNF16OC0021760 to S.C.K.) and the General Secretariat of Science and Technology Hellas (GSRT; grant SysTerp 09ΣΥΝ-23-879 to AMM and S.C.K.). C.E.V. was supported by a Queensland Government Accelerate Fellowship. </p> </div> <div class="title" tagx="title" title="title">
Author contributions</div> <p>C.I. designed experiments, expressed and analyzed <i>Sl</i>NPPS1 and monoterpene synthases in yeast and bacteria, performed mutagenesis of <i>Sf</i>CinS1, <i>Pt</i>PinS, <i>Sp</i>SabS, <i>Cl</i>LimS, <i>Se</i>CamS, produced the integration cassette for P <sub> <i>ERG1</i> </sub> and generated the MIC3-derived strains, performed the enzymatic assays for precursor flux analysis and competition experiments, analyzed the yeast growth parameters for toxicity experiments, analyzed the data, and drafted the paper; M.H.R. performed mutagenesis experiments, quantified limonene and sabinene production in yeast under shake-flask batch and semi-batch cultivation conditions and carried out toxicity and productivity evaluations, conducted the GC–qTOF analysis and contributed to the cloning of P <sub> <i>ERG1</i> </sub> and the construction of the dynamically controlled MIC3 strain; M.S.M. performed the chemical synthesis of <sup>13</sup>C-GPP and <sup>13</sup>C-NPP. A.M.M. prepared the <i>Sl</i>NPPS1 expression construct and participated in data analysis; C.E.V. provided essential reagents and assisted in data interpretation and paper revision; S.C.K. conceived the project, designed experiments, analyzed the data, and drafted the paper. </p> <div class="title" tagx="title" title="title">
Data availability</div> <p>Data supporting the findings of this work are available within the paper and its Supplementary Information files. A reporting summary for this article is available as a Supplementary Information file. The data sets generated and analyzed during this study are available from the corresponding author upon request. The source data underlying Tables <a href="#MOESM4">1</a>– <a href="#MOESM4">3</a>; Figs. <a href="#MOESM4">2</a>c, d, <a href="#MOESM4">3</a>a–e, <a href="#MOESM4">4</a>a, <a href="#MOESM4">5</a>a–d, g; Supplementary Tables <a href="#MOESM4">2</a>– <a href="#MOESM4">6</a>, and <a href="#MOESM4">8</a>; Supplementary Figs. <a href="#MOESM4">1</a>, <a href="#MOESM4">2</a>, <a href="#MOESM4">3</a>a, b, <a href="#MOESM4">4</a>a, b, <a href="#MOESM4">6</a>, <a href="#MOESM4">7</a>, <a href="#MOESM4">8</a>a, b, <a href="#MOESM4">10</a>a, b, <a href="#MOESM4">12</a>a, b are provided as a Source Data file. </p> <div class="title" tagx="title" title="title">
Competing interests</div> <p>The authors declare no competing interests.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="CR1" /> <span class="label" tagx="label" title="label">1.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">CC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jewett</span> <span class="given-names" tagx="given-names" title="given-names">MC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chin</span> <span class="given-names" tagx="given-names" title="given-names">JW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Voigt</span> <span class="given-names" tagx="given-names" title="given-names">CA</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Toward an orthogonal central dogma</span> <span class="source" tagx="source" title="source">Nat. Chem. Biol.</span> <span class="year" tagx="year" title="year">2018</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">103</span> <span class="lpage" tagx="lpage" title="lpage">106</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nchembio.2554">10.1038/nchembio.2554</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29337969">29337969</a> </span> </span> </li> <li tag="ref"> <a name="CR2" /> <span class="label" tagx="label" title="label">2.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">de Lorenzo</span> <span class="given-names" tagx="given-names" title="given-names">V</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Beware of metaphors: chasses and orthogonality in synthetic biology</span> <span class="source" tagx="source" title="source">Bioeng. Bugs</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">2</span> <span class="fpage" tagx="fpage" title="fpage">3</span> <span class="lpage" tagx="lpage" title="lpage">7</span> <span class="pub-id"> <a href="https://dx.doi.org/10.4161/bbug.2.1.13388">10.4161/bbug.2.1.13388</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21636982">21636982</a> </span> </span> </li> <li tag="ref"> <a name="CR3" /> <span class="label" tagx="label" title="label">3.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pandit</span> <span class="given-names" tagx="given-names" title="given-names">AV</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Srinivasan</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mahadevan</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Redesigning metabolism based on orthogonality principles</span> <span class="source" tagx="source" title="source">Nat. Commun.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">15188</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/ncomms15188">10.1038/ncomms15188</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28555623">28555623</a> </span> </span> </li> <li tag="ref"> <a name="CR4" /> <span class="label" tagx="label" title="label">4.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tan</span> <span class="given-names" tagx="given-names" title="given-names">SZ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Prather</span> <span class="given-names" tagx="given-names" title="given-names">KL</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Dynamic pathway regulation: recent advances and methods of construction</span> <span class="source" tagx="source" title="source">Curr. Opin. Chem. Biol.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">41</span> <span class="fpage" tagx="fpage" title="fpage">28</span> <span class="lpage" tagx="lpage" title="lpage">35</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.cbpa.2017.10.004">10.1016/j.cbpa.2017.10.004</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29059607">29059607</a> </span> </span> </li> <li tag="ref"> <a name="CR5" /> <span class="label" tagx="label" title="label">5.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dahl</span> <span class="given-names" tagx="given-names" title="given-names">RH</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Engineering dynamic pathway regulation using stress-response promoters</span> <span class="source" tagx="source" title="source">Nat. Biotechnol.</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">31</span> <span class="fpage" tagx="fpage" title="fpage">1039</span> <span class="lpage" tagx="lpage" title="lpage">1046</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nbt.2689">10.1038/nbt.2689</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24142050">24142050</a> </span> </span> </li> <li tag="ref"> <a name="CR6" /> <span class="label" tagx="label" title="label">6.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gupta</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reizman</span> <span class="given-names" tagx="given-names" title="given-names">IM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reisch</span> <span class="given-names" tagx="given-names" title="given-names">CR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Prather</span> <span class="given-names" tagx="given-names" title="given-names">KL</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Dynamic regulation of metabolic flux in engineered bacteria using a pathway-independent quorum-sensing circuit </span> <span class="source" tagx="source" title="source">Nat. Biotechnol.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">35</span> <span class="fpage" tagx="fpage" title="fpage">273</span> <span class="lpage" tagx="lpage" title="lpage">279</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nbt.3796">10.1038/nbt.3796</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28191902">28191902</a> </span> </span> </li> <li tag="ref"> <a name="CR7" /> <span class="label" tagx="label" title="label">7.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stephanopoulos</span> <span class="given-names" tagx="given-names" title="given-names">G</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koffas</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Improving fatty acids production by engineering dynamic pathway regulation and metabolic control </span> <span class="source" tagx="source" title="source">Proc. Natl Acad. Sci. USA</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">111</span> <span class="fpage" tagx="fpage" title="fpage">11299</span> <span class="lpage" tagx="lpage" title="lpage">11304</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.1406401111">10.1073/pnas.1406401111</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25049420">25049420</a> </span> </span> </li> <li tag="ref"> <a name="CR8" /> <span class="label" tagx="label" title="label">8.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Doong</span> <span class="given-names" tagx="given-names" title="given-names">SJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gupta</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Prather</span> <span class="given-names" tagx="given-names" title="given-names">KLJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Layered dynamic regulation for improving metabolic pathway productivity in Escherichia coli </span> <span class="source" tagx="source" title="source">Proc. Natl Acad. Sci. USA</span> <span class="year" tagx="year" title="year">2018</span> <span class="volume" tagx="volume" title="volume">115</span> <span class="fpage" tagx="fpage" title="fpage">2964</span> <span class="lpage" tagx="lpage" title="lpage">2969</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.1716920115">10.1073/pnas.1716920115</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29507236">29507236</a> </span> </span> </li> <li tag="ref"> <a name="CR9" /> <span class="label" tagx="label" title="label">9.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Venayak</span> <span class="given-names" tagx="given-names" title="given-names">N</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">von Kamp</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Klamt</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mahadevan</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">MoVE identifies metabolic valves to switch between phenotypic states</span> <span class="source" tagx="source" title="source">Nat. Commun.</span> <span class="year" tagx="year" title="year">2018</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">5332</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/s41467-018-07719-4">10.1038/s41467-018-07719-4</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/30552335">30552335</a> </span> </span> </li> <li tag="ref"> <a name="CR10" /> <span class="label" tagx="label" title="label">10.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vickers</span> <span class="given-names" tagx="given-names" title="given-names">CE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Williams</span> <span class="given-names" tagx="given-names" title="given-names">TC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peng</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cherry</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Recent advances in synthetic biology for engineering isoprenoid production in yeast</span> <span class="source" tagx="source" title="source">Curr. Opin. Chem. Biol.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">40</span> <span class="fpage" tagx="fpage" title="fpage">47</span> <span class="lpage" tagx="lpage" title="lpage">56</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.cbpa.2017.05.017">10.1016/j.cbpa.2017.05.017</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28623722">28623722</a> </span> </span> </li> <li tag="ref"> <a name="CR11" /> <span class="label" tagx="label" title="label">11.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zebec</span> <span class="given-names" tagx="given-names" title="given-names">Z</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Towards synthesis of monoterpenes and derivatives using synthetic biology</span> <span class="source" tagx="source" title="source">Curr. Opin. Chem. Biol.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">34</span> <span class="fpage" tagx="fpage" title="fpage">37</span> <span class="lpage" tagx="lpage" title="lpage">43</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.cbpa.2016.06.002">10.1016/j.cbpa.2016.06.002</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27315341">27315341</a> </span> </span> </li> <li tag="ref"> <a name="CR12" /> <span class="label" tagx="label" title="label">12.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">George</span> <span class="given-names" tagx="given-names" title="given-names">KW</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alonso-Gutierrez</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Keasling</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">TS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Isoprenoid drugs, biofuels, and chemicals–artemisinin, farnesene, and beyond</span> <span class="source" tagx="source" title="source">Adv. Biochem. Eng. Biotechnol.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">148</span> <span class="fpage" tagx="fpage" title="fpage">355</span> <span class="lpage" tagx="lpage" title="lpage">389</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25577395">25577395</a> </span> </span> </li> <li tag="ref"> <a name="CR13" /> <span class="label" tagx="label" title="label">13.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jongedijk</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Biotechnological production of limonene in microorganisms</span> <span class="source" tagx="source" title="source">Appl. Microbiol. Biotechnol.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">100</span> <span class="fpage" tagx="fpage" title="fpage">2927</span> <span class="lpage" tagx="lpage" title="lpage">2938</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00253-016-7337-7">10.1007/s00253-016-7337-7</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26915992">26915992</a> </span> </span> </li> <li tag="ref"> <a name="CR14" /> <span class="label" tagx="label" title="label">14.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ignea</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pontini</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maffei</span> <span class="given-names" tagx="given-names" title="given-names">ME</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Makris</span> <span class="given-names" tagx="given-names" title="given-names">AM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kampranis</span> <span class="given-names" tagx="given-names" title="given-names">SC</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Engineering monoterpene production in yeast using a synthetic dominant negative geranyl diphosphate synthase </span> <span class="source" tagx="source" title="source">ACS Synth. Biol.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="fpage" tagx="fpage" title="fpage">298</span> <span class="lpage" tagx="lpage" title="lpage">306</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/sb400115e">10.1021/sb400115e</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24847684">24847684</a> </span> </span> </li> <li tag="ref"> <a name="CR15" /> <span class="label" tagx="label" title="label">15.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alonso-Gutierrez</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Metabolic engineering of <i>Escherichia coli</i> for limonene and perillyl alcohol production </span> <span class="source" tagx="source" title="source">Metab. Eng.</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="fpage" tagx="fpage" title="fpage">33</span> <span class="lpage" tagx="lpage" title="lpage">41</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ymben.2013.05.004">10.1016/j.ymben.2013.05.004</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23727191">23727191</a> </span> </span> </li> <li tag="ref"> <a name="CR16" /> <span class="label" tagx="label" title="label">16.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">L</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Chassis and key enzymes engineering for monoterpenes production</span> <span class="source" tagx="source" title="source">Biotechnol. Adv.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">35</span> <span class="fpage" tagx="fpage" title="fpage">1022</span> <span class="lpage" tagx="lpage" title="lpage">1031</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.biotechadv.2017.09.002">10.1016/j.biotechadv.2017.09.002</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28888552">28888552</a> </span> </span> </li> <li tag="ref"> <a name="CR17" /> <span class="label" tagx="label" title="label">17.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sarria</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wong</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garcia Martin</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Keasling</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peralta-Yahya</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Microbial synthesis of pinene</span> <span class="source" tagx="source" title="source">ACS Synth. Biol.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="fpage" tagx="fpage" title="fpage">466</span> <span class="lpage" tagx="lpage" title="lpage">475</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/sb4001382">10.1021/sb4001382</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24679043">24679043</a> </span> </span> </li> <li tag="ref"> <a name="CR18" /> <span class="label" tagx="label" title="label">18.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Croteau</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Biosynthesis and catabolism of monoterpenoids</span> <span class="source" tagx="source" title="source">Chem. Rev.</span> <span class="year" tagx="year" title="year">1987</span> <span class="volume" tagx="volume" title="volume">87</span> <span class="fpage" tagx="fpage" title="fpage">929</span> <span class="lpage" tagx="lpage" title="lpage">954</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/cr00081a004">10.1021/cr00081a004</a> </span> </span> </li> <li tag="ref"> <a name="CR19" /> <span class="label" tagx="label" title="label">19.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ro</span> <span class="given-names" tagx="given-names" title="given-names">DK</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Production of the antimalarial drug precursor artemisinic acid in engineered yeast</span> <span class="source" tagx="source" title="source">Nature</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">440</span> <span class="fpage" tagx="fpage" title="fpage">940</span> <span class="lpage" tagx="lpage" title="lpage">943</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nature04640">10.1038/nature04640</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16612385">16612385</a> </span> </span> </li> <li tag="ref"> <a name="CR20" /> <span class="label" tagx="label" title="label">20.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Westfall</span> <span class="given-names" tagx="given-names" title="given-names">PJ</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Production of amorphadiene in yeast, and its conversion to dihydroartemisinic acid, precursor to the antimalarial agent artemisinin </span> <span class="source" tagx="source" title="source">Proc. Natl Acad. Sci. USA</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">109</span> <span class="fpage" tagx="fpage" title="fpage">E111</span> <span class="lpage" tagx="lpage" title="lpage">E118</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.1110740109">10.1073/pnas.1110740109</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22247290">22247290</a> </span> </span> </li> <li tag="ref"> <a name="CR21" /> <span class="label" tagx="label" title="label">21.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paddon</span> <span class="given-names" tagx="given-names" title="given-names">CJ</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">High-level semi-synthetic production of the potent antimalarial artemisinin</span> <span class="source" tagx="source" title="source">Nature</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">496</span> <span class="fpage" tagx="fpage" title="fpage">528</span> <span class="lpage" tagx="lpage" title="lpage">532</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nature12051">10.1038/nature12051</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23575629">23575629</a> </span> </span> </li> <li tag="ref"> <a name="CR22" /> <span class="label" tagx="label" title="label">22.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meadows</span> <span class="given-names" tagx="given-names" title="given-names">AL</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Rewriting yeast central carbon metabolism for industrial isoprenoid production</span> <span class="source" tagx="source" title="source">Nature</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">537</span> <span class="fpage" tagx="fpage" title="fpage">694</span> <span class="lpage" tagx="lpage" title="lpage">697</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nature19769">10.1038/nature19769</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27654918">27654918</a> </span> </span> </li> <li tag="ref"> <a name="CR23" /> <span class="label" tagx="label" title="label">23.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peng</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nielsen</span> <span class="given-names" tagx="given-names" title="given-names">LK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kampranis</span> <span class="given-names" tagx="given-names" title="given-names">SC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vickers</span> <span class="given-names" tagx="given-names" title="given-names">CE</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Engineered protein degradation of farnesyl pyrophosphate synthase is an effective regulatory mechanism to increase monoterpene production in <i>Saccharomyces cerevisiae</i> </span> <span class="source" tagx="source" title="source">Metab. Eng.</span> <span class="year" tagx="year" title="year">2018</span> <span class="volume" tagx="volume" title="volume">47</span> <span class="fpage" tagx="fpage" title="fpage">83</span> <span class="lpage" tagx="lpage" title="lpage">93</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ymben.2018.02.005">10.1016/j.ymben.2018.02.005</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29471044">29471044</a> </span> </span> </li> <li tag="ref"> <a name="CR24" /> <span class="label" tagx="label" title="label">24.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peng</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A squalene synthase protein degradation method for improved sesquiterpene production in <i>Saccharomyces cerevisiae</i> </span> <span class="source" tagx="source" title="source">Metab. Eng.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">39</span> <span class="fpage" tagx="fpage" title="fpage">209</span> <span class="lpage" tagx="lpage" title="lpage">219</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ymben.2016.12.003">10.1016/j.ymben.2016.12.003</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27939849">27939849</a> </span> </span> </li> <li tag="ref"> <a name="CR25" /> <span class="label" tagx="label" title="label">25.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Anderson</span> <span class="given-names" tagx="given-names" title="given-names">MS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yarger</span> <span class="given-names" tagx="given-names" title="given-names">JG</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burck</span> <span class="given-names" tagx="given-names" title="given-names">CL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Poulter</span> <span class="given-names" tagx="given-names" title="given-names">CD</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Farnesyl diphosphate synthetase. Molecular cloning, sequence, and expression of an essential gene from <i>Saccharomyces cerevisiae</i> </span> <span class="source" tagx="source" title="source">J. Biol. Chem.</span> <span class="year" tagx="year" title="year">1989</span> <span class="volume" tagx="volume" title="volume">264</span> <span class="fpage" tagx="fpage" title="fpage">19176</span> <span class="lpage" tagx="lpage" title="lpage">19184</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2681213">2681213</a> </span> </span> </li> <li tag="ref"> <a name="CR26" /> <span class="label" tagx="label" title="label">26.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Liebermeister, W. Enzyme economy in metabolic networks. Preprint at https://arXiv:1404.5252 [q-bio.MN] (2014). </span> </li> <li tag="ref"> <a name="CR27" /> <span class="label" tagx="label" title="label">27.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ignea</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Positive genetic interactors of HMG2 identify a new set of genetic perturbations for improving sesquiterpene production in <i>Saccharomyces cerevisiae</i> </span> <span class="source" tagx="source" title="source">Microb. Cell Factories</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="fpage" tagx="fpage" title="fpage">162</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1475-2859-11-162">10.1186/1475-2859-11-162</a> </span> </span> </li> <li tag="ref"> <a name="CR28" /> <span class="label" tagx="label" title="label">28.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schilmiller</span> <span class="given-names" tagx="given-names" title="given-names">AL</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Monoterpenes in the glandular trichomes of tomato are synthesized from a neryl diphosphate precursor rather than geranyl diphosphate </span> <span class="source" tagx="source" title="source">Proc. Natl Acad. Sci. USA</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">106</span> <span class="fpage" tagx="fpage" title="fpage">10865</span> <span class="lpage" tagx="lpage" title="lpage">10870</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.0904113106">10.1073/pnas.0904113106</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19487664">19487664</a> </span> </span> </li> <li tag="ref"> <a name="CR29" /> <span class="label" tagx="label" title="label">29.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sallaud</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A novel pathway for sesquiterpene biosynthesis from Z,Z-farnesyl pyrophosphate in the wild tomato Solanum habrochaites </span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">21</span> <span class="fpage" tagx="fpage" title="fpage">301</span> <span class="lpage" tagx="lpage" title="lpage">317</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.107.057885">10.1105/tpc.107.057885</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19155349">19155349</a> </span> </span> </li> <li tag="ref"> <a name="CR30" /> <span class="label" tagx="label" title="label">30.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gonzales-Vigil</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hufnagel</span> <span class="given-names" tagx="given-names" title="given-names">DE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Last</span> <span class="given-names" tagx="given-names" title="given-names">RL</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barry</span> <span class="given-names" tagx="given-names" title="given-names">CS</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Evolution of TPS20-related terpene synthases influences chemical diversity in the glandular trichomes of the wild tomato relative <i>Solanum habrochaites</i> </span> <span class="source" tagx="source" title="source">Plant J.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">71</span> <span class="fpage" tagx="fpage" title="fpage">921</span> <span class="lpage" tagx="lpage" title="lpage">935</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-313X.2012.05040.x">10.1111/j.1365-313X.2012.05040.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22563774">22563774</a> </span> </span> </li> <li tag="ref"> <a name="CR31" /> <span class="label" tagx="label" title="label">31.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Denby</span> <span class="given-names" tagx="given-names" title="given-names">CM</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Industrial brewing yeast engineered for the production of primary flavor determinants in hopped beer </span> <span class="source" tagx="source" title="source">Nat. Commun.</span> <span class="year" tagx="year" title="year">2018</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">965</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/s41467-018-03293-x">10.1038/s41467-018-03293-x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29559655">29559655</a> </span> </span> </li> <li tag="ref"> <a name="CR32" /> <span class="label" tagx="label" title="label">32.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brennan</span> <span class="given-names" tagx="given-names" title="given-names">TC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Turner</span> <span class="given-names" tagx="given-names" title="given-names">CD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kromer</span> <span class="given-names" tagx="given-names" title="given-names">JO</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nielsen</span> <span class="given-names" tagx="given-names" title="given-names">LK</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Alleviating monoterpene toxicity using a two-phase extractive fermentation for the bioproduction of jet fuel mixtures in <i>Saccharomyces cerevisiae</i> </span> <span class="source" tagx="source" title="source">Biotechnol. Bioeng.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">109</span> <span class="fpage" tagx="fpage" title="fpage">2513</span> <span class="lpage" tagx="lpage" title="lpage">2522</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/bit.24536">10.1002/bit.24536</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22539043">22539043</a> </span> </span> </li> <li tag="ref"> <a name="CR33" /> <span class="label" tagx="label" title="label">33.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lucker</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Monoterpene biosynthesis in lemon ( <i>Citrus limon</i>). cDNA isolation and functional analysis of four monoterpene synthases </span> <span class="source" tagx="source" title="source">Eur. J. Biochem.</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">269</span> <span class="fpage" tagx="fpage" title="fpage">3160</span> <span class="lpage" tagx="lpage" title="lpage">3171</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1046/j.1432-1033.2002.02985.x">10.1046/j.1432-1033.2002.02985.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12084056">12084056</a> </span> </span> </li> <li tag="ref"> <a name="CR34" /> <span class="label" tagx="label" title="label">34.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Faulkner</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The LPP1 and DPP1 gene products account for most of the isoprenoid phosphate phosphatase activities in <i>Saccharomyces cerevisiae</i> </span> <span class="source" tagx="source" title="source">J. Biol. Chem.</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">274</span> <span class="fpage" tagx="fpage" title="fpage">14831</span> <span class="lpage" tagx="lpage" title="lpage">14837</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1074/jbc.274.21.14831">10.1074/jbc.274.21.14831</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10329682">10329682</a> </span> </span> </li> <li tag="ref"> <a name="CR35" /> <span class="label" tagx="label" title="label">35.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stanley Fernandez</span> <span class="given-names" tagx="given-names" title="given-names">SM</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kellogg</span> <span class="given-names" tagx="given-names" title="given-names">BA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Poulter</span> <span class="given-names" tagx="given-names" title="given-names">CD</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Farnesyl diphosphate synthase. Altering the catalytic site to select for geranyl diphosphate activity </span> <span class="source" tagx="source" title="source">Biochemistry</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">39</span> <span class="fpage" tagx="fpage" title="fpage">15316</span> <span class="lpage" tagx="lpage" title="lpage">15321</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/bi0014305">10.1021/bi0014305</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11112517">11112517</a> </span> </span> </li> <li tag="ref"> <a name="CR36" /> <span class="label" tagx="label" title="label">36.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Springer</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weissman</span> <span class="given-names" tagx="given-names" title="given-names">JS</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kirschner</span> <span class="given-names" tagx="given-names" title="given-names">MW</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A general lack of compensation for gene dosage in yeast</span> <span class="source" tagx="source" title="source">Mol. Syst. Biol.</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">368</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/msb.2010.19">10.1038/msb.2010.19</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20461075">20461075</a> </span> </span> </li> <li tag="ref"> <a name="CR37" /> <span class="label" tagx="label" title="label">37.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ignea</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Efficient diterpene production in yeast by engineering Erg20p into a geranylgeranyl diphosphate synthase </span> <span class="source" tagx="source" title="source">Metab. Eng.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">27</span> <span class="fpage" tagx="fpage" title="fpage">65</span> <span class="lpage" tagx="lpage" title="lpage">75</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ymben.2014.10.008">10.1016/j.ymben.2014.10.008</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25446975">25446975</a> </span> </span> </li> <li tag="ref"> <a name="CR38" /> <span class="label" tagx="label" title="label">38.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ignea</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Overcoming the plasticity of plant specialized metabolism for selective diterpene production in yeast </span> <span class="source" tagx="source" title="source">Sci. Rep.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">8855</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/s41598-017-09592-5">10.1038/s41598-017-09592-5</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28821847">28821847</a> </span> </span> </li> <li tag="ref"> <a name="CR39" /> <span class="label" tagx="label" title="label">39.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ignea</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Synthesis of 11-carbon terpenoids in yeast using protein and metabolic engineering</span> <span class="source" tagx="source" title="source">Nat. Chem. Biol.</span> <span class="year" tagx="year" title="year">2018</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">1090</span> <span class="lpage" tagx="lpage" title="lpage">1098</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/s41589-018-0166-5">10.1038/s41589-018-0166-5</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/30429605">30429605</a> </span> </span> </li> <li tag="ref"> <a name="CR40" /> <span class="label" tagx="label" title="label">40.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">K</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Identification and characterization of a novel monoterpene synthase from soybean restricted to neryl diphosphate precursor </span> <span class="source" tagx="source" title="source">PloS One</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">e75972</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0075972">10.1371/journal.pone.0075972</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24124526">24124526</a> </span> </span> </li> <li tag="ref"> <a name="CR41" /> <span class="label" tagx="label" title="label">41.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kampranis</span> <span class="given-names" tagx="given-names" title="given-names">SC</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Rational conversion of substrate and product specificity in a salvia monoterpene synthase: structural insights into the evolution of terpene synthase function </span> <span class="source" tagx="source" title="source">Plant Cell</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">19</span> <span class="fpage" tagx="fpage" title="fpage">1994</span> <span class="lpage" tagx="lpage" title="lpage">2005</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1105/tpc.106.047779">10.1105/tpc.106.047779</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17557809">17557809</a> </span> </span> </li> <li tag="ref"> <a name="CR42" /> <span class="label" tagx="label" title="label">42.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Phillips</span> <span class="given-names" tagx="given-names" title="given-names">MA</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wildung</span> <span class="given-names" tagx="given-names" title="given-names">MR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Williams</span> <span class="given-names" tagx="given-names" title="given-names">DC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hyatt</span> <span class="given-names" tagx="given-names" title="given-names">DC</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Croteau</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">cDNA isolation, functional expression, and characterization of (+)-alpha-pinene synthase and (-)-alpha-pinene synthase from loblolly pine ( <i>Pinus taeda</i>): stereocontrol in pinene biosynthesis </span> <span class="source" tagx="source" title="source">Arch. Biochem. Biophys.</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">411</span> <span class="fpage" tagx="fpage" title="fpage">267</span> <span class="lpage" tagx="lpage" title="lpage">276</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0003-9861(02)00746-4">10.1016/S0003-9861(02)00746-4</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12623076">12623076</a> </span> </span> </li> <li tag="ref"> <a name="CR43" /> <span class="label" tagx="label" title="label">43.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tsaballa</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Use of the de novo transcriptome analysis of silver-leaf nightshade ( <i>Solanum elaeagnifolium</i>) to identify gene expression changes associated with wounding and terpene biosynthesis </span> <span class="source" tagx="source" title="source">BMC Genom.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">504</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/s12864-015-1738-3">10.1186/s12864-015-1738-3</a> </span> </span> </li> <li tag="ref"> <a name="CR44" /> <span class="label" tagx="label" title="label">44.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hyatt</span> <span class="given-names" tagx="given-names" title="given-names">DC</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Structure of limonene synthase, a simple model for terpenoid cyclase catalysis</span> <span class="source" tagx="source" title="source">Proc. Natl Acad. Sci. USA</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">104</span> <span class="fpage" tagx="fpage" title="fpage">5360</span> <span class="lpage" tagx="lpage" title="lpage">5365</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.0700915104">10.1073/pnas.0700915104</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17372193">17372193</a> </span> </span> </li> <li tag="ref"> <a name="CR45" /> <span class="label" tagx="label" title="label">45.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gesell</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The gymnosperm cytochrome P450 CYP750B1 catalyzes stereospecific monoterpene hydroxylation of (+)-sabinene in thujone biosynthesis in western redcedar </span> <span class="source" tagx="source" title="source">Plant Physiol.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">168</span> <span class="fpage" tagx="fpage" title="fpage">94</span> <span class="lpage" tagx="lpage" title="lpage">106</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.15.00315">10.1104/pp.15.00315</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25829465">25829465</a> </span> </span> </li> <li tag="ref"> <a name="CR46" /> <span class="label" tagx="label" title="label">46.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ro</span> <span class="given-names" tagx="given-names" title="given-names">DK</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ehlting</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Douglas</span> <span class="given-names" tagx="given-names" title="given-names">CJ</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Cloning, functional expression, and subcellular localization of multiple NADPH-cytochrome P450 reductases from hybrid poplar </span> <span class="source" tagx="source" title="source">Plant Physiol.</span> <span class="year" tagx="year" title="year">2002</span> <span class="volume" tagx="volume" title="volume">130</span> <span class="fpage" tagx="fpage" title="fpage">1837</span> <span class="lpage" tagx="lpage" title="lpage">1851</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1104/pp.008011">10.1104/pp.008011</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12481067">12481067</a> </span> </span> </li> <li tag="ref"> <a name="CR47" /> <span class="label" tagx="label" title="label">47.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leber</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A novel sequence element is involved in the transcriptional regulation of expression of the ERG1 (squalene epoxidase) gene in <i>Saccharomyces cerevisiae</i> </span> <span class="source" tagx="source" title="source">Eur. J. Biochem.</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">268</span> <span class="fpage" tagx="fpage" title="fpage">914</span> <span class="lpage" tagx="lpage" title="lpage">924</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1046/j.1432-1327.2001.01940.x">10.1046/j.1432-1327.2001.01940.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11179957">11179957</a> </span> </span> </li> <li tag="ref"> <a name="CR48" /> <span class="label" tagx="label" title="label">48.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yuan</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ching</span> <span class="given-names" tagx="given-names" title="given-names">CB</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Dynamic control of ERG9 expression for improved amorpha-4,11-diene production in <i>Saccharomyces cerevisiae</i> </span> <span class="source" tagx="source" title="source">Microb. Cell Factories</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">38</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/s12934-015-0220-x">10.1186/s12934-015-0220-x</a> </span> </span> </li> <li tag="ref"> <a name="CR49" /> <span class="label" tagx="label" title="label">49.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Callari</span> <span class="given-names" tagx="given-names" title="given-names">R</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meier</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ravasio</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Heider</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Dynamic control of ERG20 and ERG9 expression for improved casbene production in <i>Saccharomyces cerevisiae</i> </span> <span class="source" tagx="source" title="source">Front. Bioeng. Biotechnol.</span> <span class="year" tagx="year" title="year">2018</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">160</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3389/fbioe.2018.00160">10.3389/fbioe.2018.00160</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/30443546">30443546</a> </span> </span> </li> <li tag="ref"> <a name="CR50" /> <span class="label" tagx="label" title="label">50.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Erb</span> <span class="given-names" tagx="given-names" title="given-names">TJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jones</span> <span class="given-names" tagx="given-names" title="given-names">PR</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bar-Even</span> <span class="given-names" tagx="given-names" title="given-names">A</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Synthetic metabolism: metabolic engineering meets enzyme design</span> <span class="source" tagx="source" title="source">Curr. Opin. Biotechnol.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">37</span> <span class="fpage" tagx="fpage" title="fpage">56</span> <span class="lpage" tagx="lpage" title="lpage">62</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.cbpa.2016.12.023">10.1016/j.cbpa.2016.12.023</a> </span> </span> </li> <li tag="ref"> <a name="CR51" /> <span class="label" tagx="label" title="label">51.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lynch</span> <span class="given-names" tagx="given-names" title="given-names">MD</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Into new territory: improved microbial synthesis through engineering of the essential metabolic network </span> <span class="source" tagx="source" title="source">Curr. Opin. Biotechnol.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">38</span> <span class="fpage" tagx="fpage" title="fpage">106</span> <span class="lpage" tagx="lpage" title="lpage">111</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.copbio.2016.01.009">10.1016/j.copbio.2016.01.009</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26874263">26874263</a> </span> </span> </li> <li tag="ref"> <a name="CR52" /> <span class="label" tagx="label" title="label">52.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alonso-Gutierrez</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Principal component analysis of proteomics (PCAP) as a tool to direct metabolic engineering</span> <span class="source" tagx="source" title="source">Metab. Eng.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">28</span> <span class="fpage" tagx="fpage" title="fpage">123</span> <span class="lpage" tagx="lpage" title="lpage">133</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ymben.2014.11.011">10.1016/j.ymben.2014.11.011</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25554074">25554074</a> </span> </span> </li> <li tag="ref"> <a name="CR53" /> <span class="label" tagx="label" title="label">53.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mendez-Perez</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Production of jet fuel precursor monoterpenoids from engineered <i>Escherichia coli</i> </span> <span class="source" tagx="source" title="source">Biotechnol. Bioeng.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">114</span> <span class="fpage" tagx="fpage" title="fpage">1703</span> <span class="lpage" tagx="lpage" title="lpage">1712</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/bit.26296">10.1002/bit.26296</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28369701">28369701</a> </span> </span> </li> <li tag="ref"> <a name="CR54" /> <span class="label" tagx="label" title="label">54.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">H</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Microbial production of sabinene-a new terpene-based precursor of advanced biofuel</span> <span class="source" tagx="source" title="source">Microb. Cell Factories</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">13</span> <span class="fpage" tagx="fpage" title="fpage">20</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1475-2859-13-20">10.1186/1475-2859-13-20</a> </span> </span> </li> <li tag="ref"> <a name="CR55" /> <span class="label" tagx="label" title="label">55.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paddon</span> <span class="given-names" tagx="given-names" title="given-names">CJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Keasling</span> <span class="given-names" tagx="given-names" title="given-names">JD</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Semi-synthetic artemisinin: a model for the use of synthetic biology in pharmaceutical development </span> <span class="source" tagx="source" title="source">Nat. Rev. Microbiol.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">12</span> <span class="fpage" tagx="fpage" title="fpage">355</span> <span class="lpage" tagx="lpage" title="lpage">367</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nrmicro3240">10.1038/nrmicro3240</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24686413">24686413</a> </span> </span> </li> <li tag="ref"> <a name="CR56" /> <span class="label" tagx="label" title="label">56.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tashiro</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Bacterial production of pinene by a laboratory-evolved pinene-synthase</span> <span class="source" tagx="source" title="source">ACS Synth. Biol.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">1011</span> <span class="lpage" tagx="lpage" title="lpage">1020</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/acssynbio.6b00140">10.1021/acssynbio.6b00140</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27247193">27247193</a> </span> </span> </li> <li tag="ref"> <a name="CR57" /> <span class="label" tagx="label" title="label">57.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bakkali</span> <span class="given-names" tagx="given-names" title="given-names">F</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Averbeck</span> <span class="given-names" tagx="given-names" title="given-names">S</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Averbeck</span> <span class="given-names" tagx="given-names" title="given-names">D</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Idaomar</span> <span class="given-names" tagx="given-names" title="given-names">M</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Biological effects of essential oils-a review</span> <span class="source" tagx="source" title="source">Food Chem. Toxicol.</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">46</span> <span class="fpage" tagx="fpage" title="fpage">446</span> <span class="lpage" tagx="lpage" title="lpage">475</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.fct.2007.09.106">10.1016/j.fct.2007.09.106</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17996351">17996351</a> </span> </span> </li> <li tag="ref"> <a name="CR58" /> <span class="label" tagx="label" title="label">58.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dunlop</span> <span class="given-names" tagx="given-names" title="given-names">MJ</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Engineering microbial biofuel tolerance and export using efflux pumps</span> <span class="source" tagx="source" title="source">Mol. Syst. Biol.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">487</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/msb.2011.21">10.1038/msb.2011.21</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21556065">21556065</a> </span> </span> </li> <li tag="ref"> <a name="CR59" /> <span class="label" tagx="label" title="label">59.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Todorovic</span> <span class="given-names" tagx="given-names" title="given-names">B</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Toxicity of twenty-two plant essential oils against pathogenic bacteria of vegetables and mushrooms </span> <span class="source" tagx="source" title="source">J. Environ. Sci. Health B.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">51</span> <span class="fpage" tagx="fpage" title="fpage">832</span> <span class="lpage" tagx="lpage" title="lpage">839</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1080/03601234.2016.1208462">10.1080/03601234.2016.1208462</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27494458">27494458</a> </span> </span> </li> <li tag="ref"> <a name="CR60" /> <span class="label" tagx="label" title="label">60.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">George</span> <span class="given-names" tagx="given-names" title="given-names">KW</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Integrated analysis of isopentenyl pyrophosphate (IPP) toxicity in isoprenoid-producing <i>Escherichia coli</i> </span> <span class="source" tagx="source" title="source">Metab. Eng.</span> <span class="year" tagx="year" title="year">2018</span> <span class="volume" tagx="volume" title="volume">47</span> <span class="fpage" tagx="fpage" title="fpage">60</span> <span class="lpage" tagx="lpage" title="lpage">72</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ymben.2018.03.004">10.1016/j.ymben.2018.03.004</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29530749">29530749</a> </span> </span> </li> <li tag="ref"> <a name="CR61" /> <span class="label" tagx="label" title="label">61.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zi</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Biosynthesis of lycosantalonol, a cis-prenyl derived diterpenoid</span> <span class="source" tagx="source" title="source">J. Am. Chem. Soc.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">136</span> <span class="fpage" tagx="fpage" title="fpage">16951</span> <span class="lpage" tagx="lpage" title="lpage">16953</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/ja508477e">10.1021/ja508477e</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25406026">25406026</a> </span> </span> </li> <li tag="ref"> <a name="CR62" /> <span class="label" tagx="label" title="label">62.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Matsuba</span> <span class="given-names" tagx="given-names" title="given-names">Y</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zi</span> <span class="given-names" tagx="given-names" title="given-names">J</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jones</span> <span class="given-names" tagx="given-names" title="given-names">AD</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peters</span> <span class="given-names" tagx="given-names" title="given-names">RJ</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pichersky</span> <span class="given-names" tagx="given-names" title="given-names">E</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Biosynthesis of the diterpenoid lycosantalonol via nerylneryl diphosphate in <i>Solanum lycopersicum</i> </span> <span class="source" tagx="source" title="source">PloS ONE</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">e0119302</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0119302">10.1371/journal.pone.0119302</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25786135">25786135</a> </span> </span> </li> <li tag="ref"> <a name="CR63" /> <span class="label" tagx="label" title="label">63.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jia</span> <span class="given-names" tagx="given-names" title="given-names">Q</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Microbial-type terpene synthase genes occur widely in nonseed land plants, but not in seed plants </span> <span class="source" tagx="source" title="source">Proc. Natl Acad. Sci. USA</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">113</span> <span class="fpage" tagx="fpage" title="fpage">12328</span> <span class="lpage" tagx="lpage" title="lpage">12333</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.1607973113">10.1073/pnas.1607973113</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27791023">27791023</a> </span> </span> </li> <li tag="ref"> <a name="CR64" /> <span class="label" tagx="label" title="label">64.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Behrendorff</span> <span class="given-names" tagx="given-names" title="given-names">JB</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vickers</span> <span class="given-names" tagx="given-names" title="given-names">CE</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chrysanthopoulos</span> <span class="given-names" tagx="given-names" title="given-names">P</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nielsen</span> <span class="given-names" tagx="given-names" title="given-names">LK</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">2,2-diphenyl-1-picrylhydrazyl as a screening tool for recombinant monoterpene biosynthesis</span> <span class="source" tagx="source" title="source">Microb. Cell Fact.</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">12</span> <span class="fpage" tagx="fpage" title="fpage">76</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1475-2859-12-76">10.1186/1475-2859-12-76</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23968454">23968454</a> </span> </span> </li> <li tag="ref"> <a name="CR65" /> <span class="label" tagx="label" title="label">65.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ignea</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Improving yeast strains using recyclable integration cassettes, for the production of plant terpenoids </span> <span class="source" tagx="source" title="source">Microb. Cell Factories</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">4</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1475-2859-10-4">10.1186/1475-2859-10-4</a> </span> </span> </li> <li tag="ref"> <a name="CR66" /> <span class="label" tagx="label" title="label">66.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Trikka</span> <span class="given-names" tagx="given-names" title="given-names">FA</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Combined metabolome and transcriptome profiling provides new insights into diterpene biosynthesis in <i>S. pomifera</i> glandular trichomes </span> <span class="source" tagx="source" title="source">BMC Genom.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">16</span> <span class="fpage" tagx="fpage" title="fpage">935</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/s12864-015-2147-3">10.1186/s12864-015-2147-3</a> </span> </span> </li> <li tag="ref"> <a name="CR67" /> <span class="label" tagx="label" title="label">67.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ignea</span> <span class="given-names" tagx="given-names" title="given-names">C</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Production of the forskolin precursor 11beta-hydroxy-manoyl oxide in yeast using surrogate enzymatic activities </span> <span class="source" tagx="source" title="source">Microb. Cell Factories</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">15</span> <span class="fpage" tagx="fpage" title="fpage">46</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/s12934-016-0440-8">10.1186/s12934-016-0440-8</a> </span> </span> </li> <li tag="ref"> <a name="CR68" /> <span class="label" tagx="label" title="label">68.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pettersen</span> <span class="given-names" tagx="given-names" title="given-names">EF</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">UCSF Chimera–a visualization system for exploratory research and analysis</span> <span class="source" tagx="source" title="source">J. Comput. Chem.</span> <span class="year" tagx="year" title="year">2004</span> <span class="volume" tagx="volume" title="volume">25</span> <span class="fpage" tagx="fpage" title="fpage">1605</span> <span class="lpage" tagx="lpage" title="lpage">1612</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/jcc.20084">10.1002/jcc.20084</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15264254">15264254</a> </span> </span> </li> <li tag="ref"> <a name="CR69" /> <span class="label" tagx="label" title="label">69.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Genee</span> <span class="given-names" tagx="given-names" title="given-names">HJ</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Software-supported USER cloning strategies for site-directed mutagenesis and DNA assembly</span> <span class="source" tagx="source" title="source">ACS Synth. Biol.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">342</span> <span class="lpage" tagx="lpage" title="lpage">349</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/sb500194z">10.1021/sb500194z</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24847672">24847672</a> </span> </span> </li> <li tag="ref"> <a name="CR70" /> <span class="label" tagx="label" title="label">70.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peitsch</span> <span class="given-names" tagx="given-names" title="given-names">MC</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">ProMod and Swiss-Model: Internet-based tools for automated comparative protein modelling</span> <span class="source" tagx="source" title="source">Biochem. Soc. Trans.</span> <span class="year" tagx="year" title="year">1996</span> <span class="volume" tagx="volume" title="volume">24</span> <span class="fpage" tagx="fpage" title="fpage">274</span> <span class="lpage" tagx="lpage" title="lpage">279</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1042/bst0240274">10.1042/bst0240274</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8674685">8674685</a> </span> </span> </li> </ul> </div> </div>  </body></html>