<?xml version="1.0" encoding="UTF-8"?>
<html xmlns="http://www.w3.org/1999/xhtml">
 <head charset="UTF-8">
  <style> table { background : red; margin : 2px; } tr { margin : 2px; background : #aaffff; } td { margin : 2px; background : #ccffff; border-left: 1px solid blue; } td[colspan='2'] {background: #ddddff} td[rowspan='2'] {background: #ffdddd} tfoot tr td {background-color : #ffb; font-weight : bold;} </style>
 </head>
 <body>
  <table>
   <caption>Table 1 Responses of bacterial communities to the Deepwater Horizon oil spill and various microbiological methods applied in different studies.</caption>
   <thead>
    <tr>
     <th>target=(Habits)</th>
     <th>target=(Sampling time)</th>
     <th>target=(Elapsed time (months))</th>
     <th>target=(Dominant bacterial phylotypes)</th>
     <th>target=(Technologies)</th>
     <th>target=(References)</th>
     <th>value=(Habits)</th>
     <th>value=(Sampling time)</th>
     <th>value=(Elapsed time (months))</th>
     <th>value=(Dominant bacterial phylotypes)</th>
     <th>value=(Technologies)</th>
     <th>value=(References)</th>
    </tr>
   </thead>
   <tbody>
    <tr>
     <td>Sediments</td>
     <td>09/2010–10/2010</td>
     <td>5–6</td>
     <td>Uncultured γ-proteobacteriaandColwellia</td>
     <td>16S rRNA gene-based clone library, metagenomics, and SIP</td>
     <td>Mason et al.,2014b</td>
     <td>Sediments</td>
     <td>09/2010–10/2010</td>
     <td>5–6</td>
     <td>Uncultured γ-proteobacteriaandColwellia</td>
     <td>16S rRNA gene-based clone library, metagenomics, and SIP</td>
     <td>Mason et al.,2014b</td>
    </tr>
    <tr>
     <td/>
     <td>12/2010</td>
     <td>8</td>
     <td>Uncultured γ-proteobacteriaand δ-Proteobacteria(Desulfobacterales, Desulfuromonadales, andDesulfarculales)</td>
     <td>16S rRNA gene-based clone library, and Illumina amplicon sequencing</td>
     <td>Simister et al.,2016</td>
     <td/>
     <td>12/2010</td>
     <td>8</td>
     <td>Uncultured γ-proteobacteriaand δ-Proteobacteria(Desulfobacterales, Desulfuromonadales, andDesulfarculales)</td>
     <td>16S rRNA gene-based clone library, and Illumina amplicon sequencing</td>
     <td>Simister et al.,2016</td>
    </tr>
    <tr>
     <td/>
     <td>11/2011</td>
     <td>19</td>
     <td>OceanospirillalesandColwellia</td>
     <td>Metagenomics and metatranscriptomics</td>
     <td>Yergeau et al.,2015</td>
     <td/>
     <td>11/2011</td>
     <td>19</td>
     <td>OceanospirillalesandColwellia</td>
     <td>Metagenomics and metatranscriptomics</td>
     <td>Yergeau et al.,2015</td>
    </tr>
    <tr>
     <td>Deep water</td>
     <td/>
     <td>Before spill</td>
     <td>α-proteobacteria, PelagibacterandActinobacteria</td>
     <td>PhyloChip, 16S rRNA gene-based clone library, and SIP</td>
     <td>Hazen et al.,2010</td>
     <td>Deep water</td>
     <td/>
     <td>Before spill</td>
     <td>α-proteobacteria, PelagibacterandActinobacteria</td>
     <td>PhyloChip, 16S rRNA gene-based clone library, and SIP</td>
     <td>Hazen et al.,2010</td>
    </tr>
    <tr>
     <td/>
     <td>05/2010</td>
     <td>1</td>
     <td>γ-proteobacteria(Oceanospirillales, Alteromonadales, Cycloclasticus,</td>
     <td>454 pyrosequencing</td>
     <td>Joye et al.,2014</td>
     <td/>
     <td>05/2010</td>
     <td>1</td>
     <td>γ-proteobacteria(Oceanospirillales, Alteromonadales, Cycloclasticus,</td>
     <td>454 pyrosequencing</td>
     <td>Joye et al.,2014</td>
    </tr>
    <tr>
     <td/>
     <td>05/2010 06/2010</td>
     <td>1–2</td>
     <td>γ-proteobacteria(Oceanospirillales, Oleispira, Spongiispira, Cycloclasticus, Oleiphilus, Thalassolituus, andColwellia)</td>
     <td>PhyloChip, 16S rRNA gene-based clone library, and SIP</td>
     <td>Hazen et al.,2010; Valentine et al.,2010; Redmond and Valentine,2011; King et al.,2015</td>
     <td/>
     <td>05/2010 06/2010</td>
     <td>1–2</td>
     <td>γ-proteobacteria(Oceanospirillales, Oleispira, Spongiispira, Cycloclasticus, Oleiphilus, Thalassolituus, andColwellia)</td>
     <td>PhyloChip, 16S rRNA gene-based clone library, and SIP</td>
     <td>Hazen et al.,2010; Valentine et al.,2010; Redmond and Valentine,2011; King et al.,2015</td>
    </tr>
    <tr>
     <td/>
     <td>09/2010</td>
     <td>5</td>
     <td>Methylotrophic bacteria (Methylococcaceae, Methylophaga, andMethylophilaceae)</td>
     <td>16S rRNA gene-based clone library</td>
     <td>Kessler et al.,2011</td>
     <td/>
     <td>09/2010</td>
     <td>5</td>
     <td>Methylotrophic bacteria (Methylococcaceae, Methylophaga, andMethylophilaceae)</td>
     <td>16S rRNA gene-based clone library</td>
     <td>Kessler et al.,2011</td>
    </tr>
    <tr>
     <td/>
     <td>11/2011</td>
     <td>19</td>
     <td>γ-proteobacteria</td>
     <td>Metagenomics and metatranscriptomics</td>
     <td>Yergeau et al.,2015</td>
     <td/>
     <td>11/2011</td>
     <td>19</td>
     <td>γ-proteobacteria</td>
     <td>Metagenomics and metatranscriptomics</td>
     <td>Yergeau et al.,2015</td>
    </tr>
    <tr>
     <td>Surface water</td>
     <td/>
     <td>Before spill</td>
     <td>α-proteobacteria, Pelagibacter, andActinobacteria.</td>
     <td>PhyloChip, 16S rRNA gene-based clone library, and SIP</td>
     <td>Hazen et al.,2010</td>
     <td>Surface water</td>
     <td/>
     <td>Before spill</td>
     <td>α-proteobacteria, Pelagibacter, andActinobacteria.</td>
     <td>PhyloChip, 16S rRNA gene-based clone library, and SIP</td>
     <td>Hazen et al.,2010</td>
    </tr>
    <tr>
     <td/>
     <td>05/2010</td>
     <td>1</td>
     <td>OceanospirillalesandRhodospirillales</td>
     <td>Metagenomics and SIP</td>
     <td>Dombrowski et al.,2016</td>
     <td/>
     <td>05/2010</td>
     <td>1</td>
     <td>OceanospirillalesandRhodospirillales</td>
     <td>Metagenomics and SIP</td>
     <td>Dombrowski et al.,2016</td>
    </tr>
    <tr>
     <td/>
     <td>05/2010</td>
     <td>1</td>
     <td>Alcanivorax, Marinobacter, Alteromonas, Cycloclasticus, andColwellia)-</td>
     <td>SIP</td>
     <td>Gutierrez et al.,2013</td>
     <td/>
     <td>05/2010</td>
     <td>1</td>
     <td>Alcanivorax, Marinobacter, Alteromonas, Cycloclasticus, andColwellia)-</td>
     <td>SIP</td>
     <td>Gutierrez et al.,2013</td>
    </tr>
    <tr>
     <td/>
     <td>06/2010</td>
     <td>2</td>
     <td>Cyanobacteriaand α-proteobacteria(SAR11 clade,Rhodobacterales, andRhodospirillales)</td>
     <td>16S rRNA gene-based clone library</td>
     <td>Chakraborty et al.,2012</td>
     <td/>
     <td>06/2010</td>
     <td>2</td>
     <td>Cyanobacteriaand α-proteobacteria(SAR11 clade,Rhodobacterales, andRhodospirillales)</td>
     <td>16S rRNA gene-based clone library</td>
     <td>Chakraborty et al.,2012</td>
    </tr>
    <tr>
     <td/>
     <td>09/2010</td>
     <td>5</td>
     <td>γ-proteobacteria(Pseudoalteromonas, Pseudomonas, Vibrio, Acinetobacter, andAlteromonas)</td>
     <td>PhyloChip, 16S rRNA gene-based clone library, and SIP</td>
     <td>Redmond and Valentine,2011</td>
     <td/>
     <td>09/2010</td>
     <td>5</td>
     <td>γ-proteobacteria(Pseudoalteromonas, Pseudomonas, Vibrio, Acinetobacter, andAlteromonas)</td>
     <td>PhyloChip, 16S rRNA gene-based clone library, and SIP</td>
     <td>Redmond and Valentine,2011</td>
    </tr>
    <tr>
     <td/>
     <td>11/2011</td>
     <td>19</td>
     <td>Cyanobacteria</td>
     <td>Metagenomics and metatranscriptomics</td>
     <td>Yergeau et al.,2015</td>
     <td/>
     <td>11/2011</td>
     <td>19</td>
     <td>Cyanobacteria</td>
     <td>Metagenomics and metatranscriptomics</td>
     <td>Yergeau et al.,2015</td>
    </tr>
    <tr>
     <td>Marsh sediments</td>
     <td>06/2010, 07/2010, and 09/2010</td>
     <td>2–5</td>
     <td>Proteobacteria, Bacteroidetes, Actinobacteria, andFirmicutes(BacilliandClostridia)</td>
     <td>PhyloChip and GeoChip</td>
     <td>Beazley et al.,2012</td>
     <td>Marsh sediments</td>
     <td>06/2010, 07/2010, and 09/2010</td>
     <td>2–5</td>
     <td>Proteobacteria, Bacteroidetes, Actinobacteria, andFirmicutes(BacilliandClostridia)</td>
     <td>PhyloChip and GeoChip</td>
     <td>Beazley et al.,2012</td>
    </tr>
    <tr>
     <td/>
     <td>05/2010, 09/2010, 06/2011, 09/2011, and 08/2013</td>
     <td>1–40</td>
     <td>Proteobacteria, Firmicutes, Bacteroidetes, and/orChloroflexiphyla</td>
     <td>GS FLX amplicon pyrosequencing</td>
     <td>Engel et al.,2017</td>
     <td/>
     <td>05/2010, 09/2010, 06/2011, 09/2011, and 08/2013</td>
     <td>1–40</td>
     <td>Proteobacteria, Firmicutes, Bacteroidetes, and/orChloroflexiphyla</td>
     <td>GS FLX amplicon pyrosequencing</td>
     <td>Engel et al.,2017</td>
    </tr>
    <tr>
     <td/>
     <td>The fall of 2011, the spring of 2012, and the fall of 2013</td>
     <td>18–36</td>
     <td>Desulfococcus, Marinobacter, andMycobacterium</td>
     <td>Metagenomics</td>
     <td>Atlas et al.,2015</td>
     <td/>
     <td>The fall of 2011, the spring of 2012, and the fall of 2013</td>
     <td>18–36</td>
     <td>Desulfococcus, Marinobacter, andMycobacterium</td>
     <td>Metagenomics</td>
     <td>Atlas et al.,2015</td>
    </tr>
    <tr>
     <td/>
     <td>07/2012</td>
     <td>24</td>
     <td>Nitrosopumiluscluster</td>
     <td>TRFLP Analysis</td>
     <td>Bernhard et al.,2016</td>
     <td/>
     <td>07/2012</td>
     <td>24</td>
     <td>Nitrosopumiluscluster</td>
     <td>TRFLP Analysis</td>
     <td>Bernhard et al.,2016</td>
    </tr>
    <tr>
     <td>Beach sands</td>
     <td>07/2010</td>
     <td>3</td>
     <td>α-proteobacteria(Rhodobacteraceae) and γ-proteobacteria(Alcanivorax, Marinobacter, Pseudomonas, andAcinetobacter)</td>
     <td>Pure culture, automated ribosomal intergenic spacer analysis (ARISA), microarray, and SSU rRNA pyro sequence libraries</td>
     <td>Kostka et al.,2011</td>
     <td>Beach sands</td>
     <td>07/2010</td>
     <td>3</td>
     <td>α-proteobacteria(Rhodobacteraceae) and γ-proteobacteria(Alcanivorax, Marinobacter, Pseudomonas, andAcinetobacter)</td>
     <td>Pure culture, automated ribosomal intergenic spacer analysis (ARISA), microarray, and SSU rRNA pyro sequence libraries</td>
     <td>Kostka et al.,2011</td>
    </tr>
    <tr>
     <td/>
     <td>06/2011</td>
     <td>Over 12</td>
     <td>α-proteobacteria(Hyphomonas, Parvibaculum, andMicavibrio) and γ-proteobacteria(Alcanivorax, Pseudomonas, andMarinobacter)</td>
     <td>Metagenomics and 16S rRNA gene-based clone libraries</td>
     <td>Rodriguez-R et al.,2015</td>
     <td/>
     <td>06/2011</td>
     <td>Over 12</td>
     <td>α-proteobacteria(Hyphomonas, Parvibaculum, andMicavibrio) and γ-proteobacteria(Alcanivorax, Pseudomonas, andMarinobacter)</td>
     <td>Metagenomics and 16S rRNA gene-based clone libraries</td>
     <td>Rodriguez-R et al.,2015</td>
    </tr>
   </tbody>
  </table>
 </body>
</html>
