<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">J Biol Chem</span> <span class="iso-abbrev" title="iso-abbrev">J. Biol. Chem</span> <span class="hwp" title="hwp">jbc</span> <span class="pmc" title="pmc">jbc</span> <span class="publisher-id" title="publisher-id">JBC</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">The Journal of Biological Chemistry</span> </div> <span class="issn-ppub" tagx="issn" title="issn-ppub">0021-9258</span> <span class="issn-epub" tagx="issn" title="issn-epub">1083-351X</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">American Society for Biochemistry and Molecular Biology</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">11200 Rockville Pike, Suite 302, Rockville, MD 20852-3110, U.S.A.</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6254364">6254364</a> </span> <span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/30275012">30275012</a> </span> <span class="publisher-id" title="publisher-id">RA118.004530</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1074/jbc.RA118.004530">10.1074/jbc.RA118.004530</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Microbiology</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
The S-layer protein of a <i>Clostridium difficile</i> SLCT-11 strain displays a complex glycan required for normal cell growth and morphology </div> <div class="alt-title" title="alt-title">
S-layer glycosylation in Clostridium difficile</div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Richards</span> <span class="given-names" tagx="given-names" title="given-names">Emma</span> </span> <a href="#aff1"> <sup>‡</sup> </a> <a href="#FN1"> <sup>1</sup> </a> <a href="#FN2"> <sup>2</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bouché</span> <span class="given-names" tagx="given-names" title="given-names">Laura</span> </span> <a href="#aff1"> <sup>‡</sup> </a> <a href="#FN2"> <sup>2</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Panico</span> <span class="given-names" tagx="given-names" title="given-names">Maria</span> </span> <a href="#aff1"> <sup>‡</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arbeloa</span> <span class="given-names" tagx="given-names" title="given-names">Ana</span> </span> <a href="#aff1"> <sup>‡</sup> </a> <a href="#FN3"> <sup>3</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vinogradov</span> <span class="given-names" tagx="given-names" title="given-names">Evgeny</span> </span> <a href="#aff2"> <sup>§</sup> </a> <a href="#FN4"> <sup>4</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morris</span> <span class="given-names" tagx="given-names" title="given-names">Howard</span> </span> <a href="#aff1"> <sup>‡</sup> </a> <a href="#aff3"> <sup>¶</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wren</span> <span class="given-names" tagx="given-names" title="given-names">Brendan</span> </span> <a href="#aff4"> <sup>‖</sup> </a> <a href="#FN5"> <sup>5</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Logan</span> <span class="given-names" tagx="given-names" title="given-names">Susan M.</span> </span> <a href="#aff2"> <sup>§</sup> </a> <a href="#FN4"> <sup>4</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dell</span> <span class="given-names" tagx="given-names" title="given-names">Anne</span> </span> <a href="#aff1"> <sup>‡</sup> </a> <a href="#FN5"> <sup>5</sup> </a> <a href="#cor1"> <sup>6</sup> </a> </span> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-3224-0207</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fairweather</span> <span class="given-names" tagx="given-names" title="given-names">Neil F.</span> </span> <a href="#aff1"> <sup>‡</sup> </a> <a href="#cor2"> <sup>7</sup> </a> </span> <span class="citation_author_institution" id="aff1">[‡], </span> <span class="citation_author_institution" id="aff2">[§], </span> <span class="citation_author_institution" id="aff3">[¶], </span> <span class="citation_author_institution" id="aff4">[‖], </span> </div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"> <span class="label" tagx="label" title="label">6</span> To whom correspondence may be addressed: <span class="addr-line" title="addr-line">Dept. of Life Sciences, Sir Ernst Chain Bldg., Imperial College London, London SW7 2AZ, UK. </span> Tel.: <span class="phone" tagx="phone" title="phone">44-20-7594-5219</span>; E-mail: <span class="email" tagx="email" title="email">a.dell@imperial.ac.uk</span>. </div> <div class="corresp" title="corresp"> <span class="label" tagx="label" title="label">7</span> To whom correspondence may be addressed: <span class="addr-line" title="addr-line">Dept. of Life Sciences, Flowers Bldg., Imperial College London, London SW7 2AZ, UK.</span> Tel.: <span class="phone" tagx="phone" title="phone">44-20-7594-5247</span>; E-mail: <span class="email" tagx="email" title="email">n.fairweather@imperial.ac.uk</span>. </div> <div class="fn-type-supported-by" title="supported-by"> <span class="label" tagx="label" title="label">1</span> <p>Supported by a Medical Research Council Studentship.</p> </div> <div class="fn-type-equal" title="equal"> <span class="label" tagx="label" title="label">2</span> <p>Both authors contributed equally to this work.</p> </div> <div class="fn-type-supported-by" title="supported-by"> <span class="label" tagx="label" title="label">3</span> <p>Supported by Medical Research Council Grant G1001721 (to N. F.).</p> </div> <div class="fn-type-supported-by" title="supported-by"> <span class="label" tagx="label" title="label">4</span> <p>Supported by the National Research Council of Canada.</p> </div> <div class="fn-type-other" title="other"> <span class="label" tagx="label" title="label">5</span> <p>Recipient of funding from Wellcome Trust Award 102978/Z/13/Z.</p> </div> <div class="fn-type-edited-by" title="edited-by"> <p>Edited by Chris Whitfield</p> </div> </div> <span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2018-11-11</span> </span> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-10-10</span> </span> <span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2018-10-10</span> </span> <span class="volume" tagx="volume" title="volume">293</span> <span class="issue" tagx="issue" title="issue">47</span> <span class="fpage" tagx="fpage" title="fpage">18123</span> <span class="lpage" tagx="lpage" title="lpage">18137</span> <span class="history" title="history"> <span class="received" title="received">received: 2018-6-29</span> <span class="rev-recd" title="rev-recd">rev-recd: 2018-9-27</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Richards et al.</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2018</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p"> <i>Author's Choice</i>—Final version open access under the terms of the <a href="http://creativecommons.org/licenses/by/4.0">Creative Commons CC-BY license</a>. </span> </span> </div> <a class="self-uri" href="zbc04718018123.pdf" title="self-uri">zbc04718018123.pdf</a> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p> <i>Clostridium difficile</i> is a bacterial pathogen that causes major health challenges worldwide. It has a well-characterized surface (S)-layer, a para-crystalline proteinaceous layer surrounding the cell wall. In many bacterial and archaeal species, the S-layer is glycosylated, but no such modifications have been demonstrated in <i>C. difficile.</i> Here, we show that a <i>C. difficile</i> strain of S-layer cassette type 11, Ox247, has a complex glycan attached via an <i>O</i>-linkage to Thr-38 of the S-layer low-molecular-weight subunit. Using MS and NMR, we fully characterized this glycan. We present evidence that it is composed of three domains: (i) a core peptide–linked tetrasaccharide with the sequence -4-α-Rha-3-α-Rha-3-α-Rha-3-β-Gal-peptide; (ii) a repeating pentasaccharide with the sequence -4-β-Rha-4-α-Glc-3-β-Rha-4-(α-Rib-3-)β-Rha-; and (iii) a nonreducing end–terminal 2,3 cyclophosphoryl-rhamnose attached to a ribose-branched sub-terminal rhamnose residue. The Ox247 genome contains a 24-kb locus containing genes for synthesis and protein attachment of this glycan. Mutations in genes within this locus altered or completely abrogated formation of this glycan, and their phenotypes suggested that this S-layer modification may affect sporulation, cell length, and biofilm formation of <i>C. difficile</i>. In summary, our findings indicate that the S-layer protein of SLCT-11 strains displays a complex glycan and suggest that this glycan is required for <i>C. difficile</i> sporulation and control of cell shape, a discovery with implications for the development of antimicrobials targeting the S-layer. </p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
mass spectrometry (MS)</div> <div class="kwd" title="kwd">
nuclear magnetic resonance (NMR)</div> <div class="kwd" title="kwd">
microbiology</div> <div class="kwd" title="kwd">
glycoprotein structure</div> <div class="kwd" title="kwd">
genetics</div> <div class="kwd" title="kwd">
Clostridium difficile</div> <div class="kwd" title="kwd">
surface layer</div> <div class="kwd" title="kwd">
virulence</div> <div class="kwd" title="kwd">
glycosylation</div> <div class="kwd" title="kwd">
glycomics</div> <div class="kwd" title="kwd">
glycoproteomics</div> <div class="kwd" title="kwd">
cell wall</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"> <span class="institution" title="institution">Wellcome Trust </span> <span class="institution-id" title="institution-id">10.13039/100004440</span> </div> </div> <span class="award-id" tagx="award-id" title="award-id">102978/Z/13/Z</span> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"> <span class="institution" title="institution">RCUK | Medical Research Council (MRC) </span> <span class="institution-id" title="institution-id">10.13039/501100000265</span> </div> </div> <span class="award-id" tagx="award-id" title="award-id">MR/K500793/1</span> <span class="award-id" tagx="award-id" title="award-id">G1001721</span> </div> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>The Gram-positive anaerobe <i>Clostridium difficile</i> remains a highly problematic bacterial pathogen, both in hospital environments and in the community ( <a href="#B1">1</a>, <a href="#B2">2</a>). It can cause severe gastrointestinal disease, mediated largely through the production of two toxins, TcdA and TcdB, that glycosylate host gastrointestinal GTPases resulting in damage to gut tissues and unregulated inflammatory reactions ( <a href="#B3">3</a>). Transmission of <i>C. difficile</i> infection (CDI) <a href="#FN7"> <sup>8</sup> </a> is through robust spores produced in the intestine as part of the normal life cycle of the bacterium ( <a href="#B4">4</a>). Vegetative <i>C. difficile</i> cells are resistant to most common antibiotics, with only a few selected antimicrobials used to treat CDI ( <a href="#B1">1</a>). Animal and clinical studies have demonstrated that broad spectrum antibiotics act as predisposing agents for CDI, through selective killing of members of the normal gut microbiota, allowing germination of <i>C. difficile</i> spores and proliferation of the vegetative cells ( <a href="#B5">5</a> <a href="#B6">–</a> <a href="#B7">7</a>). </p> <p>The cell surface of <i>C. difficile</i> has been studied in some detail. Covering the entire surface of vegetative cells is an S-layer, a proteinaceous para-crystalline array composed of S-layer proteins (SLPs) and related cell wall proteins ( <a href="#B8">8</a>, <a href="#B9">9</a>). The major S-layer proteins, LMW and HMW SLPs, are derived from post-translational cleavage of an ∼95-kDa precursor, SlpA ( <a href="#B10">10</a>). The <i>slpA</i> gene is encoded within a genetic locus, termed the S-layer cassette (SLC), that encodes <i>slpA</i> and the adjacent genes <i>cwp66</i>, <i>cd2790</i>, and <i>secA2</i> ( <a href="#F1">Fig. 1</a>) ( <a href="#B11">11</a>). Genetic variation in <i>slpA</i> is observed and is reflected in amino acid variation in SlpA, particularly in the LMW SLP, which shows higher sequence diversity than the HMW SLP ( <a href="#B10">10</a>). </p> <p>Most archaeal species and many bacterial species present an S-layer on their outer surface ( <a href="#B8">8</a>, <a href="#B12">12</a>). S-layers are composed primarily of a single species of (glyco)protein covering the entire surface of the cell and present as a two-dimensional array ( <a href="#B12">12</a>, <a href="#B13">13</a>). No single function has been described for S-layers; rather a diversity of sequences is found that appear to accommodate a wide range of functions ranging from a selectivity barrier through various roles in pathogenesis ( <a href="#B8">8</a>, <a href="#B14">14</a>). In <i>C. difficile</i>, the S-layer protein SlpA is recognized by TLR4 on immune cells and may be a primary factor for the immune system to recognize and respond to <i>C. difficile</i> infection ( <a href="#B15">15</a>). In <i>Bacillus anthracis</i>, the functions of the primary S-layer proteins are unknown, but minor S-layer proteins can function in adhesion to host cells or uptake of iron and have been shown to be essential for full virulence of the bacterium ( <a href="#B16">16</a>, <a href="#B17">17</a>). Structurally, S-layer proteins are found to be diverse, although the three-dimensional crystal structures of only a few have been determined ( <a href="#B18">18</a> <a href="#B19">–</a> <a href="#B24">24</a>). Many S-layer proteins are composed of two domains: one domain anchored to the underlying cell wall and forming the two-dimensional array, and the second domain being surface-exposed and thus suitable for mediating function. In <i>B. anthracis</i> and some other species, a pyruvylated secondary cell wall polysaccharide serves as the anchor for the S-layer proteins Sap and EA1 ( <a href="#B25">25</a>), but in <i>C. difficile</i> the S-layer is noncovalently anchored to the secondary cell wall polysaccharide PSII ( <a href="#B26">26</a>). </p> <p>Some S-layers carry large, surface-exposed glycans ( <a href="#B27">27</a> <a href="#B28">–</a> <a href="#B29">29</a>). Glycans are normally <i>O-</i>linked to the S-layer protein in bacterial species but can be <i>O-</i> or <i>N</i>-linked in archaea. The sugars found on these glycosylated S-layer proteins can be unusual and not found in eukaryotes ( <a href="#B29">29</a>). Where studied, species expressing glycosylated S-layers contain a gene locus specifying the synthesis of the glycan chain, its export through the membrane(s), and ligation to the S-layer protein ( <a href="#B27">27</a>, <a href="#B29">29</a>). Glycosylation has been observed at one or several residues on S-layer proteins from several species, <i>e.g. Geobacillus stearothermophilus</i> and <i>Paenibacillus alvei</i> ( <a href="#B27">27</a>) In these cases, the glycan and the S-layer protein are not co-translocated but are exported independently across the cell wall through separate mechanisms, and it is likely this is true for many other species with glycosylated S-layers ( <a href="#B29">29</a>). </p> <p>In a genomics study of over 1000 <i>C. difficile</i> strains, variation in <i>C. difficile</i> SLCs was observed, and 13 SLC types (SLCTs) were identified, each encoding a distinct SlpA protein ( <a href="#B11">11</a>). One cassette type (SLCT-11) was found to contain a large 24-kb insertion that encoded a putative glycosylation locus. In SLCT-11 strains, this locus is located downstream of <i>secA2</i>, with its insertion causing a rearrangement of <i>cwp66</i> and <i>cd2790</i> and loss of <i>cwp2</i> ( <a href="#F1">Fig. 1</a>). The genes contained within this locus resemble most closely those present in other Gram-positive species that produce glycosylated S-layers, for example <i>G. stearothermophilus</i> and <i>P. alvei</i> ( <a href="#B27">27</a>). In the former species, the glycosylation gene cluster is located immediately downstream of the S-layer gene, <i>sgsE</i>, but in <i>P. alvei</i> the S-layer gene is unlinked. </p> <p>In this study, we analyze the S-layer structure of the <i>C. difficile</i> SLCT-11 strain Ox247 and show conclusively that the SlpA is modified by a large glycan whose composition and structure are determined. The roles of key genes are defined by deletion analysis, and the phenotypes associated with glycosylation are investigated. </p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="geneticanalysisoftheox247glycosylationlocus" title="sec"> <div class="title" tagx="title" title="title">
Genetic analysis of the Ox247 glycosylation locus</div> <p>The majority of <i>C. difficile</i> strains lack a genetic locus that could encode the machinery for glycosylation of surface proteins. An exception is SLCT-11 strains that contain a 24-kb gene cluster indicative of glycosylation of surface proteins ( <a href="#B11">11</a>). The putative glycosylation locus of SLCT-11 strain Ox247 is illustrated in <a href="#F1">Fig. 1</a>. Bioinformatics analysis ( <a href="http://www.jbc.org/cgi/content/full/RA118.004530/DC1">Table S1</a>) predicted 20 genes, including those encoding an initiating glycosyltransferase ( <i>orf2</i>), several glycosyltransferases ( <i>orfs 3, 6–10,</i> and <i>18</i>), an ABC transporter that could potentially transport a glycan to the exterior surface of the cell ( <i>orfs 11</i> and <i>12</i>), a dTDP- <div class="sc_UNKNOWN">
l</div>-rhamnose biosynthetic pathway ( <i>orfs 13, 14, 16,</i> and <i>17</i>), and a protein ligase ( <i>orf19</i>). Many of these genes have homology to biosynthetic pathways that synthesize, transport, and ligate glycans to S-layer proteins in Gram-positive or Gram-negative species ( <a href="#B11">11</a>). The 20 ORFs within the glycosylation locus are transcribed as a polycistronic operon with the adjacent genes <i>CD2790</i> (downstream) and <i>cwp66</i> (upstream) (data not shown). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1.</span> </a> </div>   <p> <b> <i>C. difficile</i> S-layer and glycosylation genetic loci. </b> The S-layer loci of strains 630 ( <i>A</i>) and Ox247 ( <i>B</i>) show the relative locations of <i>slpA</i> and surrounding genes. Insertion of a 24-kb locus harboring a putative glycosylation machinery is shown in Ox247 and in more detail in <i>C</i>. The ORFs of the glycosylation locus in <i>C</i> are numbered according to Ref. <a href="#B11">11</a> and are color-coded to indicate putative function: <i>green,</i> glycosyltransferases; <i>purple,</i> glycosyltransferases; <i>red,</i> ABC transporter; <i>pink,</i> rhamnose biosynthesis; <i>orange,</i> oligosaccharide ligase; <i>white,</i> no function predicted. </p>   </div> <p>To investigate whether this locus was involved in modification of the S-layer, mutants were constructed in Ox247 using the ClosTron mutagenesis system that inactivates genes through targeted insertion of an antibiotic resistance cassette ( <a href="#B30">30</a>). The <i>orf2, orf3, orf4, orf7, orf16,</i> and <i>orf19</i> genes were mutated in this way using an erythromycin resistance marker. S-layer proteins from Ox247 and the mutants were prepared and analyzed by SDS-PAGE ( <a href="#F2">Fig. 2</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2.</span> </a> </div>   <p> <b>S-layer proteins of <i>C. difficile</i> Ox247 and mutants. </b> Strains were cultured in BHIS broth, and the S-layer was prepared using low pH glycine extraction. The LMW (▷) and HMW (▶) S-layer proteins of strain 630 are indicated. The 20-kDa LMW SLP (→) predicted in Ox247 is not found in the WT strain, but is found in some but not all mutants. The ∼45-kDa bands in Ox247 analyzed by MS are indicated (**). <i>M</i>, <i>M</i> <sub>r</sub> markers; <i>lane 1</i>, 630; <i>lane 2</i>, Ox247 WT; <i>lane 3</i>, <i>orf2::erm</i>; <i>lane 4</i>, <i>orf2::erm</i> (pAAM008); <i>lane 5</i>, <i>orf3::erm</i>; <i>lane 6</i>, <i>orf2::erm</i> (pEJR004); <i>lane 7</i>, <i>orf4::erm</i>; <i>lane 8</i>, <i>orf4::erm</i> (pEJR007); <i>lane 9</i>, <i>orf7::erm</i>; <i>lane 10</i>, <i>orf16::erm lane 11, orf16::erm</i> (pEFR009); <i>lane 12</i>, <i>orf19::erm</i>; and <i>lane 13</i>, <i>orf19::erm</i> (pAAM011). </p>   </div> <p>S-layer proteins from WT Ox247 can be compared with those from 630, a well characterized <i>C. difficile</i> strain. In contrast to the two predominant SLPs seen in 630, migrating at ∼43 kDa (HMW SLP) and ∼35 kDa (LMW SLP), only one highly stained band is seen in Ox247, migrating at ∼42 kDa ( <a href="#F2">Fig. 2</a>). A number of minor bands were also evident both above and below the 42-kDa band in Ox247. Sequence analysis of Ox247 SlpA predicts two SLPs formed from the cleavage of the mature SlpA protein, a HMW SLP of ∼45 kDa and a LMW SLP of ∼20 kDa. The size of the LMW SLP is considerably smaller than those observed and predicted in other <i>C. difficile</i> strains. However, this ∼20-kDa species is clearly absent from the WT Ox247 S-layer extract. </p> <p>In contrast, all mutants ( <i>orf2</i>, <i>orf3</i>, <i>orf4</i>, <i>orf7</i>, <i>orf16,</i> and <i>orf19</i>) produced a band at ∼20 kDa, which was shown by MS to be the LMW SLP (see below). This strongly suggests that the genetic locus does indeed encode a glycosylation pathway that modifies SlpA and that disruption of the pathway results in the appearance of the predicted ∼20-kDa LMW SLP. Comparison of the banding pattern of Ox247, the <i>orf2</i> mutant, and the mutant complemented by a plasmid expressing Orf2 reveals a lack of several high-molecular-weight bands above the HMW SLP (∼44 kDa) in the mutant. These bands were predicted to contain glycosylated forms of the ∼20-kDa LMW SLP. Complementation of the mutants was either complete, restoring the WT banding pattern ( <i>orf2</i> and <i>orf3</i>), or incomplete ( <i>orf16</i> and <i>orf19</i>), or was not possible ( <i>orf7</i>) due to the inability to construct a stable plasmid in <i>Escherichia coli</i>. </p> </div> <div class="discoveryofcomplexs-layerglycosylation:glycoproteomicandglycomicmsanalyses" title="sec"> <div class="title" tagx="title" title="title">
Discovery of complex S-layer glycosylation: Glycoproteomic and glycomic MS analyses</div> <p>Unequivocal evidence for glycosylation of the S-layer was determined by glycoproteomic mass spectrometric strategies applied to the analyses of the S-layer proteins ( <a href="#B31">31</a> <a href="#B32">–</a> <a href="#B34">34</a>). S-layer extracts from WT and mutants were purified by gel electrophoresis, in-gel digested with trypsin, and analyzed by nano-LC coupled to electrospray MS ( <a href="#B35">35</a> <a href="#B36">–</a> <a href="#B38">38</a>). MS data on signals eluting between 37 and 40 min for the band at 42 kDa from WT Ox247 ( <a href="#F3">Fig. 3</a>) showed an interesting pattern of doubly charged ions, starting at <i>m</i>/ <i>z</i> 878 (the signal at <i>m</i>/ <i>z</i> 911.9 derives from a different peptide), separated from other prominent signals by intervals corresponding to sugars ( <i>m</i>/ <i>z</i> 81 for hexose (Hex), <i>m</i>/ <i>z</i> 73 for deoxyhexose (dHex), and <i>m</i>/ <i>z</i> 66 for pentose (Pent)). The series can be interpreted as beginning with the tryptic peptide at <i>m</i>/ <i>z</i> 878 and extending by a hexose to <i>m</i>/ <i>z</i> 959, by cleavage at the glycosidic bond with hydrogen rearrangement and charge retention on the reducing end fragment, followed by four dHex intervals (to <i>m/z</i> 1032, 1105, 1178, and 1251). Following the <i>m</i>/ <i>z</i> 1251 signal, the pattern then deviates with two possible cleavage fragments, which indicates branching. The next increment is Pent to <i>m</i>/ <i>z</i> 1317 or dHex to 1324. A common phenomenon in the MS/MS of branched saccharides is the β-elimination of side chains, sometimes giving the strongest signals not at the branching point itself, but at later cleavages. The sugar series then continues from <i>m</i>/ <i>z</i> 1324 with a further hexose residue to <i>m</i>/ <i>z</i> 1405, but this signal is also a pentose difference from <i>m</i>/ <i>z</i> 1471. This spectrum therefore affords two possible interpretations, either with a pentose branch at the <i>m</i>/ <i>z</i> 1251 (4th dHex) cleavage position to give an overall <i>m</i>/ <i>z</i> of 1317, or at the Hex <i>m</i>/ <i>z</i> 1471 position to give <i>m</i>/ <i>z</i> 1405 by β-elimination of the pentose. The <i>m</i>/ <i>z</i> 1317 is a crucial signal here, in that it cannot easily be rationalized mechanistically as deriving from the latter (Hex–Pent) possible branching series, although it can be interpreted as locating the branching point on the fourth dHex from the reducing end of the structure, with Pent loss to 1251 (see <a href="#F3">Fig. 3</a>). <i>m</i>/ <i>z</i> 1324 is then seen to derive from pentose loss from the signal at 1390 in <a href="#F3">Fig. 3</a>. This branching substitution is suggested and proven by the NMR study (see below), and the overall assignment of the glycopeptide MS/MS spectrum in <a href="#F3">Fig. 3</a> shows that the oligosaccharide chain continues out to higher mass (no nonreducing end capping observed) via signals at <i>m</i>/ <i>z</i> 1478, 1544, 1617, 1690 1771, and 1837 (assignments shown in <a href="#F3">Fig. 3</a> schematic) giving a minimal length for this novel oligosaccharide glycoprotein conjugate of 13 sugar residues at a single point of peptide <i>O-</i>linked substitution. The doubly-charged fragment ion at <i>m</i>/ <i>z</i> 878 itself, together with its subfragments, established this as a peptide with the sequence DILAAQNLTTGAVILNK, which corresponds to residues 29–45 of the low-molecular-weight subunit of the S-layer protein. Other ions in the above series selected for MS/MS analysis by the data-dependent software at this time point in the LC-MS trace were the <i>m</i>/ <i>z</i> 959, 1032, 1105, and 1178 glycopeptide fragment signals, and those data also confirmed the sequence conclusions shown in <a href="#F3">Fig. 3</a>. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3.</span> </a> </div>   <p> <b>Pattern of doubly-charged ions observed in the tryptic digest Q-TOF LC-ES-MS data set showing in-source fragmentation of a glycopeptide, the peptide component of which is observed at 878.0 <sup>2+</sup>. </b> Doubly charged ions corresponding to sugar mass intervals linked to this peptide are shown in the <i>cartoon</i> above the spectrum. These data show evidence for a minimum oligosaccharide substitution of 11 residues and a sequence (starting from the reducing (peptide-attached) end) of Hex-dHex-dHex-dHex-dHex(Pent)-dHex-Hex-dHex-dHex(Pent). See the text for details of the assignments. </p>   </div> <p>To investigate the biosynthesis of this unique type of glycopeptide found in the WT sample, the S-layers of the <i>orf2</i>, <i>orf3,</i> and <i>orf7</i> mutants were subjected to the same glycoproteomic strategy. All gave gel bands near 20 kDa ( <a href="#F2">Fig. 2</a>). In the case of the <i>orf2</i> mutant, this band was found to contain the unmodified LMW SLP. The bands near 20 kDa from the other mutants were found to contain glycosylated LMW SLP. The glycan chains ranged in size from two sugar units, specifically dHexHex ( <i>orf3</i> mutant), up to a heptasaccharide Pentd-Hex <sub>5</sub>Hex seen in the <i>orf7</i> mutant ( <a href="http://www.jbc.org/cgi/content/full/RA118.004530/DC1">Fig. S1, <i>A</i> and <i>B</i> </a>). </p> <p>To gain a better understanding of what was clearly a larger structure than shown by the electrospray MS and MS/MS data in <a href="#F3">Fig. 3</a>, full profiling of the S-layer glycoprotein using a more comprehensive range of MS techniques was employed (i) to define the precise site of attachment of the glycan chain to the S-layer proteins, (ii) to determine the carbohydrate composition and linkage of the glycan moieties observed in the MS/MS as Hex, dHex, and Pent, and (iii) to determine the overall size and sequence of the novel oligosaccharide component of this glycoconjugate. The S-layer glycoprotein sample was therefore prepared for study using different strategies and a range of analytical MS techniques, including GC-MS, MALDI-TOF MS, MS/MS, Q-TOF ES-MS, and MS/MS, also incorporating electron transfer dissociation (ETD) ( <a href="#B39">39</a>), as well as elimination, chemical derivatization, and GC-MS methods following hydrolysis. </p> <div class="" title="sec"> <div class="title" tagx="title" title="title" /> <div class="siteofattachment" title="sec"> <div class="title" tagx="title" title="title">
Site of attachment</div> <p>The ETD MS/MS spectrum of the glycopeptide [M + 4H] <sup>4+</sup> ( <i>m</i>/ <i>z</i> 516.55) at 25 min, belonging to the 20-kDa band of the <i>orf3</i> mutant, is shown in <a href="#F4">Fig. 4</a>. From this spectrum, there is excellent evidence for the substitution of dHexHex on Thr-38 of the sequence determined, supported by several fragment ions. First, considering the "c" series, there is no signal at <i>m</i>/ <i>z</i> 1265 (c <sub>9</sub> + dHexHex), but <i>m</i>/ <i>z</i> 957 (free c <sub>9</sub>) is strong, and there is a strong peak at <i>m</i>/ <i>z</i> 1366 that corresponds to c <sub>10</sub> + dHexHex (1058 + 146 + 162). Second, coming from the C terminus of the peptide, the free z <sub>8</sub> fragment ion is absent ( <i>m</i>/ <i>z</i> 798), but the glycosylated signal is seen at <i>m</i>/ <i>z</i> 1107 (z <sub>8</sub> + 1 + dHexHex) ( <a href="#F4">Fig. 4</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 4.</span> </a> </div>   <p> <b>ETD MS/MS spectrum of <i>m</i>/ <i>z</i> 516.55 <sup>4+</sup> derived from the <i>C. difficile orf3::erm</i> mutant. </b> Peptide fragmentation provides strong evidence for the sequence DILAAQNLTTGAVILNK. c-ions are labeled in <i>orange</i> and z-ions in <i>purple</i>. Following the known ETD mechanism, the cleavage of the N–Cα bond occurs producing C-type (N-terminal) and Z-type (C-terminal) fragments. The mass difference between two adjacent c or z ions provides the mass and identity of the amino acid residue and any substitution, which here identifies the glycan as being attached to threonine 38 (see the text). </p>   </div> </div> <div class="sugarcompositionandlinkage" title="sec"> <div class="title" tagx="title" title="title">
Sugar composition and linkage</div> <p>We next examined the <i>O-</i>linked glycan chain decorating the LMW SLP of <i>C. difficile</i> Ox247 via GC-MS to attempt to determine the sugar composition and, if possible, the linkages. WT Ox247 S-layer protein was extracted and purified by dialysis, reductively eliminated, and hydrolyzed into monosaccharides. Alditol acetate derivatives of the samples were then prepared and analyzed by GC-EI-MS and assigned by comparison with standards, using the GC retention times as well as the specific MS fragmentation patterns of each monosaccharide found ( <a href="#B40">40</a>). From these data, the <i>O-</i>glycan of the S-layer glycoprotein was found to contain principally rhamnose, with lesser amounts of ribose, glucose, and a small amount of galactose ( <a href="http://www.jbc.org/cgi/content/full/RA118.004530/DC1">Fig. S2</a>). For a complex natural product, such as that found here for the S-layer glycan of <i>C. difficile</i> Ox247, containing a mixture of related polymeric components ranging up to approximately 50 sugar residues or more (see below), it was not thought useful to make quantitative calculations of relative ratios. An attempt was then made to carry out linkage analysis on the β-eliminated oligosaccharide, following permethylation, by the standard hydrolysis and re-acetylation method to produce partially methylated alditol acetate. Examination of the mass spectra of each peak eluting from the GC column permitted assignment by comparison with the publicly available database on the Complex Carbohydrate Research Center website (University of Georgia, Atlanta). Positive assignments were possible for terminal ribose, 3-linked rhamnose, 3,4-linked rhamnose, and 4-linked glucose (data not shown), although other unassigned small signals were also observed. </p> <p>The data support the basic structural conclusions in the MS data set of <a href="#F3">Fig. 3</a> regarding the presence of hexose (mainly Glc but also Gal), deoxyhexose (only rhamnose), and pentose (only ribose) units in the oligosaccharide, including the facile loss of terminal pentose-branching units, but apart from the presence of some weak noninterpreted peaks, there were no further clues in the composition and linkage data as to the missing unit(s) needed to identify the nonreducing end structure. </p> </div> <div class="massandsequenceoftheoligosaccharide" title="sec"> <div class="title" tagx="title" title="title">
Mass and sequence of the oligosaccharide</div> <p>Having discovered a novel long–chain <i>O-</i>linked oligosaccharide on LMW SLP together with its precise site of attachment in the protein sequence, the objective was then to define the size of this unusual structure, including its nonreducing end and overall sequence, and for this purpose the oligosaccharide was first removed from the protein backbone by β-elimination. The sample was then derivatized by permethylation to allow examination by MALDI-TOF MS and MS/MS ( <a href="#B41">41</a>, <a href="#B42">42</a>). Interestingly, and surprisingly, in the MS spectrum coming from the 75% acetonitrile fraction of the WT sample acquired using the TOF/TOF instrument in linear MS mode ( <a href="#F5">Fig. 5</a> <i>A</i>), it is possible to see that the glycan chain decorating the S-layer protein of <i>C. difficile</i> Ox247 is much longer than the electrospray (ES-MS) data had indicated (in <a href="#F3">Fig. 3</a>), probably due to the very low internal energy transfer associated with MALDI ionization preserving the higher mass structures. In fact, from the MALDI data in <a href="#F5">Fig. 5</a> <i>A</i> there is evidence of a long glycan chain of the order of 50 sugar residues, with abundant high-mass signals around <i>m</i>/ <i>z</i> 6744 and 7632, but with the highest readily visible peak at <i>m</i>/ <i>z</i> 8520 (peak top), with weaker higher mass signals indicating that even that is not the limit of the structure of this molecule. Note also the weak satellite signals 160 Da lower and higher than the main ones, possibly indicating some heterogeneity with species lacking a ribose or carrying an additional one, because ribose has this mass difference when permethylated. Overall, these MS data illustrate a repeating pentasaccharide unit of mass 886 Da, corresponding to a permethylated [Hex·Rha <sub>3</sub>·Rib] <i> <sub>n</sub> </i> composition where the repeat number " <i>n</i>" varies, leading to different polymer lengths. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 5.</span> </a> </div>   <p> <i>A,</i> MALDI-TOF MS spectrum from <i>m</i>/ <i>z</i> 5000 to 10,000 of the permethylated glycan isolated from the S-layer protein of <i>C. difficile</i> Ox247. The data provide strong evidence for the presence of a long glycan chain of some 50 sugar residues, containing a repeating pentasaccharide unit of 886 Da within its structure (highlighted in <i>red</i>), corresponding to a Glc·Rha <sub>3</sub>·Rib composition, and leading to different polymer lengths. For details see the text. <i>B,</i> MALDI TOF/TOF MS/MS spectrum from <i>m</i>/ <i>z</i> 9 to 7122 of the signal centered around <i>m</i>/ <i>z</i> 6744 in <i>A</i> which in the calibrated MS/MS spectrum is appearing with a peak-top mass of 6736. Differences of hexose, deoxyhexose, and pentose are visible throughout the spectrum, and the sequence can be built up from the reducing end fragment (corresponding to the peptide-attached oligosaccharide sequence determined from the ES-MS data in <a href="#F3">Fig. 3</a>) as described in the text to the Rha-Glc cleavage at <i>m</i>/ <i>z</i> 1509. From there, intervals of a repeating pentasaccharide sequence (886 Da) are clearly determined to <i>m</i>/ <i>z</i> 2395 and higher masses toward the nonreducing end of the structure. Except for unassigned losses from the molecular ion, the reducing end moiety remained elusive at this stage of the study. </p>   </div> <p>Several MS/MS experiments were then carried out by MALDI TOF/TOF with refined mass calibration to define the sequence, and in these spectra the presence of certain "starting numbers" and fragment ion series are clear, as seen for 6744 in <a href="#F5">Fig. 5</a> <i>B</i>. For example, in the low mass range of the spectrum, the ion at <i>m</i>/ <i>z</i> 1509 is the more abundant fragment and is separated from <i>m</i>/ <i>z</i> 1305 by 204 atomic mass units (a permethylated Glc or Gal mass difference), which in turn is separated from <i>m</i>/ <i>z</i> 1131 by 174 atomic mass units (a Rha difference) below that. These data, combined with knowledge of sugar fragmentation mechanisms and the fact that MALDI ionization of permethylated oligosaccharides under these conditions produces M + Na <sup>+</sup> quasi-molecular ions, together also with the fact that the elimination strategy used to free the oligosaccharide from the peptide backbone leads to the production of an open-ring reducing-end residue, then allowed a test calculation to determine whether these signals came from the nonreducing or the reducing end of the overall molecular structure. In this way, <i>m</i>/ <i>z</i> 1509 could be assigned to a reduced and sodiated Glc/Gal <sub>red</sub>–Rha <sub>5</sub>–Rib–Glc/Gal fragment ion, showing that ( <i>a</i>) it is the reducing end fragment (attached originally to the peptide via the hexose) and ( <i>b</i>) it corresponds to the first 8 residues of the 13-residue structure discovered in the Q-TOF ES-MS/MS experiment seen in <a href="#F3">Fig. 3</a>, which was assigned as the branched structure Hex–dHex–dHex–dHex–dHex(Pent)-dHex–Hex-dHe-x-dHex(Pent)–dHex-Hex. Continuing the interpretation of the MS/MS spectrum in <a href="#F5">Fig. 5</a> <i>B</i> from the <i>m</i>/ <i>z</i> 1509 ion toward higher mass, a series of signals are found between that and the next most abundant fragment at <i>m</i>/ <i>z</i> 2395, corresponding to a mass difference of 886.4 Da via <i>m</i>/ <i>z</i> 1683, 2017, and 2191 assigned to a sequence of Rha-Rha(Rib)–Rha–Glc/Gal. This pentasaccharide pattern, created by a cleavage between Rha and Glc (the most abundant hexose in the compositional analysis), which would appear to be a preferred fragmentation pathway, then continues in further increments of 886 mass unit increments to <i>m</i>/ <i>z</i> 3282, <i>m</i>/ <i>z</i> 4171, and <i>m</i>/ <i>z</i> 5057 generating a polymer chain for this particular parent ion mass (6744 in linear MS mode). Note that the mass sufficiency and predominant isotope "peak top" labeled by the instrument software, which is often several mass units higher than <sup>12</sup>C, must be accounted for in the mass annotations observed in such spectra at higher masses. From these data, and the MS/MS analysis of other significantly strong signals in the MALDI MS spectrum, which themselves differ by this 886 repeating sugar units, it is clear that the novel oligosaccharide discovered here is in fact a series of closely related repeat polymer structures spanning a considerable mass range, beyond 5000 Da. </p> <p>Turning to the quasi-molecular ion region of the MS/MS spectrum in <a href="#F5">Fig. 5</a> <i>B</i>, where the "peak top" mass of the quasi-molecular ion (M + Na <sup>+</sup>) is corrected to 6736 Da, the losses observed do not fit for the now-established reducing-end structure and must therefore be coming from the unknown structure at the nonreducing end of the molecule. Auto-peak labeling at these very high masses depends on small fluctuations in <sup>13</sup>C isotope abundances, but averaging the data from this parent ion (6736) and the MS/MS spectrum of the next higher polymer in the series shows that the losses seen are principally −255 (to 6481), −283 (to 6453), and −589 (to 6147), where the 28-atomic mass unit difference between the 6481 and 6453 could be assigned to a probable ring cleavage counterpart (containing C-1 and the ring oxygen atoms of the terminal sugar) to the glycosidic bond cleavage expected for <i>m</i>/ <i>z</i> 6453. These losses were not interpretable on the basis of the standard sugars found in the composition analysis, and therefore the nonreducing end group could not be assigned from the MS data alone. To characterize the precise nature of the nonreducing end moiety, and to specify the absolute stereochemistry and linkages of the oligosaccharide, the structure was then examined by NMR spectroscopy. </p> </div> </div> </div> <div class="nmranalysisofs-layerglycan" title="sec"> <div class="title" tagx="title" title="title">
NMR analysis of S-layer glycan</div> <p>To obtain sufficient SLP glycan for NMR analysis, 36 mg of SLP protein was extracted from bacterial cells using glycine buffer ( <a href="#B43">43</a>). The SLP protein extracts were extensively digested with proteinase K for 48 h, and small nonglycosylated peptide fragments were removed using Amicon filtration (3000 Da cutoff). Digested material, which was retained in the supernatant (&amp;gt;3000 Da), was collected and lyophilized. Further purification of this larger molecular weight, soluble material was made by applying the sample to a Hitrap Q anion-exchange column, and fractions were eluted with 0–1 <div class="sc_UNKNOWN">
m</div> gradient of NaCl. Fractions were desalted on Sephadex 15, and glycan-containing fractions were identified by <sup>1</sup>H NMR. Two glycan-containing fractions were obtained, one fraction eluted in water at the beginning of the NaCl gradient (PS-1), and a second fraction eluted later in ∼0.2 <div class="sc_UNKNOWN">
m</div> NaCl (PS-2). </p> <p>2D NMR spectra (COSY, TOCSY, ROESY, <sup>1</sup>H– <sup>13</sup>C HSQC, <sup>1</sup>H– <sup>13</sup>C HMBC, <sup>1</sup>H– <sup>31</sup>P HMQC) were recorded for PS-1 and PS-2. PS-1 was identical to a previously studied <i>C. difficile</i> polysaccharide (PSII) (data not shown) ( <a href="#B44">44</a>). The second glycan was of unique structure with pentasaccharide repeating units and distinct fragments at both reducing and nonreducing ends. </p> <p>NMR signals were completely assigned for PS-1 (data not shown) and PS-2 ( <a href="#T1">Table 1</a> and <a href="#F6">Fig. 6</a> <i>A</i>). Monosaccharides were identified by COSY, TOCSY, and NOE spectroscopy cross-peak patterns and <sup>13</sup>C NMR chemical shifts. Connections between monosaccharides were determined from transglycosidic NOE and HMBC correlations. For PS-2, the following NOE correlations from H-1 of the monosaccharides were observed: A1, M1,2,3; A′1, M′2,3; B1, L1,2,3; B1, M6; C1, D3; D1, E3; E1, N3; K1, M′4,6; F1, B2,3,4; M1, F4,6; M′1, F4,6; M″1, C4; L1, M4,6; N1, Thr*3. HMBC correlations were in agreement with NOE data: A1, M3; A′1M′3; A″1M″3; B1, L3; C1, D3; D1, E3; E1, N3; K1, M′4; F1, B4; L1, M4; M1, F4; M′1, F4; M″1, C4; N1, Thr-3. These observations allowed the assembly of the structure presented in <a href="#F6">Fig. 6</a> <i>B</i>. </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  <p> <b>NMR data for the S-layer polysaccharide PS-B (ppm from acetone, 600 MHz, 35 °C)</b> </p>  <table> <thead> <tr> <th /> <th>H/C-1</th> <th>H/C-2</th> <th>H/C-3</th> <th>H/C-4</th> <th>H/C-5</th> <th>H/C-6</th> </tr> </thead> <tbody> <tr> <td>Ribf A</td> <td>5.31</td> <td>4.14</td> <td>3.99</td> <td>4.35</td> <td>3.70; 3.76</td> <td /> </tr> <tr> <td /> <td>99.0</td> <td>72.5</td> <td>71.1</td> <td>86.4</td> <td>62.6</td> <td /> </tr> <tr> <td>Ribf A′</td> <td>5.26</td> <td>4.11</td> <td>3.97</td> <td>4.46</td> <td>3.74; 3.74</td> <td /> </tr> <tr> <td /> <td>99.4</td> <td>72.6</td> <td>71.5</td> <td>86.5</td> <td>62.6</td> <td /> </tr> <tr> <td>Ribf A″</td> <td>5.30</td> <td>4.14</td> <td>3.99</td> <td>4.35</td> <td>3.70; 3.76</td> <td /> </tr> <tr> <td /> <td>99.0</td> <td>72.5</td> <td>71.1</td> <td>86.4</td> <td>62.6</td> <td /> </tr> <tr> <td>α-Glc B</td> <td>5.10</td> <td>3.59</td> <td>3.93</td> <td>3.68</td> <td>4.01</td> <td>3.80; 3.89</td> </tr> <tr> <td /> <td>96.5</td> <td>72.5</td> <td>73.9</td> <td>77.9</td> <td>71.6</td> <td>61.6</td> </tr> <tr> <td>α-Rha C</td> <td>5.06</td> <td>4.14</td> <td>3.96</td> <td>3.62</td> <td>3.95</td> <td>1.32</td> </tr> <tr> <td /> <td>103.3</td> <td>71.2</td> <td>70.2</td> <td>83.8</td> <td>68.7</td> <td>17.9</td> </tr> <tr> <td>α-Rha D</td> <td>5.04</td> <td>4.15</td> <td>3.92</td> <td>3.56</td> <td>3.86</td> <td>1.31</td> </tr> <tr> <td /> <td>103.3</td> <td>71.2</td> <td>79.7</td> <td>72.5</td> <td>70.5</td> <td>17.9</td> </tr> <tr> <td>α-Rha E</td> <td>5.03</td> <td>4.17</td> <td>3.91</td> <td>3.58</td> <td>3.88</td> <td>1.30</td> </tr> <tr> <td /> <td>103.3</td> <td>71.2</td> <td>79.4</td> <td>72.5</td> <td>70.4</td> <td>17.9</td> </tr> <tr> <td>β-Rha F</td> <td>4.90</td> <td>4.15</td> <td>3.72</td> <td>3.54</td> <td>3.54</td> <td>1.34</td> </tr> <tr> <td /> <td>101.9</td> <td>71.2</td> <td>72.7</td> <td>83.7</td> <td>72.1</td> <td>17.9</td> </tr> <tr> <td>β-Rha K</td> <td>4.97</td> <td>4.68</td> <td>4.34</td> <td>3.62</td> <td>3.45</td> <td>1.33</td> </tr> <tr> <td /> <td>99.6</td> <td>75.8</td> <td>81.1</td> <td>74.1</td> <td>72.3</td> <td>17.9</td> </tr> <tr> <td>β-Rha L</td> <td>4.77</td> <td>4.24</td> <td>3.68</td> <td>3.44</td> <td>3.43</td> <td>1.34</td> </tr> <tr> <td /> <td>101.0</td> <td>68.1</td> <td>78.9</td> <td>73.5</td> <td>73.6</td> <td>17.9</td> </tr> <tr> <td>β-Rha M</td> <td>4.76</td> <td>4.34</td> <td>3.94</td> <td>3.74</td> <td>3.62</td> <td>1.38</td> </tr> <tr> <td /> <td>101.4</td> <td>68.1</td> <td>75.5</td> <td>77.8</td> <td>72.9</td> <td>18.2</td> </tr> <tr> <td>β-Rha M′</td> <td>4.78</td> <td>4.33</td> <td>3.90</td> <td>3.80</td> <td>3.62</td> <td>1.40</td> </tr> <tr> <td /> <td>101.4</td> <td>68.4</td> <td>75.5</td> <td>78.2</td> <td>72.9</td> <td>18.2</td> </tr> <tr> <td>β-Rha M″</td> <td>4.77</td> <td>4.34</td> <td>3.94</td> <td>3.74</td> <td>3.62</td> <td>1.38</td> </tr> <tr> <td /> <td>101.4</td> <td>68.1</td> <td>75.5</td> <td>77.8</td> <td>72.9</td> <td>18.2</td> </tr> <tr> <td>β-Gal N</td> <td>4.56</td> <td>3.65</td> <td>3.72</td> <td>4.03</td> <td>3.73</td> <td>3.78; 3.78</td> </tr> <tr> <td /> <td>102.0</td> <td>71.3</td> <td>81.3</td> <td>69.6</td> <td>76.3</td> <td>62.1</td> </tr> <tr> <td>Thr*</td> <td /> <td>4.60</td> <td>4.34</td> <td>1.29</td> <td /> <td /> </tr> <tr> <td /> <td /> <td>59.5</td> <td>75.3</td> <td>17.0</td> <td /> <td /> </tr> <tr> <td>Thr</td> <td /> <td>4.00</td> <td>4.22</td> <td>1.34</td> <td /> <td /> </tr> <tr> <td /> <td /> <td>59.7</td> <td>67.3</td> <td>19.9</td> <td /> <td /> </tr> <tr> <td>Gly</td> <td /> <td>3.98; 3.98</td> <td /> <td /> <td /> <td /> </tr> <tr> <td /> <td /> <td>43.8</td> <td /> <td /> <td /> <td /> </tr> <tr> <td>Ala</td> <td /> <td>4.20</td> <td>1.33</td> <td /> <td /> <td /> </tr> <tr> <td /> <td /> <td>52.0</td> <td>18.8</td> <td /> <td /> <td /> </tr> </tbody> </table> </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 6.</span> </a> </div>   <p> <i>A,</i> <sup>1</sup>H– <sup>13</sup>C HSQC spectrum of the polysaccharide from S-layer protein SlpA (PS-2). CH signals are <i>black,</i> and CH <sub>2</sub> signals are <i>green. B,</i> structure of PS-2. </p>   </div> <p>For PS-2, signals from the internal repeating units as well as reducing and nonreducing ends of the polymer were visible. The reducing end was occupied by a tetrapeptide containing Ala, Gly, and two Thr residues. The reducing end sequence of monosaccharides -3-α-Rha-3-α-Rha-3-β-Gal- was different from that of the repeating units. Terminal Gal was linked to one of the Thr residues of the peptide, as assignable from NOE and HMBC data. This is a typical <i>O-</i>linkage of glycan to protein in S-layer proteins. </p> <p>The nonreducing end of this unique glycan was occupied by a β-Rha residue K. This monosaccharide had <sup>1</sup>H and <sup>13</sup>C signals of H,C-2,3 strongly shifted to low field when compared with the expected values for nonsubstituted β-Rha ( <a href="#T1">Table 1</a>). The <sup>31</sup>P spectrum contained a signal at 16.2 ppm, correlating in <sup>1</sup>H– <sup>31</sup>P HSQC to K1 and K3 but not to K2 ( <a href="#F7">Fig. 7</a>). H-3 signal had additional 18 Hz splitting due to H–P coupling. The nature of this phosphate remained unclear at this stage; In addition the ∼10 ppm downfield shift of H-3 signal and the huge coupling constant with <sup>31</sup>P are also difficult to explain. Particularly intriguing is the correlation H-1–P with no trace of H-2–P correlation. At this point, we were unable to assign a definitive structure for this phospho-derivatized rhamnose nonreducing end residue but made a tentative 3-phospho-Rha assignment ( <a href="#F6">Fig. 6</a> <i>B</i>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 7.</span> </a> </div>   <p> <b> <sup>1</sup>H– <sup>31</sup>P HSQC spectrum of PS-2. </b> </p>   </div> </div> <div class="correlationofthemassspectrometryandnmrdata" title="sec"> <div class="title" tagx="title" title="title">
Correlation of the mass spectrometry and NMR data</div> <p>The detection of a phosphate proton resonance in the NMR then allowed a re-examination of the MALDI MS/MS data to correlate masses observed with a possible phosphate-substituted rhamnose nonreducing end moiety. First, in the interpretation of the data in the quasi-molecular ion region of the permethylated sample in <a href="#F5">Fig. 5</a> <i>B</i>, the loss of 283 mass units to give <i>m</i>/ <i>z</i> 6453 can now be rationalized as the loss of a dimethyl-phospho-rhamnose terminal unit via glycosidic bond cleavage with hydrogen transfer. <i>m</i>/ <i>z</i> 6481 is the ring-cleavage fragment retaining H–C=O on the reducing end referred to earlier. The remaining abundant ion in this region at <i>m</i>/ <i>z</i> 6147 is then assigned as the equivalent ring fragment to 6481 at the next residue, which is assigned as a branched Rha-(Rib), 334-atomic mass unit difference. </p> <p>Importantly, this assignment of a terminal Rha-phosphate moiety (which from sub-fragment data could be positioned at the 2- or 3-position in the ring) did not correlate with the overall mass data coming from both the MALDI MS of permethylated derivatives or from the data on the intact glycopeptide itself, and this now required explanation. <a href="#F9">Fig. 9</a> shows the negative ion MALDI MS/MS spectrum of the native glycopeptide isolated by proteinase K digestion in preparing the NMR sample above. This spectrum was determined by collisionally activated decomposition of the parent ion observed at <i>m</i>/ <i>z</i> 6563.6 (peak top) for the [M − H] <sup>−</sup> species. In arithmetic terms, this would correspond to a <sup>12</sup>C mass of 6560.6 for the glycopeptide. Because in other (electrospray) data (not shown) we had definitively assigned the peptide portion of the molecule as the sequence TTGA (with no ragged ends), the theoretical <sup>12</sup>C mass for the glycopeptide, containing a terminal Rha-phosphate and with the repeat unit <i>n</i> = 7, is calculated as 6578.4. Although mass accuracy in this difficult type of high-mass negative ion MS/MS experiment on small amounts of material can be questionable, within plus or minus a mass unit or so, it is nevertheless clear that the intact mass data on the native glycopeptide does not correlate with a phosphate substitution of the terminal rhamnose and is roughly 18 mass units lower than the theoretical mass overall. </p> <p> <a href="#F8">Fig. 8</a> explains the solution to this conundrum, via interpretation of the important major fragment ions at <i>m</i>/ <i>z</i> 225 and 631 in the MS/MS spectrum of 6564 [M − H] <sup>−</sup>. <i>m</i>/ <i>z</i> 225 actually provides the evidence for the source of the 18-atomic mass unit difference between experimental and theoretical masses discussed above, because it is 18 atomic mass units below what would be expected for a phosphate-substituted rhamnose residue. This mass was assigned as cyclophospho-rhamnose as shown in <a href="#F8">Fig. 8</a>, and the fragment ion itself derives from glycosidic bond cleavage with retention of the glycosidic oxygen and concomitant hydrogen transfer to that atom from the second sugar in the chain. The next significant fragment is seen at <i>m</i>/ <i>z</i> 631, which is due to cleavage and charge retention on the nonreducing end again, but is now between a Rha(Rib) and Glc residue with retention of the glycosidic oxygen on the reducing end (uncharged) fragment with hydrogen transfer from the Rha to that oxygen. Interestingly, from this point, the principal fragment ions are all due to similar Rha–Glc cleavages along the oligosaccharide backbone, separated by the repeat pentasaccharide unit Rha–Rha(Rib)–Rha–Glc to give <i>m</i>/ <i>z</i> 1363, 2095, 2829, 3562, and 4295. A single (less favored) reducing end fragment, which corresponds to the partial sequence identified in <a href="#F3">Figs. 3</a> and <a href="#F6">6</a>, is observed at <i>m</i>/ <i>z</i> 1533 in this spectrum, whereby the negative charge would most probably reside on the peptide carboxylate anion. The 6564.6 peak top assignment in the TOF MS calculates for a <sup>12</sup>C value of 6559.6 for [M − H] <sup>−</sup> and thus a molecular mass of 6560.6 for the glycopeptide, which is close to the theoretical calculated mass for the new structure. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 8.</span> </a> </div>   <p> <b>Negative ion MALDI TOF/TOF MS/MS spectrum of the native glycopeptide of Ox247 isolated by proteinase K digestion. </b> The data show the detailed fragmentation produced from a quasi-molecular ion signal found in the MS spectrum at (peak-top) <i>m</i>/ <i>z</i> 6563.6, and clearly identify (i) a nonreducing end structure 18 mass units below that expected for phospho-rhamnose and (ii) an intact mass some 18 atomic mass units lower than expected for the overall predicted structure in <a href="#F7">Fig. 7</a>. These data suggest a cyclophospho-rhamnose nonreducing terminus. For detailed interpretation see the fragmentation cartoon above the spectrum and the text. </p>   </div> <p>Interestingly, the apparent anomaly of the MALDI TOF/TOF data on the β-eliminated permethylated samples correlating with a dimethylphosphate-substituted terminal rhamnose moiety is explained by the chemistry of the permethylation reaction and its predictable effect on a cyclic phosphate substitution of this type. The rhamnose cyclophosphate would be subject to nucleophilic attack by hydroxide anion at either of the sugar ring attachment points (C-2 or C-3), which would lead to ring opening to produce a linear phosphate substitution with two hydroxyls for subsequent methylation. </p> <p>Following the assignment of the overall masses of the oligosaccharide polymers, together with the specific nonreducing terminal mass assignable as corresponding to a possible cyclophospho-deoxyhexose in the corresponding negative ion MALDI TOF/TOF MS/MS data in <a href="#F8">Fig. 8</a>, the NMR data in <a href="#F6">Figs. 6</a> <i>A</i> and <a href="#F7">7</a> were re-evaluated for evidence of such a moiety. Cyclophosphate substituents have been found on some naturally occurring sugars or have been synthesized, and they appear to be characterized by chemical <sup>31</sup>P shifts of 18–29 ppm and large coupling constants from 12 to 22 Hz. For example, cyclophosphate at positions 1,2 of mannose demonstrated a chemical shift of 19.7 ppm ( <a href="#B45">45</a>), whereas methyl- <div class="sc_UNKNOWN">
l</div>-glycero-α- <div class="sc_UNKNOWN">
d</div>-manno-heptopyranoside-6,7-cyclophosphate had a <sup>31</sup>P chemical shift of 18.3 ppm ( <a href="#B46">46</a>). The 16 ppm chemical shift and 18 Hz coupling observed in the S-layer structure are within these ranges and allow an assignment of a 2,3 cyclophosphate on the terminal rhamnose residue Lys. The overall structure of the glycan attached to SlpA predicted from the NMR and MS interpretations is depicted in <a href="#F9">Fig. 9</a>. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 9.</span> </a> </div>   <p> <b>Deduced overall structure of the glycan attached to SlpA in Ox247 from MS and NMR data. </b> The structure contains ∼8 copies of a repeat containing Rha, Glc, and Rib <i>f</i> attached via a linker containing Rha and Gal and attached via an <i>O-</i>linkage to Thr-38 of SlpA. The identity of the capping residue is 2,3 cyclophosphoryl-rhamnose. The genes identified as expressing enzymes mediating steps in the pathway are indicated. </p>   </div> </div> <div class="phenotypesofglycosylation-defectivemutants" title="sec"> <div class="title" tagx="title" title="title">
Phenotypes of glycosylation-defective mutants</div> <p>To investigate the functional relevance of S-layer glycosylation, a variety of experiments were carried out on the physiology and behavior of <i>C. difficile</i> Ox247 compared with mutants in the glycosylation locus. </p> <p>Ox247 and mutants in <i>orf2, orf3, orf4, orf7, orf16,</i> and <i>orf19</i> grew normally and did not exhibit any growth defect in liquid media (data not shown). However, we noticed that the cells of every mutant were shorter than the WT ( <a href="#F10">Fig. 10</a>). The average cell of WT Ox247 cells was 6.52 μm, with the mutants having average lengths of 5.34 μm ( <i>orf2</i>), 5.79 μm ( <i>orf3</i>), 5.56 μm ( <i>orf4</i>), 5.68 μm ( <i>orf7</i>), 5.75 μm ( <i>orf16</i>), and 5.14 μm ( <i>orf19</i>). In all mutants (except <i>orf7</i> where complementation was not possible), the cell-length defect in the mutant was complemented by a plasmid expressing the WT gene, with a significant ( <i>p</i> &amp;lt; 0.001) increase in mean cell length occurring ( <a href="#F10">Fig. 10</a> <i>B</i> and <a href="http://www.jbc.org/cgi/content/full/RA118.004530/DC1">Fig. S4, <i>A–D</i> </a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 10.</span> </a> </div>   <p> <b>S-layer glycosylation affects cell length.</b> Box and whisker plots of cell length of Ox247 and S-layer glycosylation mutants are shown. <i>A,</i> Ox247 and mutants. <i>B,</i> Ox247, its <i>orf2</i> mutant and the <i>orf2</i> mutant containing pAAM008 expressing WT <i>orf2</i> gene. Strains were grown overnight in BHI broth, and cell lengths were measured from phase-contrast micrographs of 450 cells per strain, using ImageJ. Analysis of variance revealed a statistically significant difference between Ox247 and all mutants (***, <i>p</i> &amp;lt; 0.001). Complementation of the mutants in <i>orf3</i>, <i>orf4</i>, <i>orf16,</i> and <i>orf19</i> is shown in <a href="http://www.jbc.org/cgi/content/full/RA118.004530/DC1">Fig. S4, <i>A–D</i> </a>. The Ox247 and <i>orf2::erm</i> data points in <i>A</i> are reused in <i>B</i>, because the data were acquired from the same experiment. </p>   </div> <p>A key phenotype of the <i>Clostridium</i> genus is the ability to differentiate to form heat-resistant spores ( <a href="#B4">4</a>). We found that although Ox247 produced spores at a normal rate, the <i>orf2</i> and <i>orf19</i> mutants were deficient in this process ( <a href="#F11">Fig. 11</a>). After 120 h in liquid culture, Ox247 produced ∼10 <sup>6</sup> spores/ml. In contrast, the Ox247 <i>orf2</i> mutant produced no detectable spores at all during this period. Interestingly, the <i>orf2</i> mutant containing the WT <i>orf2</i> gene behaved similarly to the <i>orf2</i> mutant, producing no spores, despite this strain being able to complement the S-layer cell wall protein phenotype as determined by SDS-PAGE ( <a href="#F2">Fig. 2</a>). The sporulation-defective phenotype could not be complemented by plasmids expressing <i>orf2</i> from either a constitutive promoter ( <i>P <sub>cwp</sub> </i>; <a href="#F11">Fig. 11</a> <i>C</i>) or from the anhydrotetracycline-inducible promoter, <i>P <sub>tet</sub> </i> (data not shown). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 11.</span> </a> </div>   <p> <b>Sporulation is compromised in glycosylation-deficient mutants.</b> Overnight cultures of Ox247 ( <i>A</i>), Ox247 <i>orf2::erm</i> ( <i>B</i>), Ox247 <i>orf2::erm</i> (PAAM008) ( <i>C</i>), and R20291 ( <i>D</i>) were grown in BHIS broth, and sub-cultured three times. After the final sub-culture (time = 0), samples were taken and assayed for total cells and spores. The R20291 culture was included as a reference to a well-characterized strain. Date are the average ± S.E. </p>   </div> <p>Several studies have implicated the S-layer of <i>C. difficile</i> to play a key role in its adhesion to host intestinal cells ( <a href="#B47">47</a>, <a href="#B48">48</a>). To investigate whether glycosylation of the S-layer affects adhesion to intestinal cells, adhesion of Ox247 and the <i>orf2</i> mutant to intestinal Caco-2 cells was compared. As shown in <a href="#F12">Fig. 12</a>, adhesion was greater in Ox247 compared with its <i>orf2</i> mutant cells. A slight restoration in adhesion was observed by the Ox247 <i>orf2::erm</i> mutant when complemented with pAAM008 expressing Orf2, but this was not statistically significant. Additionally, we assessed the effect of S-layer glycosylation on the ability to form biofilms. A small but significant increase in biofilm formation was observed in the <i>orf2</i> mutant that lacks the glycosylated S-layer ( <a href="#F12">Fig. 12</a>, <i>B</i> and <i>C</i>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 12.</span> </a> </div>   <p> <b>Adhesion and biofilm formation of <i>C. difficile</i> Ox247. </b> <i>A,</i> adhesion to Caco2 cells. Caco2 cell monolayers were infected with <i>C. difficile</i> strains with a multiplicity of infection of 5 and incubated for 2 h. After washing, the viable adherent bacteria were measured by growth on BHIS agar. Data are representative of seven individual assays, and the S.E. is shown. ***, <i>p</i> &amp;lt; 0.001 when compared with Ox247. <i>B</i> and <i>C,</i> biofilm formation of Ox247 and the <i>orf2</i> mutant was assessed at 24 h ( <i>B</i>) and 72 h ( <i>C</i>). *, <i>p</i> &amp;lt; 0.05. </p>   </div> <p>Several studies have implicated cell-surface components in the cellular behavior of <i>C. difficile</i>. For example, overexpression of the cell wall protein CwpV induces aggregation of cells ( <a href="#B28">28</a>), as does increasing the intracellular level of cyclic di-GMP ( <a href="#B29">29</a>). We saw no difference between Ox247 and any of the S-layer glycosylation mutants in aggregation or in flagellar-mediated motility (data not shown). </p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p> <i>C. difficile</i> is an important bacterial pathogen that presents major healthcare problems around the world ( <a href="#B1">1</a>, <a href="#B2">2</a>). Pathogenic strains produce toxins that are the major virulence factors, but cell-surface components are involved in survival <i>in vivo</i> and in the transition between a sessile and motile life and between planktonic and biofilm growth. The <i>C. difficile</i> S-layer is known to vary between strains, with at least 13 SLCTs identified to date ( <a href="#B11">11</a>, <a href="#B49">49</a>). Recently, a promising family of novel antimicrobials, diffocins, have been described that have high activity against <i>C. difficile</i> ( <a href="#B50">50</a>). Diffocins, which are derived from phage tails, are highly specific for certain <i>C. difficile</i> strains and have been shown to target specifically the <i>C. difficile</i> S-layer protein SlpA ( <a href="#B49">49</a>). The discovery that SLCT-11 is glycosylated could have important consequences for the use of diffocins to effectively target the S-layer. </p> <p>Our study reveals that the S-layer protein of SLCT-11 strains is unique in being a glycoprotein displaying a complex glycan on its surface. The organizational structure of the glycan bears some resemblance to previously described glycans in Gram-positive bacteria ( <a href="#B29">29</a>). MS and NMR analysis revealed the structure of the glycan and showed that it is composed of three domains: (i) a core tetrasaccharide with the sequence -4-α-Rha-3-α-Rha-3-α-Rha-3-β-Gal-; (ii) a repeating pentasaccharide with the sequence -4-β-Rha-4-α-Glc-3-β-Rha-4- (α-Rib-3-)β-Rha-; and (iii) a terminal 2,3 cyclophosphoryl-rhamnose attached to a ribose-branched sub-terminal rhamnose. The overall mass of the glycan identified by MS suggests the presence of at least ∼8 repeat units linked in tandem and attached at the reducing end to the tetrasaccharide core that is <i>O-</i>linked to Thr-38 in the SlpA protein ( <a href="#F9">Fig. 9</a>.) Thr-38 is present within the LMW SLP of the S-layer protein, which is located on the external face of the S-layer, whereas the HMW SLP is located on the inner face of the S-layer and is in contact with the underlying polysaccharide–peptidoglycan layer ( <a href="#B26">26</a>). </p> <p>Our genetic analysis combined with bioinformatics analysis and MS has cast light on the functions of several of the genes in the glycosylation cluster. The <i>orf2</i> mutants fail to synthesize a glycan as evidenced by the presence of the 20-kDa form of LMW SLP and the lack of higher molecular weight proteins migrating at ∼45 kDa. Consistent with bioinformatic analysis, Orf2 is predicted to be a undecaprenyl phosphate galactose phosphotransferase that would transfer galactose to undecaprenyl phosphate on the inner face of the cytoplasmic membrane. A similar phenotype is seen with <i>orf19</i> mutants, consistent with its role as a ligase, acting on the external face of the membrane to transfer the complete glycan chain to the S-layer at position Thr-38. The <i>orf3</i> mutants produce an S-layer with a truncated glycan, containing dHex-Hex as visualized by MS and confirmed by NMR analysis of the WT glycan to be α-Rha-3-β-Gal. Thus, Orf 3 is predicted to be a 1,3-rhamnose rhamnosyltransferase responsible for the addition of 1–3 rhamnose residues to the core reducing end tetrasaccharide glycan. MS analysis of the <i>orf4</i> mutant showed glycans that consistently lacked ribose, pointing to this protein being a 1,3-ribosyltransferase, transferring the Ribf to the Rha ( <a href="http://www.jbc.org/cgi/content/full/RA118.004530/DC1">Fig. S5</a>). Finally <i>orf7</i> mutants produced a truncated oligosaccharide, containing pentose dHex <sub>5</sub>Hex. This suggests that Orf7 may be a glycosyltransferase that attaches glucose to the rhamnose in the repeating structure of the glycan. </p> <p>The machinery described here for synthesis, assembly, and transport of the glycan and its ligation to the S-layer protein most resembles those described for glycosylation of S-layers from <i>G. stearothermophilus</i> and <i>P. alvei</i> ( <a href="#B27">27</a>, <a href="#B29">29</a>). The glycan is built on the cytoplasmic side of the membrane using endogenous undecaprenyl phosphate as an acceptor, first by addition of a core tetrasaccharide, followed by addition of repeating polysaccharide, and finally a capping moiety that presumably terminates synthesis. The glycan is exported via a two-component ABC transporter and ligated to the protein on the external surface of the cell. Termination of glycan synthesis in the system we describe here appears to involve incorporation of a modified phosphate group on the nonreducing end of the capping group. Phosphate was identified at the nonreducing end of the O9 <i>O</i>-polysaccharide of <i>E. coli</i> and is required for termination of glycan chain elongation and for subsequent export via the ABC transporter ( <a href="#B51">51</a>). To our knowledge, the presence of phosphate as a constituent of the terminal group of an S-layer glycan has not previously been described, and we speculate that this moiety serves a similar purpose to that in O9a polysaccharide biosynthesis. </p> <p> <i>C. difficile</i> strains can glycosylate other surface structures, notably flagella, by addition of <i>O</i>-linked glycans. In some strains, exemplified by strain 630, a phosphorylated <i>N</i>-methyl- <div class="sc_UNKNOWN">
l</div>-threonine–substituted GlcNAc is present. In ribotype 027 strains, a distinctive modification is found consisting of a novel sulfonated peptidylamido-glycan containing rhamnose residues ( <a href="#B52">52</a>). </p> <p>Approximately 10–15% of <i>C. difficile</i> strains carry SLCT-11, including the genes that compose the glycan biosynthetic machinery ( <a href="#B11">11</a>), and therefore they are expected to display the glycan we describe here. Our previous study showed that other strains of <i>C. difficile</i> do not contain a glycosylated S-layer, consistent with their lack of glycosylation machinery ( <a href="#B53">53</a>). These nonglycosylated strains were of ribotypes 001, 012, 010, 016, 017, 027, and 053. However, because the S-layer cassette is thought to recombine by homologous recombination, there is not an absolute association of SLCT with ribotype. The strain we have described here, Ox247, is of ribotype 005 and is within clade 1. However, further analysis of strains within clade 3 has revealed a predominance of strains with SLCT-11, <a href="#FN8"> <sup>9</sup> </a> suggesting that this glycosylated S-layer does to some extent associate more stably with some strains than others. </p> <p>Analysis of mutants defective in glycosylation of SlpA revealed some interesting phenotypes, suggesting a possible role for the glycan in the fitness and physiology of <i>C. difficile.</i> Glycan-negative strains were consistently unable to form spores, suggesting a role for the S-layer in sporulation. Glycan-deficient cells were also shorter than WT cells suggesting a link between cell division or cell architecture and the presence of a complete S-layer. Glycan-deficient cells were also less proficient at adhering to the intestinal Caco2 cell line, suggesting the glycan may be involved in adhesion to gut cells and colonization <i>in vivo</i> during infection. </p> </div> <div class="methods" title="methods"> <div class="title" tagx="title" title="title">
Experimental procedures</div> <div class="bacterialgrowthandculture" title="sec"> <div class="title" tagx="title" title="title">
Bacterial growth and culture</div> <p> <i>C. difficile</i> was routinely cultured in TY broth and on BHIS agar (brain heart infusion agar supplemented with 0.1% <div class="sc_UNKNOWN">
l</div>-cysteine and 5 mg/ml yeast extract) at 37 °C without shaking, under anaerobic conditions (80% N <sub>2</sub>, 10% CO <sub>2</sub>, 10% H <sub>2</sub>.) Media were supplemented where appropriate with thiamphenicol (15 μg/ml), cycloserine (250 μg/ml), cefoxitin (81 μg/ml), or lincomycin (20 μg/ml) or erythromycin (5 μg/ml). All <i>E. coli</i> strains were grown using Luria-Bertani broth or agar supplemented, where necessary, with 12.5 mg/ml chloramphenicol (Sigma) at 37 °C. Strains and plasmids used in this study are shown in <a href="http://www.jbc.org/cgi/content/full/RA118.004530/DC1">Table S2</a>. </p> </div> <div class="genetictechniques" title="sec"> <div class="title" tagx="title" title="title">
Genetic techniques</div> <p> <i>C. difficile</i> genomic DNA was isolated as described previously ( <a href="#B54">54</a>) and plasmids were transferred from <i>E. coli</i> CA434 to <i>C. difficile</i> strains by conjugation as described previously ( <a href="#B30">30</a>). PCRs were performed with KOD Hot Start polymerase (Novagen) using primers as detailed in <a href="http://www.jbc.org/cgi/content/full/RA118.004530/DC1">Table S2</a>. For construction of insertional mutants in <i>C. difficile,</i> the ClosTron method was used ( <a href="#B30">30</a>) with modifications. Retargeted plasmids and the relevant oligonucleotides were designed using the algorithm at ClosTron, and the plasmids were synthesized by DNA 2.0. Plasmids were transformed into <i>E. coli</i> CA4343 and then transferred by conjugation to <i>C. difficile</i> Ox247, where transconjugants were selected on BHIS agar containing 15 μg/ml thiamphenicol (Sigma). Insertion of the intron into the chromosome was by selection of erythromycin (5 μg/ml), and colonies were screened by PCR and DNA sequencing. Mutants were confirmed to have only one insertion by Southern blot analysis ( <a href="http://www.jbc.org/cgi/content/full/RA118.004530/DC1">Fig. S3</a>). Mutants were complemented by introducing the WT gene under the control of the constitutive P <i> <sub>cwp2</sub> </i> promoter or the anhydrotetracycline-inducible promoter P <i> <sub>tet</sub> </i>, as described previously ( <a href="#B26">26</a>). </p> </div> <div class="s-layerproteinpurification" title="sec"> <div class="title" tagx="title" title="title">
S-layer protein purification</div> <p>S-layer proteins and associated cell wall proteins were prepared by growth of <i>C. difficile</i> strains overnight in 50 ml of BHI broth. Cells were centrifuged, washed in 0.1 volume of PBS, and resuspended in 0.01 volume of 0.2 <div class="sc_UNKNOWN">
m</div> glycine, pH 2.2. Cells were mixed by rotation for 30 min and centrifuged, and the supernatant containing the SLPs was removed and neutralized by the addition of 2 <div class="sc_UNKNOWN">
m</div> Tris base. </p> </div> <div class="sporulationandcell-lengthassays" title="sec"> <div class="title" tagx="title" title="title">
Sporulation and cell-length assays</div> <p>To enumerate spores, overnight cultures of <i>C. difficile</i> grown in BHIS medium supplemented with thiamphenicol, where necessary, were used to inoculate 10 ml of fresh BHIS medium. At intervals, samples of culture were taken and divided. To determine the total number of colony-forming units, one sample was serially diluted and plated on BHIS medium plus 0.1% taurocholate (Sigma). To determine the number of spores, the second sample was heat-killed by incubation for 25 min at 65 °C prior to plating on BHIS medium plus 0.1% taurocholate. The number of spores was subtracted from the total cell counts to give the vegetative cell numbers. </p> <p>To analyze cell length, <i>C. difficile</i> strains were grown in BHIS broth at 37 °C in an anaerobic chamber for 12 h. 0.5 ml of each culture was centrifuged at 4000 × <i>g</i> for 2 min. Following gentle resuspension in 150 μl of 1× PBS, 3 μl of culture was placed on top of 1.2% agarose pads and left to dry for 15 min. Slides were visualized using an Eclipse E600 microscope (Nikon), using a Plan Fluor ×100 objective lens, using oil immersion. Images were captured using a Retiga 2000R camera, using the QCapture Pro Software (QImaging), and images were processed using the ImageJ software. </p> </div> <div class="biofilmandadhesionassays" title="sec"> <div class="title" tagx="title" title="title">
Biofilm and adhesion assays</div> <div class="" title="sec"> <div class="title" tagx="title" title="title" /> <div class="biofilmformation" title="sec"> <div class="title" tagx="title" title="title">
Biofilm formation</div> <p>Single colonies <i>C. difficile</i> grown on BHIS agar were used to inoculate pre-reduced BHIS broth. After overnight incubation, 200 μl of each culture at OD <sub>600</sub> 0.1 was used to inoculate 1.8 ml of pre-reduced BHIS broth on a 24-well plate (Costar).The plates were incubated anaerobically at 37 °C. After 24 or 72 h, the media were gently removed from each well, and the wells were gently washed with 1 ml of pre-reduced PBS. Biofilms were stained with 500 μl of 0.1% filter-sterilized crystal violet for 30 min. 500 μl of 100% methanol was added to solubilize the crystal violet stain followed by incubation for 30 min at room temperature. The OD <sub>595</sub> of the solution was then measured using a microplate reader (Bio-Rad). Three biological replicates were performed for each strain, and the assay was replicated six times. </p> </div> <div class="adhesionassay" title="sec"> <div class="title" tagx="title" title="title">
Adhesion assay</div> <p>TC7 cells, a sub-clone of the Caco-2 cell line, were maintained using Dulbecco's modified Eagle's medium (DMEM), containing 15% fetal bovine saline (ThermoFisher Scientific), 1% minimum Eagle's medium nonessential amino acids (Life Technologies, Inc.), and 2 m <div class="sc_UNKNOWN">
m</div> GlutaMAX (ThermoFisher Scientific). Monolayers were used at 8 days post-confluency, and cells were given nonsupplemented DMEM (containing no fetal bovine serum) 24 h prior to infection. Cells at passage 30–40 were washed with PBS prior to infection. <i>C. difficile</i> bacteria were prepared at a multiplicity of infection of 5 in pre-reduced and pre-warmed DMEM. 1 ml of bacterial cells was added to each well of TC7 cells and incubated for 2 h. Bacterial suspensions were then removed, and cells were washed three times with pre-reduced BHIS media. Following vigorous pipetting, TC7 cells with adherent bacteria were removed from the wells, gently vortexed to remove cell clumping, and enumerated on BHIS agar. </p> </div> </div> </div> <div class="massspectrometry" title="sec"> <div class="title" tagx="title" title="title">
Mass spectrometry</div> <div class="" title="sec"> <div class="title" tagx="title" title="title" /> <div class="discoveryandglycoproteomicanalyses" title="sec"> <div class="title" tagx="title" title="title">
Discovery and glycoproteomic analyses</div> <p>For discovery and glycoproteomic experiments, the bands of interest from polyacrylamide gels were in-gel digested with trypsin, and the tryptic peptide/glycopeptide digest mixtures, from S-layer <i>C. difficile</i> Ox247 both WT and mutants, were analyzed directly by either micro- or nano-LC-ES-MS and MS/MS using Q-TOF mass spectrometers as described previously ( <a href="#B31">31</a> <a href="#B32">–</a> <a href="#B38">38</a>): ( <i>a</i>) a reverse-phase nano-HPLC system ((15 cm × 75 mm inner diameter) PepMap column (LC Packings, Dionex)) connected to a Q-STAR Pulsar (ABI/MDS SCIEX) and/or ( <i>b</i>) a reverse-phase LC Acquity column (BEH C-18 1 × 50 mm, Waters) connected to a Xevo G2 (Waters) instrument. All interpretations of glycoproteomic data were made manually by visual inspection ( <a href="#B36">36</a> <a href="#B37">–</a> <a href="#B38">38</a>). The site of glycan substitution was determined using electron capture dissociation on <i>C. difficile</i> WT samples, and both for this work and studies on the <i>orf3::erm</i> mutant a Synapt G2-S instrument (Waters) fitted with a nanoAcquity Ultra Performance LC C <sub>18</sub> column (15 cm length, 75 μm inner diameter) was used. ETD was achieved using glow discharge and <i>m</i>-nitrobenzyl alcohol (Sigma) to supercharge the peptides before mass spectrometric detection ( <a href="#B39">39</a>). </p> </div> <div class="sugarcompositionandlinkage" title="sec"> <div class="title" tagx="title" title="title">
Sugar composition and linkage</div> <p>GC-MS compositional analysis of alditol acetates, following TFA hydrolysis, borodeuteride reduction, and re- <i>N</i>-acetylation was carried out as described previously ( <a href="#B36">36</a>). Linkage analysis was achieved by the study of partially methylated alditol acetate derivatives as described ( <a href="#B40">40</a>). GC-MS analysis was carried out on a Bruker SCION SQ 456 instrument. </p> </div> <div class="oligosaccharideandglycopeptidemassandsequenceanalysis" title="sec"> <div class="title" tagx="title" title="title">
Oligosaccharide and glycopeptide mass and sequence analysis</div> <p>The preparation, purification, and permethylation of glycans β-eliminated from the protein backbone was carried out as described previously ( <a href="#B41">41</a>, <a href="#B42">42</a>). MALDI-TOF MS of both free and derivatized oligosaccharides and of free intact glycopeptides was carried out in linear and reflectron modes in both positive and negative ionization, as appropriate, using a 4800 MALDI TOF/TOF (ABI SCIEX) mass spectrometer. For MALDI TOF/TOF MS/MS selected quasi-molecular ions of either permethylated or underivatized species observed in the MS spectra were subjected to collision-induced dissociation, at a collision energy of 1 kV with argon as the collision gas ( <a href="#B41">41</a>, <a href="#B42">42</a>). </p> </div> </div> </div> <div class="large-scaleglycanpurificationfornmranalysis" title="sec"> <div class="title" tagx="title" title="title">
Large-scale glycan purification for NMR analysis</div> <p> <i>C. difficile</i> Ox247 was grown in BHIS broth (6× 1 liter) for 16 h at 37 °C in an anaerobic chamber. Bacterial cells were harvested, and the cell pellets were resuspended in 0.2 <div class="sc_UNKNOWN">
m</div> glycine buffer and incubated at room temperature for 20 min with gentle mixing. Cells were removed by centrifugation for 15 min in an Eppendorf microcentrifuge, and glycine supernatants were neutralized with 1 <div class="sc_UNKNOWN">
m</div> Tris. Following dialysis against dH <sub>2</sub>O for 48 h, the S-layer protein extract was digested with proteinase K (Sigma) in 10 m <div class="sc_UNKNOWN">
m</div> NaPO <sub>4</sub>, pH 8.0, at 37 °C for 48 h (5:1 ratio S-layer protein/proteinase K). The digested samples were then subjected to Amicon filtration (3000-Da cutoff), and the retentate was lyophilized. The lyophilized sample was resuspended in dH <sub>2</sub>O and further purified as described below. </p> <p>Polysaccharides were purified by anion-exchange chromatography on Hitrap Q column in a linear gradient from water to 1 <div class="sc_UNKNOWN">
m</div> NaCl over 1 h with UV detection at 220 nm and a spot test on TLC plate with development by dipping in 5% H <sub>2</sub>SO <sub>4</sub> in ethanol and heating with a heat gun until brown spots became visible, and products were desalted by gel chromatography. Samples were desalted on Sephadex G-15 column (1.6 × 60 cm) in 1% AcOH with refractive index detector. </p> </div> <div class="nmrspectroscopy" title="sec"> <div class="title" tagx="title" title="title">
NMR spectroscopy</div> <p>NMR experiments were carried out on a Bruker AVANCE III 600 MHz ( <sup>1</sup>H) spectrometer with a 5-mm Z-gradient probe with acetone internal reference (2.225 ppm for <sup>1</sup>H and 31.45 ppm for <sup>13</sup>C) using standard pulse sequences cosygpprqf (gCOSY), mlevphpr (TOCSY, mixing time 120 ms), roesyphpr (ROESY, mixing time 500 ms), hsqcedetgp (HSQC), hsqcetgpml (HSQC-TOCSY, 80 ms TOCSY delay), and hmbcgplpndqf (HMBC, 100-ms long-range transfer delay). Resolution was kept &amp;lt;3 Hz/point in F2 in proton–proton correlations and &amp;lt;5 Hz/pt in F2 of H–C correlations. The spectra were processed and analyzed using the Bruker Topspin 2.1 program. </p> </div> </div> <div class="authorcontributions" title="sec"> <div class="title" tagx="title" title="title">
Author contributions</div> <p>E. R., L. B., M. P., A. A., S. M. L., and A. D. data curation; E. R., M. P., A. A., E. V., H. M., S. M. L., A. D., and N. F. F. formal analysis; E. R., L. B., M. P., A. A., E. V., H. M., A. D., and N. F. F. investigation; E. R., L. B., M. P., E. V., H. M., and S. M. L. methodology; E. R., L. B., E. V., H. M., B. W., S. M. L., A. D., and N. F. F. writing-original draft; H. M., S. M. L., A. D., and N. F. F. supervision; H. M., B. W., S. M. L., A. D., and N. F. F. writing-review and editing; B. W., S. M. L., A. D., and N. F. F. conceptualization; B. W., S. M. L., A. D., and N. F. F. funding acquisition; B. W., A. D., and N. F. F. project administration. </p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supporting Information</div>  <div class="media" title="media"> <a href="supp_293_47_18123__index.html">LINK</a> </div> <div class="media" title="media"> <a href="supp_RA118.004530_138905_1_supp_211971_pfx22m.pdf">LINK</a> </div> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-COI-statement" title="COI-statement"> <p> <span class="named-content" title="named-content">The authors declare that they have no conflicts of interest with the contents of this article </span>. </p> </div> <div class="fn-type-supplementary-material" title="supplementary-material"> <p>This article contains <a href="http://www.jbc.org/cgi/content/full/RA118.004530/DC1">Figs. S1–S5 and Tables S1–S2</a>. </p> </div> <div class="fn-type-other" title="other"> <span class="label" tagx="label" title="label">9</span> <p>A. Shaw and B. Wren, manuscript in preparation.</p> </div> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label">8</span> <p>The abbreviations used are: </p><div class="def-list" title="def-list"> <div class="def-item" title="def-item"> <div class="term" title="term">
CDI</div> <div class="def" title="def"> <p> <i>C. difficile</i> infection </p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SLP</div> <div class="def" title="def"> <p>S-layer protein</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
ETD</div> <div class="def" title="def"> <p>electron transfer dissociation</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
LMW</div> <div class="def" title="def"> <p>low-molecular-weight</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
HMW</div> <div class="def" title="def"> <p>high-molecular-weight</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SLC</div> <div class="def" title="def"> <p>S-layer cassette</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SLCT</div> <div class="def" title="def"> <p>SLC type</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
dH <sub>2</sub>O </div> <div class="def" title="def"> <p>distilled H <sub>2</sub>O </p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
DMEM</div> <div class="def" title="def"> <p>Dulbecco's modified Eagle's medium</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
HSQC</div> <div class="def" title="def"> <p>heteronuclear single quantum coherence</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
TOCSY</div> <div class="def" title="def"> <p>total correlation spectroscopy</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
ROESY</div> <div class="def" title="def"> <p>rotating frame nuclear Overhauser enhancement spectroscopy</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
COSY</div> <div class="def" title="def"> <p>two-dimensional correlated spectroscopy</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
HMBC</div> <div class="def" title="def"> <p>heteronuclear multiple bond correlation</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
Hex</div> <div class="def" title="def"> <p>hexose</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
dHex</div> <div class="def" title="def"> <p>deoxyhexose</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
Pent</div> <div class="def" title="def"> <p>pentose.</p> </div> </div> </div> <p /> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>We thank Kate Dingle for strains and discussion and Houmar Hussain for help with creating the orf2 and orf19 mutants. </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="B1" /> <span class="label" tagx="label" title="label">1.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smits</span> <span class="given-names" tagx="given-names" title="given-names">W. K.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lyras</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lacy</span> <span class="given-names" tagx="given-names" title="given-names">D. B.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wilcox</span> <span class="given-names" tagx="given-names" title="given-names">M. H.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kuijper</span> <span class="given-names" tagx="given-names" title="given-names">E. J.</span> </span> </span> ( <span class="year" tagx="year" title="year">2016</span>) <span class="mixed-article-title" title="mixed-article-title"> <i>Clostridium difficile</i> infection </span>. <span class="source" tagx="source" title="source">Nat. Rev. Dis. Primers</span> <span class="volume" tagx="volume" title="volume">2</span>, <span class="fpage" tagx="fpage" title="fpage">16021</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nrdp.2016.21">10.1038/nrdp.2016.21</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27227752">27227752</a> </span> </span> </li> <li tag="ref"> <a name="B2" /> <span class="label" tagx="label" title="label">2.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Abt</span> <span class="given-names" tagx="given-names" title="given-names">M. C.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McKenney</span> <span class="given-names" tagx="given-names" title="given-names">P. T.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pamer</span> <span class="given-names" tagx="given-names" title="given-names">E. G.</span> </span> </span> ( <span class="year" tagx="year" title="year">2016</span>) <span class="mixed-article-title" title="mixed-article-title"> <i>Clostridium difficile</i> colitis: pathogenesis and host defence </span>. <span class="source" tagx="source" title="source">Nat. Rev. Microbiol</span>. <span class="volume" tagx="volume" title="volume">14</span>, <span class="fpage" tagx="fpage" title="fpage">609</span>– <span class="lpage" tagx="lpage" title="lpage">620</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nrmicro.2016.108">10.1038/nrmicro.2016.108</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27573580">27573580</a> </span> </span> </li> <li tag="ref"> <a name="B3" /> <span class="label" tagx="label" title="label">3.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jank</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aktories</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> </span> ( <span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">Structure and mode of action of clostridial glucosylating toxins: the ABCD model</span>. <span class="source" tagx="source" title="source">Trends Microbiol</span>. <span class="volume" tagx="volume" title="volume">16</span>, <span class="fpage" tagx="fpage" title="fpage">222</span>– <span class="lpage" tagx="lpage" title="lpage">229</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.tim.2008.01.011">10.1016/j.tim.2008.01.011</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18394902">18394902</a> </span> </span> </li> <li tag="ref"> <a name="B4" /> <span class="label" tagx="label" title="label">4.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deakin</span> <span class="given-names" tagx="given-names" title="given-names">L. J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Clare</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fagan</span> <span class="given-names" tagx="given-names" title="given-names">R. P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dawson</span> <span class="given-names" tagx="given-names" title="given-names">L. F.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pickard</span> <span class="given-names" tagx="given-names" title="given-names">D. J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">West</span> <span class="given-names" tagx="given-names" title="given-names">M. R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wren</span> <span class="given-names" tagx="given-names" title="given-names">B. W.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fairweather</span> <span class="given-names" tagx="given-names" title="given-names">N. F.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dougan</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lawley</span> <span class="given-names" tagx="given-names" title="given-names">T. D.</span> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">The <i>Clostridium difficile spo0A</i> gene is a persistence and transmission factor </span>. <span class="source" tagx="source" title="source">Infect. Immun</span>. <span class="volume" tagx="volume" title="volume">80</span>, <span class="fpage" tagx="fpage" title="fpage">2704</span>– <span class="lpage" tagx="lpage" title="lpage">2711</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/IAI.00147-12">10.1128/IAI.00147-12</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22615253">22615253</a> </span> </span> </li> <li tag="ref"> <a name="B5" /> <span class="label" tagx="label" title="label">5.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lawley</span> <span class="given-names" tagx="given-names" title="given-names">T. D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Clare</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Walker</span> <span class="given-names" tagx="given-names" title="given-names">A. W.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goulding</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stabler</span> <span class="given-names" tagx="given-names" title="given-names">R. A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Croucher</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mastroeni</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Scott</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raisen</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mottram</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fairweather</span> <span class="given-names" tagx="given-names" title="given-names">N. F.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wren</span> <span class="given-names" tagx="given-names" title="given-names">B. W.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Parkhill</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dougan</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> </span> ( <span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Antibiotic treatment of <i>Clostridium difficile</i> carrier mice triggers a supershedder state, spore-mediated transmission, and severe disease in immunocompromised hosts </span>. <span class="source" tagx="source" title="source">Infect. Immun</span>. <span class="volume" tagx="volume" title="volume">77</span>, <span class="fpage" tagx="fpage" title="fpage">3661</span>– <span class="lpage" tagx="lpage" title="lpage">3669</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/IAI.00558-09">10.1128/IAI.00558-09</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19564382">19564382</a> </span> </span> </li> <li tag="ref"> <a name="B6" /> <span class="label" tagx="label" title="label">6.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chang</span> <span class="given-names" tagx="given-names" title="given-names">J. Y.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Antonopoulos</span> <span class="given-names" tagx="given-names" title="given-names">D. A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kalra</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tonelli</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khalife</span> <span class="given-names" tagx="given-names" title="given-names">W. T.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmidt</span> <span class="given-names" tagx="given-names" title="given-names">T. M.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Young</span> <span class="given-names" tagx="given-names" title="given-names">V. B.</span> </span> </span> ( <span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">Decreased diversity of the fecal Microbiome in recurrent <i>Clostridium difficile</i>-associated diarrhea </span>. <span class="source" tagx="source" title="source">J. Infect. Dis</span>. <span class="volume" tagx="volume" title="volume">197</span>, <span class="fpage" tagx="fpage" title="fpage">435</span>– <span class="lpage" tagx="lpage" title="lpage">438</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1086/525047">10.1086/525047</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18199029">18199029</a> </span> </span> </li> <li tag="ref"> <a name="B7" /> <span class="label" tagx="label" title="label">7.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seekatz</span> <span class="given-names" tagx="given-names" title="given-names">A. M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rao</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Santhosh</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Young</span> <span class="given-names" tagx="given-names" title="given-names">V. B.</span> </span> </span> ( <span class="year" tagx="year" title="year">2016</span>) <span class="mixed-article-title" title="mixed-article-title">Dynamics of the fecal microbiome in patients with recurrent and nonrecurrent <i>Clostridium difficile</i> infection </span>. <span class="source" tagx="source" title="source">Genome Med</span>. <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">47</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/s13073-016-0298-8">10.1186/s13073-016-0298-8</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27121861">27121861</a> </span> </span> </li> <li tag="ref"> <a name="B8" /> <span class="label" tagx="label" title="label">8.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fagan</span> <span class="given-names" tagx="given-names" title="given-names">R. P.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fairweather</span> <span class="given-names" tagx="given-names" title="given-names">N. F.</span> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">Biogenesis and functions of bacterial S-layers</span>. <span class="source" tagx="source" title="source">Nat. Rev. Microbiol</span>. <span class="volume" tagx="volume" title="volume">12</span>, <span class="fpage" tagx="fpage" title="fpage">211</span>– <span class="lpage" tagx="lpage" title="lpage">222</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nrmicro3213">10.1038/nrmicro3213</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24509785">24509785</a> </span> </span> </li> <li tag="ref"> <a name="B9" /> <span class="label" tagx="label" title="label">9.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kirk</span> <span class="given-names" tagx="given-names" title="given-names">J. A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Banerji</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fagan</span> <span class="given-names" tagx="given-names" title="given-names">R. P.</span> </span> </span> ( <span class="year" tagx="year" title="year">2017</span>) <span class="mixed-article-title" title="mixed-article-title">Characteristics of the <i>Clostridium difficile</i> cell envelope and its importance in therapeutics </span>. <span class="source" tagx="source" title="source">Microb. Biotechnol</span>. <span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">76</span>– <span class="lpage" tagx="lpage" title="lpage">90</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27311697">27311697</a> </span> </span> </li> <li tag="ref"> <a name="B10" /> <span class="label" tagx="label" title="label">10.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Calabi</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ward</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wren</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paxton</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Panico</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morris</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dell</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dougan</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fairweather</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> </span> ( <span class="year" tagx="year" title="year">2001</span>) <span class="mixed-article-title" title="mixed-article-title">Molecular characterization of the surface layer proteins from <i>Clostridium difficile</i> </span>. <span class="source" tagx="source" title="source">Mol. Microbiol</span>. <span class="volume" tagx="volume" title="volume">40</span>, <span class="fpage" tagx="fpage" title="fpage">1187</span>– <span class="lpage" tagx="lpage" title="lpage">1199</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1046/j.1365-2958.2001.02461.x">10.1046/j.1365-2958.2001.02461.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11401722">11401722</a> </span> </span> </li> <li tag="ref"> <a name="B11" /> <span class="label" tagx="label" title="label">11.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dingle</span> <span class="given-names" tagx="given-names" title="given-names">K. E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Didelot</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ansari</span> <span class="given-names" tagx="given-names" title="given-names">M. A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eyre</span> <span class="given-names" tagx="given-names" title="given-names">D. W.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vaughan</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Griffiths</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ip</span> <span class="given-names" tagx="given-names" title="given-names">C. L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Batty</span> <span class="given-names" tagx="given-names" title="given-names">E. M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Golubchik</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bowden</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jolley</span> <span class="given-names" tagx="given-names" title="given-names">K. A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hood</span> <span class="given-names" tagx="given-names" title="given-names">D. W.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fawley</span> <span class="given-names" tagx="given-names" title="given-names">W. N.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Walker</span> <span class="given-names" tagx="given-names" title="given-names">A. S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peto</span> <span class="given-names" tagx="given-names" title="given-names">T. E.</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Recombinational switching of the <i>Clostridium difficile</i> S-layer and a novel glycosylation gene cluster revealed by large-scale whole-genome sequencing </span>. <span class="source" tagx="source" title="source">J. Infect. Dis</span>. <span class="volume" tagx="volume" title="volume">207</span>, <span class="fpage" tagx="fpage" title="fpage">675</span>– <span class="lpage" tagx="lpage" title="lpage">686</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/infdis/jis734">10.1093/infdis/jis734</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23204167">23204167</a> </span> </span> </li> <li tag="ref"> <a name="B12" /> <span class="label" tagx="label" title="label">12.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sleytr</span> <span class="given-names" tagx="given-names" title="given-names">U. B.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Messner</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> </span> ( <span class="year" tagx="year" title="year">1988</span>) <span class="mixed-article-title" title="mixed-article-title">Crystalline surface layers in procaryotes</span>. <span class="source" tagx="source" title="source">J. Bacteriol</span>. <span class="volume" tagx="volume" title="volume">170</span>, <span class="fpage" tagx="fpage" title="fpage">2891</span>– <span class="lpage" tagx="lpage" title="lpage">2897</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/jb.170.7.2891-2897.1988">10.1128/jb.170.7.2891-2897.1988</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/3290191">3290191</a> </span> </span> </li> <li tag="ref"> <a name="B13" /> <span class="label" tagx="label" title="label">13.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sára</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sleytr</span> <span class="given-names" tagx="given-names" title="given-names">U. B.</span> </span> </span> ( <span class="year" tagx="year" title="year">2000</span>) <span class="mixed-article-title" title="mixed-article-title">S-layer proteins</span>. <span class="source" tagx="source" title="source">J. Bacteriol</span>. <span class="volume" tagx="volume" title="volume">182</span>, <span class="fpage" tagx="fpage" title="fpage">859</span>– <span class="lpage" tagx="lpage" title="lpage">868</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JB.182.4.859-868.2000">10.1128/JB.182.4.859-868.2000</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10648507">10648507</a> </span> </span> </li> <li tag="ref"> <a name="B14" /> <span class="label" tagx="label" title="label">14.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beveridge</span> <span class="given-names" tagx="given-names" title="given-names">T. J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pouwels</span> <span class="given-names" tagx="given-names" title="given-names">P. H.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sára</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kotiranta</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lounatmaa</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kari</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kerosuo</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Haapasalo</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Egelseer</span> <span class="given-names" tagx="given-names" title="given-names">E. M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schocher</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sleytr</span> <span class="given-names" tagx="given-names" title="given-names">U. B.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morelli</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Callegari</span> <span class="given-names" tagx="given-names" title="given-names">M. L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nomellini</span> <span class="given-names" tagx="given-names" title="given-names">J. F.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bingle</span> <span class="given-names" tagx="given-names" title="given-names">W. H.</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">1997</span>) <span class="mixed-article-title" title="mixed-article-title">Functions of S-layers</span>. <span class="source" tagx="source" title="source">FEMS Microbiol. Rev</span>. <span class="volume" tagx="volume" title="volume">20</span>, <span class="fpage" tagx="fpage" title="fpage">99</span>– <span class="lpage" tagx="lpage" title="lpage">149</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0168-6445(97)00043-0,10.1111/j.1574-6976.1997.tb00305.x">10.1016/S0168-6445(97)00043-0,10.1111/j.1574-6976.1997.tb00305.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9276929">9276929</a> </span> </span> </li> <li tag="ref"> <a name="B15" /> <span class="label" tagx="label" title="label">15.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ryan</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lynch</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smith</span> <span class="given-names" tagx="given-names" title="given-names">S. M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Amu</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nel</span> <span class="given-names" tagx="given-names" title="given-names">H. J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McCoy</span> <span class="given-names" tagx="given-names" title="given-names">C. E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dowling</span> <span class="given-names" tagx="given-names" title="given-names">J. K.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Draper</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">O'Reilly</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McCarthy</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">O'Brien</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ní Eidhin</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">O'Connell</span> <span class="given-names" tagx="given-names" title="given-names">M. J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Keogh</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morton</span> <span class="given-names" tagx="given-names" title="given-names">C. O.</span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">A role for TLR4 in <i>Clostridium difficile</i> infection and the recognition of surface layer proteins </span>. <span class="source" tagx="source" title="source">PLoS Pathog</span>. <span class="volume" tagx="volume" title="volume">7</span>, <span class="fpage" tagx="fpage" title="fpage">e1002076</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.ppat.1002076">10.1371/journal.ppat.1002076</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21738466">21738466</a> </span> </span> </li> <li tag="ref"> <a name="B16" /> <span class="label" tagx="label" title="label">16.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kern</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schneewind</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span> </span> ( <span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">BslA, the S-layer adhesin of B. anthracis, is a virulence factor for anthrax pathogenesis</span>. <span class="source" tagx="source" title="source">Mol. Microbiol</span>. <span class="volume" tagx="volume" title="volume">75</span>, <span class="fpage" tagx="fpage" title="fpage">324</span>– <span class="lpage" tagx="lpage" title="lpage">332</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-2958.2009.06958.x">10.1111/j.1365-2958.2009.06958.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19906175">19906175</a> </span> </span> </li> <li tag="ref"> <a name="B17" /> <span class="label" tagx="label" title="label">17.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tarlovsky</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fabian</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Solomaha</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Honsa</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Olson</span> <span class="given-names" tagx="given-names" title="given-names">J. S.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maresso</span> <span class="given-names" tagx="given-names" title="given-names">A. W.</span> </span> </span> ( <span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">A <i>Bacillus anthracis</i> S-layer homology protein that binds heme and mediates heme delivery to IsdC </span>. <span class="source" tagx="source" title="source">J. Bacteriol</span>. <span class="volume" tagx="volume" title="volume">192</span>, <span class="fpage" tagx="fpage" title="fpage">3503</span>– <span class="lpage" tagx="lpage" title="lpage">3511</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JB.00054-10">10.1128/JB.00054-10</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20435727">20435727</a> </span> </span> </li> <li tag="ref"> <a name="B18" /> <span class="label" tagx="label" title="label">18.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baranova</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fronzes</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garcia-Pino</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van Gerven</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Papapostolou</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Péhau-Arnaudet</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pardon</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Steyaert</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Howorka</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Remaut</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">SbsB structure and lattice reconstruction unveil Ca <sup>2+</sup> triggered S-layer assembly </span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">487</span>, <span class="fpage" tagx="fpage" title="fpage">119</span>– <span class="lpage" tagx="lpage" title="lpage">122</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nature11155">10.1038/nature11155</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22722836">22722836</a> </span> </span> </li> <li tag="ref"> <a name="B19" /> <span class="label" tagx="label" title="label">19.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arbing</span> <span class="given-names" tagx="given-names" title="given-names">M. A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chan</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shin</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Phan</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ahn</span> <span class="given-names" tagx="given-names" title="given-names">C. J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rohlin</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gunsalus</span> <span class="given-names" tagx="given-names" title="given-names">R. P.</span> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Structure of the surface layer of the methanogenic archaean <i>Methanosarcina acetivorans</i> </span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U.S.A</span>. <span class="volume" tagx="volume" title="volume">109</span>, <span class="fpage" tagx="fpage" title="fpage">11812</span>– <span class="lpage" tagx="lpage" title="lpage">11817</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.1120595109">10.1073/pnas.1120595109</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22753492">22753492</a> </span> </span> </li> <li tag="ref"> <a name="B20" /> <span class="label" tagx="label" title="label">20.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jing</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Takagi</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">J. H.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lindgren</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">R. G.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Joachimiak</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">J. H.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Springer</span> <span class="given-names" tagx="given-names" title="given-names">T. A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2002</span>) <span class="mixed-article-title" title="mixed-article-title">Archaeal surface layer proteins contain β propeller, PKD, and β helix domains and are related to metazoan cell surface proteins </span>. <span class="source" tagx="source" title="source">Structure</span> <span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">1453</span>– <span class="lpage" tagx="lpage" title="lpage">1464</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0969-2126(02)00840-7">10.1016/S0969-2126(02)00840-7</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12377130">12377130</a> </span> </span> </li> <li tag="ref"> <a name="B21" /> <span class="label" tagx="label" title="label">21.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Anderson</span> <span class="given-names" tagx="given-names" title="given-names">V. J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kern</span> <span class="given-names" tagx="given-names" title="given-names">J. W.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McCool</span> <span class="given-names" tagx="given-names" title="given-names">J. W.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schneewind</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Missiakas</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">The SLH-domain protein BslO is a determinant of <i>Bacillus anthracis</i> chain length </span>. <span class="source" tagx="source" title="source">Mol. Microbiol</span>. <span class="volume" tagx="volume" title="volume">81</span>, <span class="fpage" tagx="fpage" title="fpage">192</span>– <span class="lpage" tagx="lpage" title="lpage">205</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-2958.2011.07688.x">10.1111/j.1365-2958.2011.07688.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21585566">21585566</a> </span> </span> </li> <li tag="ref"> <a name="B22" /> <span class="label" tagx="label" title="label">22.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kern</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wilton</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Binkowski</span> <span class="given-names" tagx="given-names" title="given-names">T. A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Joachimiak</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schneewind</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Structure of the SLH domains from <i>Bacillus anthracis</i> surface array protein </span>. <span class="source" tagx="source" title="source">J. Biol. Chem</span>. <span class="volume" tagx="volume" title="volume">286</span>, <span class="fpage" tagx="fpage" title="fpage">26042</span>– <span class="lpage" tagx="lpage" title="lpage">26049</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1074/jbc.M111.248070">10.1074/jbc.M111.248070</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21572039">21572039</a> </span> </span> </li> <li tag="ref"> <a name="B23" /> <span class="label" tagx="label" title="label">23.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fagan</span> <span class="given-names" tagx="given-names" title="given-names">R. P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Albesa-Jové</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qazi</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Svergun</span> <span class="given-names" tagx="given-names" title="given-names">D. I.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brown</span> <span class="given-names" tagx="given-names" title="given-names">K. A.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fairweather</span> <span class="given-names" tagx="given-names" title="given-names">N. F.</span> </span> </span> ( <span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Structural insights into the molecular organization of the S-layer from <i>Clostridium difficile</i> </span>. <span class="source" tagx="source" title="source">Mol. Microbiol</span>. <span class="volume" tagx="volume" title="volume">71</span>, <span class="fpage" tagx="fpage" title="fpage">1308</span>– <span class="lpage" tagx="lpage" title="lpage">1322</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-2958.2009.06603.x">10.1111/j.1365-2958.2009.06603.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19183279">19183279</a> </span> </span> </li> <li tag="ref"> <a name="B24" /> <span class="label" tagx="label" title="label">24.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bharat</span> <span class="given-names" tagx="given-names" title="given-names">T. A. M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kureisaite-Ciziene</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hardy</span> <span class="given-names" tagx="given-names" title="given-names">G. G.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">E. W.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Devant</span> <span class="given-names" tagx="given-names" title="given-names">J. M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hagen</span> <span class="given-names" tagx="given-names" title="given-names">W. J. H.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brun</span> <span class="given-names" tagx="given-names" title="given-names">Y. V.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Briggs</span> <span class="given-names" tagx="given-names" title="given-names">J. A. G.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Löwe</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span> ( <span class="year" tagx="year" title="year">2017</span>) <span class="mixed-article-title" title="mixed-article-title">Structure of the hexagonal surface layer on <i>Caulobacter crescentus</i> cells </span>. <span class="source" tagx="source" title="source">Nat. Microbiol</span>. <span class="volume" tagx="volume" title="volume">2</span>, <span class="fpage" tagx="fpage" title="fpage">17059</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nmicrobiol.2017.59">10.1038/nmicrobiol.2017.59</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28418382">28418382</a> </span> </span> </li> <li tag="ref"> <a name="B25" /> <span class="label" tagx="label" title="label">25.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mesnage</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tosi-Couture</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mock</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fouet</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> </span> ( <span class="year" tagx="year" title="year">1999</span>) <span class="mixed-article-title" title="mixed-article-title">The S-layer homology domain as a means for anchoring heterologous proteins on the cell surface of <i>Bacillus anthracis</i> </span>. <span class="source" tagx="source" title="source">J. Appl. Microbiol</span>. <span class="volume" tagx="volume" title="volume">87</span>, <span class="fpage" tagx="fpage" title="fpage">256</span>– <span class="lpage" tagx="lpage" title="lpage">260</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1046/j.1365-2672.1999.00880.x">10.1046/j.1365-2672.1999.00880.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10475961">10475961</a> </span> </span> </li> <li tag="ref"> <a name="B26" /> <span class="label" tagx="label" title="label">26.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Willing</span> <span class="given-names" tagx="given-names" title="given-names">S. E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Candela</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shaw</span> <span class="given-names" tagx="given-names" title="given-names">H. A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seager</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mesnage</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fagan</span> <span class="given-names" tagx="given-names" title="given-names">R. P.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fairweather</span> <span class="given-names" tagx="given-names" title="given-names">N. F.</span> </span> </span> ( <span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title"> <i>Clostridium difficile</i> surface proteins are anchored to the cell wall using CWB2 motifs that recognise the anionic polymer PSII </span>. <span class="source" tagx="source" title="source">Mol. Microbiol</span>. <span class="volume" tagx="volume" title="volume">96</span>, <span class="fpage" tagx="fpage" title="fpage">596</span>– <span class="lpage" tagx="lpage" title="lpage">608</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/mmi.12958">10.1111/mmi.12958</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25649385">25649385</a> </span> </span> </li> <li tag="ref"> <a name="B27" /> <span class="label" tagx="label" title="label">27.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ristl</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Steiner</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zarschler</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zayni</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Messner</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schäffer</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">The S-layer glycome-adding to the sugar coat of bacteria</span>. <span class="source" tagx="source" title="source">Int. J. Microbiol</span>. <span class="volume" tagx="volume" title="volume">2011</span>, <span class="fpage" tagx="fpage" title="fpage">127870</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20871840">20871840</a> </span> </span> </li> <li tag="ref"> <a name="B28" /> <span class="label" tagx="label" title="label">28.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Abu-Qarn</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eichler</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharon</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> </span> ( <span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">Not just for Eukarya anymore: protein glycosylation in bacteria and archaea</span>. <span class="source" tagx="source" title="source">Curr. Opin. Struct. Biol</span>. <span class="volume" tagx="volume" title="volume">18</span>, <span class="fpage" tagx="fpage" title="fpage">544</span>– <span class="lpage" tagx="lpage" title="lpage">550</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.sbi.2008.06.010">10.1016/j.sbi.2008.06.010</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18694827">18694827</a> </span> </span> </li> <li tag="ref"> <a name="B29" /> <span class="label" tagx="label" title="label">29.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schäffer</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Messner</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> </span> ( <span class="year" tagx="year" title="year">2017</span>) <span class="mixed-article-title" title="mixed-article-title">Emerging facets of prokaryotic glycosylation</span>. <span class="source" tagx="source" title="source">FEMS Microbiol. Rev</span>. <span class="volume" tagx="volume" title="volume">41</span>, <span class="fpage" tagx="fpage" title="fpage">49</span>– <span class="lpage" tagx="lpage" title="lpage">91</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27566466">27566466</a> </span> </span> </li> <li tag="ref"> <a name="B30" /> <span class="label" tagx="label" title="label">30.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Heap</span> <span class="given-names" tagx="given-names" title="given-names">J. T.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pennington</span> <span class="given-names" tagx="given-names" title="given-names">O. J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cartman</span> <span class="given-names" tagx="given-names" title="given-names">S. T.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carter</span> <span class="given-names" tagx="given-names" title="given-names">G. P.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Minton</span> <span class="given-names" tagx="given-names" title="given-names">N. P.</span> </span> </span> ( <span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">The ClosTron: a universal gene knock-out system for the genus <i>Clostridium</i> </span>. <span class="source" tagx="source" title="source">J. Microbiol. Methods</span> <span class="volume" tagx="volume" title="volume">70</span>, <span class="fpage" tagx="fpage" title="fpage">452</span>– <span class="lpage" tagx="lpage" title="lpage">464</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.mimet.2007.05.021">10.1016/j.mimet.2007.05.021</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17658189">17658189</a> </span> </span> </li> <li tag="ref"> <a name="B31" /> <span class="label" tagx="label" title="label">31.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dell</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morris</span> <span class="given-names" tagx="given-names" title="given-names">H. R.</span> </span> </span> ( <span class="year" tagx="year" title="year">2001</span>) <span class="mixed-article-title" title="mixed-article-title">Glycoprotein structure determination by mass spectrometry</span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">291</span>, <span class="fpage" tagx="fpage" title="fpage">2351</span>– <span class="lpage" tagx="lpage" title="lpage">2356</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.1058890">10.1126/science.1058890</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11269315">11269315</a> </span> </span> </li> <li tag="ref"> <a name="B32" /> <span class="label" tagx="label" title="label">32.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moody</span> <span class="given-names" tagx="given-names" title="given-names">A. M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">North</span> <span class="given-names" tagx="given-names" title="given-names">S. J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reinhold</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van Dyken</span> <span class="given-names" tagx="given-names" title="given-names">S. J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rogers</span> <span class="given-names" tagx="given-names" title="given-names">M. E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Panico</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dell</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morris</span> <span class="given-names" tagx="given-names" title="given-names">H. R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marth</span> <span class="given-names" tagx="given-names" title="given-names">J. D.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reinherz</span> <span class="given-names" tagx="given-names" title="given-names">E. L.</span> </span> </span> ( <span class="year" tagx="year" title="year">2003</span>) <span class="mixed-article-title" title="mixed-article-title">Sialic acid capping of CD8β core 1- <i>O</i>-glycans controls thymocyte-major histocompatibility complex class I interaction </span>. <span class="source" tagx="source" title="source">J. Biol. Chem</span>. <span class="volume" tagx="volume" title="volume">278</span>, <span class="fpage" tagx="fpage" title="fpage">7240</span>– <span class="lpage" tagx="lpage" title="lpage">7246</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1074/jbc.M210468200">10.1074/jbc.M210468200</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12459555">12459555</a> </span> </span> </li> <li tag="ref"> <a name="B33" /> <span class="label" tagx="label" title="label">33.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">van Der Wel</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morris</span> <span class="given-names" tagx="given-names" title="given-names">H. R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Panico</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paxton</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">North</span> <span class="given-names" tagx="given-names" title="given-names">S. J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dell</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thomson</span> <span class="given-names" tagx="given-names" title="given-names">J. M.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">West</span> <span class="given-names" tagx="given-names" title="given-names">C. M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2001</span>) <span class="mixed-article-title" title="mixed-article-title">A non-Golgi α1,2-fucosyltransferase that modifies Skp1 in the cytoplasm of <i>Dictyostelium</i> </span>. <span class="source" tagx="source" title="source">J. Biol. Chem</span>. <span class="volume" tagx="volume" title="volume">276</span>, <span class="fpage" tagx="fpage" title="fpage">33952</span>– <span class="lpage" tagx="lpage" title="lpage">33963</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1074/jbc.M102555200">10.1074/jbc.M102555200</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11423539">11423539</a> </span> </span> </li> <li tag="ref"> <a name="B34" /> <span class="label" tagx="label" title="label">34.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morris</span> <span class="given-names" tagx="given-names" title="given-names">H. R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chalabi</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Panico</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sutton-Smith</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Clark</span> <span class="given-names" tagx="given-names" title="given-names">G. F.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goldberg</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dell</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">Glycoproteomics: past, present and future</span>. <span class="source" tagx="source" title="source">Int. J. Mass Spectrom</span>. <span class="volume" tagx="volume" title="volume">259</span>, <span class="fpage" tagx="fpage" title="fpage">16</span>– <span class="lpage" tagx="lpage" title="lpage">31</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ijms.2006.09.002">10.1016/j.ijms.2006.09.002</a> </span> </span> </li> <li tag="ref"> <a name="B35" /> <span class="label" tagx="label" title="label">35.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morris</span> <span class="given-names" tagx="given-names" title="given-names">H. R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paxton</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dell</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Langhorne</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Berg</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bordoli</span> <span class="given-names" tagx="given-names" title="given-names">R. S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hoyes</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bateman</span> <span class="given-names" tagx="given-names" title="given-names">R. H.</span> </span> </span> ( <span class="year" tagx="year" title="year">1996</span>) <span class="mixed-article-title" title="mixed-article-title">High sensitivity collisionally-activated decomposition tandem mass spectrometry on a novel quadrupole/orthogonal-acceleration time-of-flight mass spectrometer </span>. <span class="source" tagx="source" title="source">Rapid Commun. Mass Spectrom</span>. <span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">889</span>– <span class="lpage" tagx="lpage" title="lpage">896</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/(SICI)1097-0231(19960610)10:8%3C889::AID-RCM615%3E3.0.CO%3B2-F">10.1002/(SICI)1097-0231(19960610)10:8%3C889::AID-RCM615%3E3.0.CO%3B2-F</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8777321">8777321</a> </span> </span> </li> <li tag="ref"> <a name="B36" /> <span class="label" tagx="label" title="label">36.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bouché</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Panico</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hitchen</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Binet</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sastre</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Faulds-Pain</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Valiente</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vinogradov</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aubry</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fulton</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Twine</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Logan</span> <span class="given-names" tagx="given-names" title="given-names">S. M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wren</span> <span class="given-names" tagx="given-names" title="given-names">B. W.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dell</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morris</span> <span class="given-names" tagx="given-names" title="given-names">H. R.</span> </span> </span> ( <span class="year" tagx="year" title="year">2016</span>) <span class="mixed-article-title" title="mixed-article-title">The type B flagellin of hypervirulent <i>Clostridium difficile</i> is modified with novel sulfonated peptidylamido-glycans </span>. <span class="source" tagx="source" title="source">J. Biol. Chem</span>. <span class="volume" tagx="volume" title="volume">291</span>, <span class="fpage" tagx="fpage" title="fpage">25439</span>– <span class="lpage" tagx="lpage" title="lpage">25449</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1074/jbc.M116.749481">10.1074/jbc.M116.749481</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27758867">27758867</a> </span> </span> </li> <li tag="ref"> <a name="B37" /> <span class="label" tagx="label" title="label">37.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Panico</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bouché</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Binet</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">O'Connor</span> <span class="given-names" tagx="given-names" title="given-names">M. J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rahman</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pang</span> <span class="given-names" tagx="given-names" title="given-names">P. C.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Canis</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">North</span> <span class="given-names" tagx="given-names" title="given-names">S. J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Desrosiers</span> <span class="given-names" tagx="given-names" title="given-names">R. C.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chertova</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Keele</span> <span class="given-names" tagx="given-names" title="given-names">B. F.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bess</span> <span class="given-names" tagx="given-names" title="given-names">J. W.</span> <span class="suffix" tagx="suffix" title="suffix">Jr.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lifson</span> <span class="given-names" tagx="given-names" title="given-names">J. D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Haslam</span> <span class="given-names" tagx="given-names" title="given-names">S. M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dell</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morris</span> <span class="given-names" tagx="given-names" title="given-names">H. R.</span> </span> </span> ( <span class="year" tagx="year" title="year">2016</span>) <span class="mixed-article-title" title="mixed-article-title">Mapping the complete glycoproteome of virion-derived HIV-1 gp120 provides insights into broadly neutralizing antibody binding </span>. <span class="source" tagx="source" title="source">Sci. Rep</span>. <span class="volume" tagx="volume" title="volume">6</span>, <span class="fpage" tagx="fpage" title="fpage">32956</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/srep32956">10.1038/srep32956</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27604319">27604319</a> </span> </span> </li> <li tag="ref"> <a name="B38" /> <span class="label" tagx="label" title="label">38.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morris</span> <span class="given-names" tagx="given-names" title="given-names">H. R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paxton</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Panico</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McDowell</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dell</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> </span> ( <span class="year" tagx="year" title="year">1997</span>) <span class="mixed-article-title" title="mixed-article-title">A novel geometry mass spectrometer, the Q-TOF, for low-femtomole/attomole-range biopolymer sequencing </span>. <span class="source" tagx="source" title="source">J. Protein Chem</span>. <span class="volume" tagx="volume" title="volume">16</span>, <span class="fpage" tagx="fpage" title="fpage">469</span>– <span class="lpage" tagx="lpage" title="lpage">479</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1023/A:1026309410737">10.1023/A:1026309410737</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9246631">9246631</a> </span> </span> </li> <li tag="ref"> <a name="B39" /> <span class="label" tagx="label" title="label">39.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Iavarone</span> <span class="given-names" tagx="given-names" title="given-names">A. T.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Williams</span> <span class="given-names" tagx="given-names" title="given-names">E. R.</span> </span> </span> ( <span class="year" tagx="year" title="year">2002</span>) <span class="mixed-article-title" title="mixed-article-title">Supercharging in electrospray ionization: effects on signal and charge</span>. <span class="source" tagx="source" title="source">Int. J. Mass Spectrom</span>. <span class="volume" tagx="volume" title="volume">219</span>, <span class="fpage" tagx="fpage" title="fpage">63</span>– <span class="lpage" tagx="lpage" title="lpage">72</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S1387-3806(02)00587-0">10.1016/S1387-3806(02)00587-0</a> </span> </span> </li> <li tag="ref"> <a name="B40" /> <span class="label" tagx="label" title="label">40.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sweet</span> <span class="given-names" tagx="given-names" title="given-names">D. P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shapiro</span> <span class="given-names" tagx="given-names" title="given-names">R. H.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Albersheim</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> </span> ( <span class="year" tagx="year" title="year">1974</span>) <span class="mixed-article-title" title="mixed-article-title">The mass spectral fragmentation of partially ethylated alditol acetates, a derivative used in determining the glycosyl linkage composition of polysaccharides </span>. <span class="source" tagx="source" title="source">Biomed. Mass. Spectrom</span>. <span class="volume" tagx="volume" title="volume">1</span>, <span class="fpage" tagx="fpage" title="fpage">263</span>– <span class="lpage" tagx="lpage" title="lpage">268</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/bms.1200010410">10.1002/bms.1200010410</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/4441621">4441621</a> </span> </span> </li> <li tag="ref"> <a name="B41" /> <span class="label" tagx="label" title="label">41.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jang-Lee</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">North</span> <span class="given-names" tagx="given-names" title="given-names">S. J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sutton-Smith</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goldberg</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Panico</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morris</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Haslam</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dell</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2006</span>) <span class="mixed-article-title" title="mixed-article-title">Glycomic profiling of cells and tissues by mass spectrometry: fingerprinting and sequencing methodologies </span>. <span class="source" tagx="source" title="source">Methods Enzymol</span>. <span class="volume" tagx="volume" title="volume">415</span>, <span class="fpage" tagx="fpage" title="fpage">59</span>– <span class="lpage" tagx="lpage" title="lpage">86</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0076-6879(06)15005-3">10.1016/S0076-6879(06)15005-3</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17116468">17116468</a> </span> </span> </li> <li tag="ref"> <a name="B42" /> <span class="label" tagx="label" title="label">42.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Canis</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McKinnon</span> <span class="given-names" tagx="given-names" title="given-names">T. A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nowak</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Haslam</span> <span class="given-names" tagx="given-names" title="given-names">S. M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Panico</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morris</span> <span class="given-names" tagx="given-names" title="given-names">H. R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Laffan</span> <span class="given-names" tagx="given-names" title="given-names">M. A.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dell</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Mapping the <i>N</i>-glycome of human von Willebrand factor </span>. <span class="source" tagx="source" title="source">Biochem. J</span>. <span class="volume" tagx="volume" title="volume">447</span>, <span class="fpage" tagx="fpage" title="fpage">217</span>– <span class="lpage" tagx="lpage" title="lpage">228</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1042/BJ20120810">10.1042/BJ20120810</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22849435">22849435</a> </span> </span> </li> <li tag="ref"> <a name="B43" /> <span class="label" tagx="label" title="label">43.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fagan</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fairweather</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> </span> ( <span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Dissecting the cell surface</span>. <span class="source" tagx="source" title="source">Methods Mol. Biol</span>. <span class="volume" tagx="volume" title="volume">646</span>, <span class="fpage" tagx="fpage" title="fpage">117</span>– <span class="lpage" tagx="lpage" title="lpage">134</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/978-1-60327-365-7_8">10.1007/978-1-60327-365-7_8</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20597006">20597006</a> </span> </span> </li> <li tag="ref"> <a name="B44" /> <span class="label" tagx="label" title="label">44.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ganeshapillai</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vinogradov</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rousseau</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weese</span> <span class="given-names" tagx="given-names" title="given-names">J. S.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Monteiro</span> <span class="given-names" tagx="given-names" title="given-names">M. A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title"> <i>Clostridium difficile</i> cell-surface polysaccharides composed of pentaglycosyl and hexaglycosyl phosphate repeating units </span>. <span class="source" tagx="source" title="source">Carbohydr. Res</span>. <span class="volume" tagx="volume" title="volume">343</span>, <span class="fpage" tagx="fpage" title="fpage">703</span>– <span class="lpage" tagx="lpage" title="lpage">710</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.carres.2008.01.002">10.1016/j.carres.2008.01.002</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18237724">18237724</a> </span> </span> </li> <li tag="ref"> <a name="B45" /> <span class="label" tagx="label" title="label">45.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sernee</span> <span class="given-names" tagx="given-names" title="given-names">M. F.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ralton</span> <span class="given-names" tagx="given-names" title="given-names">J. E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dinev</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khairallah</span> <span class="given-names" tagx="given-names" title="given-names">G. N.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">O'Hair</span> <span class="given-names" tagx="given-names" title="given-names">R. A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Williams</span> <span class="given-names" tagx="given-names" title="given-names">S. J.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McConville</span> <span class="given-names" tagx="given-names" title="given-names">M. J.</span> </span> </span> ( <span class="year" tagx="year" title="year">2006</span>) <span class="mixed-article-title" title="mixed-article-title"> <i>Leishmania</i> β-1,2-mannan is assembled on a mannose-cyclic phosphate primer </span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U.S.A</span>. <span class="volume" tagx="volume" title="volume">103</span>, <span class="fpage" tagx="fpage" title="fpage">9458</span>– <span class="lpage" tagx="lpage" title="lpage">9463</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.0603539103">10.1073/pnas.0603539103</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16766650">16766650</a> </span> </span> </li> <li tag="ref"> <a name="B46" /> <span class="label" tagx="label" title="label">46.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stewart</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bernlind</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martin</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oscarson</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Richards</span> <span class="given-names" tagx="given-names" title="given-names">J. C.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schweda</span> <span class="given-names" tagx="given-names" title="given-names">E. K. H.</span> </span> </span> ( <span class="year" tagx="year" title="year">1998</span>) <span class="mixed-article-title" title="mixed-article-title">Studies of alkaline mediated phosphate migration in synthetic phosphoethanolamine <div class="sc_UNKNOWN">
l</div>-glycero- <div class="sc_UNKNOWN">
d</div>-manno-heptoside derivatives </span>. <span class="source" tagx="source" title="source">Carbohydr. Res</span>. <span class="volume" tagx="volume" title="volume">313</span>, <span class="fpage" tagx="fpage" title="fpage">193</span>– <span class="lpage" tagx="lpage" title="lpage">202</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0008-6215(98)00271-7">10.1016/S0008-6215(98)00271-7</a> </span> </span> </li> <li tag="ref"> <a name="B47" /> <span class="label" tagx="label" title="label">47.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Calabi</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Calabi</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Phillips</span> <span class="given-names" tagx="given-names" title="given-names">A. D.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fairweather</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> </span> ( <span class="year" tagx="year" title="year">2002</span>) <span class="mixed-article-title" title="mixed-article-title">Binding of <i>Clostridium difficile</i> surface layer proteins to gastrointestinal tissues </span>. <span class="source" tagx="source" title="source">Infect. Immun</span>. <span class="volume" tagx="volume" title="volume">70</span>, <span class="fpage" tagx="fpage" title="fpage">5770</span>– <span class="lpage" tagx="lpage" title="lpage">5778</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/IAI.70.10.5770-5778.2002">10.1128/IAI.70.10.5770-5778.2002</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12228307">12228307</a> </span> </span> </li> <li tag="ref"> <a name="B48" /> <span class="label" tagx="label" title="label">48.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Merrigan</span> <span class="given-names" tagx="given-names" title="given-names">M. M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Venugopal</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Roxas</span> <span class="given-names" tagx="given-names" title="given-names">J. L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Anwar</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mallozzi</span> <span class="given-names" tagx="given-names" title="given-names">M. J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Roxas</span> <span class="given-names" tagx="given-names" title="given-names">B. A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gerding</span> <span class="given-names" tagx="given-names" title="given-names">D. N.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Viswanathan</span> <span class="given-names" tagx="given-names" title="given-names">V. K.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vedantam</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> </span> ( <span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Surface-layer protein A (SlpA) is a major contributor to host-cell adherence of <i>Clostridium difficile</i> </span>. <span class="source" tagx="source" title="source">PLoS ONE</span> <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">e78404</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0078404">10.1371/journal.pone.0078404</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24265687">24265687</a> </span> </span> </li> <li tag="ref"> <a name="B49" /> <span class="label" tagx="label" title="label">49.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kirk</span> <span class="given-names" tagx="given-names" title="given-names">J. A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gebhart</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buckley</span> <span class="given-names" tagx="given-names" title="given-names">A. M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lok</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Scholl</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Douce</span> <span class="given-names" tagx="given-names" title="given-names">G. R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Govoni</span> <span class="given-names" tagx="given-names" title="given-names">G. R.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fagan</span> <span class="given-names" tagx="given-names" title="given-names">R. P.</span> </span> </span> ( <span class="year" tagx="year" title="year">2017</span>) <span class="mixed-article-title" title="mixed-article-title">New class of precision antimicrobials redefines role of <i>Clostridium difficile</i> S-layer in virulence and viability </span>. <span class="source" tagx="source" title="source">Sci. Transl. Med</span>. <span class="volume" tagx="volume" title="volume">6</span>, <span class="fpage" tagx="fpage" title="fpage">aah6813</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1126/scitranslmed.aah6813">10.1126/scitranslmed.aah6813</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28878013">28878013</a> </span> </span> </li> <li tag="ref"> <a name="B50" /> <span class="label" tagx="label" title="label">50.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gebhart</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Williams</span> <span class="given-names" tagx="given-names" title="given-names">S. R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bishop-Lilly</span> <span class="given-names" tagx="given-names" title="given-names">K. A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Govoni</span> <span class="given-names" tagx="given-names" title="given-names">G. R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Willner</span> <span class="given-names" tagx="given-names" title="given-names">K. M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Butani</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sozhamannan</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martin</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fortier</span> <span class="given-names" tagx="given-names" title="given-names">L.-C.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Scholl</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Novel high-molecular-weight, R-type bacteriocins of <i>Clostridium difficile</i> </span>. <span class="source" tagx="source" title="source">J. Bacteriol</span>. <span class="volume" tagx="volume" title="volume">194</span>, <span class="fpage" tagx="fpage" title="fpage">6240</span>– <span class="lpage" tagx="lpage" title="lpage">6247</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/JB.01272-12">10.1128/JB.01272-12</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22984261">22984261</a> </span> </span> </li> <li tag="ref"> <a name="B51" /> <span class="label" tagx="label" title="label">51.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Clarke</span> <span class="given-names" tagx="given-names" title="given-names">B. R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Richards</span> <span class="given-names" tagx="given-names" title="given-names">M. R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Greenfield</span> <span class="given-names" tagx="given-names" title="given-names">L. K.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hou</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lowary</span> <span class="given-names" tagx="given-names" title="given-names">T. L.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Whitfield</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title"> <i>In vitro</i> reconstruction of the chain termination reaction in biosynthesis of the <i>Escherichia coli</i> O9a <i>O</i>-polysaccharide: the chain-length regulator, WbdD, catalyzes the addition of methyl phosphate to the non-reducing terminus of the growing glycan </span>. <span class="source" tagx="source" title="source">J. Biol. Chem</span>. <span class="volume" tagx="volume" title="volume">286</span>, <span class="fpage" tagx="fpage" title="fpage">41391</span>– <span class="lpage" tagx="lpage" title="lpage">41401</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1074/jbc.M111.295857">10.1074/jbc.M111.295857</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21990359">21990359</a> </span> </span> </li> <li tag="ref"> <a name="B52" /> <span class="label" tagx="label" title="label">52.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Faulds-Pain</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Twine</span> <span class="given-names" tagx="given-names" title="given-names">S. M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vinogradov</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Strong</span> <span class="given-names" tagx="given-names" title="given-names">P. C.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dell</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buckley</span> <span class="given-names" tagx="given-names" title="given-names">A. M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Douce</span> <span class="given-names" tagx="given-names" title="given-names">G. R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Valiente</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Logan</span> <span class="given-names" tagx="given-names" title="given-names">S. M.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wren</span> <span class="given-names" tagx="given-names" title="given-names">B. W.</span> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">The post-translational modification of the <i>Clostridium difficile</i> flagellin affects motility, cell surface properties and virulence </span>. <span class="source" tagx="source" title="source">Mol. Microbiol</span>. <span class="volume" tagx="volume" title="volume">94</span>, <span class="fpage" tagx="fpage" title="fpage">272</span>– <span class="lpage" tagx="lpage" title="lpage">289</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/mmi.12755">10.1111/mmi.12755</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25135277">25135277</a> </span> </span> </li> <li tag="ref"> <a name="B53" /> <span class="label" tagx="label" title="label">53.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Qazi</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hitchen</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tissot</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Panico</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morris</span> <span class="given-names" tagx="given-names" title="given-names">H. R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dell</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fairweather</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> </span> ( <span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Mass spectrometric analysis of the S-layer proteins from <i>Clostridium difficile</i> demonstrates the absence of glycosylation </span>. <span class="source" tagx="source" title="source">J. Mass Spectrom</span>. <span class="volume" tagx="volume" title="volume">44</span>, <span class="fpage" tagx="fpage" title="fpage">368</span>– <span class="lpage" tagx="lpage" title="lpage">374</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/jms.1514">10.1002/jms.1514</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18932172">18932172</a> </span> </span> </li> <li tag="ref"> <a name="B54" /> <span class="label" tagx="label" title="label">54.</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Emerson</span> <span class="given-names" tagx="given-names" title="given-names">J. E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reynolds</span> <span class="given-names" tagx="given-names" title="given-names">C. B.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fagan</span> <span class="given-names" tagx="given-names" title="given-names">R. P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shaw</span> <span class="given-names" tagx="given-names" title="given-names">H. A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goulding</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>, and <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fairweather</span> <span class="given-names" tagx="given-names" title="given-names">N. F.</span> </span> </span> ( <span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">A novel genetic switch controls phase variable expression of CwpV, a <i>Clostridium difficile</i> cell wall protein </span>. <span class="source" tagx="source" title="source">Mol. Microbiol</span>. <span class="volume" tagx="volume" title="volume">74</span>, <span class="fpage" tagx="fpage" title="fpage">541</span>– <span class="lpage" tagx="lpage" title="lpage">556</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1365-2958.2009.06812.x">10.1111/j.1365-2958.2009.06812.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19656296">19656296</a> </span> </span> </li> </ul> </div> </div>  </body></html>